<!DOCTYPE html>
<html xmlns:og="http://ogp.me/ns#" xmlns:fb="https://www.facebook.com/2008/fbml" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<meta content="mdpi" name="sso-service">
<meta content="width=device-width, initial-scale=1.0" name="viewport">
<title>Crystals  | Free Full-Text | The Resolution in X-ray Crystallography and Single-Particle Cryogenic Electron Microscopy | HTML</title><link rel="stylesheet" href="font-awesome.css">
<link rel="stylesheet" href="jquery.css">
<link rel="stylesheet" href="chosen.css">
<link rel="stylesheet" href="main2.css">
<link rel="mask-icon" href="https://www.mdpi.com/img/mask-icon-128.svg?c1c7eca266cd7013" color="#4f5671">
<link rel="apple-touch-icon" sizes="180x180" href="https://www.mdpi.com/icon/apple-touch-icon-180x180.png">
<link rel="apple-touch-icon" sizes="152x152" href="https://www.mdpi.com/icon/apple-touch-icon-152x152.png">
<link rel="apple-touch-icon" sizes="144x144" href="https://www.mdpi.com/icon/apple-touch-icon-144x144.png">
<link rel="apple-touch-icon" sizes="120x120" href="https://www.mdpi.com/icon/apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" sizes="114x114" href="https://www.mdpi.com/icon/apple-touch-icon-114x114.png">
<link rel="apple-touch-icon" sizes="76x76" href="https://www.mdpi.com/icon/apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" sizes="72x72" href="https://www.mdpi.com/icon/apple-touch-icon-72x72.png">
<link rel="apple-touch-icon" sizes="57x57" href="https://www.mdpi.com/icon/apple-touch-icon-57x57.png">
<link rel="apple-touch-icon" href="https://www.mdpi.com/icon/apple-touch-icon-57x57.png">
<link rel="apple-touch-icon-precomposed" href="https://www.mdpi.com/icon/apple-touch-icon-57x57.png">
<link rel="manifest" href="https://www.mdpi.com/manifest.json">
<meta name="theme-color" content="#ffffff">
<meta name="application-name" content="&nbsp;">
<link rel="apple-touch-startup-image" href="https://www.mdpi.com/img/journals/crystals-logo-sq.png?da4be6f5caf6fc17">
<link rel="apple-touch-icon" href="https://www.mdpi.com/img/journals/crystals-logo-sq.png?da4be6f5caf6fc17">
<meta name="msapplication-TileImage" content="/img/journals/crystals-logo-sq.png?da4be6f5caf6fc17">
<style>

h2, #abstract .related_suggestion_title {
    color: rgba(51,109,175,0.75);
}

.batch_articles a {
    color: #000;
}

    a, .batch_articles .authors a, a:focus, a:hover, a:active, .batch_articles a:focus, .batch_articles a:hover, li.side-menu-li a {
        color: rgba(51,109,175,0.75);
    }

    span.label a {
        color: #fff;
    }

    #main-content a.title-link:hover,
    #main-content a.title-link:focus,
    #main-content div.generic-item a.title-link:hover,
    #main-content div.generic-item a.title-link:focus {
        color: rgba(51,109,175,0.75);
    }

    #main-content #middle-column .generic-item.article-item a.title-link:hover,
    #main-content #middle-column .generic-item.article-item a.title-link:focus {
        color: rgba(51,109,175,0.75);
    }

    .art-authors a.toEncode {
        color: #333;
        font-weight: 700;
    }

    #main-content #middle-column ul li::before {
        color: rgba(51,109,175,0.75);
    }

    .accordion-navigation.active a.accordion__title,
    .accordion-navigation.active a.accordion__title::after {
        color: rgba(51,109,175,0.75) !important;
    }

    .accordion-navigation li:hover::before,
    .accordion-navigation li:hover a,
    .accordion-navigation li:focus a {
        color: rgba(51,109,175,0.75) !important;
    }

    .relative-size-container .relative-size-image .relative-size {
        background-color: rgba(51,109,175,0.75);
    }

    .middle-column__help__fixed a:hover i,
        color: rgba(51,109,175,0.75) !important;
    }

    input[type="checkbox"]:checked:after {
        color: rgba(51,109,175,0.75) !important;
    }

    input[type="checkbox"]:not(:disabled):hover:before {
        border-color: rgba(51,109,175,0.75) !important;
    }

    #main-content .bolded-text {
        color: rgba(51,109,175,0.75);
    }


#main-content .hypothesis-count-container {
    background-color: rgba(51,109,175,0.75) !important;
}

#main-content .hypothesis-count-container:before {
    border-bottom-color: rgba(51,109,175,0.75) !important;
}

#main-content .trendmd-widget-cookie-notification__info a {
    color: rgba(51,109,175,0.75) !important;
}

.full-size-menu ul li.menu-item .dropdown-wrapper {
    background-color: #336DAF !important;
}

.full-size-menu ul li.menu-item > a.open::after {
    border-bottom: 10px solid #336DAF;
}

#title-story .title-story-orbit .orbit-caption {
    #background: url('/img/design/000000_background.png') !important;
    background: url('/img/design/ffffff_background.png') !important;
    color: rgb(51, 51, 51) !important;
}

#main-content .content__container__orbit {
    background-color: #000 !important;
}

#main-content .content__container__journal {
    background-color: #336DAF;
    color: #fff;
}

.html-article-menu .row span {
    color: rgba(51,109,175,0.75);
}

.html-article-menu .row span.active {
    }

.side-menu-li.active::before,
.side-menu-li.active a {
    color: rgba(51,109,175,0.75) !important;
}

.side-menu-ul li.active a, .side-menu-ul li.active, .side-menu-ul li.active::before {
    color: rgba(51,109,175,0.75) !important;
}

.side-menu-ul li.active a {
    font-weight: 700 !important;
}

.result-selected, .active-result.highlighted, .active-result:hover,
.result-selected, .active-result.highlighted, .active-result:focus {
    background-color: rgba(51,109,175,0.75) !important;
    color: #fff;
}

.search-container.search-container__default-scheme {
    background-color: #336DAF;
}

nav.tab-bar .open-small-search.active:after {
    border-bottom-color: #336DAF;
}

.search-container.search-container__default-scheme .custom-accordion-for-small-screen-link::after {
    color: #fff;
}

@media only screen and (max-width: 50em) {
    #main-content .content__container.journal-info {
        color: #fff;
        background-color: rgba(51,109,175,0.75);
    }

    #main-content .content__container.journal-info a {
        color: #fff;
    }
} 

.button.button--color {
    color: rgba(51,109,175,0.75);
    border-color: rgba(51,109,175,0.75);
}

.button.button--color:hover,
.button.button--color:focus {
    color: #fff;
    border-color: rgba(51,109,175,0.75);
    background-color: rgba(51,109,175,0.75);
}

.button.button--color-inversed {
    background-color: rgba(51,109,175,0.75);
    border-color: #fff;
    color: #fff;
}

.button.button--color-inversed:hover,
.button.button--color-inversed:focus {
    background-color: rgba(51,109,175,0.75);
    border-color: #fff;
    color: #fff;
}

.button.button--color path {
    fill: rgba(51,109,175,0.75);
}

.button.button--color:hover path {
    fill: #fff;
}

#main-content #search-refinements .ui-slider-horizontal .ui-slider-range {
    background-color: rgba(51,109,175,0.75);
}

.breadcrumb__element:last-of-type a {
    color: rgba(51,109,175,0.75);
}

    
#main-header {
    background-color: rgba(51,109,175,0.75);
}

#full-size-menu .top-bar, #full-size-menu li.menu-item span.user-email {
    background-color:   rgba(51,109,175,0.75)
}

.top-bar-section li:not(.has-form) a:not(.button) {
    background-color:   rgba(51,109,175,0.75)
}

#full-size-menu li.menu-item .dropdown-wrapper li a:hover {
    color:   rgba(51,109,175,0.75)
}

#full-size-menu li.menu-item a:hover, #full-size-menu li.menu.item a:focus, nav.tab-bar a:hover {
    }
#full-size-menu li.menu.item a:active, #full-size-menu li.menu.item a.active {
    background-color: #9FCBFF;
}

#full-size-menu li.menu-item a.open-mega-menu.active, #full-size-menu li.menu-item div.mega-menu, a.open-mega-menu.active {
    background-color: #9FCBFF;
    color: #000;
}

#full-size-menu li.menu-item div.mega-menu li, #full-size-menu li.menu-item div.mega-menu a {
    background-color: #9FCBFF;
    color: #000;
    border-color: #9a9a9a;
}

div.type-section h1 {
    padding-left: 15px !important;
    background-color: rgba(51,109,175,0.75);
    color: #fff;
    font-size: 20px;
    line-height: 26px;
    font-weight: 300;
}

div.type-section h3 {
    margin-left: 15px;
    margin-bottom: 0px;
    font-weight: 300;
}

.journal-tabs .tab-title.active a {
    background-color: rgba(51,109,175,0.75);
    border-color: rgba(51,109,175,0.75);
    color: #fff;
}

</style>
<link rel="stylesheet" href="jquery-ui-1.css">
<link rel="stylesheet" href="magnific-popup.css">
<link rel="stylesheet" href="article-html.css">
<style>
                                </style>
<meta name="title" content="The Resolution in X-ray Crystallography and Single-Particle Cryogenic Electron Microscopy">
<meta name="description" content="X-ray crystallography and single-particle analysis cryogenic electron microscopy are essential techniques for uncovering the three-dimensional structures of biological macromolecules. Both techniques rely on the Fourier transform to calculate experimental maps. However, one of the crucial parameters, resolution, is rather broadly defined. Here, the methods to determine the resolution in X-ray crystallography and single-particle analysis are summarized. In X-ray crystallography, it is becoming increasingly more common to include reflections discarded previously by traditionally used standards, allowing for the inclusion of incomplete and anisotropic reflections into the refinement process. In general, the resolution is the smallest lattice spacing given by Bragg&amp;rsquo;s law for a particular set of X-ray diffraction intensities; however, typically the resolution is truncated by the user during the data processing based on certain parameters and later it is used during refinement. However, at which resolution to perform such a truncation is not always clear and this makes it very confusing for the novices entering the structural biology field. Furthermore, it is argued that the effective resolution should be also reported as it is a more descriptive measure accounting for anisotropy and incompleteness of the data. In single particle cryo-EM, the situation is not much better, as multiple ways exist to determine the resolution, such as Fourier shell correlation, spectral signal-to-noise ratio and the Fourier neighbor correlation. The most widely accepted is the Fourier shell correlation using a threshold of 0.143 to define the resolution (so-called &amp;ldquo;gold-standard&amp;rdquo;), although it is still debated whether this is the correct threshold. Besides, the resolution obtained from the Fourier shell correlation is an estimate of varying resolution across the density map. In reality, the interpretability of the map is more important than the numerical value of the resolution.">
<link rel="image_src" href="crystals-logo.png">
<meta name="dc.title" content="The Resolution in X-ray Crystallography and Single-Particle Cryogenic Electron Microscopy">
<meta name="dc.creator" content="Victor  R.A. Dubach">
<meta name="dc.creator" content="Albert Guskov">
<meta name="dc.type" content="Review">
<meta name="dc.source" content="Crystals 2020, Vol. 10, Page 580">
<meta name="dc.date" content="2020-07-05">
<meta name="dc.identifier" content="10.3390/cryst10070580">
<meta name="dc.publisher" content="Multidisciplinary Digital Publishing Institute">
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/3.0/">
<meta name="dc.format" content="application/pdf">
<meta name="dc.language" content="en">
<meta name="dc.description" content="X-ray crystallography and single-particle analysis cryogenic electron microscopy are essential techniques for uncovering the three-dimensional structures of biological macromolecules. Both techniques rely on the Fourier transform to calculate experimental maps. However, one of the crucial parameters, resolution, is rather broadly defined. Here, the methods to determine the resolution in X-ray crystallography and single-particle analysis are summarized. In X-ray crystallography, it is becoming increasingly more common to include reflections discarded previously by traditionally used standards, allowing for the inclusion of incomplete and anisotropic reflections into the refinement process. In general, the resolution is the smallest lattice spacing given by Bragg&amp;rsquo;s law for a particular set of X-ray diffraction intensities; however, typically the resolution is truncated by the user during the data processing based on certain parameters and later it is used during refinement. However, at which resolution to perform such a truncation is not always clear and this makes it very confusing for the novices entering the structural biology field. Furthermore, it is argued that the effective resolution should be also reported as it is a more descriptive measure accounting for anisotropy and incompleteness of the data. In single particle cryo-EM, the situation is not much better, as multiple ways exist to determine the resolution, such as Fourier shell correlation, spectral signal-to-noise ratio and the Fourier neighbor correlation. The most widely accepted is the Fourier shell correlation using a threshold of 0.143 to define the resolution (so-called &amp;ldquo;gold-standard&amp;rdquo;), although it is still debated whether this is the correct threshold. Besides, the resolution obtained from the Fourier shell correlation is an estimate of varying resolution across the density map. In reality, the interpretability of the map is more important than the numerical value of the resolution.">
<meta name="dc.subject" content="X-ray crystallography">
<meta name="dc.subject" content="single-particle analysis">
<meta name="dc.subject" content="resolution">
<meta name="prism.publicationName" content="Crystals">
<meta name="prism.publicationDate" content="2020-07-05">
<meta name="prism.volume" content="10">
<meta name="prism.number" content="7">
<meta name="prism.section" content="Review">
<meta name="prism.startingPage" content="580">
<meta name="citation_journal_title" content="Crystals">
<meta name="citation_publisher" content="Multidisciplinary Digital Publishing Institute">
<meta name="citation_title" content="The Resolution in X-ray Crystallography and Single-Particle Cryogenic Electron Microscopy">
<meta name="citation_publication_date" content="2020/7">
<meta name="citation_online_date" content="2020/07/05">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="7">
<meta name="citation_firstpage" content="580">
<meta name="citation_author" content="Dubach, Victor  R.A.">
<meta name="citation_author" content="Guskov, Albert">
<meta name="citation_doi" content="10.3390/cryst10070580">
<meta name="citation_id" content="mdpi-cryst10070580">
<meta name="citation_abstract_html_url" content="https://www.mdpi.com/2073-4352/10/7/580">
<meta name="citation_pdf_url" content="https://www.mdpi.com/2073-4352/10/7/580/pdf">
<link rel="alternate" type="application/pdf" title="PDF Full-Text" href="https://www.mdpi.com/2073-4352/10/7/580/pdf">
<meta name="fulltext_pdf" content="https://www.mdpi.com/2073-4352/10/7/580/pdf">
<meta name="citation_fulltext_html_url" content="https://www.mdpi.com/2073-4352/10/7/580/htm">
<link rel="alternate" type="text/html" title="HTML Full-Text" href="https://www.mdpi.com/2073-4352/10/7/580/htm">
<meta name="fulltext_html" content="https://www.mdpi.com/2073-4352/10/7/580/htm">
<link rel="alternate" type="text/xml" title="XML Full-Text" href="https://www.mdpi.com/2073-4352/10/7/580/xml">
<meta name="fulltext_xml" content="https://www.mdpi.com/2073-4352/10/7/580/xml">
<meta name="twitter:card" content="summary">
<meta name="twitter:site" content="@MDPIOpenAccess">
<meta name="twitter:image" content="https://www.mdpi.com/img/journals/crystals-logo-sq.png?da4be6f5caf6fc17">
<meta property="fb:app_id" content="131189377574">
<meta property="og:site_name" content="MDPI">
<meta property="og:type" content="article">
<meta property="og:url" content="https://www.mdpi.com/2073-4352/10/7/580">
<meta property="og:title" content="The Resolution in X-ray Crystallography and Single-Particle Cryogenic Electron Microscopy">
<meta property="og:description" content="X-ray crystallography and single-particle analysis cryogenic electron microscopy are essential techniques for uncovering the three-dimensional structures of biological macromolecules. Both techniques rely on the Fourier transform to calculate experimental maps. However, one of the crucial parameters, resolution, is rather broadly defined. Here, the methods to determine the resolution in X-ray crystallography and single-particle analysis are summarized. In X-ray crystallography, it is becoming increasingly more common to include reflections discarded previously by traditionally used standards, allowing for the inclusion of incomplete and anisotropic reflections into the refinement process. In general, the resolution is the smallest lattice spacing given by Bragg&amp;rsquo;s law for a particular set of X-ray diffraction intensities; however, typically the resolution is truncated by the user during the data processing based on certain parameters and later it is used during refinement. However, at which resolution to perform such a truncation is not always clear and this makes it very confusing for the novices entering the structural biology field. Furthermore, it is argued that the effective resolution should be also reported as it is a more descriptive measure accounting for anisotropy and incompleteness of the data. In single particle cryo-EM, the situation is not much better, as multiple ways exist to determine the resolution, such as Fourier shell correlation, spectral signal-to-noise ratio and the Fourier neighbor correlation. The most widely accepted is the Fourier shell correlation using a threshold of 0.143 to define the resolution (so-called &amp;ldquo;gold-standard&amp;rdquo;), although it is still debated whether this is the correct threshold. Besides, the resolution obtained from the Fourier shell correlation is an estimate of varying resolution across the density map. In reality, the interpretability of the map is more important than the numerical value of the resolution.">
<meta property="og:image" content="https://www.mdpi.com/crystals/crystals-10-00580/article_deploy/html/images/crystals-10-00580-g001-550.jpg">
<link rel="alternate" type="application/rss+xml" title="MDPI Publishing - Latest articles" href="https://www.mdpi.com/rss">
<script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-WPK7SW5"></script><script async="" src="https://www.google-analytics.com/analytics.js"></script><script type="text/x-mathjax-config">
    MathJax.Hub.processSectionDelay = 0;
      MathJax.Hub.Config({
        menuSettings: {
          CHTMLpreview: false
        },
        "CHTML-preview":{
          disabled: true
        },
        "HTML-CSS": {
          availableFonts: ["TeX"],
          preferredFonts: "TeX",
          webFont:"TeX",
          imageFont:"TeX",
          undefinedFamily:"'Arial Unicode MS',serif"
        },
        "TeX": {
          extensions: ["noErrors.js"],
          noErrors: {
            inlineDelimiters: ["",""],
            multiLine: true,
            style: {
              "font-size":   "90%",
              "text-align":  "left",
              "color":       "black",
              "padding":     "1px 3px",
              "border":      "1px solid"
            }
          }
        }
      });
    </script>
<script type="text/javascript" src="MathJax.js"></script>
<meta name="google-site-verification" content="PxTlsg7z2S00aHroktQd57fxygEjMiNHydKn3txhvwY">
<!--[if lt IE 9]>
            <script>var browserIe8 = true;</script>
            <link rel="stylesheet" href="https://www.mdpi.com/assets/css/ie8foundationfix.css?8ad845af75faa4fd">
            <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
            <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.2/html5shiv.js"></script>
            <script src="//s3.amazonaws.com/nwapi/nwmatcher/nwmatcher-1.2.5-min.js"></script>
            <script src="//html5base.googlecode.com/svn-history/r38/trunk/js/selectivizr-1.0.3b.js"></script>
            <script src="//cdnjs.cloudflare.com/ajax/libs/respond.js/1.1.0/respond.min.js"></script>
            <script src="https://www.mdpi.com/assets/js/ie8/ie8patch.js?9e1d3c689a0471df"></script>
            <script src="https://www.mdpi.com/assets/js/ie8/rem.min.js?94b62787dcd6d2f2"></script>            
                                                        <![endif]-->
<script type="text/javascript">
            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
            })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

            ga('create', 'UA-5824718-2', 'auto');
            ga('require', 'displayfeatures');
            ga('send', 'pageview');
        </script>
<script type="text/javascript">
                (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
                new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
                j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
                'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
                })(window,document,'script','dataLayer','GTM-WPK7SW5');
            </script>
<link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/articles.js?a171333ed3a7c6c5"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/jquery-ui-1.10.4.custom.min.js?9fb4d59ff745b497"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/underscore.js?f893e294cde60c24"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/magnific-popup.js?4a09c18460afb26c"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/jquery-scrollspy.js?09cbaec0dbb35a67"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/storage.js?e9b262d3a3476d25"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/affix.js?ac4ea55275297c15"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/jquery-ui-1.12.0.min.js?4c2c52cc11e5a28d"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/clipboard.min.js?3f3688138a1b9fc4"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/affix.js?ac4ea55275297c15"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/ifvisible.min.js?c621d19ecb761212"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/mdpi.js?c5e55b6509d6aab3"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/lib.js?82799462daed82e0"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/jquery.cycle2.min.js?63413052928f97ee"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/jquery.multiselect.js?0edd3998731d1091"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/foundation-5.5.3.equalizer.min.js?0f6c549b75ec554c"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/foundation-5.5.3.min.js?6b2ec41c18b29054"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/jquery-1.12.4.min.js?4f252523d4af0b47"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/modernizr-2.8.3.min.js?5227e0738f7f421d"><link rel="preload" as="script" href="MathJax.js"><script type="text/javascript" src="https://www.mdpi.com/bundles/mathjax/config/TeX-AMS-MML_HTMLorMML.js?V=2.7.1"></script></head>
<body>
<noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WPK7SW5" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
<div class="direction direction_right" id="small_right" style="border-right-width: 0px; padding:0;">
<i class="fa fa-caret-right fa-2x"></i>
</div>
<div class="big_direction direction_right" id="big_right" style="border-right-width: 0px;">
<div style="text-align: right;">
Next Article in Journal<br>
<div><a href="https://www.mdpi.com/2073-4352/10/7/581">Principal Component Analysis (PCA) for Powder Diffraction Data: Towards Unblinded Applications</a></div>
Next Article in Special Issue<br>
<div><a href="https://www.mdpi.com/2073-4352/10/7/583">In Situ Imaging of Domain Structure Evolution in LaBGeO<sub>5</sub> Single Crystals</a></div>
</div>
</div>
<div class="direction" id="small_left" style="border-left-width: 0px">
<i class="fa fa-caret-left fa-2x"></i>
</div>
<div class="big_direction" id="big_left" style="border-left-width: 0px;">
<div>
Previous Article in Journal / Special Issue<br>
<div><a href="https://www.mdpi.com/2073-4352/10/7/578">Partial and Total Substitution of Zn by Mg in the Cu<sub>2</sub>ZnSnS<sub>4</sub> Structure</a></div>
</div>
</div>
<div style="clear: both;"></div>
<div id="menuModal" class="reveal-modal reveal-modal-new reveal-modal-menu" aria-hidden="true" data-reveal="" role="dialog">
<div class="menu-container">
<div class="UI_NavMenu">
<a href="https://www.mdpi.com/about/journals">
<h2>Journals</h2>
</a>
<div class="content__container ">
<div class="custom-accordion-for-small-screen-link ">
<h2>Information</h2>
</div>
<div class="target-item custom-accordion-for-small-screen-content show-for-medium-up">
<div class="menu-container__links">
<div style="width: 100%; max-width: 120px; float: left;">
<a href="https://www.mdpi.com/authors">For Authors</a>
<a href="https://www.mdpi.com/reviewers">For Reviewers</a>
<a href="https://www.mdpi.com/editors">For Editors</a>
<a href="https://www.mdpi.com/librarians">For Librarians</a>
<a href="https://www.mdpi.com/publishing_services">For Publishers</a>
<a href="https://www.mdpi.com/societies">For Societies</a>
</div>
<div style="width: 100%; max-width: 250px; float: left;">
<a href="https://www.mdpi.com/apc">Article Processing Charges</a>
<a href="https://www.mdpi.com/openaccess">Open Access Policy</a>
<a href="https://www.mdpi.com/ioap">Institutional Open Access Program</a>
<a href="https://www.mdpi.com/editorial_process">Editorial Process</a>
<a href="https://www.mdpi.com/awards">Awards</a>
<a href="https://www.mdpi.com/ethics">Research and Publication Ethics</a>
</div>
</div>
</div>
</div>
<a href="https://www.mdpi.com/authors/english">
<h2>Author Services</h2>
</a>
<div class="content__container ">
<div class="custom-accordion-for-small-screen-link ">
<h2>Initiatives</h2>
</div>
<div class="target-item custom-accordion-for-small-screen-content show-for-medium-up">
<div class="menu-container__links">
<div style="width: 100%; float: left;">
<a href="https://sciforum.net/" target="_blank" rel="noopener noreferrer">Sciforum</a>
<a href="https://www.preprints.org/" target="_blank" rel="noopener noreferrer">Preprints</a>
<a href="https://www.scilit.net/" target="_blank" rel="noopener noreferrer">Scilit</a>
<a href="https://sciprofiles.com/" target="_blank" rel="noopener noreferrer">SciProfiles</a>
<a href="https://www.mdpi.com/books" target="_blank" rel="noopener noreferrer">MDPI Books</a>
<a href="https://encyclopedia.pub/" target="_blank" rel="noopener noreferrer">Encyclopedia</a>
<a href="https://jams.pub/" target="_blank" rel="noopener noreferrer">JAMS</a>
<a href="https://www.mdpi.com/about/proceedings">Proceedings</a>
</div>
</div>
</div>
</div>
<a href="https://www.mdpi.com/about">
<h2>About</h2>
</a>
</div>
<div class="menu-container__buttons">
<a class="button UA_SignInUpButton" href="https://www.mdpi.com/user/login">Sign In / Sign Up</a>
</div>
</div>
</div>
<div id="captchaModal" class="reveal-modal reveal-modal-new reveal-modal-new--small" data-reveal="" aria-label="Captcha" aria-hidden="true" role="dialog"></div>
<div id="actionDisabledModal" class="reveal-modal" data-reveal="" aria-labelledby="actionDisableModalTitle" aria-hidden="true" role="dialog" style="width: 300px;">
<h2 id="actionDisableModalTitle">Notice</h2>
<form action="/email/captcha" method="post" id="emailCaptchaForm">
<div class="row">
<div id="js-action-disabled-modal-text" class="small-12 columns">
</div>
<div id="js-action-disabled-modal-submit" class="small-12 columns" style="margin-top: 10px; display: none;">
You can make submissions to other journals
<a href="https://susy.mdpi.com/user/manuscripts/upload" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'event', 'Submit Paper', 'Send', 'Generic');">here</a>.
</div>
</div>
</form>
<a class="close-reveal-modal" aria-label="Close">
<i class="material-icons">clear</i>
</a>
</div>
<div id="container">
<noscript>
                    <div id="no-javascript">
                        You seem to have javascript disabled. Please note that many of the page functionalities won't work as expected without javascript enabled.
                    </div>
                </noscript>
<div class="fixed">
<nav class="tab-bar show-for-medium-down">
<div class="row full-width collapse">
<div class="medium-3 small-4 columns">
<a href="https://www.mdpi.com/">
<img class="full-size-menu__mdpi-logo" src="mdpi-pub-logo-blue-small4.png" style="max-width: 65px;" title="MDPI Open Access Journals">
</a>
</div>
 <div class="medium-3 small-4 columns right-aligned">
<div class="show-for-medium-down">
<a href="#" style="display: none;">
<i class="material-icons" onclick="if (!window.__cfRLUnblockHandlers) return false; $('#menuModal').foundation('reveal', 'close'); return false;">clear</i>
</a>
<a href="#" class="js-open-small-search open-small-search">
<i class="material-icons show-for-small only">search</i>
</a>
<a title="MDPI main page" class="js-open-menu" data-reveal-id="menuModal" href="#">
<i class="material-icons">menu</i>
</a>
</div>
</div>
</div>
</nav>
</div>
<section class="main-section">
<header>
<div class="full-size-menu show-for-large-up">
<div class="row full-width">
<div class="large-3 columns">
<a href="https://www.mdpi.com/">
<img class="full-size-menu__mdpi-logo" src="mdpi-pub-logo-blue-small4.png" title="MDPI Open Access Journals">
</a>
</div>
<div class="large-6 columns text-center UI_NavMenu">
<ul>
<li class="menu-item">
<a href="https://www.mdpi.com/about/journals">Journals</a>
</li>
<li class="menu-item">
<a href="https://www.mdpi.com/authors" data-dropdown="information-dropdown" aria-controls="information-dropdown" aria-expanded="false" data-options="is_hover:true; hover_timeout:200">Information</a>
<ul id="information-dropdown" class="f-dropdown dropdown-wrapper" data-dropdown-content="" aria-hidden="true" tabindex="-1">
<li>
<div class="row">
<div class="small-4 columns right-border">
<ul>
<li>
<a href="https://www.mdpi.com/authors">For Authors</a>
</li>
<li>
<a href="https://www.mdpi.com/reviewers">For Reviewers</a>
</li>
<li>
<a href="https://www.mdpi.com/editors">For Editors</a>
</li>
<li>
<a href="https://www.mdpi.com/librarians">For Librarians</a>
</li>
<li>
<a href="https://www.mdpi.com/publishing_services">For Publishers</a>
</li>
<li>
<a href="https://www.mdpi.com/societies">For Societies</a>
</li>
</ul>
</div>
<div class="small-8 columns">
<ul>
<li>
<a href="https://www.mdpi.com/apc">Article Processing Charges</a>
</li>
<li>
<a href="https://www.mdpi.com/openaccess">Open Access Policy</a>
</li>
<li>
<a href="https://www.mdpi.com/ioap">Institutional Open Access Program</a>
</li>
<li>
<a href="https://www.mdpi.com/editorial_process">Editorial Process</a>
</li>
<li>
<a href="https://www.mdpi.com/awards">Awards</a>
</li>
<li>
<a href="https://www.mdpi.com/ethics">Research and Publication Ethics</a>
</li>
</ul>
</div>
</div>
</li>
</ul>
</li>
<li class="menu-item">
<a href="https://www.mdpi.com/authors/english">Author Services</a>
</li>
<li class="menu-item">
<a href="#" data-dropdown="initiatives-dropdown" aria-controls="initiatives-dropdown" aria-expanded="false" data-options="is_hover: true; hover_timeout: 200">Initiatives</a>
<ul id="initiatives-dropdown" class="f-dropdown dropdown-wrapper dropdown-wrapper__small" data-dropdown-content="" aria-hidden="true" tabindex="-1">
<li>
<div class="row">
<div class="small-12 columns">
<ul>
<li>
<a href="https://sciforum.net/" target="_blank" rel="noopener noreferrer">
Sciforum
</a>
</li>
<li>
<a href="https://www.preprints.org/" target="_blank" rel="noopener noreferrer">
 Preprints
</a>
</li>
<li>
<a href="https://www.scilit.net/" target="_blank" rel="noopener noreferrer">
Scilit
</a>
</li>
<li>
<a href="https://sciprofiles.com/" target="_blank" rel="noopener noreferrer">
SciProfiles
</a>
</li>
<li>
<a href="https://www.mdpi.com/books" target="_blank" rel="noopener noreferrer">
MDPI Books
</a>
 </li>
<li>
<a href="https://encyclopedia.pub/" target="_blank" rel="noopener noreferrer">
Encyclopedia
</a>
</li>
<li>
<a href="https://jams.pub/" target="_blank" rel="noopener noreferrer">
JAMS
</a>
</li>
<li>
<a href="https://www.mdpi.com/about/proceedings">
Proceedings
</a>
</li>
</ul>
</div>
</div>
</li>
</ul>
</li>
<li class="menu-item">
<a href="https://www.mdpi.com/about">About</a>
</li>
</ul>
</div>
<div class="large-3 columns text-right full-size-menu__buttons">
<div>
<a class="button button--default-inversed UA_SignInUpButton" href="https://www.mdpi.com/user/login">Sign In / Sign Up</a>
<a class="button button--default js-journal-active-only-link js-journal-active-only-submit-link UC_NavSubmitButton" href="https://susy.mdpi.com/user/manuscripts/upload?journal=crystals" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'event', 'Submit Paper', 'Send', 'Crystals');" data-disabledmessage="new submissions are not possible.">Submit</a>
</div>
</div>
</div>
</div>
<div class="search-container hide-for-small-down row search-container__default-scheme">
<form id="basic_search" style="background-color: inherit !important;" class="large-12 medium-12 columns " action="/search" method="get">
<div class="row search-container__main-elements">
<div class="large-2 medium-2 small-12 columns text-right1 small-only-text-left">
<div class="show-for-medium-up">
<div class="search-input-label">&nbsp;</div>
</div>
<span class="search-container__title">Search<span class="hide-for-medium"> for Articles</span><span class="hide-for-small">:</span></span>
</div>
<div class="custom-accordion-for-small-screen-content">
<div class="large-2 medium-2 small-6 columns ">
<div class="">
<div class="search-input-label">Title / Keyword</div>
</div>
<input placeholder="Title / Keyword" id="q" tabindex="1" name="q" type="text">
</div>
<div class="large-2 medium-2 small-6 columns ">
<div class="">
<div class="search-input-label">Author / Affiliation</div>
</div>
<input id="authors" placeholder="Author / Affiliation" tabindex="2" name="authors" type="text">
</div>
<div class="large-2 medium-2 small-6 columns ">
<div class="">
<div class="search-input-label">Journal</div>
</div>
<select id="journal" tabindex="3" name="journal" class="chosen-select">
<option value="">All Journals</option>
<option value="acoustics">
Acoustics
</option>
<option value="actuators">
Actuators
</option>
<option value="admsci">
Administrative Sciences
</option>
<option value="adolescents">
Adolescents
</option>
<option value="aerospace">
Aerospace
</option>
<option value="agriculture">
Agriculture
</option>
<option value="agriengineering">
AgriEngineering
</option>
<option value="agronomy">
Agronomy
</option>
<option value="ai">
AI
</option>
<option value="algorithms">
Algorithms
</option>
<option value="allergies">
Allergies
</option>
<option value="analytica">
Analytica
</option>
<option value="animals">
Animals
</option>
<option value="antibiotics">
Antibiotics
</option>
<option value="antibodies">
Antibodies
</option>
<option value="antioxidants">
Antioxidants
</option>
<option value="applmech">
Applied Mechanics
</option>
<option value="applnano">
Applied Nano
</option>
<option value="applsci">
Applied Sciences
</option>
<option value="asi">
Applied System Innovation (ASI)
</option>
<option value="arts">
Arts
</option>
<option value="atmosphere">
Atmosphere
</option>
<option value="atoms">
Atoms
</option>
<option value="audiolres">
Audiology Research
</option>
<option value="automation">
Automation
</option>
<option value="axioms">
Axioms
</option>
<option value="batteries">
Batteries
</option>
<option value="behavsci">
Behavioral Sciences
</option>
<option value="beverages">
Beverages
</option>
<option value="BDCC">
Big Data and Cognitive Computing (BDCC)
</option>
<option value="biochem">
BioChem
</option>
<option value="bioengineering">
Bioengineering
</option>
<option value="biology">
Biology
</option>
<option value="biomechanics">
Biomechanics
</option>
<option value="biomedicines">
Biomedicines
</option>
<option value="biomimetics">
Biomimetics
</option>
<option value="biomolecules">
Biomolecules
</option>
<option value="biosensors">
Biosensors
</option>
<option value="biotech">
BioTech
</option>
<option value="birds">
Birds
</option>
<option value="brainsci">
Brain Sciences
</option>
<option value="buildings">
Buildings
</option>
<option value="businesses">
Businesses
</option>
<option value="carbon">
C
</option>
<option value="cancers">
Cancers
</option>
<option value="cardiogenetics">
Cardiogenetics
</option>
<option value="catalysts">
Catalysts
</option>
<option value="cells">
Cells
</option>
<option value="ceramics">
Ceramics
</option>
<option value="challenges">
Challenges
</option>
<option value="ChemEngineering">
ChemEngineering
</option>
<option value="chemistry">
Chemistry
</option>
<option value="chemproc">
Chemistry Proceedings
</option>
<option value="chemosensors">
Chemosensors
</option>
<option value="children">
Children
</option>
<option value="civileng">
CivilEng
</option>
<option value="cleantechnol">
Clean Technologies (Clean Technol.)
</option>
<option value="climate">
Climate
</option>
<option value="clinpract">
Clinics and Practice
</option>
<option value="clockssleep">
Clocks &amp; Sleep
</option>
<option value="coatings">
Coatings
</option>
<option value="colloids">
Colloids and Interfaces
</option>
<option value="compounds">
Compounds
</option>
<option value="computation">
Computation
</option>
<option value="computers">
Computers
</option>
<option value="condensedmatter">
Condensed Matter
</option>
<option value="constrmater">
Construction Materials
</option>
<option value="cmd">
Corrosion and Materials Degradation (CMD)
</option>
<option value="cosmetics">
Cosmetics
</option>
<option value="cryptography">
Cryptography
</option>
<option value="crystals" selected="selected">
Crystals
</option>
<option value="curroncol">
Current Oncology
</option>
<option value="dairy">
Dairy
</option>
<option value="data">
Data
</option>
<option value="dentistry">
Dentistry Journal
</option>
<option value="dermatopathology">
Dermatopathology
</option>
<option value="designs">
Designs
</option>
<option value="diabetology">
Diabetology
</option>
<option value="diagnostics">
Diagnostics
</option>
<option value="diseases">
Diseases
</option>
<option value="diversity">
Diversity
</option>
<option value="drones">
Drones
</option>
<option value="earth">
Earth
</option>
<option value="ecologies">
Ecologies
</option>
<option value="econometrics">
Econometrics
</option>
<option value="economies">
Economies
</option>
<option value="education">
Education Sciences
</option>
<option value="electricity">
Electricity
</option>
<option value="electrochem">
Electrochem
</option>
<option value="electronicmat">
Electronic Materials
</option>
<option value="electronics">
Electronics
</option>
<option value="encyclopedia">
Encyclopedia
</option>
<option value="endocrines">
Endocrines
</option>
<option value="energies">
Energies
</option>
<option value="eng">
Eng
</option>
<option value="engproc">
Engineering Proceedings
</option>
<option value="entropy">
Entropy
</option>
<option value="environsciproc">
Environmental Sciences Proceedings
</option>
<option value="environments">
Environments
</option>
<option value="epidemiologia">
Epidemiologia
</option>
<option value="epigenomes">
Epigenomes
</option>
<option value="ebj">
European Burn Journal (EBJ)
</option>
<option value="ejihpe">
European Journal of Investigation in Health, Psychology and Education (EJIHPE)
</option>
<option value="fermentation">
Fermentation
</option>
<option value="fibers">
Fibers
</option>
<option value="fire">
Fire
</option>
<option value="fishes">
Fishes
</option>
<option value="fluids">
Fluids
</option>
<option value="foods">
Foods
</option>
<option value="forecasting">
Forecasting
</option>
<option value="forensicsci">
Forensic Sciences
</option>
<option value="forests">
Forests
</option>
<option value="fractalfract">
Fractal and Fractional (Fractal Fract)
</option>
<option value="fuels">
Fuels
</option>
<option value="futureinternet">
Future Internet
</option>
<option value="galaxies">
Galaxies
</option>
<option value="games">
Games
</option>
<option value="gases">
Gases
</option>
<option value="gastroent">
Gastroenterology Insights
</option>
<option value="gastrointestdisord">
Gastrointestinal Disorders (GastrointestDisord)
</option>
<option value="gels">
Gels
</option>
<option value="genealogy">
Genealogy
</option>
<option value="genes">
Genes
</option>
<option value="geographies">
Geographies
</option>
<option value="geohazards">
GeoHazards
</option>
<option value="geosciences">
Geosciences
</option>
<option value="geriatrics">
Geriatrics
</option>
<option value="healthcare">
Healthcare
</option>
<option value="hearts">
Hearts
</option>
<option value="hemato">
Hemato
</option>
<option value="heritage">
Heritage
</option>
<option value="histories">
Histories
</option>
<option value="horticulturae">
Horticulturae
</option>
<option value="humanities">
Humanities
</option>
<option value="hydrogen">
Hydrogen
</option>
<option value="hydrology">
Hydrology
</option>
<option value="idr">
Infectious Disease Reports
</option>
<option value="informatics">
Informatics
</option>
<option value="information">
Information
</option>
<option value="infrastructures">
Infrastructures
</option>
<option value="inorganics">
Inorganics
</option>
<option value="insects">
Insects
</option>
<option value="instruments">
Instruments
</option>
<option value="ijerph">
International Journal of Environmental Research and Public Health (IJERPH)
</option>
<option value="ijfs">
International Journal of Financial Studies (IJFS)
</option>
<option value="ijms">
International Journal of Molecular Sciences (IJMS)
</option>
<option value="IJNS">
International Journal of Neonatal Screening (IJNS)
</option>
<option value="ijtpp">
International Journal of Turbomachinery, Propulsion and Power (IJTPP)
</option>
<option value="inventions">
Inventions
</option>
<option value="IoT">
IoT
</option>
<option value="ijgi">
ISPRS International Journal of Geo-Information (IJGI)
</option>
<option value="J">
J
</option>
<option value="jcdd">
Journal of Cardiovascular Development and Disease (JCDD)
 </option>
<option value="jcm">
Journal of Clinical Medicine (JCM)
</option>
<option value="jcs">
Journal of Composites Science (J. Compos. Sci.)
</option>
<option value="jcp">
Journal of Cybersecurity and Privacy (JCP)
</option>
<option value="jdb">
Journal of Developmental Biology (JDB)
</option>
<option value="jfb">
Journal of Functional Biomaterials (JFB)
</option>
<option value="jfmk">
Journal of Functional Morphology and Kinesiology (JFMK)
</option>
<option value="jof">
Journal of Fungi (JoF)
</option>
<option value="jimaging">
Journal of Imaging (J. Imaging)
</option>
<option value="jintelligence">
Journal of Intelligence (J. Intell.)
</option>
<option value="jlpea">
Journal of Low Power Electronics and Applications (JLPEA)
</option>
<option value="jmmp">
Journal of Manufacturing and Materials Processing (JMMP)
</option>
<option value="jmse">
Journal of Marine Science and Engineering (JMSE)
</option>
<option value="jmp">
Journal of Molecular Pathology (JMP)
</option>
<option value="jnt">
Journal of Nanotheranostics (JNT)
</option>
<option value="jne">
Journal of Nuclear Engineering (JNE)
</option>
<option value="JOItmC">
Journal of Open Innovation: Technology, Market, and Complexity (JOItmC)
</option>
<option value="ohbm">
Journal of Otorhinolaryngology, Hearing and Balance Medicine (OHBM)
</option>
<option value="jpm">
Journal of Personalized Medicine (JPM)
</option>
<option value="jor">
Journal of Respiration (JoR)
</option>
<option value="jrfm">
Journal of Risk and Financial Management (JRFM)
</option>
<option value="jsan">
Journal of Sensor and Actuator Networks (JSAN)
</option>
<option value="jtaer">
Journal of Theoretical and Applied Electronic Commerce Research (JTAER)
</option>
<option value="jox">
Journal of Xenobiotics (JoX)
</option>
<option value="jzbg">
Journal of Zoological and Botanical Gardens (JZBG)
</option>
<option value="journalmedia">
Journalism and Media
</option>
<option value="land">
Land
</option>
<option value="languages">
Languages
</option>
<option value="laws">
Laws
</option>
<option value="life">
Life
</option>
<option value="livers">
Livers
</option>
<option value="logistics">
Logistics
</option>
<option value="lubricants">
Lubricants
</option>
<option value="make">
Machine Learning and Knowledge Extraction (MAKE)
</option>
<option value="machines">
Machines
</option>
<option value="macromol">
Macromol
</option>
<option value="magnetochemistry">
Magnetochemistry
</option>
<option value="marinedrugs">
Marine Drugs
</option>
<option value="materials">
Materials
</option>
<option value="materproc">
Materials Proceedings
</option>
<option value="mca">
Mathematical and Computational Applications (MCA)
</option>
<option value="mathematics">
Mathematics
</option>
<option value="medsci">
Medical Sciences
</option>
<option value="medicina">
Medicina
</option>
<option value="medicines">
Medicines
</option>
<option value="membranes">
Membranes
</option>
<option value="metabolites">
Metabolites
</option>
<option value="metals">
Metals
</option>
<option value="mps">
Methods and Protocols (MPs)
</option>
<option value="microbiolres">
Microbiology Research
</option>
<option value="micromachines">
Micromachines
</option>
<option value="microorganisms">
Microorganisms
</option>
<option value="minerals">
Minerals
</option>
<option value="mining">
Mining
</option>
<option value="modelling">
Modelling
</option>
<option value="molbank">
Molbank
</option>
<option value="molecules">
Molecules
</option>
<option value="mti">
Multimodal Technologies and Interaction (MTI)
</option>
<option value="nanoenergyadv">
Nanoenergy Advances
</option>
<option value="nanomanufacturing">
Nanomanufacturing
</option>
<option value="nanomaterials">
Nanomaterials
</option>
<option value="neuroglia">
Neuroglia
</option>
<option value="neurolint">
Neurology International
</option>
<option value="neurosci">
NeuroSci
</option>
<option value="nitrogen">
Nitrogen
</option>
<option value="ncrna">
Non-Coding RNA (ncRNA)
</option>
<option value="nursrep">
Nursing Reports
</option>
<option value="nutraceuticals">
Nutraceuticals
</option>
<option value="nutrients">
Nutrients
</option>
<option value="obesities">
Obesities
</option>
<option value="oceans">
Oceans
</option>
<option value="optics">
Optics
</option>
<option value="oral">
Oral
</option>
<option value="organics">
Organics
</option>
<option value="osteology">
Osteology
</option>
<option value="parasitologia">
Parasitologia
</option>
<option value="particles">
Particles
</option>
<option value="pathogens">
Pathogens
</option>
<option value="pathophysiology">
Pathophysiology
</option>
<option value="pediatrrep">
Pediatric Reports
</option>
<option value="pharmaceuticals">
Pharmaceuticals
</option>
<option value="pharmaceutics">
Pharmaceutics
</option>
<option value="pharmacy">
Pharmacy
</option>
<option value="philosophies">
Philosophies
</option>
<option value="photonics">
Photonics
</option>
<option value="physchem">
Physchem
</option>
<option value="physics">
Physics
</option>
<option value="plants">
Plants
</option>
<option value="plasma">
Plasma
</option>
<option value="polymers">
Polymers
</option>
<option value="polysaccharides">
Polysaccharides
</option>
<option value="proceedings">
Proceedings
</option>
<option value="processes">
Processes
</option>
<option value="prosthesis">
Prosthesis
</option>
<option value="proteomes">
Proteomes
</option>
<option value="psych">
Psych
</option>
<option value="psychiatryint">
Psychiatry International
</option>
<option value="publications">
Publications
</option>
<option value="qubs">
Quantum Beam Science (QuBS)
</option>
<option value="quantumrep">
Quantum Reports
</option>
<option value="quaternary">
Quaternary
</option>
<option value="radiation">
Radiation
</option>
<option value="reactions">
Reactions
</option>
<option value="recycling">
Recycling
</option>
<option value="religions">
Religions
</option>
<option value="remotesensing">
Remote Sensing
</option>
<option value="reports">
Reports
</option>
<option value="reprodmed">
Reproductive Medicine (Reprod. Med.)
</option>
<option value="resources">
Resources
</option>
<option value="risks">
Risks
</option>
<option value="robotics">
Robotics
</option>
<option value="safety">
Safety
</option>
<option value="sci">
Sci
</option>
<option value="scipharm">
Scientia Pharmaceutica (Sci. Pharm.)
</option>
<option value="sensors">
Sensors
</option>
<option value="separations">
Separations
</option>
<option value="sexes">
Sexes
</option>
<option value="signals">
Signals
</option>
<option value="sinusitis">
Sinusitis
</option>
<option value="smartcities">
Smart Cities
</option>
<option value="socsci">
Social Sciences
</option>
<option value="societies">
Societies
</option>
<option value="soilsystems">
Soil Systems
</option>
<option value="solids">
Solids
</option>
<option value="sports">
Sports
</option>
<option value="stats">
Stats
</option>
<option value="stresses">
Stresses
</option>
<option value="surfaces">
Surfaces
</option>
<option value="surgeries">
Surgeries
</option>
<option value="sustainability">
Sustainability
</option>
<option value="suschem">
Sustainable Chemistry
</option>
<option value="symmetry">
Symmetry
</option>
<option value="systems">
Systems
</option>
<option value="taxonomy">
Taxonomy
</option>
<option value="technologies">
Technologies
</option>
<option value="telecom">
Telecom
</option>
<option value="tourismhosp">
Tourism and Hospitality
</option>
<option value="toxics">
Toxics
</option>
<option value="toxins">
Toxins
</option>
<option value="transplantology">
Transplantology
</option>
<option value="tropicalmed">
Tropical Medicine and Infectious Disease (TropicalMed)
</option>
<option value="universe">
Universe
</option>
<option value="urbansci">
Urban Science
</option>
<option value="uro">
Uro
</option>
<option value="vaccines">
Vaccines
</option>
<option value="vehicles">
Vehicles
</option>
<option value="vetsci">
Veterinary Sciences
</option>
<option value="vibration">
Vibration
</option>
<option value="viruses">
Viruses
</option>
<option value="vision">
Vision
</option>
<option value="water">
Water
</option>
<option value="women">
Women
</option>
<option value="world">
World
</option>
<option value="wevj">
World Electric Vehicle Journal (WEVJ)
</option>
</select>
</div>
<div class="large-2 medium-2 small-6 columns ">
<div class="">
<div class="search-input-label">Article Type</div>
</div>
<select id="article_type" tabindex="4" name="article_type" class="chosen-select">
<option value="" selected="selected">All Article Types</option>
<option value="research-article">Article</option>
<option value="review-article">Review</option>
<option value="rapid-communication">Communication</option>
<option value="editorial">Editorial</option>
<option value="addendum">Addendum</option>
<option value="book-review">Book Review</option>
<option value="brief-report">Brief Report</option>
<option value="case-report">Case Report</option>
<option value="article-commentary">Comment</option>
<option value="commentary">Commentary</option>
<option value="concept-paper">Concept Paper</option>
<option value="conference-report">Conference Report</option>
<option value="correction">Correction</option>
<option value="creative">Creative</option>
<option value="discussion">Discussion</option>
<option value="erratum">Erratum</option>
<option value="essay">Essay</option>
<option value="expression-of-concern">Expression of Concern</option>
<option value="extended-abstract">Extended Abstract</option>
<option value="hypothesis">Hypothesis</option>
<option value="interesting-image">Interesting Images</option>
<option value="letter">Letter</option>
<option value="books-received">New Book Received</option>
<option value="opinion">Opinion</option>
<option value="perspective">Perspective</option>
<option value="proceedings">Proceedings</option>
<option value="project-report">Project Report</option>
<option value="reply">Reply</option>
<option value="retraction">Retraction</option>
<option value="note">Short Note</option>
<option value="study-protocol">Study Protocol</option>
<option value="technical-note">Technical Note</option>
<option value="viewpoint">Viewpoint</option>
</select>
</div>
<div class="large-1 medium-1 small-6 end columns small-push-6 medium-reset-order large-reset-order js-search-collapsed-button-container">
<div class="search-input-label">&nbsp;</div>
<input id="search" value="Search" class="button button--dark button--full-width searchButton1 US_SearchButton" tabindex="12" name="search" type="submit">
</div>
<div class="large-1 medium-1 small-6 end columns large-text-left small-only-text-center small-pull-6 medium-reset-order large-reset-order js-search-collapsed-link-container">
<div class="search-input-label">&nbsp;</div>
<a class="main-search-clear search-container__link" href="#" onclick="if (!window.__cfRLUnblockHandlers) return false; openAdvanced(''); return false;">Advanced<span class="show-for-small-only"> Search</span></a>
</div>
</div>
</div>
<div class="search-container__advanced" style="margin-top: 0; padding-top: 0px; background-color: inherit; color: inherit;">
<div class="row">
<div class="large-2 medium-2 columns show-for-medium-up">&nbsp;</div>
<div class="large-2 medium-2 small-6 columns ">
<div class="">
<div class="search-input-label">Section</div>
</div>
<select id="section" tabindex="5" name="section" class="chosen-select">
<option value="" selected="selected"></option>
</select>
</div>
<div class="large-2 medium-2 small-6 columns ">
<div class="">
<div class="search-input-label">Special Issue</div>
</div>
<select id="special_issue" tabindex="6" name="special_issue" class="chosen-select">
<option value="" selected="selected"></option>
</select>
</div>
<div class="large-1 medium-1 small-6 end columns ">
<div class="search-input-label">Volume</div>
<input id="volume" tabindex="7" name="volume" placeholder="..." value="10" type="text">
</div>
<div class="large-1 medium-1 small-6 end columns ">
<div class="search-input-label">Issue</div>
<input id="issue" tabindex="8" name="issue" placeholder="..." value="7" type="text">
</div>
<div class="large-1 medium-1 small-6 end columns ">
<div class="search-input-label">Number</div>
<input id="number" tabindex="9" name="number" placeholder="..." type="text">
</div>
<div class="large-1 medium-1 small-6 end columns ">
<div class="search-input-label">Page</div>
<input id="page" tabindex="10" name="page" placeholder="..." type="text">
</div>
<div class="large-1 medium-1 small-6 columns small-push-6 medium-reset order large-reset-order medium-reset-order js-search-expanded-button-container"></div>
<div class="large-1 medium-1 small-6 columns large-text-left small-only-text-center small-pull-6 medium-reset-order large-reset-order js-search-expanded-link-container"></div>
</div>
</div>
</form>
<form id="advanced-search" class="large-12 medium-12 columns">
<div class="search-container__advanced">
<div id="advanced-search-template" class="row advanced-search-row">
<div class="large-2 medium-2 small-12 columns show-for-medium-up">&nbsp;</div>
<div class="large-2 medium-2 small-3 columns connector-div">
<div class="search-input-label"><span class="show-for-medium-up">Logical Operator</span><span class="show-for-small">Operator</span></div>
<select class="connector">
<option value="and" selected="selected">AND</option>
<option value="or">OR</option>
</select>
</div>
<div class="large-3 medium-3 small-6 columns search-text-div">
<div class="search-input-label">Search Text</div>
<input class="search-text" placeholder="Search text" type="text">
</div>
<div class="large-2 medium-2 small-6 large-offset-0 medium-offset-0 small-offset-3 columns search-field-div">
<div class="search-input-label">Search Type</div>
<select class="search-field">
<option value="all" selected="selected">All fields</option>
<option value="title">Title</option>
<option value="abstract">Abstract</option>
<option value="keywords">Keywords</option>
<option value="authors">Authors</option>
<option value="affiliations">Affiliations</option>
<option value="doi">Doi</option>
<option value="full_text">Full Text</option>
<option value="references">References</option>
</select>
</div>
<div class="large-1 medium-1 small-3 columns">
<div class="search-input-label">&nbsp;</div>
<div class="search-action-div">
<div class="search-plus">
<i class="material-icons">add_circle_outline</i>
</div>
</div>
<div class="search-action-div">
<div class="search-minus">
<i class="material-icons">remove_circle_outline</i>
</div>
</div>
</div>
<div class="large-1 medium-1 small-6 large-offset-0 medium-offset-0 small-offset-3 end columns">
<div class="search-input-label">&nbsp;</div>
<input class="advanced-search-button button button--dark search-submit" value="Search" type="submit">
</div>
<div class="large-1 medium-1 small-6 end columns show-for-medium-up"></div>
</div>
</div>
</form>
</div>
<div class="breadcrumb row full-row">
<div class="breadcrumb__element">
<a href="https://www.mdpi.com/about/journals">Journals</a>
</div>
<div class="breadcrumb__element">
<a href="https://www.mdpi.com/journal/crystals">Crystals</a>
</div>
<div class="breadcrumb__element">
<a href="https://www.mdpi.com/2073-4352/10">Volume 10</a>
</div>
<div class="breadcrumb__element">
<a href="https://www.mdpi.com/2073-4352/10/7">Issue 7</a>
</div>
<div class="breadcrumb__element">
<a href="#">10.3390/cryst10070580</a>
</div>
</div>
</header>
<div id="main-content" class="">
<div class="row full-width">
<div id="left-column" class="content__column large-3 medium-3 small-12 columns">
<div class="content__container">
<a href="https://www.mdpi.com/journal/crystals">
<img src="crystals-logo.png" alt="crystals-logo" title="Crystals" style="max-height: 60px; margin: 0 0 0 0;">
</a>
<div class="generic-item no-border">
<a class="button button--color button--full-width js-journal-active-only-link js-journal-active-only-submit-link UC_ArticleSubmitButton" href="https://susy.mdpi.com/user/manuscripts/upload?form%5Bjournal_id%5D%3D28" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'event', 'Submit Paper', 'Send', 'Crystals');" data-disabledmessage="creating new submissions is not possible.">
Submit to this Journal
</a>
<a class="button button--color button--full-width js-journal-active-only-link UC_ArticleReviewButton" href="https://susy.mdpi.com/volunteer/journals/review" data-disabledmessage="volunteering as journal reviewer is not possible.">
Review for this Journal
</a>
<a class="button button--color button--full-width js-journal-active-only-link UC_ArticleEditIssueButton" href="https://www.mdpi.com/journalproposal/sendproposalspecialissue/crystals" data-path="/2073-4352/10/7/580/htm" data-disabledmessage="proposing new special issue is not possible.">
Edit a Special Issue
</a>
</div>
<div class="generic-item link-article-menu show-for-small">
<a href="#" class="link-article-menu show-for-small">
<span class="closed"></span>
<span class="open" style="display: none;"></span>
Article Menu
</a>
</div>
<div class="hide-small-down-initially UI_ArticleMenu">
<div class="generic-item">
<h2>Article Menu</h2>
</div>
<ul class="accordion accordion__menu" data-accordion="" data-options="multi_expand:false;toggleable: true">
<li class="accordion-navigation">
<a href="#overview" class="accordion__title">Article Overview</a>
<div id="overview" class="content  UI_ArticleMenu_Overview">
<ul>
<li>
<a href="https://www.mdpi.com/2073-4352/10/7/580#abstract">Abstract</a>
</li>
<li>
<a href="https://www.mdpi.com/openaccess">Open Access and Permissions</a>
</li>
<li>
<a href="https://www.mdpi.com/2073-4352/10/7/580#cite">Share and Cite</a>
</li>
<li>
<a href="https://www.mdpi.com/2073-4352/10/7/580#metrics">Article Metrics</a>
</li>
<li>
<a href="https://www.mdpi.com/2073-4352/10/7/580#related_content">Related Articles</a>
</li>
<li>
<a href="https://www.mdpi.com/2073-4352/10/7/580/reprints">Order Article Reprints</a>
</li>
</ul>
</div>
</li>
<li class="accordion-navigation">
<a href="#versions-div" class="accordion__title">Article Versions</a>
<div id="versions-div" class="content  UI_ArticleMenu_Versions">
<ul>
<li>
<a href="https://www.mdpi.com/2073-4352/10/7/580">Abstract</a>
</li>
<li>
<a href="https://www.mdpi.com/2073-4352/10/7/580/notes" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', '/2073-4352/10/7/580/notes');">Article Versions Notes</a>
</li>
<li>
<a href="https://www.mdpi.com/2073-4352/10/7/580/htm" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', $(this).attr('href'));" id="html_link">Full-Text HTML</a>
</li>
<li class="li-download">
<a href="https://www.mdpi.com/2073-4352/10/7/580/pdf" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', $(this).attr('href'));" id="pdf_link">Full-Text PDF</a>
</li>
<li class="li-download">
<a id="js-xml-access-captcha" href="#" data-target="/2073-4352/10/7/580/xml" class="accessCaptcha" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', '/2073-4352/10/7/580/xml');">Full-Text XML</a>
</li>
<li class="li-download">
<a href="https://www.mdpi.com/2073-4352/10/7/580/epub" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', '/2073-4352/10/7/580/epub');" id="epub_link">Full-Text Epub</a>
</li>
</ul>
</div>
</li>
<li class="accordion-navigation">
<a href="#exports_menu" class="accordion__title">Export Article</a>
<div id="exports_menu" class="content  UI_ArticleMenu_Export">

<form style="margin:0; padding:0; display:inline;" name="export-bibtex" method="POST" action="/export">
<input name="articles_ids[]" value="380528" type="hidden">
<input name="export_format_top" value="bibtex" type="hidden">
<input name="export_submit_top" value="" type="hidden">
</form>

<form style="margin:0; padding:0; display:inline;" name="export-endnote" method="POST" action="/export">
<input name="articles_ids[]" value="380528" type="hidden">
<input name="export_format_top" value="endnote_no_abstract" type="hidden">
<input name="export_submit_top" value="" type="hidden">
</form>

<form style="margin:0; padding:0; display:inline;" name="export-ris" method="POST" action="/export">
<input name="articles_ids[]" value="380528" type="hidden">
<input name="export_format_top" value="ris" type="hidden">
<input name="export_submit_top" value="" type="hidden">
</form>
<ul>
<li class="li-link">
<a href="javascript:window.document.forms['export-bibtex'].submit()">BibTeX</a>
</li>
<li class="li-link">
<a href="javascript:window.document.forms['export-endnote'].submit()">EndNote</a>
</li>
<li class="li-link">
<a href="javascript:window.document.forms['export-ris'].submit()">RIS</a>
</li>
</ul>
</div>
</li>
<li class="accordion-navigation">
<a href="#related" class="accordion__title">Related Info Links</a>
<div id="related" class="content  UI_ArticleMenu_RelatedLinks">
<ul>
<li class="li-link">
<a href="https://scholar.google.com/scholar?q=The%20Resolution%20in%20X-ray%20Crystallography%20and%20Single-Particle%20Cryogenic%20Electron%20Microscopy" target="_blank" rel="noopener noreferrer">Google Scholar</a>
</li>
</ul>
</div>
</li>
<li class="accordion-navigation">
<a href="#authors" class="accordion__title">More by Authors Links</a>
<div id="authors" class="content  UI_ArticleMenu_AuthorsLinks">
 <ul class="side-menu-ul">
<li>
<a class="expand" onclick='if (!window.__cfRLUnblockHandlers) return false; $(this).closest("li").next("div").toggle(); return false;'>on DOAJ</a>
</li>
<div id="AuthorDOAJExpand" style="display:none;">
<ul class="submenu">
<li class="li-link">
<a href="http://doaj.org/search?source=%7B%22query%22%3A%7B%22query_string%22%3A%7B%22query%22%3A%22Victor%20%20R.A.%20Dubach%22%2C%22default_operator%22%3A%22AND%22%7D%7D%7D" target="_blank" rel="noopener noreferrer">Dubach, V. R.A.</a>
</li><li>
</li>
<li class="li-link">
<a href="http://doaj.org/search?source=%7B%22query%22%3A%7B%22query_string%22%3A%7B%22query%22%3A%22Albert%20Guskov%22%2C%22default_operator%22%3A%22AND%22%7D%7D%7D" target="_blank" rel="noopener noreferrer">Guskov, A.</a>
</li><li>
</li>
</ul>
</div>
<li>
<a class="expand" onclick='if (!window.__cfRLUnblockHandlers) return false; $(this).closest("li").next("div").toggle(); return false;'>on Google Scholar</a>
</li>
<div id="AuthorGoogleExpand" style="display:none;">
<ul class="submenu">
<li class="li-link">
<a href="https://scholar.google.com/scholar?q=Victor%20%20R.A.%20Dubach" target="_blank" rel="noopener noreferrer">Dubach, V. R.A.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://scholar.google.com/scholar?q=Albert%20Guskov" target="_blank" rel="noopener noreferrer">Guskov, A.</a>
</li><li>
</li>
</ul>
</div>
<li>
<a class="expand" onclick='if (!window.__cfRLUnblockHandlers) return false; $(this).closest("li").next("div").toggle(); return false;'>on PubMed</a>
</li>
<div id="AuthorPubMedExpand" style="display:none;">
<ul class="submenu">
<li class="li-link">
<a href="https://www.pubmed.gov/?cmd=Search&amp;term=Victor%20%20R.A.%20Dubach" target="_blank" rel="noopener noreferrer">Dubach, V. R.A.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://www.pubmed.gov/?cmd=Search&amp;term=Albert%20Guskov" target="_blank" rel="noopener noreferrer">Guskov, A.</a>
</li><li>
</li>
</ul>
</div>
</ul>
</div>
</li>
<li class="accordion-navigation">
<a href="#html-links" class="accordion__title">Full Article Text</a>
<div id="html-links" class="content active">
<div class="menu-caption" id="html-quick-links-title"></div>
</div>
</li>
</ul>
<div id="scifeed-modal" class="reveal-modal reveal-modal-new" data-reveal="" aria-labelledby="modalTitle" aria-hidden="true" role="dialog">
</div>
<span style="display:none" id="scifeed_hidden_flag"></span>
<span style="display:none" id="scifeed_subscribe_url">/ajax/scifeed/subscribe</span>
</div>
</div>
</div>
<div id="middle-column" class="content__column large-9 medium-9 small-12 columns end middle-bordered">
<div class="middle-column__help">
<div class="middle-column__help__fixed show-for-medium-up">
<a href="#" class="UA_ShareButton" data-reveal-id="main-share-modal" title="Share">
<i class="material-icons">share</i>
</a>
<a href="#" data-reveal-id="main-help-modal" title="Help">
<i class="material-icons">announcement</i>
</a>
<a href="#" class="" data-hypothesis-trigger-endorses-tab="" title="Endorse">
<i data-hypothesis-endorse-trigger="" class="material-icons">thumb_up</i>
<div data-hypothesis-endorsement-count="" data-hypothesis-trigger-endorses-tab="" class="hypothesis-count-container">
...
</div>
</a>
<a href="#" data-hypothesis-trigger="" class="js-hypothesis-open UI_ArticleAnnotationsButton" title="Comment">
<i class="material-icons">textsms</i>
<div data-hypothesis-annotation-count="" class="hypothesis-count-container">
...
</div>
</a>
</div>
<div id="main-help-modal" class="reveal-modal reveal-modal-new" data-reveal="" aria-labelledby="modalTitle" aria-hidden="true" role="dialog">
<div class="row">
<div class="small-12 columns">
<h2 style="margin: 0;">Need Help?</h2>
</div>
<div class="small-6 columns">
<h3>Support</h3>
<p>
Find support for a specific problem on the support section of our website.
</p>
<a target="_blank" href="https://www.mdpi.com/about/contactform" class="button button--color button--full-width">
Get Support
</a>
</div>
<div class="small-6 columns">
<h3>Feedback</h3>
<p>
Please let us know what you think of our products and services.
</p>
<a target="_blank" href="https://www.mdpi.com/feedback/send" class="button button--color button--full-width">
Give Feedback
</a>
</div>
<div class="small-6 columns end">
<h3>Information</h3>
<p>
Our dedicated information section provides allows you to learn more about MDPI.
</p>
<a target="_blank" href="https://www.mdpi.com/authors" class="button button--color button--full-width">
Get Information
</a>
</div>
</div>
<a class="close-reveal-modal" aria-label="Close">
<i class="material-icons">clear</i>
</a>
</div>
</div>
<div class="middle-column__main ">
<div class="html-content__container content__container content__container__combined-for-large content__container__combined-for-large__first" style="overflow: auto;">
<article>
<div class="html-article-content">
<div itemscope="" itemtype="http://schema.org/ScholarlyArticle" id="abstract" class="abstract_div">
<div class="js-check-update-container"></div>
<div>
<span itemprop="publisher" content="Multidisciplinary Digital Publishing Institute"></span><span itemprop="url" content="https://www.mdpi.com/2073-4352/10/7/580"></span>
<div class="article-icons"><span class="label openaccess">Open Access</span><span class="label articletype">Review</span></div>
<h1 class="title hypothesis_container" itemprop="name">
The Resolution in X-ray Crystallography and Single-Particle Cryogenic Electron Microscopy </h1>
<div class="art-authors hypothesis_container">

by
<span class="inlineblock"><div class="sciprofiles-link" style="display: inline-block"><a class="sciprofiles-link__link" href="https://sciprofiles.com/profile/1129436" target="_blank" rel="noopener noreferrer"><img class="sciprofiles-link__image" src="masked-unknown-user.png" style="width: auto; height: 16px;"><span class="sciprofiles-link__name">Victor R.A. Dubach</span></a></div><sup> 1</sup><span style="display: inline; margin-left: 5px;"></span><a class="toEncode emailCaptcha" data-author-id="3143585" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a><a href="https://orcid.org/0000-0002-1657-7184" target="_blank" rel="noopener noreferrer"><img src="orcid.png" title="OrcID" style="position: relative; width: 13px; margin-left: 3px; max-width: 13px !important; height: auto; top: -5px;"></a> and </span><span class="inlineblock"><div class="sciprofiles-link" style="display: inline-block"><a class="sciprofiles-link__link" href="https://sciprofiles.com/profile/221505" target="_blank" rel="noopener noreferrer"><img class="sciprofiles-link__image" src="masked-unknown-user.png" style="width: auto; height: 16px;"><span class="sciprofiles-link__name">Albert Guskov</span></a></div><sup> 2,3,*</sup><span style="display: inline; margin-left: 5px;"></span><a class="toEncode emailCaptcha" data-author-id="3143586" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a><a href="https://orcid.org/0000-0003-2340-2216" target="_blank" rel="noopener noreferrer"><img src="orcid.png" title="OrcID" style="position: relative; width: 13px; margin-left: 3px; max-width: 13px !important; height: auto; top: -5px;"></a></span> 
</div>
<div class="nrm"></div>
<span style="display:block; height:6px;"></span>
<div></div>
<div style="margin: 5px 0 15px 0;" class="hypothesis_container">
<div class="art-affiliations">
<div class="affiliation">
<div class="affiliation-item"><sup>1</sup></div>
<div class="affiliation-name ">Faculty of Science and Engineering, University of Groningen, Nijenborgh 4, 9747 AG Groningen, The Netherlands</div>
</div>
<div class="affiliation">
<div class="affiliation-item"><sup>2</sup></div>
<div class="affiliation-name ">Groningen Biomolecular Sciences &amp; 
Biotechnology Institute (GBB), University of Groningen, Nijenborgh 4, 
9747 AG Groningen, The Netherlands</div>
</div>
<div class="affiliation">
<div class="affiliation-item"><sup>3</sup></div>
<div class="affiliation-name ">Moscow Institute of Physics and Technology, 141701 Dolgoprudny, Russia</div>
</div>
<div class="affiliation">
<div class="affiliation-item"><sup>*</sup></div>
<div class="affiliation-name ">Author to whom correspondence should be addressed. </div>
</div>
</div>
</div>
<div class="bib-identity" style="margin-bottom: 10px;">
<em>Crystals</em> <b>2020</b>, <em>10</em>(7), 580; <a href="https://doi.org/10.3390/cryst10070580">https://doi.org/10.3390/cryst10070580</a>
</div>
<div class="pubhistory" style="font-weight: bold; padding-bottom: 10px;">
Received: 13 May 2020 / Revised: 25 June 2020 / Accepted: 3 July 2020 / Published: 5 July 2020
</div>
<div class="belongsTo" style="margin-bottom: 10px;">
(This article belongs to the Special Issue <a href="https://www.mdpi.com/journal/crystals/special_issues/10-Year_Crystals">A 10 Years Journey: Chemical, Physical, and Biological Properties and Applications of Crystals</a>)
</div>
<div class="highlight-box1">
<div class="download">
<a class="button button--color-inversed margin-bottom-10 UD_ArticlePDF" href="https://www.mdpi.com/2073-4352/10/7/580/pdf" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', '/2073-4352/10/7/580/pdf');">Download PDF</a>
<div class="js-browse-figures" style="display: inline-block;">
<a href="#" class="button button--color-inversed margin-bottom-10 openpopupgallery UI_BrowseArticleFigures" data-target="article-popup">Browse Figure</a>
</div>
<div id="article-popup" class="popupgallery" style="display: inline; line-height: 200%">
<a href="https://www.mdpi.com/crystals/crystals-10-00580/article_deploy/html/images/crystals-10-00580-g001.png" title="
                        &lt;strong&gt;Figure 1&lt;/strong&gt;&lt;br/&gt;
                                                    &lt;p&gt;A typical FSC curve with the thresholds 0.5 and 0.143 indicated.&lt;/p&gt;
                                                ">
</a>
</div>
</div>
</div>
<div class="responsive-moving-container small hidden" data-id="article-counters" style="margin-top: 15px;"></div>
</div>
<div class="html-abstract-title">
<a name="abstractc"></a>
<h2>Abstract</h2>
</div>
<div class="art-abstract in-tab hypothesis_container">
X-ray crystallography and single-particle analysis cryogenic electron 
microscopy are essential techniques for uncovering the three-dimensional
 structures of biological macromolecules. Both techniques rely on the 
Fourier transform to calculate experimental maps. However, one of the 
crucial parameters, resolution, is rather broadly defined. Here, the 
methods to determine the resolution in X-ray crystallography and 
single-particle analysis are summarized. In X-ray crystallography, it is
 becoming increasingly more common to include reflections discarded 
previously by traditionally used standards, allowing for the inclusion 
of incomplete and anisotropic reflections into the refinement process. 
In general, the resolution is the smallest lattice spacing given by 
Braggs law for a particular set of X-ray diffraction intensities; 
however, typically the resolution is truncated by the user during the 
data processing based on certain parameters and later it is used during 
refinement. However, at which resolution to perform such a truncation is
 not always clear and this makes it very confusing for the novices 
entering the structural biology field. Furthermore, it is argued that 
the effective resolution should be also reported as it is a more 
descriptive measure accounting for anisotropy and incompleteness of the 
data. In single particle cryo-EM, the situation is not much better, as 
multiple ways exist to determine the resolution, such as Fourier shell 
correlation, spectral signal-to-noise ratio and the Fourier neighbor 
correlation. The most widely accepted is the Fourier shell correlation 
using a threshold of 0.143 to define the resolution (so-called 
gold-standard), although it is still debated whether this is the 
correct threshold. Besides, the resolution obtained from the Fourier 
shell correlation is an estimate of varying resolution across the 
density map. In reality, the interpretability of the map is more 
important than the numerical value of the resolution.
</div>
<div class="art-keywords in-tab hypothesis_container"><em>Keywords: </em>
<span itemprop="keywords" style="display:none">X-ray crystallography; single-particle analysis; resolution</span>
<span> <a href="https://www.mdpi.com/search?q=X-ray%20crystallography">X-ray crystallography</a>; <a href="https://www.mdpi.com/search?q=single-particle%20analysis">single-particle analysis</a>; <a href="https://www.mdpi.com/search?q=resolution">resolution</a></span>
</div>
</div>
</div> <div class="hypothesis_container">
<ul class="menu html-nav" data-prev-node="#html-quick-links-title">
</ul>
<div class="html-body">
<section id="sec1-crystals-10-00580" type="intro"><h2 data-nested="1"> 1. Introduction</h2><div class="html-p">Seeing
 is believing is at the heart of structural biology. Both X-ray 
crystallography and single-particle cryogenic electron microscopy 
(cryo-EM) have become essential for uncovering the three-dimensional 
(3D) structures of biological macromolecules. With both techniques it is
 possible to obtain the structures with high resolution with the current
 absolute record of 0.48  for crystallography [<a href="#B1-crystals-10-00580" class="html-bibr">1</a>] and near atomic resolution of 1.54  achieved by single particle analysis [<a href="#B2-crystals-10-00580" class="html-bibr">2</a>]
 and ~1  resolution with micro-ED. Resolution in nuclear magnetic 
resonance (NMR) is an entirely different concept and more of a 
philosophical question, and will not be discussed here. Atomic 
resolution allows one to distinguish individual atoms and has its 
certain benefits, for refinement and model building [<a href="#B3-crystals-10-00580" class="html-bibr">3</a>].
 Atomic resolution is not a strictly defined term though. It is 
regularly thought that a resolution of 1.2  or higher is an atomic 
resolution, better known as Sheldricks criterion [<a href="#B4-crystals-10-00580" class="html-bibr">4</a>].
 Near-atomic resolution usually describes maps which are of a resolution
 of 2  or better but is not strictly defined. The terms atomic and 
near-atomic are occasionally misused describing maps not close to 
their respective resolutions [<a href="#B5-crystals-10-00580" class="html-bibr">5</a>].
 Furthermore, to many non-structural biologists it may be very confusing
 how the resolution is defined in X-ray crystallography or cryo-EM.</div><div class="html-p">Resolution
 in X-ray crystallography and cryo-EM is different from the usual 
interpretation of resolution as generally accepted in light microscopy. 
In the light microscopy field, the resolution was first defined by Lord 
Rayleigh as the smallest distance at which two point sources can be 
still distinguished [<a href="#B6-crystals-10-00580" class="html-bibr">6</a>].
 At this distance, the maximum of one point source coincides with the 
first minimum of the other. However, the definition of Lord Rayleigh is 
not applicable in X-ray crystallography and cryo-EM, because both 
techniques make use of Fourier space to determine the resolution of 
data. Fourier space and the nature of the experimental data make the 
resolution determination less clear and can confuse anyone without 
experience with these techniques. The recent technological progress in 
the structural biology and the enormous effort of software developers to
 make their software user-friendly have brought many new users sometimes
 without deep understanding of techniques. The aim of this mini-review 
is to give an introductory overview on how the resolution of data is 
determined in X-ray crystallography and single particle cryo-EM.</div></section><section id="sec2-crystals-10-00580" type=""><h2 data-nested="1"> 2. X-ray&nbsp;Crystallography</h2><div class="html-p">X-ray
 crystallography is the oldest and most productive field of structural 
biology (~145.00 entries in the protein data bank), where crystals of 
the protein of interest are irradiated with X-ray photons. The crystal 
diffracts the X-ray beam into discrete diffraction spots, also called 
reflections. The amplitudes are measured during the experiment and the 
missing phases are obtained via Molecular Replacement (MR) [<a href="#B7-crystals-10-00580" class="html-bibr">7</a>,<a href="#B8-crystals-10-00580" class="html-bibr">8</a>], single- or multiple-wavelength anomalous dispersion (SAD or MAD) [<a href="#B9-crystals-10-00580" class="html-bibr">9</a>,<a href="#B10-crystals-10-00580" class="html-bibr">10</a>], multiple isomorphous replacement (MIR) [<a href="#B11-crystals-10-00580" class="html-bibr">11</a>] or ab initio [<a href="#B12-crystals-10-00580" class="html-bibr">12</a>,<a href="#B13-crystals-10-00580" class="html-bibr">13</a>].
 The farther away the reflections are from the center of the detector, 
the higher resolution information they contain. However, with increasing
 resolution, the signal decreases. At a certain distance from the center
 of the detector, the signal will be too weak and this, in principle, 
will be the resolution limit of the dataset. There are numerous 
statistics for the quality of the data including, but not limited to, 
R-factors, signal-to-noise ratio (<math display="inline"><semantics>
<mrow>
<mi>I</mi>
<mo>/</mo>
<mi></mi>
</mrow>
</semantics></math>) and completeness. These statistical measures are 
also used to truncate the data and thus to decide which reflections are 
not of sufficient quality for the consequent map calculation, followed 
by model building and refinement. This, in a way, also sets the 
resolution limit of the dataset. However, it was shown that in many 
cases inclusion of weak incomplete high-resolution data still improves 
the quality of the model. The standards derived in the past are often 
too strict and underestimate the information in the excluded data [<a href="#B14-crystals-10-00580" class="html-bibr">14</a>,<a href="#B15-crystals-10-00580" class="html-bibr">15</a>,<a href="#B16-crystals-10-00580" class="html-bibr">16</a>].
 This ignited a debate about the usefulness of information contained in 
the weak high-resolution reflections and the general consensus now is to
 use all the available data with a few considerations (see below) [<a href="#B17-crystals-10-00580" class="html-bibr">17</a>,<a href="#B18-crystals-10-00580" class="html-bibr">18</a>,<a href="#B19-crystals-10-00580" class="html-bibr">19</a>].
 It also showed a flaw in the resolution normally reported in Table 1 or
 in the Protein Data Bank (PDB) entries. The current recommendation is 
to diligently report if incomplete anisotropic data were used.</div><section id="sec2dot1-crystals-10-00580" type=""><h4 class="html-italic" data-nested="2"> 2.1. Resolution&nbsp;Cutoff</h4><div class="html-p">Where
 to truncate my data? (Also known as resolution cut-off) - Is one of 
the most popular questions asked when people start learning protein 
crystallography. Old textbooks recommend to keep only the strongest 
reflections and truncate the data at the threshold where the 
signal-to-noise ratio equals 2 for the highest resolution shell; 
gradually this requirement was relaxed to about 1.5 and sometimes even 
to 1.0 but recently even this threshold has been questioned [<a href="#B14-crystals-10-00580" class="html-bibr">14</a>,<a href="#B15-crystals-10-00580" class="html-bibr">15</a>,<a href="#B16-crystals-10-00580" class="html-bibr">16</a>]. The classic indicators used to determine where to truncate the data are the signal-to-noise ratio &lt;<math display="inline"><semantics>
<mrow>
<mi>I</mi>
<mo>/</mo>
<mi></mi>
<mo>(</mo>
<mi>I</mi>
<mo>)</mo>
</mrow>
</semantics></math>&gt; and the <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math>, which was first introduced as <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>s</mi>
<mi>y</mi>
<mi>m</mi>
</mrow>
</msub>
</semantics></math> [<a href="#B20-crystals-10-00580" class="html-bibr">20</a>]. Nowadays, <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> and <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>s</mi>
<mi>y</mi>
<mi>m</mi>
</mrow>
</msub>
</semantics></math> are used interchangeably; however, historically <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>s</mi>
<mi>y</mi>
<mi>m</mi>
</mrow>
</msub>
</semantics></math> was used for symmetry-related reflections, whereas <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> was introduced to evaluate the difference between 
different datasets. The truncation deemed necessary as at low 
signal-to-noise ratio it would be hard to differentiate the signal from 
the stochastic noise, hence bringing the risk that noise is mixed with 
the signal and that it will be incorporated into the calculated electron
 density map. Therefore, the data is truncated and with signal-to-noise 
threshold of 2 all reflections with a signal less than twice the 
intensity of the estimated noise will be discarded as a safety measure 
irrespective whether they contain useful information or not. The <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> is a statistic for the precision of the measurements
 of each unique reflection (i.e., it is a measure of agreement among 
multiple measurements of the same reflection):<div class="html-disp-formula-info" id="FD1-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
<mo>=</mo>
<mfrac>
<mrow>
<mstyle displaystyle="true">
<munder>
 <mo></mo>
<mrow>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
</mrow>
</munder>
</mstyle>
<mstyle displaystyle="true">
<munderover>
<mo></mo>
<mrow>
<mi>i</mi>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mi>n</mi>
</munderover>
</mstyle>
<mrow>
<mo>|</mo>
<msub>
<mi>I</mi>
<mi>i</mi>
</msub>
<mrow>
<mo>(</mo>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
<mo>)</mo>
</mrow>
<mo></mo>
<mover>
<mi>I</mi>
<mo></mo>
</mover>
<mrow>
<mo>(</mo>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
<mo>)</mo>
</mrow>
<mo>|</mo>
</mrow>
</mrow>
<mrow>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
</mrow>
</munder>
</mstyle>
<mstyle displaystyle="true">
<munderover>
<mo></mo>
<mrow>
<mi>i</mi>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mi>n</mi>
</munderover>
</mstyle>
<msub>
<mi>I</mi>
<mi>i</mi>
</msub>
<mrow>
<mo>(</mo>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
<mo>)</mo>
</mrow>
</mrow>
</mfrac>
<mo>,</mo>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(1)</label>
</div>
</div>
where <math display="inline"><semantics>
<mrow>
<msub>
<mi>I</mi>
<mi>i</mi>
</msub>
<mrow>
<mo>(</mo>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
<mo>)</mo>
</mrow>
</mrow>
</semantics></math> is the intensity of the reflections, <math display="inline"><semantics>
<mrow>
<mover>
<mi>I</mi>
<mo></mo>
</mover>
<mrow>
<mo>(</mo>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
<mo>)</mo>
</mrow>
</mrow>
</semantics></math> is the average intensity, and they are summed over 
the measured reflections. Many reflections are measured more than once 
as they are symmetry-related. <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> indicates how much measurements of the same 
reflection differ in intensity from the average intensity of that 
reflection. It was postulated that large <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> shows that the measurements of the same reflection 
are not similar (error prone), hence the data should be truncated at the
 resolution shell where <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> is over an arbitrary limit (typically 4060%).</div><div class="html-p">However, <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> is inherently flawed [<a href="#B21-crystals-10-00580" class="html-bibr">21</a>],
 as it is dependent on multiplicity (also called redundancy) and its 
value increases with more measurements of the same reflection, even 
though the precision of the measurement goes up. Diederichs and Karplus 
(1997) introduced a multiplicity-independent R-factor named <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>a</mi>
<mi>s</mi>
</mrow>
</msub>
</semantics></math> [<a href="#B21-crystals-10-00580" class="html-bibr">21</a>]:<div class="html-disp-formula-info" id="FD2-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>a</mi>
<mi>s</mi>
</mrow>
</msub>
<mo>=</mo>
<mfrac>
<mrow>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
</mrow>
</munder>
</mstyle>
<msqrt>
<mfrac>
<msub>
<mi>n</mi>
<mrow>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
</mrow>
</msub>
<mrow>
<msub>
<mi>n</mi>
<mrow>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
</mrow>
</msub>
<mo></mo>
<mn>1</mn>
</mrow>
</mfrac>
</msqrt>
<mstyle displaystyle="true">
<munderover>
<mo></mo>
<mrow>
<mi>i</mi>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mi>n</mi>
</munderover>
</mstyle>
<mrow>
<mo>|</mo>
<msub>
<mi>I</mi>
<mi>i</mi>
</msub>
<mrow>
<mo>(</mo>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
<mo>)</mo>
</mrow>
<mo></mo>
<mover>
<mi>I</mi>
<mo></mo>
</mover>
<mrow>
<mo>(</mo>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
<mo>)</mo>
</mrow>
<mo>|</mo>
</mrow>
</mrow>
<mrow>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
</mrow>
</munder>
</mstyle>
<mstyle displaystyle="true">
<munderover>
<mo></mo>
<mrow>
<mi>i</mi>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mi>n</mi>
</munderover>
</mstyle>
<msub>
<mi>I</mi>
<mi>i</mi>
</msub>
<mrow>
<mo>(</mo>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
<mo>)</mo>
</mrow>
</mrow>
</mfrac>
<mo>.</mo>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(2)</label>
</div>
</div>
where each reflection is corrected with a factor of <math display="inline"><semantics>
<msqrt>
<mfrac>
<msub>
<mi>n</mi>
<mrow>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
</mrow>
</msub>
<mrow>
<msub>
<mi>n</mi>
<mrow>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
</mrow>
</msub>
<mo></mo>
<mn>1</mn>
</mrow>
</mfrac>
</msqrt>
</semantics></math>, where <math display="inline"><semantics>
<msub>
<mi>n</mi>
<mrow>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
</mrow>
</msub>
</semantics></math> denotes the multiplicity of the reflection. When corrected, the outcome stays constant with varying multiplicity, while <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> would increase or decrease. Therefore, <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>a</mi>
<mi>s</mi>
</mrow>
</msub>
</semantics></math> indicates the real precision of the measurement, 
independent of the multiplicity of the reflection. Unfortunately, many 
users keep resisting to report <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>a</mi>
<mi>s</mi>
</mrow>
</msub>
</semantics></math> simply because it shows a higher (but more realistic) value than <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> and many erroneously think that the lower value the better.</div><div class="html-p"><math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> (<math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>s</mi>
<mi>y</mi>
<mi>m</mi>
</mrow>
</msub>
</semantics></math>) and <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>a</mi>
<mi>s</mi>
</mrow>
</msub>
</semantics></math> are used to evaluate individual (unmerged) 
reflections and for merged (i.e., averaged) reflections another R-factor
 was introduced [<a href="#B22-crystals-10-00580" class="html-bibr">22</a>], namely <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>p</mi>
<mo>.</mo>
<mi>i</mi>
<mo>.</mo>
<mi>m</mi>
<mo>.</mo>
</mrow>
</msub>
</semantics></math> (precision-indicating merging R-factor)
<div class="html-disp-formula-info" id="FD3-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<msub>
<mi>R</mi>
<mrow>
<mi>p</mi>
<mo>.</mo>
<mi>i</mi>
<mo>.</mo>
<mi>m</mi>
<mo>.</mo>
</mrow>
</msub>
<mo>=</mo>
<mfrac>
<mrow>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
</mrow>
</munder>
</mstyle>
<msqrt>
<mfrac>
<mn>1</mn>
<mrow>
<msub>
<mi>n</mi>
<mrow>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
</mrow>
</msub>
<mo></mo>
<mn>1</mn>
</mrow>
</mfrac>
</msqrt>
<mstyle displaystyle="true">
<munderover>
<mo></mo>
<mrow>
<mi>i</mi>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mi>n</mi>
</munderover>
</mstyle>
<mrow>
<mo>|</mo>
<msub>
<mi>I</mi>
<mi>i</mi>
</msub>
<mrow>
<mo>(</mo>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
<mo>)</mo>
</mrow>
<mo></mo>
<mover>
<mi>I</mi>
<mo></mo>
</mover>
<mrow>
<mo>(</mo>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
<mo>)</mo>
</mrow>
<mo>|</mo>
</mrow>
</mrow>
<mrow>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
</mrow>
</munder>
</mstyle>
<mstyle displaystyle="true">
<munderover>
<mo></mo>
<mrow>
<mi>i</mi>
<mo>=</mo>
<mn>1</mn>
</mrow>
<mi>n</mi>
</munderover>
</mstyle>
<msub>
<mi>I</mi>
<mi>i</mi>
</msub>
<mrow>
<mo>(</mo>
<mi>h</mi>
<mi>k</mi>
<mi>l</mi>
<mo>)</mo>
</mrow>
</mrow>
</mfrac>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(3)</label>
</div>
</div></div><div class="html-p">The <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> or <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>a</mi>
<mi>s</mi>
</mrow>
</msub>
</semantics></math> were often used in tandem with the signal-to-noise ratio as a resolution cut-off. Commonly, if the <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>a</mi>
<mi>s</mi>
</mrow>
</msub>
</semantics></math> rose above <math display="inline"><semantics>
<mrow>
<mn>60</mn>
<mo>%</mo>
</mrow>
</semantics></math> or &lt;<math display="inline"><semantics>
<mrow>
<mi>I</mi>
<mo>/</mo>
<mi></mi>
<mo>(</mo>
<mi>I</mi>
<mo>)</mo>
</mrow>
</semantics></math>&gt; dropped below 2, the reflections would be 
considered not good enough to be included in the map calculation and 
would be discarded. It seemed to be a good way to determine where to 
truncate the data; however, several scientists have shown that the 
inclusion into refinement of weak high-resolution data, with an &lt;<math display="inline"><semantics>
<mrow>
<mi>I</mi>
<mo>/</mo>
<mi></mi>
<mo>(</mo>
<mi>I</mi>
<mo>)</mo>
</mrow>
</semantics></math>&gt; below 1 and an <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>a</mi>
<mi>s</mi>
</mrow>
</msub>
</semantics></math> well over 100%, might be beneficial [<a href="#B14-crystals-10-00580" class="html-bibr">14</a>,<a href="#B15-crystals-10-00580" class="html-bibr">15</a>,<a href="#B16-crystals-10-00580" class="html-bibr">16</a>].</div><div class="html-p">Karplus and Diederichs proposed a new statistic as a quality indicator to define the resolution limit of a dataset [<a href="#B14-crystals-10-00580" class="html-bibr">14</a>]. They introduced a Pearsons correlation coefficient, <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mn>1</mn>
<mo>/</mo>
<mn>2</mn>
</mrow>
</msub>
</mrow>
</semantics></math>. Correlation coefficients are widely used in cryo-EM
 (discussed below) and in X-ray crystallography for anomalous phasing. 
Furthermore, as pointed out by Evans and Murshudov, correlation 
coefficients report the degree of linear dependence between data and are
 less dependent on the distribution of the data [<a href="#B15-crystals-10-00580" class="html-bibr">15</a>], and therefore they are better indicators. <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mn>1</mn>
<mo>/</mo>
<mn>2</mn>
</mrow>
</msub>
</mrow>
</semantics></math> is based on the random division of the complete set 
of reflections in two equal parts and calculating the correlation 
between the intensity estimates of the two subsets (i.e., how well one 
half of the data predicts the other half). A value of 1 indicates a 
perfect correlation, whereas 0 indicates no correlation at all. At low 
resolution (where the strongest reflections are measured), the 
correlation is around 1 and it goes down with the higher resolution 
approaching zero at the highest resolution (where the weakest 
reflections are measured). <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mn>1</mn>
<mo>/</mo>
<mn>2</mn>
</mrow>
</msub>
</mrow>
</semantics></math>, by definition cannot be used though to estimate the
 quality of the data after merging, thus Karplus and Diederichs also 
introduced <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msup>
<mi>C</mi>
<mo>*</mo>
</msup>
</mrow>
</semantics></math>,
<div class="html-disp-formula-info" id="FD4-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<mi>C</mi>
<msup>
<mi>C</mi>
<mo>*</mo>
</msup>
<mo>=</mo>
<msqrt>
<mfrac>
<mrow>
<mn>2</mn>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mn>1</mn>
<mo>/</mo>
<mn>2</mn>
</mrow>
</msub>
</mrow>
<mrow>
<mn>1</mn>
<mo>+</mo>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mn>1</mn>
<mo>/</mo>
<mn>2</mn>
</mrow>
</msub>
</mrow>
</mfrac>
</msqrt>
<mo>.</mo>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(4)</label>
</div>
</div></div><div class="html-p"><math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msup>
<mi>C</mi>
<mo>*</mo>
</msup>
</mrow>
</semantics></math> provides an estimate of the <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<mi>C</mi>
</mrow>
</semantics></math> that would be obtained between the final merged dataset and the unknown true values that they are representing [<a href="#B14-crystals-10-00580" class="html-bibr">14</a>]. Importantly, it allows comparing data and model quality by calculating <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mi>f</mi>
<mi>r</mi>
<mi>e</mi>
<mi>e</mi>
</mrow>
</msub>
</mrow>
</semantics></math> and <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mi>w</mi>
<mi>o</mi>
<mi>r</mi>
<mi>k</mi>
</mrow>
</msub>
</mrow>
</semantics></math> as the CC between <math display="inline"><semantics>
<msubsup>
<mi>F</mi>
<mrow>
<mi>c</mi>
<mi>a</mi>
<mi>l</mi>
<mi>c</mi>
</mrow>
<mn>2</mn>
</msubsup>
</semantics></math> (square of the calculated structure factor) and <math display="inline"><semantics>
<msubsup>
<mi>F</mi>
<mrow>
<mi>o</mi>
<mi>b</mi>
<mi>s</mi>
</mrow>
<mn>2</mn>
</msubsup>
</semantics></math> (square of the observed structure factor), respectively.</div><div class="html-p">It
 has been shown by many that using this novel statistic for resolution 
truncation is very beneficial for getting better resolved maps, improved
 refinement and improved models [<a href="#B23-crystals-10-00580" class="html-bibr">23</a>,<a href="#B24-crystals-10-00580" class="html-bibr">24</a>] and it allowed for better merging of datasets from different crystals [<a href="#B25-crystals-10-00580" class="html-bibr">25</a>,<a href="#B26-crystals-10-00580" class="html-bibr">26</a>]. If classical criteria for resolution truncation were considered in those cases the numbers would be shockingwith <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>a</mi>
<mi>s</mi>
</mrow>
</msub>
</semantics></math> values up to 1000% and signal-to-noise ratio of 0.3. However, as elaborated by Karplus &amp; Diederichs, <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>a</mi>
<mi>s</mi>
</mrow>
</msub>
</semantics></math> or <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> both approach infinity with increasing resolution, thus rendering them useless as quality indicators.</div><div class="html-p">Nevertheless, there is a problem with <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mn>1</mn>
<mo>/</mo>
<mn>2</mn>
</mrow>
</msub>
</mrow>
</semantics></math> as it does not provide one rock solid threshold, e.g., that data below <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mn>1</mn>
<mo>/</mo>
<mn>2</mn>
</mrow>
</msub>
<mo>=</mo>
<mn>0.3</mn>
</mrow>
</semantics></math> shall be discarded. In reality depending on data quality, the useful range of <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mn>1</mn>
<mo>/</mo>
<mn>2</mn>
</mrow>
</msub>
</mrow>
</semantics></math> lies somewhere between 0.1 and 0.5 [<a href="#B18-crystals-10-00580" class="html-bibr">18</a>,<a href="#B27-crystals-10-00580" class="html-bibr">27</a>]. To determine the exact value of <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mn>1</mn>
<mo>/</mo>
<mn>2</mn>
</mrow>
</msub>
</mrow>
</semantics></math> for the particular dataset the paired refinement procedure is recommended [<a href="#B14-crystals-10-00580" class="html-bibr">14</a>].
 In brief, the quality of a model is evaluated against the dataset 
truncated at different resolution cut-offs and if any additional batch 
of high-resolution reflections does not contribute positively in the 
model quality, those reflections are discarded. The easiest way to 
perform it, is to run it via PDB Redo server [<a href="#B28-crystals-10-00580" class="html-bibr">28</a>], which is also useful to correct modelling errors.</div></section><section id="sec2dot2-crystals-10-00580" type=""><h4 class="html-italic" data-nested="2"> 2.2. Resolution of the&nbsp;Dataset</h4><div class="html-p">First
 and foremost, we need to understand how the resolution is defined in 
X-ray crystallography. What is normally referred to as resolution is the
 nominal resolution (<math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>h</mi>
<mi>i</mi>
<mi>g</mi>
<mi>h</mi>
</mrow>
</msub>
</semantics></math>). This is defined by the smallest distance 
(typically measured in ) between crystal lattice planes that is 
resolved in the diffraction pattern, i.e., if this number is lower, the 
resolution is higher and vice versa. Generally resolution of 0.51  is 
called subatomic (or ultra-high), 11.5  (1.2  according to Sheldrick [<a href="#B4-crystals-10-00580" class="html-bibr">4</a>]) atomic, 1.52  high, 23  medium, 35  low, and worse than 5  very low resolution. In a way, <math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>h</mi>
<mi>i</mi>
<mi>g</mi>
<mi>h</mi>
</mrow>
</msub>
</semantics></math> is dictated by the resolution cut-off (truncation) 
which was applied during the data processing. However, this is very 
subjective (see above). Furthermore, <math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>h</mi>
<mi>i</mi>
<mi>g</mi>
<mi>h</mi>
</mrow>
</msub>
</semantics></math> is not accommodating the data completeness or 
anisotropy, but it merely reflects the highest resolution shells used 
regardless of the completeness and anisotropy of the dataset. The 
nominal resolution is therefore much more an indicator of where someone 
has truncated their data and says nothing about its quality.</div><div class="html-p">As an alternative, the optical resolution (<math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>o</mi>
<mi>p</mi>
<mi>t</mi>
</mrow>
</msub>
</semantics></math>) can be used [<a href="#B29-crystals-10-00580" class="html-bibr">29</a>,<a href="#B30-crystals-10-00580" class="html-bibr">30</a>].
 It should not be confused though with the optical resolution in light 
microscopy, where it can be defined by the minimum distance <span class="html-italic">r</span>
 at which the points on a specimen can be distinguished as individual 
entities. In X-ray crystallography Vaguine et al. (1999) defined the 
optical resolution as the expected minimum distance between two resolved
 peaks (which shapes are fitted by a Gaussian) in the electron-density 
map, i.e., it shows the smallest distance at which two peaks can still 
be seen as separate, if one had a perfect electron-density map with all 
the exact phases [<a href="#B30-crystals-10-00580" class="html-bibr">30</a>].
<div class="html-disp-formula-info" id="FD5-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<msub>
<mi>d</mi>
<mrow>
<mi>o</mi>
<mi>p</mi>
<mi>t</mi>
</mrow>
</msub>
<mo>=</mo>
<msqrt>
<mrow>
<mn>2</mn>
<mo stretchy="false">(</mo>
<msubsup>
<mi></mi>
<mrow>
<mi>P</mi>
<mi>a</mi>
<mi>t</mi>
<mi>t</mi>
</mrow>
<mn>2</mn>
</msubsup>
<mo>+</mo>
<msubsup>
<mi></mi>
<mrow>
<mi>s</mi>
<mi>p</mi>
<mi>h</mi>
</mrow>
<mn>2</mn>
</msubsup>
<mo stretchy="false">)</mo>
</mrow>
</msqrt>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(5)</label>
</div>
</div>
where <math display="inline"><semantics>
<msub>
<mi></mi>
<mrow>
<mi>P</mi>
<mi>a</mi>
<mi>t</mi>
<mi>t</mi>
</mrow>
</msub>
</semantics></math> is the standard deviation of the Gaussian function fitted to the Patterson origin peak and <math display="inline"><semantics>
<msub>
<mi></mi>
<mrow>
<mi>s</mi>
<mi>p</mi>
<mi>h</mi>
</mrow>
</msub>
</semantics></math> is the standard deviation of the Gaussian function 
fitted to the origin peak of the spherical interference function, 
representing the Fourier transform of a sphere with radius <math display="inline"><semantics>
<mrow>
<mn>1</mn>
<mo>/</mo>
<msub>
<mi>d</mi>
<mrow>
<mi>h</mi>
<mi>i</mi>
<mi>g</mi>
<mi>h</mi>
</mrow>
</msub>
</mrow>
</semantics></math>.</div><div class="html-p">The optical resolution is 
always higher than the nominal resolution of a dataset. However, an 
increase in the nominal resolution gives a smaller increase in the 
optical resolution [<a href="#B18-crystals-10-00580" class="html-bibr">18</a>]. According to Vaguine et al. (1999), <math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>o</mi>
<mi>p</mi>
<mi>t</mi>
</mrow>
</msub>
</semantics></math> equals twice the standard deviation of a fit to 
Gaussian; however, later Urzhemtseva et al. (2013) noticed that such an 
approach is sub-optimal as it demonstrated inconsistent results [<a href="#B29-crystals-10-00580" class="html-bibr">29</a>], e.g., the <math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>o</mi>
<mi>p</mi>
<mi>t</mi>
</mrow>
</msub>
</semantics></math> of an incomplete dataset is higher than that of the 
complete dataset. Urzhemtseva et al. (2013) suggested an improved method
 to determine the <math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>o</mi>
<mi>p</mi>
<mi>t</mi>
</mrow>
</msub>
</semantics></math>, without a need to perform a fit to Gaussian, by calculating <math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>o</mi>
<mi>p</mi>
<mi>t</mi>
</mrow>
</msub>
</semantics></math> as the minimum distance at which two C atoms could still be resolved with a typical B-factor [<a href="#B29-crystals-10-00580" class="html-bibr">29</a>]
 (also called atomic displacement parameter (ADP), which describes the 
displacement of an atomi.e., the atoms cannot be considered as immobile
 point scatterers). The B-factor are generally estimated from the Wilson
 plot [<a href="#B31-crystals-10-00580" class="html-bibr">31</a>]. 
Unfortunately, the optical resolution does suffer from the same flaws as
 the nominal resolution does: it does not reflect the completeness or 
the anisotropy of the dataset. Moreover, the <math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>o</mi>
<mi>p</mi>
<mi>t</mi>
</mrow>
</msub>
</semantics></math> is calculated using the Wilson plot which describes 
the typical B-factor of a scatterer at a given resolution. However, in 
this way the real B-factors at high resolution are systematically 
underestimated. Nevertheless, it can be used to compare different 
datasets, especially in combination with other statistical indicators, 
such as the <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mn>1</mn>
<mo>/</mo>
<mn>2</mn>
</mrow>
</msub>
</mrow>
</semantics></math> and the <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msup>
<mi>C</mi>
<mo>*</mo>
</msup>
</mrow>
</semantics></math>, to evaluate the dataset.</div><div class="html-p">To account for the completeness of the data, Weiss suggested yet another resolution, termed effective resolution (<math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>e</mi>
<mi>f</mi>
<mi>f</mi>
</mrow>
</msub>
</semantics></math>) and it has a semi-empirical correlation with the completeness of the dataset [<a href="#B32-crystals-10-00580" class="html-bibr">32</a>]. It was defined as the nominal resolution multiplied with the cube root of the completeness of the dataset:<div class="html-disp-formula-info" id="FD6-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<msub>
<mi>d</mi>
<mrow>
<mi>e</mi>
<mi>f</mi>
<mi>f</mi>
</mrow>
</msub>
<mo>=</mo>
<msub>
<mi>d</mi>
<mrow>
<mi>h</mi>
<mi>i</mi>
<mi>g</mi>
<mi>h</mi>
</mrow>
</msub>
<msup>
<mi>C</mi>
<mrow>
<mo></mo>
<mn>1</mn>
<mo>/</mo>
<mn>3</mn>
</mrow>
</msup>
<mo>.</mo>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(6)</label>
</div>
</div></div><div class="html-p">In such a form <math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>e</mi>
<mi>f</mi>
<mi>f</mi>
</mrow>
</msub>
</semantics></math> only accounts for the completeness but not the 
anisotropy of the data. To overcome this limitation a more complete 
derivation was suggested [<a href="#B29-crystals-10-00580" class="html-bibr">29</a>,<a href="#B33-crystals-10-00580" class="html-bibr">33</a>]. In short, <math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>e</mi>
<mi>f</mi>
<mi>f</mi>
</mrow>
</msub>
</semantics></math> can be defined via comparison of the calculated 
minimum distance for point scatterers with the theoretical values for 
the complete dataset [<a href="#B29-crystals-10-00580" class="html-bibr">29</a>]. Furthermore, the anisotropy of the dataset can be characterized by calculating <math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>e</mi>
<mi>f</mi>
<mi>f</mi>
</mrow>
</msub>
</semantics></math> in different directions. The ratio of the highest effective resolution (<math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>e</mi>
<mi>f</mi>
<mi>f</mi>
<mo>,</mo>
<mi>h</mi>
<mi>i</mi>
<mi>g</mi>
<mi>h</mi>
<mi>e</mi>
<mi>s</mi>
<mi>t</mi>
</mrow>
</msub>
</semantics></math>) and the lowest effective resolution (<math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>e</mi>
<mi>f</mi>
<mi>f</mi>
<mo>,</mo>
<mi>l</mi>
<mi>o</mi>
<mi>w</mi>
<mi>e</mi>
<mi>s</mi>
<mi>t</mi>
</mrow>
</msub>
</semantics></math>) defines the anisotropy <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>e</mi>
<mi>f</mi>
<mi>f</mi>
</mrow>
</msub>
</semantics></math>, where
<div class="html-disp-formula-info" id="FD7-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<msub>
<mi>R</mi>
<mrow>
<mi>e</mi>
<mi>f</mi>
<mi>f</mi>
</mrow>
</msub>
<mo>=</mo>
<msub>
<mi>d</mi>
<mrow>
<mi>e</mi>
<mi>f</mi>
<mi>f</mi>
<mo>,</mo>
<mi>h</mi>
<mi>i</mi>
<mi>g</mi>
<mi>h</mi>
<mi>e</mi>
<mi>s</mi>
<mi>t</mi>
</mrow>
</msub>
<mo>/</mo>
<msub>
<mi>d</mi>
<mrow>
<mi>e</mi>
<mi>f</mi>
<mi>f</mi>
<mo>,</mo>
<mi>l</mi>
<mi>o</mi>
<mi>w</mi>
<mi>e</mi>
<mi>s</mi>
<mi>t</mi>
</mrow>
</msub>
<mo>.</mo>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(7)</label>
</div>
</div></div><div class="html-p">The effective resolution coincides with <math display="inline"><semantics>
<msub>
<mi>d</mi>
<mrow>
<mi>h</mi>
<mi>i</mi>
<mi>g</mi>
<mi>h</mi>
</mrow>
</msub>
</semantics></math> if the dataset is complete and does not have anisotropy.</div></section></section><section id="sec3-crystals-10-00580" type=""><h2 data-nested="1"> 3. Electron&nbsp;Microscopy</h2><div class="html-p">Cryo-EM
 has recently become a very popular structural biology technique due to 
the several technological advances, such as direct electron detectors [<a href="#B34-crystals-10-00580" class="html-bibr">34</a>,<a href="#B35-crystals-10-00580" class="html-bibr">35</a>,<a href="#B36-crystals-10-00580" class="html-bibr">36</a>], new generation of microscopes, more advanced and used-friendly software [<a href="#B37-crystals-10-00580" class="html-bibr">37</a>,<a href="#B38-crystals-10-00580" class="html-bibr">38</a>],
 etc. The amount of structures solved with cryo-EM is steadily rising as
 well as their resolution, which allowed this technique to shake the 
partial blobology label. With the increasing resolution though the 
same old problem re-emergedwhat is actually the resolution in cryo-EM? 
Currently, multiple resolution criteria are in use, with Fourier shell 
correlation (FSC) being by far the most used one [<a href="#B39-crystals-10-00580" class="html-bibr">39</a>,<a href="#B40-crystals-10-00580" class="html-bibr">40</a>].
 Even though FSC is widely accepted by the scientific community, a 
never-ending discussion continues about a threshold at which the 
resolution should be defined. Alternatives for the FSC, such as Spectral
 Signal-to-Noise Ratio (SSNR) [<a href="#B41-crystals-10-00580" class="html-bibr">41</a>,<a href="#B42-crystals-10-00580" class="html-bibr">42</a>,<a href="#B43-crystals-10-00580" class="html-bibr">43</a>] and Fourier Neighbor Correlation (FNC) [<a href="#B44-crystals-10-00580" class="html-bibr">44</a>]
 exist but are not common. Each method has its own limitations and is 
based on different assumptions, which makes it difficult to reach a 
consensus. For an extensive statistical description and behavior of the 
most commonly used resolution measures, see the recent review by Sorzano
 et al. (2017) [<a href="#B45-crystals-10-00580" class="html-bibr">45</a>].
 To add to the complexity, the resolution determined as a single 
numerical value is, in a way, an average of the density map; however, in
 cryo-EM the resolution varies within a map itself, thus it is essential
 to calculate the local resolution of the density map as well.</div><section id="sec3dot1-crystals-10-00580" type=""><h4 class="html-italic" data-nested="2"> 3.1. Fourier Shell&nbsp;Correlation</h4><div class="html-p">The Fourier shell correlation was first introduced for 2D reconstructions as the Fourier Ring Correlation (FRC) [<a href="#B39-crystals-10-00580" class="html-bibr">39</a>,<a href="#B40-crystals-10-00580" class="html-bibr">40</a>] and was later extended to 3D reconstructions [<a href="#B46-crystals-10-00580" class="html-bibr">46</a>].
 The basis of FSC is the comparison of independently calculated maps 
from half the dataset and determining where these maps stop correlating
<div class="html-disp-formula-info" id="FD8-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<mi>F</mi>
<mi>S</mi>
<mi>C</mi>
<mrow>
<mo>(</mo>
<msub>
<mi>r</mi>
<mi>i</mi>
</msub>
<mo>)</mo>
</mrow>
<mo>=</mo>
<mfrac>
<mrow>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>r</mi>
<mo></mo>
<msub>
<mi>r</mi>
<mi>i</mi>
</msub>
</mrow>
</munder>
</mstyle>
<msub>
<mi>F</mi>
<mn>1</mn>
</msub>
<mrow>
<mo>(</mo>
<mi>r</mi>
<mo>)</mo>
</mrow>
<msubsup>
<mi>F</mi>
<mn>2</mn>
<mo>*</mo>
</msubsup>
<mrow>
<mo>(</mo>
<mi>r</mi>
<mo>)</mo>
</mrow>
</mrow>
<msqrt>
<mrow>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>r</mi>
<mo></mo>
<msub>
<mi>r</mi>
<mi>i</mi>
</msub>
</mrow>
</munder>
</mstyle>
<mrow>
<mo stretchy="false">|</mo>
</mrow>
<msub>
<mi>F</mi>
<mn>1</mn>
</msub>
<msup>
<mrow>
<mrow>
<mo>(</mo>
<mi>r</mi>
<mo>)</mo>
</mrow>
<mo>|</mo>
</mrow>
<mn>2</mn>
</msup>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>r</mi>
<mo></mo>
<msub>
<mi>r</mi>
<mi>i</mi>
</msub>
</mrow>
</munder>
</mstyle>
<msup>
<mrow>
<mo>|</mo>
<msubsup>
<mi>F</mi>
<mn>2</mn>
<mo>*</mo>
</msubsup>
<mrow>
<mo>(</mo>
<mi>r</mi>
<mo>)</mo>
</mrow>
<mo>|</mo>
</mrow>
<mn>2</mn>
</msup>
</mrow>
</msqrt>
</mfrac>
<mo>.</mo>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(8)</label>
</div>
</div></div><div class="html-p">The correlation is calculated over each shell (<math display="inline"><semantics>
<msub>
<mi>r</mi>
<mi>i</mi>
</msub>
</semantics></math>) in the Fourier transform, where <math display="inline"><semantics>
<mrow>
<msub>
<mi>F</mi>
<mn>1</mn>
</msub>
<mrow>
<mo>(</mo>
<mi>r</mi>
<mo>)</mo>
</mrow>
</mrow>
</semantics></math> and <math display="inline"><semantics>
<mrow>
<msubsup>
<mi>F</mi>
<mn>2</mn>
<mo>*</mo>
</msubsup>
<mrow>
<mo>(</mo>
<mi>r</mi>
<mo>)</mo>
</mrow>
</mrow>
</semantics></math> are the structure factors of half map 1 and 2, respectively, at position <span class="html-italic">r</span> within Fourier shell <math display="inline"><semantics>
<msub>
<mi>r</mi>
<mi>i</mi>
</msub>
</semantics></math> and where * indicates the complex conjugate.</div><div class="html-p">At
 the start of refinement, the complete dataset is split in half and a 3D
 map is calculated from each half. Each half map is then refined 
individually and treated separately without any interactions between the
 two half maps to guarantee independence. The correlation of these two 
half maps is calculated, from low spatial frequency (low resolution) to 
high spatial frequency (high-resolution). The FSC curve is normalized 
between 1, for highly correlating maps, and 0, for non-correlating maps.
 The signal of the protein contribution decreases with increasing 
spatial frequency, thus the map will be less and less defined by the 
protein and more by the stochastic noise at higher spatial frequencies. 
As a result of that, the individual half maps correlate less and the 
correlation drops (<a href="#crystals-10-00580-f001" class="html-fig">Figure 1</a>).</div><div class="html-p">The
 discussion concerning the resolution limit of EM maps is actually about
 the threshold value where two half maps do not correlate anymore, in 
other words, the spatial frequency at which two half maps are considered
 to be not correlated (this immediately indicates that this is 
subjective) defines the resolution of the map. Initially a value of 0.5 
was chosen as the threshold at which half maps were considered as not 
correlated. This is still one of the most often used (and the most 
conservative) thresholds for the FSC curve and it corresponds to an SNR<sub>C</sub> of 1 (SNR<sub>C</sub> being the signal-to-noise ratio in the particular Fourier shell) [<a href="#B45-crystals-10-00580" class="html-bibr">45</a>]. However, this threshold value of 0.5 is said to underestimate the resolution [<a href="#B47-crystals-10-00580" class="html-bibr">47</a>,<a href="#B48-crystals-10-00580" class="html-bibr">48</a>],
 as during this procedure the data are split randomly in halves. This 
means that during refinement there are half less particles than when the
 complete dataset were used. Having fewer particles makes the dataset 
noisier, as there are fewer particles to average with. To overcome this 
problem, a new processing method (named the gold standard method) was 
introduced [<a href="#B47-crystals-10-00580" class="html-bibr">47</a>,<a href="#B48-crystals-10-00580" class="html-bibr">48</a>],
 which currently is the most accepted and widely used in the EM 
community. The resolution determined by the gold standard method is 
often the one reported for structures in the PDB or the Electron 
Microscopy Data Bank (EMDB). In this gold standard method the models 
derived from each half dataset are refined independently opposed to 
having two maps and a single model. This decreases unintended 
correlation of the compared maps and lessens the effect of data 
overfitting but does not eliminate it entirely [<a href="#B48-crystals-10-00580" class="html-bibr">48</a>,<a href="#B49-crystals-10-00580" class="html-bibr">49</a>].
 The suggested threshold value of 0.143 originates from a 0.5 value of 
correlation of the complete dataset and an unknown perfect map of the 
macromolecule. The threshold of 0.5 for the estimated correlation was 
chosen for two main reasons. First, the estimated correlation can be 
written out as a function of the phase error. This is equivalent to an 
X-ray crystallographic measure of the accuracy of the phases, the figure
 of merit (FOM). The FOM is commonly used in X-ray crystallography as an
 indication if the map is interpretable enough to build a structure in 
it. A value of 0.5 corresponds to a phase error of <math display="inline"><semantics>
<mrow>
<mn>60</mn>
<mo form="prefix">deg</mo>
</mrow>
</semantics></math> which is considered as interpretable enough to build a structure into the density map [<a href="#B50-crystals-10-00580" class="html-bibr">50</a>]. Second, the FSC can be related to the real space correlation coefficient (<span class="html-italic">R</span>),
 a measure of similarity between two density maps. When there are no 
amplitude errors present, the real space correlation coefficient <span class="html-italic">R</span>
 corresponds to the estimated correlation of the complete dataset and 
the unknown perfect map. Using this, one can predict if the addition of a
 Fourier shell will have a positive effect on the correlation of the map
 and the perfect map and thus improve the map itself with 0.5 being the 
threshold [<a href="#B50-crystals-10-00580" class="html-bibr">50</a>].</div><div class="html-p">An alternative threshold used is the sigma (<math display="inline"><semantics>
<mi></mi>
</semantics></math>) criterion. It uses the standard deviation, <math display="inline"><semantics>
<mrow>
<mi></mi>
<mo>=</mo>
<mfrac>
<mn>1</mn>
<msqrt>
<msub>
<mi>N</mi>
<mi>r</mi>
</msub>
</msqrt>
</mfrac>
</mrow>
</semantics></math> where <math display="inline"><semantics>
<msub>
<mi>N</mi>
<mi>r</mi>
</msub>
</semantics></math> denotes the number samples in the Fourier shell with radius <span class="html-italic">r</span>.
 The FSC of the density maps is then compared to an FSC based entirely 
on noise. The obtained value ranges from 1 to 5, meaning that the 
correlation needs to be bigger than the signal of pure noise by 1 to 5 
times. Even though the criterion has not been widely accepted, some 
advocate for its use because it is not a fixed value threshold but it 
depends on the number of samples in the Fourier shells [<a href="#B51-crystals-10-00580" class="html-bibr">51</a>,<a href="#B52-crystals-10-00580" class="html-bibr">52</a>].
 A fixed value is one of the main critiques of the 0.143 and 0.5 
thresholds, as they do not account for the amount of Fourier components 
in the shells or symmetry of the structure [<a href="#B51-crystals-10-00580" class="html-bibr">51</a>]. This makes its outcome less reproducible than a varying threshold.</div><div class="html-p">To overcome this issue, Van Heel and Schatz introduced a new modified version of the <math display="inline"><semantics>
<mi></mi>
</semantics></math> criterion [<a href="#B51-crystals-10-00580" class="html-bibr">51</a>,<a href="#B52-crystals-10-00580" class="html-bibr">52</a>].
 It is a bit-based information criterion imposing a variable threshold 
depending whether there is still enough information in the signal to 
improve the calculated density map. This threshold should correct a 
presumed wrong assumption of the FSC calculation and make the resolution
 calculation independent of the number of voxels (a 3D pixel in Fourier 
space) in each Fourier shell and symmetry of the structure. For the full
 details and statistical basis of the bit-based information criterion, 
the reader is referred to Van Heel &amp; Schatz (2005) [<a href="#B51-crystals-10-00580" class="html-bibr">51</a>].
 A 1/2-bit threshold is proposed to define the resolution of the density
 map. This is not a fixed value threshold as it varies with symmetry, 
box size, and voxels in the Fourier shell. However, as identified by 
Sorzano et al. (2017) [<a href="#B45-crystals-10-00580" class="html-bibr">45</a>],
 this an almost arbitrarily chosen threshold that corresponds to an SNR 
of 0.4142, while a 1-bit threshold corresponds to an SNR of 1.</div><div class="html-p">Apart
 from the discussion on the correct threshold, the FSC curve anyway is 
not an ideal tool to determine the resolution. First, the complete 
dataset is processed before the splitting it in equal halves. This means
 that the two half datasets are not fully independent and can carry some
 biases increasing the estimated resolution [<a href="#B53-crystals-10-00580" class="html-bibr">53</a>]. Secondly, the resolution estimate is not affected by isotropic filtering of the complete dataset [<a href="#B42-crystals-10-00580" class="html-bibr">42</a>,<a href="#B45-crystals-10-00580" class="html-bibr">45</a>].
 When a low-pass filter is applied, meaning only the low spatial 
frequencies are left, the FSC curve stays the same and thus the same 
resolution estimate is obtained. Furthermore, the values of the 
resolution that the aforementioned criteria produce seem not differ 
greatly. In some specific cases, some criteria may function better than 
others but there is most likely no perfect criterion. The behavior of 
the complete FSC curve is a better indicator of the quality of the 
reconstruction instead of an exact coordinate where it crosses the 
threshold value: the curve should stay as high as possible before it 
passes the chosen threshold value.</div></section><section id="sec3dot2-crystals-10-00580" type=""><h4 class="html-italic" data-nested="2"> 3.2. Spectral Signal-To-Noise&nbsp;Ratio</h4><div class="html-p">The spectral signal-to-noise ratio was first introduced for 2D reconstructions in 1987 [<a href="#B41-crystals-10-00580" class="html-bibr">41</a>] and, similar to the FSC, was sequentially extended for 3D reconstructions [<a href="#B42-crystals-10-00580" class="html-bibr">42</a>,<a href="#B43-crystals-10-00580" class="html-bibr">43</a>].
 It is one of the easiest methods conceptually, as a similar criterion 
is also used in X-ray crystallography. However, it is not as simple as 
in X-ray crystallography because the signal in cryo-EM is significantly 
noisier. Additionally, both the amplitude and the phase of the wave are 
measured and both are affected by noise. Therefore, it is not possible 
to merely look at the intensities as in X-ray crystallography.</div><div class="html-p">The SSNR is based on the assumption that the signal and noise are additive <math display="inline"><semantics>
<mrow>
<msubsup>
<mi>F</mi>
<mi>n</mi>
<mi>k</mi>
</msubsup>
<mo>=</mo>
<msubsup>
<mi>F</mi>
<mi>T</mi>
<mi>k</mi>
</msubsup>
<mo>+</mo>
<msubsup>
<mi>N</mi>
<mi>n</mi>
<mi>k</mi>
</msubsup>
</mrow>
</semantics></math>, where <math display="inline"><semantics>
<msubsup>
<mi>F</mi>
<mi>n</mi>
<mi>k</mi>
</msubsup>
</semantics></math> is the signal, or more correctly the Fourier transform of the recorded signal in the microscope, <math display="inline"><semantics>
<msubsup>
<mi>F</mi>
<mi>T</mi>
<mi>k</mi>
</msubsup>
</semantics></math> is the unknown true signal without noise and <math display="inline"><semantics>
<msubsup>
<mi>N</mi>
<mi>n</mi>
<mi>k</mi>
</msubsup>
</semantics></math> is the added Gaussian noise which is unique for each image. The <span class="html-italic">n</span> and <span class="html-italic">k</span> denote the n-th projection at voxel <span class="html-italic">k</span>. Here, the SSNR for a shell with radius <span class="html-italic">r</span> is
<div class="html-disp-formula-info" id="FD9-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<mi>S</mi>
<mi>S</mi>
<mi>N</mi>
<mi>R</mi>
<mrow>
<mo>(</mo>
<mi>r</mi>
<mo>)</mo>
</mrow>
<mo>=</mo>
<mfrac>
<mrow>
<mstyle displaystyle="true">
<munderover>
<mo></mo>
<mrow>
<mi>k</mi>
<mo>=</mo>
<mn>1</mn>
</mrow>
<msub>
<mi>n</mi>
<mi>r</mi>
</msub>
</munderover>
</mstyle>
<msup>
<mrow>
<mo>|</mo>
<msubsup>
<mi>F</mi>
<mi>T</mi>
<mi>k</mi>
</msubsup>
<mo>|</mo>
</mrow>
<mn>2</mn>
</msup>
</mrow>
<mrow>
<mstyle displaystyle="true">
<munderover>
<mo></mo>
<mrow>
<mi>k</mi>
<mo>=</mo>
<mn>1</mn>
</mrow>
<msub>
<mi>n</mi>
<mi>r</mi>
</msub>
</munderover>
</mstyle>
<mfrac>
<mn>1</mn>
<msub>
<mi>L</mi>
<mi>k</mi>
</msub>
</mfrac>
<msup>
<mrow>
<mo>(</mo>
<msup>
<mi></mi>
<mi>k</mi>
</msup>
<mo>)</mo>
</mrow>
<mn>2</mn>
</msup>
</mrow>
</mfrac>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(9)</label>
</div>
</div>
where <span class="html-italic">L<sub>k</sub></span> denotes the amount of Fourier components per voxel and <math display="inline"><semantics>
<msup>
<mi></mi>
<mi>r</mi>
</msup>
</semantics></math> is equal to <math display="inline"><semantics>
<mrow>
<msubsup>
<mi>N</mi>
<mi>n</mi>
<mi>k</mi>
</msubsup>
<msup>
<mrow></mrow>
<mn>2</mn>
</msup>
</mrow>
</semantics></math>. This is the ratio of the energy of the signal and 
the energy of the noise, and it is adjusted with the size of the 
dataset. However, this cannot be known because <math display="inline"><semantics>
<msubsup>
<mi>F</mi>
<mi>T</mi>
<mi>k</mi>
</msubsup>
</semantics></math> cannot be known. Therefore, the SSNR is estimated using:<div class="html-disp-formula-info" id="FD10-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<mi>S</mi>
<mrow>
<mo>(</mo>
<mi>r</mi>
<mo>)</mo>
</mrow>
<mo>=</mo>
<mfrac>
<mrow>
<mstyle displaystyle="true">
<munderover>
<mo></mo>
<mrow>
<mi>k</mi>
<mo>=</mo>
<mn>1</mn>
</mrow>
<msub>
<mi>n</mi>
<mi>r</mi>
</msub>
</munderover>
</mstyle>
<msup>
<mfenced separators="" open="|" close="|">
<msup>
<mi>F</mi>
<mi>k</mi>
</msup>
</mfenced>
<mn>2</mn>
</msup>
</mrow>
<mrow>
<mstyle displaystyle="true">
<munderover>
<mo></mo>
<mrow>
<mi>k</mi>
<mo>=</mo>
<mn>1</mn>
</mrow>
<msub>
<mi>n</mi>
<mi>r</mi>
</msub>
</munderover>
</mstyle>
<mfrac>
<mn>1</mn>
<mrow>
<msub>
<mi>L</mi>
<mi>k</mi>
</msub>
<mrow>
<mo>(</mo>
<msub>
<mi>L</mi>
<mi>k</mi>
</msub>
<mo></mo>
<mn>1</mn>
<mo>)</mo>
</mrow>
</mrow>
</mfrac>
<mstyle displaystyle="true">
<munderover>
<mo></mo>
<mi>n</mi>
<msub>
<mi>L</mi>
<mi>k</mi>
</msub>
</munderover>
</mstyle>
<msup>
<mrow>
<mo>|</mo>
<msubsup>
<mi>F</mi>
<mi>n</mi>
<mi>k</mi>
</msubsup>
<mo></mo>
<msup>
<mi>F</mi>
<mi>k</mi>
</msup>
<mo>|</mo>
</mrow>
<mn>2</mn>
</msup>
</mrow>
</mfrac>
<mo>,</mo>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(10)</label>
</div>
</div>
where subtraction of 1 is necessary to produce an unbiased estimate [<a href="#B42-crystals-10-00580" class="html-bibr">42</a>]. Once <math display="inline"><semantics>
<mrow>
<mi>S</mi>
<mo>(</mo>
<mi>r</mi>
<mo>)</mo>
</mrow>
</semantics></math> becomes <math display="inline"><semantics>
<mrow>
<mo>&lt;</mo>
<mn>1</mn>
</mrow>
</semantics></math>, then the <math display="inline"><semantics>
<mrow>
<mi>S</mi>
<mi>S</mi>
<mi>N</mi>
<mi>R</mi>
<mo>(</mo>
<mi>r</mi>
<mo>)</mo>
</mrow>
</semantics></math> is made to be equal to 0. Here, <math display="inline"><semantics>
<msup>
<mi>F</mi>
<mi>k</mi>
</msup>
</semantics></math> is an estimator of <math display="inline"><semantics>
<msubsup>
<mi>F</mi>
<mi>T</mi>
<mi>k</mi>
</msubsup>
</semantics></math> with the relation being
<div class="html-disp-formula-info" id="FD11-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<msup>
<mi>F</mi>
<mi>k</mi>
</msup>
<mo>=</mo>
<mfrac>
<mn>1</mn>
<msub>
<mi>L</mi>
<mi>k</mi>
</msub>
</mfrac>
<mstyle displaystyle="true">
<munderover>
<mo></mo>
<mi>n</mi>
<msub>
<mi>L</mi>
<mi>k</mi>
</msub>
</munderover>
</mstyle>
<msubsup>
<mi>F</mi>
<mi>n</mi>
<mi>k</mi>
</msubsup>
<mo>.</mo>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(11)</label>
</div>
</div></div><div class="html-p">The SSNR indicates how consistent the 
input and the calculated map are. It does so by estimating the 
signal-to-noise ratio over the resolution of the map. The resolution of 
the density map can be defined when the SNR drops below a certain 
threshold usually 1, where the signal and noise have equal strength. A 
threshold of 1 corresponds to a 0.30.5 threshold of the FSC curve [<a href="#B42-crystals-10-00580" class="html-bibr">42</a>], so it could be too conservative as seen above. This is calculated by the relationship between SSNR and FSC [<a href="#B41-crystals-10-00580" class="html-bibr">41</a>,<a href="#B45-crystals-10-00580" class="html-bibr">45</a>],
<div class="html-disp-formula-info" id="FD12-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<mi>S</mi>
<mi>S</mi>
<mi>N</mi>
<mi>R</mi>
<mo>=</mo>
<mfrac>
<mrow>
<mn>2</mn>
<mi>F</mi>
<mi>S</mi>
<mi>C</mi>
</mrow>
<mrow>
<mn>1</mn>
<mo></mo>
<mi>F</mi>
<mi>S</mi>
<mi>C</mi>
</mrow>
</mfrac>
<mo>.</mo>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(12)</label>
</div>
</div></div><div class="html-p">However, this is only an approximation, 
not an inherent relationship of the two criteria. The approximation was 
made for stationary signals (signals whose frequency or spectral 
contents remain unchanged), but the signals in cryo-EM are not 
stationary. The main advantage of SSNR is that it does not require the 
dataset to be split in two random halves as is needed for the FSC, 
allowing one to use the entire dataset. Using the SSNR, one can 
calculate directional resolution of the density map, which can be used 
to reveal anisotropy in the dataset. At the basis of SSNR lies the 
assumption that the noise in each image is uncorrelated and random. 
Although this is generally the case for noise, it can start correlating 
during alignment of the particles, and then SSNR tends to overestimate 
the true signal-to-noise ratio [<a href="#B53-crystals-10-00580" class="html-bibr">53</a>].</div></section><section id="sec3dot3-crystals-10-00580" type=""><h4 class="html-italic" data-nested="2"> 3.3. Fourier Neighbor&nbsp;Correlation</h4><div class="html-p">A
 relatively new method to determine the resolution of a cryo-EM density 
map is the Fourier neighbor correlation (FNC). It calculates the 
correlation of neighboring voxels in the Fourier image of the density 
map [<a href="#B44-crystals-10-00580" class="html-bibr">44</a>]:<div class="html-disp-formula-info" id="FD13-crystals-10-00580">
<div class="f">
<math display="block"><semantics>
<mrow>
<mi>F</mi>
<mi>N</mi>
<mi>C</mi>
<mrow>
<mo>(</mo>
<mi>r</mi>
<mo>)</mo>
</mrow>
<mo>=</mo>
<mfrac>
<mrow>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>p</mi>
<mo></mo>
<mi>r</mi>
</mrow>
</munder>
</mstyle>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>h</mi>
<mo></mo>
<mi>N</mi>
<mo>(</mo>
<mi>p</mi>
<mo>)</mo>
</mrow>
</munder>
</mstyle>
<msub>
<mi>F</mi>
<mi>p</mi>
</msub>
<msubsup>
<mi>F</mi>
<mi>h</mi>
<mo>*</mo>
</msubsup>
</mrow>
<msqrt>
<mrow>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>p</mi>
<mo></mo>
<mi>r</mi>
</mrow>
</munder>
</mstyle>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>h</mi>
<mo></mo>
<mi>N</mi>
<mo>(</mo>
<mi>p</mi>
<mo>)</mo>
</mrow>
</munder>
</mstyle>
<mrow>
<mo stretchy="false">|</mo>
</mrow>
<msub>
<mi>F</mi>
<mi>p</mi>
</msub>
<msup>
<mrow>
<mo stretchy="false">|</mo>
</mrow>
<mn>2</mn>
</msup>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>p</mi>
<mo></mo>
<mi>r</mi>
</mrow>
</munder>
</mstyle>
<mstyle displaystyle="true">
<munder>
<mo></mo>
<mrow>
<mi>h</mi>
<mo></mo>
<mi>N</mi>
<mo>(</mo>
<mi>p</mi>
<mo>)</mo>
</mrow>
</munder>
</mstyle>
<msup>
<mrow>
<mo>|</mo>
<msub>
<mi>F</mi>
<mi>h</mi>
</msub>
<mo>|</mo>
</mrow>
<mn>2</mn>
</msup>
</mrow>
</msqrt>
</mfrac>
<mo>.</mo>
</mrow>
</semantics></math>
</div>
<div class="l">
<label>(13)</label>
</div>
</div></div><div class="html-p">It compares one voxel (<math display="inline"><semantics>
<msub>
<mi>F</mi>
<mi>p</mi>
</msub>
</semantics></math>) with its six closest neighbors (<math display="inline"><semantics>
<msub>
<mi>F</mi>
<mi>h</mi>
</msub>
</semantics></math>) denoted by <math display="inline"><semantics>
<mrow>
<mi>N</mi>
<mo>(</mo>
<mi>p</mi>
<mo>)</mo>
</mrow>
</semantics></math>, where <span class="html-italic">r</span> is the 
radius of the Fourier shell and * denotes the complex conjugate. From 
the local correlation of the voxels, the resolution is estimated without
 splitting the dataset in halves.</div><div class="html-p">The FNC is related to the SSNR and the SSNR is in turn related to the FSC (<a href="#FD12-crystals-10-00580" class="html-disp-formula">12</a>).
 This allows one to estimate the FSC without splitting the dataset into 
two equal parts. Additionally, the FNC is less susceptible to noise 
overfitting than the FSC. For the FNC, only the density map is required 
to calculate the resolution, not the half maps or the raw experimental 
data. This allows one to recalculate the resolution of a density map 
downloaded from the PDB or EMDB. This is not always possible for the 
FSC, as it is not mandatory to deposit each half map when depositing a 
structure, although this is strongly advocated by the community. This 
also allows one to check the reported resolution with the deposited map,
 improving transparency. Calculating the FSC from FNC gives slightly 
higher resolutions than when the FSC is calculated directly using the 
gold standard method [<a href="#B54-crystals-10-00580" class="html-bibr">54</a>,<a href="#B55-crystals-10-00580" class="html-bibr">55</a>], but it might be overoptimistic for some reconstructions [<a href="#B56-crystals-10-00580" class="html-bibr">56</a>].
 Overall, the FNC seems to be a good alternative method for calculating 
the FSC curve without having to split the dataset in halves or the 
availability of the experimental data (half maps) for the posterior 
analysis.</div></section><section id="sec3dot4-crystals-10-00580" type=""><h4 class="html-italic" data-nested="2"> 3.4. Local&nbsp;Resolution</h4><div class="html-p">In
 cryo-EM, unlike in X-ray crystallography, the calculated density map 
does not have the same resolution all over the map, i.e., it varies 
across the map. The single number obtained by application of any of 
those criteria discussed above gives a general estimate how reliable the
 experimental map is as a whole, but tells nothing about the reliability
 of a single voxel. In fact, as there is always some kind of 
heterogeneity coming from a specimen itself (e.g., as some parts of the 
molecule are more flexible) and/or as the result of imaging (radiation 
damage), data processing (e.g., small alignment errors [<a href="#B57-crystals-10-00580" class="html-bibr">57</a>]), etc., we need to estimate the local resolution of a map.</div><div class="html-p">MonoRes and ResMap are the two most commonly used programs to determine the local resolution of a density map [<a href="#B58-crystals-10-00580" class="html-bibr">58</a>,<a href="#B59-crystals-10-00580" class="html-bibr">59</a>].
 Both generate a colored density map, where the color gradient indicates
 the local resolution of the map. In ResMap the local resolution is 
defined by the smallest wavelength where its three-dimensional sinusoid 
is still detectable above the noise level at a given map voxel [<a href="#B59-crystals-10-00580" class="html-bibr">59</a>].
 The main advantage of the given algorithm that it provides a robust 
false discovery rate control and that it accounts for data dependency 
between neighboring points [<a href="#B59-crystals-10-00580" class="html-bibr">59</a>].
 MonoRes has a different approach: it uses the signal-to-noise ratio to 
determine the local resolution. This is done by decomposing the signal 
in a voxel using the Riesz transform [<a href="#B60-crystals-10-00580" class="html-bibr">60</a>].
 From the decomposed signal, a monogenic signal is generated, defined as
 a signal without negative values or oscillations. Its amplitude at this
 spatial frequency can be compared to the monogenic signal of the noise,
 where the latter is estimated from the two half maps. MonoRes and 
ResMap generate maps with slightly different resolutions, yet the 
resulting resolutions are comparable. No user input is required when 
using MonoRes, as opposed to ResMap, which can increase reproducibility 
in the resulting resolutions [<a href="#B58-crystals-10-00580" class="html-bibr">58</a>].
 MonoRes has been recently expanded to account for directionality (now 
named MonoDir), i.e., it additionally calculates the local resolution 
along a set of directions in 3D [<a href="#B61-crystals-10-00580" class="html-bibr">61</a>].
 Importantly it requires only a final map without a need for the 
original particles or their assigned projection directions. Introduction
 of directionality in local resolution opens up the new possibilities 
for validation, e.g., by analyzing angular alignment errors and data 
anisotropy.</div><div class="html-p">The local resolution map is the 
most useful map to view next to the map used in refinement and model 
building. It shows the viewer clearly which regions have better or worse
 defined density. Such a map is always recommended for inspection when 
looking at a cryo-EM-obtained structure.</div></section></section><section id="sec4-crystals-10-00580" type="conclusions"><h2 data-nested="1"> 4. Conclusions</h2><div class="html-p">The
 determination of resolution in X-ray crystallography and cryo-EM is not
 a trivial task. In both fields there is still ongoing (perhaps even 
never-ending) discussion what is the actual resolution at which any 
given structure is solved.</div><div class="html-p">In X-ray 
crystallography, the main issue is what is the best threshold to 
truncate the data? In other words, to what extent the collected data 
still contains useful information. Traditionally, only data with the 
signal-to-noise ratio  2 and with the values of <math display="inline"><semantics>
<msub>
<mi>R</mi>
<mrow>
<mi>m</mi>
<mi>e</mi>
<mi>r</mi>
<mi>g</mi>
<mi>e</mi>
</mrow>
</msub>
</semantics></math> 40% were used. However, this approach is outdated and barely justifiable (see above) and novel more robust criteria such as <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msub>
<mi>C</mi>
<mrow>
<mn>1</mn>
<mo>/</mo>
<mn>2</mn>
</mrow>
</msub>
</mrow>
</semantics></math> and <math display="inline"><semantics>
<mrow>
<mi>C</mi>
<msup>
<mi>C</mi>
<mo>*</mo>
</msup>
</mrow>
</semantics></math> shall be used to determine the resolution cut-off. 
However, there is no universal fixed value for this criteria, and one 
possible way to overcome this problem is to do so-called paired 
refinement to determine the actual best cut-off for a given dataset. 
Hopefully this approach will become standard practice in the near 
future.</div><div class="html-p">In the cryo-EM field, the main 
discussion resolves around how to determine the actual resolution of a 
reconstruction. The most commonly used method is the gold-standard FSC
 with a threshold of 0.143. Yet, the question remains whether this is 
the correct threshold and whether FSC is the most appropriate way to 
determine the resolution. The recently introduced FNC is a good 
alternative to FSC, as it is less susceptible to over-fitting and it can
 be calculated without splitting the data and hence in absence of half 
maps. This allows one to recalculate the resolution of the density map 
even after it has been deposited to the PDB or EMDB. Nevertheless, 
majority keeps using FSC which is far from ideal. To improve it, 
recently Steven Ludtke, proposed to add error bars to the FSC curve 
(Ludtke, personal communication). The advantage is that it would add an 
error margin to the resolution estimate using the FSC. Moreover, 
reporting the local resolution is always good practice as a single value
 of the resolution gives a rather poor description.</div><div class="html-p">However,
 perhaps what is more important for any novice in the structural biology
 field is to realize that the resolution itself is a poor indicator of 
quality and that the end goal is not to show the highest resolution of 
ones structure but to produce a structure as close describing the 
experimental data as possible.</div></section>
</div>
<div class="html-back">
<section class="html-notes"><h2>Author Contributions</h2><div class="html-p">V.R.A.D. and A.G. wrote the manuscript. All authors have read and agreed to the published version of the manuscript.</div></section><section class="html-notes"><h2>Funding</h2><div class="html-p">This research received no external funding.</div></section><section class="html-notes"><h2>Conflicts of Interest</h2><div class="html-p">The authors declare no conflicts of interest.</div></section><section id="html-references_list"><h2>References</h2><ol class="html-xx"><li id="B1-crystals-10-00580" class="html-x" data-content="1.">Schmidt, A.; Teeter, M.; Weckert, E.; Lamzin, V.S. Crystal structure of small protein crambin at 0.48  resolution. <span class="html-italic">Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun.</span> <b>2011</b>, <span class="html-italic">67</span>, 424428. [<a href="https://scholar.google.com/scholar_lookup?title=Crystal+structure+of+small+protein+crambin+at+0.48+%C3%85+resolution&amp;author=Schmidt,+A.&amp;author=Teeter,+M.&amp;author=Weckert,+E.&amp;author=Lamzin,+V.S.&amp;publication_year=2011&amp;journal=Acta+Crystallogr.+Sect.+F+Struct.+Biol.+Cryst.+Commun.&amp;volume=67&amp;pages=424%E2%80%93428&amp;doi=10.1107/S1744309110052607&amp;pmid=21505232" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S1744309110052607" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/21505232" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B2-crystals-10-00580" class="html-x" data-content="2.">Kato,
 T.; Makino, F.; Nakane, T.; Terahara, N.; Kaneko, T.; Shimizu, Y.; 
Motoki, S.; Ishikawa, I.; Yonekura, K.; Namba, K. CryoTEM with a Cold 
Field Emission Gun That Moves Structural Biology into a New Stage. <span class="html-italic">Microsc. Microanal.</span> <b>2019</b>, <span class="html-italic">25</span>, 998999. [<a href="https://scholar.google.com/scholar_lookup?title=CryoTEM+with+a+Cold+Field+Emission+Gun+That+Moves+Structural+Biology+into+a+New+Stage&amp;author=Kato,+T.&amp;author=Makino,+F.&amp;author=Nakane,+T.&amp;author=Terahara,+N.&amp;author=Kaneko,+T.&amp;author=Shimizu,+Y.&amp;author=Motoki,+S.&amp;author=Ishikawa,+I.&amp;author=Yonekura,+K.&amp;author=Namba,+K.&amp;publication_year=2019&amp;journal=Microsc.+Microanal.&amp;volume=25&amp;pages=998%E2%80%93999&amp;doi=10.1017/S1431927619005725" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1017/S1431927619005725" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B3-crystals-10-00580" class="html-x" data-content="3.">Dauter, Z.; Lamzin, V.S.; Wilson, K.S. The benefits of atomic resolution. <span class="html-italic">Curr. Opin. Struct. Biol.</span> <b>1997</b>, <span class="html-italic">7</span>, 681688. [<a href="https://scholar.google.com/scholar_lookup?title=The+benefits+of+atomic+resolution&amp;author=Dauter,+Z.&amp;author=Lamzin,+V.S.&amp;author=Wilson,+K.S.&amp;publication_year=1997&amp;journal=Curr.+Opin.+Struct.+Biol.&amp;volume=7&amp;pages=681%E2%80%93688&amp;doi=10.1016/S0959-440X(97)80078-4" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/S0959-440X(97)80078-4" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B4-crystals-10-00580" class="html-x" data-content="4.">Sheldrick, G.M. Phase annealing in SHELX-90: Direct methods for larger structures. <span class="html-italic">Acta Crystallogr. Sect. A Found. Crystallogr.</span> <b>1990</b>, <span class="html-italic">46</span>, 467473. [<a href="https://scholar.google.com/scholar_lookup?title=Phase+annealing+in+SHELX-90:+Direct+methods+for+larger+structures&amp;author=Sheldrick,+G.M.&amp;publication_year=1990&amp;journal=Acta+Crystallogr.+Sect.+A+Found.+Crystallogr.&amp;volume=46&amp;pages=467%E2%80%93473&amp;doi=10.1107/S0108767390000277" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0108767390000277" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B5-crystals-10-00580" class="html-x" data-content="5.">Wlodawer, A.; Dauter, Z. Atomic resolution: A badly abused term in structural biology. <span class="html-italic">Acta Crystallogr. Sect. D Struct. Biol.</span> <b>2017</b>, <span class="html-italic">73</span>, 379. [<a href="https://scholar.google.com/scholar_lookup?title=%E2%80%98Atomic+resolution%E2%80%99:+A+badly+abused+term+in+structural+biology&amp;author=Wlodawer,+A.&amp;author=Dauter,+Z.&amp;publication_year=2017&amp;journal=Acta+Crystallogr.+Sect.+D+Struct.+Biol.&amp;volume=73&amp;pages=379&amp;doi=10.1107/S205979831700225X" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S205979831700225X" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B6-crystals-10-00580" class="html-x" data-content="6.">Rayleigh, L. XXXI Investigations in optics, with special reference to the spectroscope. <span class="html-italic">Lond. Edinb. Dublin Philos. Mag. J. Sci.</span> <b>1879</b>, <span class="html-italic">8</span>, 261274. [<a href="https://scholar.google.com/scholar_lookup?title=XXXI+Investigations+in+optics,+with+special+reference+to+the+spectroscope&amp;author=Rayleigh,+L.&amp;publication_year=1879&amp;journal=Lond.+Edinb.+Dublin+Philos.+Mag.+J.+Sci.&amp;volume=8&amp;pages=261%E2%80%93274&amp;doi=10.1080/14786447908639684" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1080/14786447908639684" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B7-crystals-10-00580" class="html-x" data-content="7.">McCoy, A.J. Solving structures of protein complexes by molecular replacement with Phaser. <span class="html-italic">Acta Crystallogr. Sect. D Biol. Crystallogr.</span> <b>2007</b>, <span class="html-italic">63</span>, 3241. [<a href="https://scholar.google.com/scholar_lookup?title=Solving+structures+of+protein+complexes+by+molecular+replacement+with+Phaser&amp;author=McCoy,+A.J.&amp;publication_year=2007&amp;journal=Acta+Crystallogr.+Sect.+D+Biol.+Crystallogr.&amp;volume=63&amp;pages=32%E2%80%9341&amp;doi=10.1107/S0907444906045975" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0907444906045975" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B8-crystals-10-00580" class="html-x" data-content="8.">Read, R.J. Pushing the boundaries of molecular replacement with maximum likelihood. <span class="html-italic">Acta Crystallogr. Sect. D Biol. Crystallogr.</span> <b>2001</b>, <span class="html-italic">57</span>, 13731382. [<a href="https://scholar.google.com/scholar_lookup?title=Pushing+the+boundaries+of+molecular+replacement+with+maximum+likelihood&amp;author=Read,+R.J.&amp;publication_year=2001&amp;journal=Acta+Crystallogr.+Sect.+D+Biol.+Crystallogr.&amp;volume=57&amp;pages=1373%E2%80%931382&amp;doi=10.1107/S0907444901012471" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0907444901012471" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B9-crystals-10-00580" class="html-x" data-content="9.">Hendrickson, W.A.; Ogata, C.M. [28] Phase determination from multiwavelength anomalous diffraction measurements. In <span class="html-italic">Methods in Enzymology</span>; Elsevier: Amsterdam, The Netherlands, 1997; Volume 276, pp. 494523. [<a href="https://scholar.google.com/scholar_lookup?title=[28]+Phase+determination+from+multiwavelength+anomalous+diffraction+measurements&amp;author=Hendrickson,+W.A.&amp;author=Ogata,+C.M.&amp;publication_year=1997&amp;pages=494%E2%80%93523" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B10-crystals-10-00580" class="html-xx" data-content="10.">Rose, J.P.; Wang, B.C. SAD phasing: History, current impact and future opportunities. <span class="html-italic">Arch. Biochem. Biophys.</span> <b>2016</b>, <span class="html-italic">602</span>, 8094. [<a href="https://scholar.google.com/scholar_lookup?title=SAD+phasing:+History,+current+impact+and+future+opportunities&amp;author=Rose,+J.P.&amp;author=Wang,+B.C.&amp;publication_year=2016&amp;journal=Arch.+Biochem.+Biophys.&amp;volume=602&amp;pages=80%E2%80%9394&amp;doi=10.1016/j.abb.2016.03.018" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.abb.2016.03.018" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B11-crystals-10-00580" class="html-xx" data-content="11.">Ke, H. [25] Overview of isomorphous replacement phasing. In <span class="html-italic">Methods in Enzymology</span>; Elsevier: Amsterdam, The Netherlands, 1997; Volume 276, pp. 448461. [<a href="https://scholar.google.com/scholar_lookup?title=[25]+Overview+of+isomorphous+replacement+phasing&amp;author=Ke,+H.&amp;publication_year=1997&amp;pages=448%E2%80%93461" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B12-crystals-10-00580" class="html-xx" data-content="12.">Rodrguez,
 D.D.; Grosse, C.; Himmel, S.; Gonzlez, C.; De Ilarduya, I.M.; Becker, 
S.; Sheldrick, G.M.; Usn, I. Crystallographic ab initio protein 
structure solution below atomic resolution. <span class="html-italic">Nat. Methods</span> <b>2009</b>, <span class="html-italic">6</span>, 651653. [<a href="https://scholar.google.com/scholar_lookup?title=Crystallographic+ab+initio+protein+structure+solution+below+atomic+resolution&amp;author=Rodr%C3%ADguez,+D.D.&amp;author=Grosse,+C.&amp;author=Himmel,+S.&amp;author=Gonz%C3%A1lez,+C.&amp;author=De+Ilarduya,+I.M.&amp;author=Becker,+S.&amp;author=Sheldrick,+G.M.&amp;author=Us%C3%B3n,+I.&amp;publication_year=2009&amp;journal=Nat.+Methods&amp;volume=6&amp;pages=651%E2%80%93653&amp;doi=10.1038/nmeth.1365" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1038/nmeth.1365" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B13-crystals-10-00580" class="html-xx" data-content="13.">Sheldrick, G.; Gilmore, C.; Hauptman, H.; Weeks, C.; Miller, R.; Usn, I. Ab initio phasing. <span class="html-italic">Int. Tables Crystallogr.</span> <b>2006</b>, 413432. [<a href="https://scholar.google.com/scholar_lookup?title=Ab+initio+phasing&amp;author=Sheldrick,+G.&amp;author=Gilmore,+C.&amp;author=Hauptman,+H.&amp;author=Weeks,+C.&amp;author=Miller,+R.&amp;author=Us%C3%B3n,+I.&amp;publication_year=2006&amp;journal=Int.+Tables+Crystallogr.&amp;pages=413%E2%80%93432&amp;doi=10.1107/97809553602060000689" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/97809553602060000689" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B14-crystals-10-00580" class="html-xx" data-content="14.">Karplus, P.A.; Diederichs, K. Linking crystallographic model and data quality. <span class="html-italic">Science</span> <b>2012</b>, <span class="html-italic">336</span>, 10301033. [<a href="https://scholar.google.com/scholar_lookup?title=Linking+crystallographic+model+and+data+quality&amp;author=Karplus,+P.A.&amp;author=Diederichs,+K.&amp;publication_year=2012&amp;journal=Science&amp;volume=336&amp;pages=1030%E2%80%931033&amp;doi=10.1126/science.1218231" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1126/science.1218231" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B15-crystals-10-00580" class="html-xx" data-content="15.">Evans, P.R.; Murshudov, G.N. How good are my data and what is the resolution? <span class="html-italic">Acta Crystallogr. Sect. D Biol. Crystallogr.</span> <b>2013</b>, <span class="html-italic">69</span>, 12041214. [<a href="https://scholar.google.com/scholar_lookup?title=How+good+are+my+data+and+what+is+the+resolution?&amp;author=Evans,+P.R.&amp;author=Murshudov,+G.N.&amp;publication_year=2013&amp;journal=Acta+Crystallogr.+Sect.+D+Biol.+Crystallogr.&amp;volume=69&amp;pages=1204%E2%80%931214&amp;doi=10.1107/S0907444913000061&amp;pmid=23793146" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0907444913000061" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/23793146" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B16-crystals-10-00580" class="html-xx" data-content="16.">Wang, J.; Wing, R.A. Diamonds in the rough: A strong case for the inclusion of weak-intensity X-ray diffraction data. <span class="html-italic">Acta Crystallogr. Sect. D Biol. Crystallogr.</span> <b>2014</b>, <span class="html-italic">70</span>, 14911497. [<a href="https://scholar.google.com/scholar_lookup?title=Diamonds+in+the+rough:+A+strong+case+for+the+inclusion+of+weak-intensity+X-ray+diffraction+data&amp;author=Wang,+J.&amp;author=Wing,+R.A.&amp;publication_year=2014&amp;journal=Acta+Crystallogr.+Sect.+D+Biol.+Crystallogr.&amp;volume=70&amp;pages=1491%E2%80%931497&amp;doi=10.1107/S1399004714005318&amp;pmid=24816117" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S1399004714005318" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/24816117" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B17-crystals-10-00580" class="html-xx" data-content="17.">Diederichs, K.; Karplus, P.A. Better models by discarding data? <span class="html-italic">Acta Crystallogr. Sect. D Biol. Crystallogr.</span> <b>2013</b>, <span class="html-italic">69</span>, 12151222. [<a href="https://scholar.google.com/scholar_lookup?title=Better+models+by+discarding+data?&amp;author=Diederichs,+K.&amp;author=Karplus,+P.A.&amp;publication_year=2013&amp;journal=Acta+Crystallogr.+Sect.+D+Biol.+Crystallogr.&amp;volume=69&amp;pages=1215%E2%80%931222&amp;doi=10.1107/S0907444913001121" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0907444913001121" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B18-crystals-10-00580" class="html-xx" data-content="18.">Luo, Z.; Rajashankar, K.; Dauter, Z. Weak data do not make a free lunch, only a cheap meal. <span class="html-italic">Acta Crystallogr. Sect. D Biol. Crystallogr.</span> <b>2014</b>, <span class="html-italic">70</span>, 253260. [<a href="https://scholar.google.com/scholar_lookup?title=Weak+data+do+not+make+a+free+lunch,+only+a+cheap+meal&amp;author=Luo,+Z.&amp;author=Rajashankar,+K.&amp;author=Dauter,+Z.&amp;publication_year=2014&amp;journal=Acta+Crystallogr.+Sect.+D+Biol.+Crystallogr.&amp;volume=70&amp;pages=253%E2%80%93260&amp;doi=10.1107/S1399004713026680" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S1399004713026680" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B19-crystals-10-00580" class="html-xx" data-content="19.">Wang, J. Estimation of the quality of refined protein crystal structures. <span class="html-italic">Protein Sci.</span> <b>2015</b>, <span class="html-italic">24</span>, 661669. [<a href="https://scholar.google.com/scholar_lookup?title=Estimation+of+the+quality+of+refined+protein+crystal+structures&amp;author=Wang,+J.&amp;publication_year=2015&amp;journal=Protein+Sci.&amp;volume=24&amp;pages=661%E2%80%93669&amp;doi=10.1002/pro.2639" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1002/pro.2639" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B20-crystals-10-00580" class="html-xx" data-content="20.">Arndt, U.; Crowther, R.; Mallett, J. A computer-linked cathode-ray tube microdensitometer for x-ray crystallography. <span class="html-italic">J. Phys. E Sci. Instrum.</span> <b>1968</b>, <span class="html-italic">1</span>, 510. [<a href="https://scholar.google.com/scholar_lookup?title=A+computer-linked+cathode-ray+tube+microdensitometer+for+x-ray+crystallography&amp;author=Arndt,+U.&amp;author=Crowther,+R.&amp;author=Mallett,+J.&amp;publication_year=1968&amp;journal=J.+Phys.+E+Sci.+Instrum.&amp;volume=1&amp;pages=510&amp;doi=10.1088/0022-3735/1/5/303" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1088/0022-3735/1/5/303" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B21-crystals-10-00580" class="html-xx" data-content="21.">Diederichs, K.; Karplus, P.A. Improved R-factors for diffraction data analysis in macromolecular crystallography. <span class="html-italic">Nat. Struct. Biol.</span> <b>1997</b>, <span class="html-italic">4</span>, 269275. [<a href="https://scholar.google.com/scholar_lookup?title=Improved+R-factors+for+diffraction+data+analysis+in+macromolecular+crystallography&amp;author=Diederichs,+K.&amp;author=Karplus,+P.A.&amp;publication_year=1997&amp;journal=Nat.+Struct.+Biol.&amp;volume=4&amp;pages=269%E2%80%93275&amp;doi=10.1038/nsb0497-269" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1038/nsb0497-269" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B22-crystals-10-00580" class="html-xx" data-content="22.">Weiss, M.; Hilgenfeld, R. On the use of the merging R factor as a quality indicator for X-ray data. <span class="html-italic">J. Appl. Crystallogr.</span> <b>1997</b>, <span class="html-italic">30</span>, 203205. [<a href="https://scholar.google.com/scholar_lookup?title=On+the+use+of+the+merging+R+factor+as+a+quality+indicator+for+X-ray+data&amp;author=Weiss,+M.&amp;author=Hilgenfeld,+R.&amp;publication_year=1997&amp;journal=J.+Appl.+Crystallogr.&amp;volume=30&amp;pages=203%E2%80%93205&amp;doi=10.1107/S0021889897003907" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0021889897003907" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B23-crystals-10-00580" class="html-xx" data-content="23.">Bae,
 B.; Davis, E.; Brown, D.; Campbell, E.A.; Wigneshweraraj, S.; Darst, 
S.A. Phage T7 Gp2 inhibition of Escherichia coli RNA polymerase involves
 misappropriation of <span class="html-italic"></span>70 domain 1.1. <span class="html-italic">Proc. Natl. Acad. Sci. USA</span> <b>2013</b>, <span class="html-italic">110</span>, 1977219777. [<a href="https://scholar.google.com/scholar_lookup?title=Phage+T7+Gp2+inhibition+of+Escherichia+coli+RNA+polymerase+involves+misappropriation+of+%CF%8370+domain+1.1&amp;author=Bae,+B.&amp;author=Davis,+E.&amp;author=Brown,+D.&amp;author=Campbell,+E.A.&amp;author=Wigneshweraraj,+S.&amp;author=Darst,+S.A.&amp;publication_year=2013&amp;journal=Proc.+Natl.+Acad.+Sci.+USA&amp;volume=110&amp;pages=19772%E2%80%9319777&amp;doi=10.1073/pnas.1314576110" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1073/pnas.1314576110" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B24-crystals-10-00580" class="html-xx" data-content="24.">Shaya,
 D.; Findeisen, F.; Abderemane-Ali, F.; Arrigoni, C.; Wong, S.; Nurva, 
S.R.; Loussouarn, G.; Minor, D.L., Jr. Structure of a prokaryotic sodium
 channel pore reveals essential gating elements and an outer ion binding
 site common to eukaryotic channels. <span class="html-italic">J. Mol. Biol.</span> <b>2014</b>, <span class="html-italic">426</span>, 467483. [<a href="https://scholar.google.com/scholar_lookup?title=Structure+of+a+prokaryotic+sodium+channel+pore+reveals+essential+gating+elements+and+an+outer+ion+binding+site+common+to+eukaryotic+channels&amp;author=Shaya,+D.&amp;author=Findeisen,+F.&amp;author=Abderemane-Ali,+F.&amp;author=Arrigoni,+C.&amp;author=Wong,+S.&amp;author=Nurva,+S.R.&amp;author=Loussouarn,+G.&amp;author=Minor,+D.L.,+Jr.&amp;publication_year=2014&amp;journal=J.+Mol.+Biol.&amp;volume=426&amp;pages=467%E2%80%93483&amp;doi=10.1016/j.jmb.2013.10.010" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.jmb.2013.10.010" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B25-crystals-10-00580" class="html-xx" data-content="25.">Akey,
 D.L.; Brown, W.C.; Konwerski, J.R.; Ogata, C.M.; Smith, J.L. Use of 
massively multiple merged data for low-resolution S-SAD phasing and 
refinement of flavivirus NS1. <span class="html-italic">Acta Crystallogr. Sect. D Biol. Crystallogr.</span> <b>2014</b>, <span class="html-italic">70</span>, 27192729. [<a href="https://scholar.google.com/scholar_lookup?title=Use+of+massively+multiple+merged+data+for+low-resolution+S-SAD+phasing+and+refinement+of+flavivirus+NS1&amp;author=Akey,+D.L.&amp;author=Brown,+W.C.&amp;author=Konwerski,+J.R.&amp;author=Ogata,+C.M.&amp;author=Smith,+J.L.&amp;publication_year=2014&amp;journal=Acta+Crystallogr.+Sect.+D+Biol.+Crystallogr.&amp;volume=70&amp;pages=2719%E2%80%932729&amp;doi=10.1107/S1399004714017556&amp;pmid=25286855" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S1399004714017556" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/25286855" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B26-crystals-10-00580" class="html-xx" data-content="26.">Liu,
 Q.; Hendrickson, W. Robust structural analysis of native biological 
macromolecules from multi-crystal anomalous diffraction data. <span class="html-italic">Acta Crystallogr. Sect. D Biol. Crystallogr.</span> <b>2013</b>, <span class="html-italic">69</span>, 13141332. [<a href="https://scholar.google.com/scholar_lookup?title=Robust+structural+analysis+of+native+biological+macromolecules+from+multi-crystal+anomalous+diffraction+data&amp;author=Liu,+Q.&amp;author=Hendrickson,+W.&amp;publication_year=2013&amp;journal=Acta+Crystallogr.+Sect.+D+Biol.+Crystallogr.&amp;volume=69&amp;pages=1314%E2%80%931332&amp;doi=10.1107/S0907444913001479&amp;pmid=23793158" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0907444913001479" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/23793158" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B27-crystals-10-00580" class="html-xx" data-content="27.">Karplus, P.A.; Diederichs, K. Assessing and maximizing data quality in macromolecular crystallography. <span class="html-italic">Curr. Opin. Struct. Biol.</span> <b>2015</b>, <span class="html-italic">34</span>, 6068. [<a href="https://scholar.google.com/scholar_lookup?title=Assessing+and+maximizing+data+quality+in+macromolecular+crystallography&amp;author=Karplus,+P.A.&amp;author=Diederichs,+K.&amp;publication_year=2015&amp;journal=Curr.+Opin.+Struct.+Biol.&amp;volume=34&amp;pages=60%E2%80%9368&amp;doi=10.1016/j.sbi.2015.07.003" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.sbi.2015.07.003" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B28-crystals-10-00580" class="html-xx" data-content="28.">Joosten, R.P.; Long, F.; Murshudov, G.N.; Perrakis, A. The PDB_REDO server for macromolecular structure model optimization. <span class="html-italic">IUCrJ</span> <b>2014</b>, <span class="html-italic">1</span>, 213220. [<a href="https://scholar.google.com/scholar_lookup?title=The+PDB_REDO+server+for+macromolecular+structure+model+optimization&amp;author=Joosten,+R.P.&amp;author=Long,+F.&amp;author=Murshudov,+G.N.&amp;author=Perrakis,+A.&amp;publication_year=2014&amp;journal=IUCrJ&amp;volume=1&amp;pages=213%E2%80%93220&amp;doi=10.1107/S2052252514009324&amp;pmid=25075342" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S2052252514009324" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/25075342" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B29-crystals-10-00580" class="html-xx" data-content="29.">Urzhumtseva, L.; Klaholz, B.; Urzhumtsev, A. On effective and optical resolutions of diffraction data sets. <span class="html-italic">Acta Crystallogr. Sect. D Biol. Crystallogr.</span> <b>2013</b>, <span class="html-italic">69</span>, 19211934. [<a href="https://scholar.google.com/scholar_lookup?title=On+effective+and+optical+resolutions+of+diffraction+data+sets&amp;author=Urzhumtseva,+L.&amp;author=Klaholz,+B.&amp;author=Urzhumtsev,+A.&amp;publication_year=2013&amp;journal=Acta+Crystallogr.+Sect.+D+Biol.+Crystallogr.&amp;volume=69&amp;pages=1921%E2%80%931934&amp;doi=10.1107/S0907444913016673" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0907444913016673" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B30-crystals-10-00580" class="html-xx" data-content="30.">Vaguine,
 A.A.; Richelle, J.; Wodak, S. SFCHECK: A unified set of procedures for 
evaluating the quality of macromolecular structure-factor data and their
 agreement with the atomic model. <span class="html-italic">Acta Crystallogr. Sect. D Biol. Crystallogr.</span> <b>1999</b>, <span class="html-italic">55</span>, 191205. [<a href="https://scholar.google.com/scholar_lookup?title=SFCHECK:+A+unified+set+of+procedures+for+evaluating+the+quality+of+macromolecular+structure-factor+data+and+their+agreement+with+the+atomic+model&amp;author=Vaguine,+A.A.&amp;author=Richelle,+J.&amp;author=Wodak,+S.&amp;publication_year=1999&amp;journal=Acta+Crystallogr.+Sect.+D+Biol.+Crystallogr.&amp;volume=55&amp;pages=191%E2%80%93205&amp;doi=10.1107/S0907444998006684&amp;pmid=10089410" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0907444998006684" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/10089410" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B31-crystals-10-00580" class="html-xx" data-content="31.">Wilson, A. The probability distribution of X-ray intensities. <span class="html-italic">Acta Crystallogr.</span> <b>1949</b>, <span class="html-italic">2</span>, 318321. [<a href="https://scholar.google.com/scholar_lookup?title=The+probability+distribution+of+X-ray+intensities&amp;author=Wilson,+A.&amp;publication_year=1949&amp;journal=Acta+Crystallogr.&amp;volume=2&amp;pages=318%E2%80%93321&amp;doi=10.1107/S0365110X49000813" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0365110X49000813" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B32-crystals-10-00580" class="html-xx" data-content="32.">Weiss, M.S. Global indicators of X-ray data quality. <span class="html-italic">J. Appl. Crystallogr.</span> <b>2001</b>, <span class="html-italic">34</span>, 130135. [<a href="https://scholar.google.com/scholar_lookup?title=Global+indicators+of+X-ray+data+quality&amp;author=Weiss,+M.S.&amp;publication_year=2001&amp;journal=J.+Appl.+Crystallogr.&amp;volume=34&amp;pages=130%E2%80%93135&amp;doi=10.1107/S0021889800018227" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0021889800018227" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B33-crystals-10-00580" class="html-xx" data-content="33.">Urzhumtseva, L.; Urzhumtsev, A. EFRESOL: Effective resolution of a diffraction data set. <span class="html-italic">J. Appl. Crystallogr.</span> <b>2015</b>, <span class="html-italic">48</span>, 589597. [<a href="https://scholar.google.com/scholar_lookup?title=EFRESOL:+Effective+resolution+of+a+diffraction+data+set&amp;author=Urzhumtseva,+L.&amp;author=Urzhumtsev,+A.&amp;publication_year=2015&amp;journal=J.+Appl.+Crystallogr.&amp;volume=48&amp;pages=589%E2%80%93597&amp;doi=10.1107/S1600576715001648" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S1600576715001648" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B34-crystals-10-00580" class="html-xx" data-content="34.">Veesler,
 D.; Campbell, M.G.; Cheng, A.; Fu, C.y.; Murez, Z.; Johnson, J.E.; 
Potter, C.S.; Carragher, B. Maximizing the potential of electron 
cryomicroscopy data collected using direct detectors. <span class="html-italic">J. Struct. Biol.</span> <b>2013</b>, <span class="html-italic">184</span>, 193202. [<a href="https://scholar.google.com/scholar_lookup?title=Maximizing+the+potential+of+electron+cryomicroscopy+data+collected+using+direct+detectors&amp;author=Veesler,+D.&amp;author=Campbell,+M.G.&amp;author=Cheng,+A.&amp;author=Fu,+C.y.&amp;author=Murez,+Z.&amp;author=Johnson,+J.E.&amp;author=Potter,+C.S.&amp;author=Carragher,+B.&amp;publication_year=2013&amp;journal=J.+Struct.+Biol.&amp;volume=184&amp;pages=193%E2%80%93202&amp;doi=10.1016/j.jsb.2013.09.003&amp;pmid=24036281" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.jsb.2013.09.003" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/24036281" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B35-crystals-10-00580" class="html-xx" data-content="35.">McMullan,
 G.; Chen, S.; Henderson, R.; Faruqi, A. Detective quantum efficiency of
 electron area detectors in electron microscopy. <span class="html-italic">Ultramicroscopy</span> <b>2009</b>, <span class="html-italic">109</span>, 11261143. [<a href="https://scholar.google.com/scholar_lookup?title=Detective+quantum+efficiency+of+electron+area+detectors+in+electron+microscopy&amp;author=McMullan,+G.&amp;author=Chen,+S.&amp;author=Henderson,+R.&amp;author=Faruqi,+A.&amp;publication_year=2009&amp;journal=Ultramicroscopy&amp;volume=109&amp;pages=1126%E2%80%931143&amp;doi=10.1016/j.ultramic.2009.04.002&amp;pmid=19497671" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ultramic.2009.04.002" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/19497671" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B36-crystals-10-00580" class="html-xx" data-content="36.">Milazzo,
 A.C.; Leblanc, P.; Duttweiler, F.; Jin, L.; Bouwer, J.C.; Peltier, S.; 
Ellisman, M.; Bieser, F.; Matis, H.S.; Wieman, H.; et al. Active pixel 
sensor array as a detector for electron microscopy. <span class="html-italic">Ultramicroscopy</span> <b>2005</b>, <span class="html-italic">104</span>, 152159. [<a href="https://scholar.google.com/scholar_lookup?title=Active+pixel+sensor+array+as+a+detector+for+electron+microscopy&amp;author=Milazzo,+A.C.&amp;author=Leblanc,+P.&amp;author=Duttweiler,+F.&amp;author=Jin,+L.&amp;author=Bouwer,+J.C.&amp;author=Peltier,+S.&amp;author=Ellisman,+M.&amp;author=Bieser,+F.&amp;author=Matis,+H.S.&amp;author=Wieman,+H.&amp;publication_year=2005&amp;journal=Ultramicroscopy&amp;volume=104&amp;pages=152%E2%80%93159&amp;doi=10.1016/j.ultramic.2005.03.006&amp;pmid=15890445" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ultramic.2005.03.006" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/15890445" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B37-crystals-10-00580" class="html-xx" data-content="37.">Campbell,
 M.G.; Cheng, A.; Brilot, A.F.; Moeller, A.; Lyumkis, D.; Veesler, D.; 
Pan, J.; Harrison, S.C.; Potter, C.S.; Carragher, B.; et al. Movies of 
ice-embedded particles enhance resolution in electron cryo-microscopy. <span class="html-italic">Structure</span> <b>2012</b>, <span class="html-italic">20</span>, 18231828. [<a href="https://scholar.google.com/scholar_lookup?title=Movies+of+ice-embedded+particles+enhance+resolution+in+electron+cryo-microscopy&amp;author=Campbell,+M.G.&amp;author=Cheng,+A.&amp;author=Brilot,+A.F.&amp;author=Moeller,+A.&amp;author=Lyumkis,+D.&amp;author=Veesler,+D.&amp;author=Pan,+J.&amp;author=Harrison,+S.C.&amp;author=Potter,+C.S.&amp;author=Carragher,+B.&amp;publication_year=2012&amp;journal=Structure&amp;volume=20&amp;pages=1823%E2%80%931828&amp;doi=10.1016/j.str.2012.08.026&amp;pmid=23022349" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.str.2012.08.026" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/23022349" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B38-crystals-10-00580" class="html-xx" data-content="38.">Scheres, S.H. A Bayesian view on cryo-EM structure determination. <span class="html-italic">J. Mol. Biol.</span> <b>2012</b>, <span class="html-italic">415</span>, 406418. [<a href="https://scholar.google.com/scholar_lookup?title=A+Bayesian+view+on+cryo-EM+structure+determination&amp;author=Scheres,+S.H.&amp;publication_year=2012&amp;journal=J.+Mol.+Biol.&amp;volume=415&amp;pages=406%E2%80%93418&amp;doi=10.1016/j.jmb.2011.11.010&amp;pmid=22100448" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.jmb.2011.11.010" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/22100448" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B39-crystals-10-00580" class="html-xx" data-content="39.">Saxton, W.; Baumeister, W. The correlation averaging of a regularly arranged bacterial cell envelope protein. <span class="html-italic">J. Microsc.</span> <b>1982</b>, <span class="html-italic">127</span>, 127138. [<a href="https://scholar.google.com/scholar_lookup?title=The+correlation+averaging+of+a+regularly+arranged+bacterial+cell+envelope+protein&amp;author=Saxton,+W.&amp;author=Baumeister,+W.&amp;publication_year=1982&amp;journal=J.+Microsc.&amp;volume=127&amp;pages=127%E2%80%93138&amp;doi=10.1111/j.1365-2818.1982.tb00405.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1365-2818.1982.tb00405.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B40-crystals-10-00580" class="html-xx" data-content="40.">Van Heel, M.; Keegstra, W.; Schutter, W.; Van Bruggen, E. Arthropod hemocyanin structures studied by image analysis. <span class="html-italic">Life Chem. Rep. Suppl</span> <b>1982</b>, <span class="html-italic">1</span>, 6973. [<a href="https://scholar.google.com/scholar_lookup?title=Arthropod+hemocyanin+structures+studied+by+image+analysis&amp;author=Van+Heel,+M.&amp;author=Keegstra,+W.&amp;author=Schutter,+W.&amp;author=Van+Bruggen,+E.&amp;publication_year=1982&amp;journal=Life+Chem.+Rep.+Suppl&amp;volume=1&amp;pages=69%E2%80%9373" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B41-crystals-10-00580" class="html-xx" data-content="41.">Unser, M.; Trus, B.L.; Steven, A.C. A new resolution criterion based on spectral signal-to-noise ratios. <span class="html-italic">Ultramicroscopy</span> <b>1987</b>, <span class="html-italic">23</span>, 3951. [<a href="https://scholar.google.com/scholar_lookup?title=A+new+resolution+criterion+based+on+spectral+signal-to-noise+ratios&amp;author=Unser,+M.&amp;author=Trus,+B.L.&amp;author=Steven,+A.C.&amp;publication_year=1987&amp;journal=Ultramicroscopy&amp;volume=23&amp;pages=39%E2%80%9351&amp;doi=10.1016/0304-3991(87)90225-7" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/0304-3991(87)90225-7" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B42-crystals-10-00580" class="html-xx" data-content="42.">Unser,
 M.; Sorzano, C.S.; Thevenaz, P.; Joni, S.; El-Bez, C.; De Carlo, S.; 
Conway, J.; Trus, B. Spectral signal-to-noise ratio and resolution 
assessment of 3D reconstructions. <span class="html-italic">J. Struct. Biol.</span> <b>2005</b>, <span class="html-italic">149</span>, 243255. [<a href="https://scholar.google.com/scholar_lookup?title=Spectral+signal-to-noise+ratio+and+resolution+assessment+of+3D+reconstructions&amp;author=Unser,+M.&amp;author=Sorzano,+C.S.&amp;author=Thevenaz,+P.&amp;author=Joni%C4%87,+S.&amp;author=El-Bez,+C.&amp;author=De+Carlo,+S.&amp;author=Conway,+J.&amp;author=Trus,+B.&amp;publication_year=2005&amp;journal=J.+Struct.+Biol.&amp;volume=149&amp;pages=243%E2%80%93255&amp;doi=10.1016/j.jsb.2004.10.011" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.jsb.2004.10.011" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B43-crystals-10-00580" class="html-xx" data-content="43.">Penczek, P.A. Three-dimensional spectral signal-to-noise ratio for a class of reconstruction algorithms. <span class="html-italic">J. Struct. Biol.</span> <b>2002</b>, <span class="html-italic">138</span>, 3446. [<a href="https://scholar.google.com/scholar_lookup?title=Three-dimensional+spectral+signal-to-noise+ratio+for+a+class+of+reconstruction+algorithms&amp;author=Penczek,+P.A.&amp;publication_year=2002&amp;journal=J.+Struct.+Biol.&amp;volume=138&amp;pages=34%E2%80%9346&amp;doi=10.1016/S1047-8477(02)00033-3" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/S1047-8477(02)00033-3" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B44-crystals-10-00580" class="html-xx" data-content="44.">Sousa, D.; Grigorieff, N. Ab initio resolution measurement for single particle structures. <span class="html-italic">J. Struct. Biol.</span> <b>2007</b>, <span class="html-italic">157</span>, 201210. [<a href="https://scholar.google.com/scholar_lookup?title=Ab+initio+resolution+measurement+for+single+particle+structures&amp;author=Sousa,+D.&amp;author=Grigorieff,+N.&amp;publication_year=2007&amp;journal=J.+Struct.+Biol.&amp;volume=157&amp;pages=201%E2%80%93210&amp;doi=10.1016/j.jsb.2006.08.003&amp;pmid=17029845" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.jsb.2006.08.003" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/17029845" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B45-crystals-10-00580" class="html-xx" data-content="45.">Sorzano,
 C.; Vargas, J.; Otn, J.; Abrishami, V.; de la Rosa-Trevn, J.; 
Gmez-Blanco, J.; Vilas, J.; Marabini, R.; Carazo, J. A review of 
resolution measures and related aspects in 3D Electron Microscopy. <span class="html-italic">Prog. Biophys. Mol. Biol.</span> <b>2017</b>, <span class="html-italic">124</span>, 130. [<a href="https://scholar.google.com/scholar_lookup?title=A+review+of+resolution+measures+and+related+aspects+in+3D+Electron+Microscopy&amp;author=Sorzano,+C.&amp;author=Vargas,+J.&amp;author=Ot%C3%B3n,+J.&amp;author=Abrishami,+V.&amp;author=de+la+Rosa-Trev%C3%ADn,+J.&amp;author=G%C3%B3mez-Blanco,+J.&amp;author=Vilas,+J.&amp;author=Marabini,+R.&amp;author=Carazo,+J.&amp;publication_year=2017&amp;journal=Prog.+Biophys.+Mol.+Biol.&amp;volume=124&amp;pages=1%E2%80%9330&amp;doi=10.1016/j.pbiomolbio.2016.09.005&amp;pmid=27666962" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.pbiomolbio.2016.09.005" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/27666962" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B46-crystals-10-00580" class="html-xx" data-content="46.">Harauz, G.; van Heel, M. Exact filters for general geometry three dimensional reconstruction. <span class="html-italic">Optik (Stuttg.)</span> <b>1986</b>, <span class="html-italic">73</span>, 146156. [<a href="https://scholar.google.com/scholar_lookup?title=Exact+filters+for+general+geometry+three+dimensional+reconstruction&amp;author=Harauz,+G.&amp;author=van+Heel,+M.&amp;publication_year=1986&amp;journal=Optik+(Stuttg.)&amp;volume=73&amp;pages=146%E2%80%93156" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B47-crystals-10-00580" class="html-xx" data-content="47.">Rosenthal,
 P.B.; Henderson, R. Optimal determination of particle orientation, 
absolute hand, and contrast loss in single-particle electron 
cryomicroscopy. <span class="html-italic">J. Mol. Biol.</span> <b>2003</b>, <span class="html-italic">333</span>, 721745. [<a href="https://scholar.google.com/scholar_lookup?title=Optimal+determination+of+particle+orientation,+absolute+hand,+and+contrast+loss+in+single-particle+electron+cryomicroscopy&amp;author=Rosenthal,+P.B.&amp;author=Henderson,+R.&amp;publication_year=2003&amp;journal=J.+Mol.+Biol.&amp;volume=333&amp;pages=721%E2%80%93745&amp;doi=10.1016/j.jmb.2003.07.013" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.jmb.2003.07.013" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B48-crystals-10-00580" class="html-xx" data-content="48.">Scheres, S.H.; Chen, S. Prevention of overfitting in cryo-EM structure determination. <span class="html-italic">Nat. Methods</span> <b>2012</b>, <span class="html-italic">9</span>, 853. [<a href="https://scholar.google.com/scholar_lookup?title=Prevention+of+overfitting+in+cryo-EM+structure+determination&amp;author=Scheres,+S.H.&amp;author=Chen,+S.&amp;publication_year=2012&amp;journal=Nat.+Methods&amp;volume=9&amp;pages=853&amp;doi=10.1038/nmeth.2115" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1038/nmeth.2115" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B49-crystals-10-00580" class="html-xx" data-content="49.">Henderson,
 R.; Sali, A.; Baker, M.L.; Carragher, B.; Devkota, B.; Downing, K.H.; 
Egelman, E.H.; Feng, Z.; Frank, J.; Grigorieff, N.; et al. Outcome of 
the first electron microscopy validation task force meeting. <span class="html-italic">Structure</span> <b>2012</b>, <span class="html-italic">20</span>, 205214. [<a href="https://scholar.google.com/scholar_lookup?title=Outcome+of+the+first+electron+microscopy+validation+task+force+meeting&amp;author=Henderson,+R.&amp;author=Sali,+A.&amp;author=Baker,+M.L.&amp;author=Carragher,+B.&amp;author=Devkota,+B.&amp;author=Downing,+K.H.&amp;author=Egelman,+E.H.&amp;author=Feng,+Z.&amp;author=Frank,+J.&amp;author=Grigorieff,+N.&amp;publication_year=2012&amp;journal=Structure&amp;volume=20&amp;pages=205%E2%80%93214&amp;doi=10.1016/j.str.2011.12.014" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.str.2011.12.014" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B50-crystals-10-00580" class="html-xx" data-content="50.">Lunin, V.Y.; Woolfson, M. Mean phase error and the map-correlation coefficient. <span class="html-italic">Acta Crystallogr. Sect. D Biol. Crystallogr.</span> <b>1993</b>, <span class="html-italic">49</span>, 530533. [<a href="https://scholar.google.com/scholar_lookup?title=Mean+phase+error+and+the+map-correlation+coefficient&amp;author=Lunin,+V.Y.&amp;author=Woolfson,+M.&amp;publication_year=1993&amp;journal=Acta+Crystallogr.+Sect.+D+Biol.+Crystallogr.&amp;volume=49&amp;pages=530%E2%80%93533&amp;doi=10.1107/S0907444993005852" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0907444993005852" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B51-crystals-10-00580" class="html-xx" data-content="51.">Van Heel, M.; Schatz, M. Fourier shell correlation threshold criteria. <span class="html-italic">J. Struct. Biol.</span> <b>2005</b>, <span class="html-italic">151</span>, 250262. [<a href="https://scholar.google.com/scholar_lookup?title=Fourier+shell+correlation+threshold+criteria&amp;author=Van+Heel,+M.&amp;author=Schatz,+M.&amp;publication_year=2005&amp;journal=J.+Struct.+Biol.&amp;volume=151&amp;pages=250%E2%80%93262&amp;doi=10.1016/j.jsb.2005.05.009" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.jsb.2005.05.009" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B52-crystals-10-00580" class="html-xx" data-content="52.">van Heel, M.; Schatz, M. Reassessing the revolutions resolutions. <span class="html-italic">BioRxiv</span> <b>2017</b>, 224402. [<a href="https://scholar.google.com/scholar_lookup?title=Reassessing+the+revolutions+resolutions&amp;author=van+Heel,+M.&amp;author=Schatz,+M.&amp;publication_year=2017&amp;journal=BioRxiv&amp;pages=224402&amp;doi=10.1101/224402" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1101/224402" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B53-crystals-10-00580" class="html-xx" data-content="53.">Grigorieff, N. Resolution measurement in structures derived from single particles. <span class="html-italic">Acta Crystallogr. Sect. D Biol. Crystallogr.</span> <b>2000</b>, <span class="html-italic">56</span>, 12701277. [<a href="https://scholar.google.com/scholar_lookup?title=Resolution+measurement+in+structures+derived+from+single+particles&amp;author=Grigorieff,+N.&amp;publication_year=2000&amp;journal=Acta+Crystallogr.+Sect.+D+Biol.+Crystallogr.&amp;volume=56&amp;pages=1270%E2%80%931277&amp;doi=10.1107/S0907444900009549&amp;pmid=10998623" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1107/S0907444900009549" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/10998623" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B54-crystals-10-00580" class="html-xx" data-content="54.">Sindelar, C.V.; Downing, K.H. The beginning of kinesins force-generating cycle visualized at 9-A resolution. <span class="html-italic">J. Cell Biol.</span> <b>2007</b>, <span class="html-italic">177</span>, 377385. [<a href="https://scholar.google.com/scholar_lookup?title=The+beginning+of+kinesin%E2%80%99s+force-generating+cycle+visualized+at+9-A+resolution&amp;author=Sindelar,+C.V.&amp;author=Downing,+K.H.&amp;publication_year=2007&amp;journal=J.+Cell+Biol.&amp;volume=177&amp;pages=377%E2%80%93385&amp;doi=10.1083/jcb.200612090&amp;pmid=17470637" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1083/jcb.200612090" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/17470637" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B55-crystals-10-00580" class="html-xx" data-content="55.">Lau,
 W.C.; Rubinstein, J.L. Structure of intact Thermus thermophilus 
V-ATPase by cryo-EM reveals organization of the membrane-bound VO motor.
 <span class="html-italic">Proc. Natl. Acad. Sci. USA</span> <b>2010</b>, <span class="html-italic">107</span>, 13671372. [<a href="https://scholar.google.com/scholar_lookup?title=Structure+of+intact+Thermus+thermophilus+V-ATPase+by+cryo-EM+reveals+organization+of+the+membrane-bound+VO+motor&amp;author=Lau,+W.C.&amp;author=Rubinstein,+J.L.&amp;publication_year=2010&amp;journal=Proc.+Natl.+Acad.+Sci.+USA&amp;volume=107&amp;pages=1367%E2%80%931372&amp;doi=10.1073/pnas.0911085107&amp;pmid=20080582" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1073/pnas.0911085107" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/20080582" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B56-crystals-10-00580" class="html-xx" data-content="56.">Yuan, S.; Yu, X.; Topf, M.; Ludtke, S.J.; Wang, X.; Akey, C.W. Structure of an apoptosome-procaspase-9 CARD complex. <span class="html-italic">Structure</span> <b>2010</b>, <span class="html-italic">18</span>, 571583. [<a href="https://scholar.google.com/scholar_lookup?title=Structure+of+an+apoptosome-procaspase-9+CARD+complex&amp;author=Yuan,+S.&amp;author=Yu,+X.&amp;author=Topf,+M.&amp;author=Ludtke,+S.J.&amp;author=Wang,+X.&amp;author=Akey,+C.W.&amp;publication_year=2010&amp;journal=Structure&amp;volume=18&amp;pages=571%E2%80%93583&amp;doi=10.1016/j.str.2010.04.001" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.str.2010.04.001" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B57-crystals-10-00580" class="html-xx" data-content="57.">Cardone,
 G.; Heymann, J.B.; Steven, A.C. One number does not fit all: Mapping 
local variations in resolution in cryo-EM reconstructions. <span class="html-italic">J. Struct. Biol.</span> <b>2013</b>, <span class="html-italic">184</span>, 226236. [<a href="https://scholar.google.com/scholar_lookup?title=One+number+does+not+fit+all:+Mapping+local+variations+in+resolution+in+cryo-EM+reconstructions&amp;author=Cardone,+G.&amp;author=Heymann,+J.B.&amp;author=Steven,+A.C.&amp;publication_year=2013&amp;journal=J.+Struct.+Biol.&amp;volume=184&amp;pages=226%E2%80%93236&amp;doi=10.1016/j.jsb.2013.08.002" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.jsb.2013.08.002" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B58-crystals-10-00580" class="html-xx" data-content="58.">Vilas,
 J.L.; Gmez-Blanco, J.; Conesa, P.; Melero, R.; de la Rosa-Trevn, 
J.M.; Otn, J.; Cuenca, J.; Marabini, R.; Carazo, J.M.; Vargas, J.; et 
al. MonoRes: Automatic and accurate estimation of local resolution for 
electron microscopy maps. <span class="html-italic">Structure</span> <b>2018</b>, <span class="html-italic">26</span>, 337344. [<a href="https://scholar.google.com/scholar_lookup?title=MonoRes:+Automatic+and+accurate+estimation+of+local+resolution+for+electron+microscopy+maps&amp;author=Vilas,+J.L.&amp;author=G%C3%B3mez-Blanco,+J.&amp;author=Conesa,+P.&amp;author=Melero,+R.&amp;author=de+la+Rosa-Trev%C3%ADn,+J.M.&amp;author=Ot%C3%B3n,+J.&amp;author=Cuenca,+J.&amp;author=Marabini,+R.&amp;author=Carazo,+J.M.&amp;author=Vargas,+J.&amp;publication_year=2018&amp;journal=Structure&amp;volume=26&amp;pages=337%E2%80%93344&amp;doi=10.1016/j.str.2017.12.018" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.str.2017.12.018" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B59-crystals-10-00580" class="html-xx" data-content="59.">Kucukelbir, A.; Sigworth, F.J.; Tagare, H.D. Quantifying the local resolution of cryo-EM density maps. <span class="html-italic">Nat. Methods</span> <b>2014</b>, <span class="html-italic">11</span>, 6365. [<a href="https://scholar.google.com/scholar_lookup?title=Quantifying+the+local+resolution+of+cryo-EM+density+maps&amp;author=Kucukelbir,+A.&amp;author=Sigworth,+F.J.&amp;author=Tagare,+H.D.&amp;publication_year=2014&amp;journal=Nat.+Methods&amp;volume=11&amp;pages=63%E2%80%9365&amp;doi=10.1038/nmeth.2727" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1038/nmeth.2727" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B60-crystals-10-00580" class="html-xx" data-content="60.">Unser, M.; Van De Ville, D. Wavelet steerability and the higher-order Riesz transform. <span class="html-italic">IEEE Trans. Image Process.</span> <b>2009</b>, <span class="html-italic">19</span>, 636652. [<a href="https://scholar.google.com/scholar_lookup?title=Wavelet+steerability+and+the+higher-order+Riesz+transform&amp;author=Unser,+M.&amp;author=Van+De+Ville,+D.&amp;publication_year=2009&amp;journal=IEEE+Trans.+Image+Process.&amp;volume=19&amp;pages=636%E2%80%93652&amp;doi=10.1109/TIP.2009.2038832" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1109/TIP.2009.2038832" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B61-crystals-10-00580" class="html-xx" data-content="61.">Vilas,
 J.L.; Tagare, H.D.; Vargas, J.; Carazo, J.M.; Sorzano, C.O.S. Measuring
 local-directional resolution and local anisotropy in cryo-EM maps. <span class="html-italic">Nat. Commun.</span> <b>2020</b>, <span class="html-italic">11</span>, 17. [<a href="https://scholar.google.com/scholar_lookup?title=Measuring+local-directional+resolution+and+local+anisotropy+in+cryo-EM+maps&amp;author=Vilas,+J.L.&amp;author=Tagare,+H.D.&amp;author=Vargas,+J.&amp;author=Carazo,+J.M.&amp;author=Sorzano,+C.O.S.&amp;publication_year=2020&amp;journal=Nat.+Commun.&amp;volume=11&amp;pages=1%E2%80%937&amp;doi=10.1038/s41467-019-13742-w" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1038/s41467-019-13742-w" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li></ol></section><section id="Figure" type="display-objects"><div class="html-fig-wrap" id="crystals-10-00580-f001">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_crystals-10-00580-f001">
<img alt="Crystals 10 00580 g001 550" data-large="/crystals/crystals-10-00580/article_deploy/html/images/crystals-10-00580-g001.png" data-original="/crystals/crystals-10-00580/article_deploy/html/images/crystals-10-00580-g001.png" src="crystals-10-00580-g001-550.jpg">
<a class="html-expand html-figpopup" href="#fig_body_display_crystals-10-00580-f001"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 1.</b>
A typical FSC curve with the thresholds 0.5 and 0.143 indicated.

</div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_crystals-10-00580-f001">
<div class="html-caption"> <b>Figure 1.</b>
A typical FSC curve with the thresholds 0.5 and 0.143 indicated.</div>
<div class="html-img"><img alt="Crystals 10 00580 g001" data-large="/crystals/crystals-10-00580/article_deploy/html/images/crystals-10-00580-g001.png" data-original="/crystals/crystals-10-00580/article_deploy/html/images/crystals-10-00580-g001.png" src="crystals-10-00580-g001.png"></div>
</div></section>
<section id="html-copyright"><br> 2020 by the authors. Licensee MDPI, 
Basel, Switzerland. This article is an open access article distributed 
under the terms and conditions of the Creative Commons Attribution (CC 
BY) license (<a href="https://creativecommons.org/licenses/by/4.0/" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>).</section>
</div>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</section>
<div id="footer">
<div class="journal-info">
<span>
<em><a class="Var_JournalInfo" href="https://www.mdpi.com/journal/crystals">Crystals</a></em>,
EISSN 2073-4352,
Published by MDPI
</span>
<span>
<a data-dropdown="drop-view-disclaimer-journal" aria-controls="drop-view-disclaimer-journal" aria-expanded="false" data-options="align:top; is_hover:true; hover_timeout:2000;">
Disclaimer
</a>
<div id="drop-view-disclaimer-journal" class="f-dropdown label__btn__dropdown label__btn__dropdown--wide text-left" data-dropdown-content="" aria-hidden="true" tabindex="-1">
The statements, opinions and data contained in the journal <em>Crystals</em> are solely
those of the individual authors and contributors and not of the publisher and the editor(s).
MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.
</div>
</span>
<div class="large-right">
<span>
<a href="https://www.mdpi.com/rss/journal/crystals">RSS</a>
</span>
<span>
<a href="https://www.mdpi.com/journal/crystals/toc-alert">Content Alert</a>
</span>
</div>
</div>
<div class="row full-width footer-links" data-equalizer="footer" data-equalizer-mq="small">
<div class="large-2 large-push-4 medium-3 small-6 columns" data-equalizer-watch="footer">
<h3>
Further Information
</h3>
<a href="https://www.mdpi.com/apc">
Article Processing Charges
</a>
<a href="https://www.mdpi.com/about/payment">
Pay an Invoice
</a>
<a href="https://www.mdpi.com/openaccess">
Open Access Policy
</a>
<a href="https://www.mdpi.com/about/contact">
Contact MDPI
</a>
<a href="https://careers.mdpi.com/" target="_blank" rel="noopener noreferrer">
Jobs at MDPI
</a>
</div>
<div class="large-2 large-push-4 medium-3 small-6 columns" data-equalizer-watch="footer">
<h3>
Guidelines
</h3>
<a href="https://www.mdpi.com/authors">
For Authors
</a>
<a href="https://www.mdpi.com/reviewers">
For Reviewers
</a>
<a href="https://www.mdpi.com/editors">
For Editors
</a>
<a href="https://www.mdpi.com/librarians">
For Librarians
</a>
<a href="https://www.mdpi.com/publishing_services">
For Publishers
</a>
<a href="https://www.mdpi.com/societies">
For Societies
</a>
</div>
<div class="large-2 large-push-4 medium-3 small-6 columns">
<h3>
MDPI Initiatives
</h3>
<a href="https://www.mdpi.com/ioap">
Institutional Open Access Program (IOAP)
</a>
<a href="https://sciforum.net/" target="_blank" rel="noopener noreferrer">
Sciforum
</a>
<a href="https://www.preprints.org/" target="_blank" rel="noopener noreferrer">
Preprints
</a>
<a href="https://www.scilit.net/" target="_blank" rel="noopener noreferrer">
Scilit
</a>
<a href="https://sciprofiles.com/" target="_blank" rel="noopener noreferrer">
SciProfiles
</a>
<a href="https://www.mdpi.com/books" target="_blank" rel="noopener noreferrer">
MDPI Books
</a>
<a href="https://encyclopedia.pub/" target="_blank" rel="noopener noreferrer">
Encyclopedia
</a>
<a href="https://jams.pub/" target="_blank" rel="noopener noreferrer">
JAMS
</a>
<a href="https://www.mdpi.com/about/proceedings">
Proceedings
</a>
<a href="http://blog.mdpi.com/" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', 'homeBlog');" target="_blank" rel="noopener noreferrer">
MDPI Blog
</a>
</div>
<div class="large-2 large-push-4 medium-3 small-6 right-border-large-without columns UA_FooterFollowMDPI">
<h3>
Follow MDPI
</h3>
<a href="https://www.linkedin.com/company/mdpi" target="_blank" rel="noopener noreferrer">
LinkedIn
</a>
<a href="https://www.facebook.com/MDPIOpenAccessPublishing" target="_blank" rel="noopener noreferrer">
Facebook
</a>
<a href="https://twitter.com/MDPIOpenAccess" target="_blank" rel="noopener noreferrer">
Twitter
</a>
</div>
<div id="footer-subscribe" class="large-4 large-pull-8 medium-12 small-12 left-border-large columns">
<div class="footer-subscribe__container">
<img class="show-for-large-up" src="mdpi-pub-logo-white-small.png" alt="MDPI" title="MDPI Open Access Journals" style="height: 50px; margin-bottom: 10px;">
<form id="newsletter" method="POST" action="/subscribe">
<p>
Subscribe to receive issue release notifications and newsletters from MDPI journals
</p>
<select multiple="multiple" id="newsletter-journal" class="foundation-select" name="journals[]">
<option value="acoustics">Acoustics</option>
<option value="actuators">Actuators</option>
<option value="admsci">Administrative Sciences</option>
<option value="adolescents">Adolescents</option>
<option value="aerospace">Aerospace</option>
<option value="agriculture">Agriculture</option>
<option value="agriengineering">AgriEngineering</option>
<option value="agronomy">Agronomy</option>
<option value="ai">AI</option>
<option value="algorithms">Algorithms</option>
<option value="allergies">Allergies</option>
<option value="analytica">Analytica</option>
<option value="animals">Animals</option>
<option value="antibiotics">Antibiotics</option>
<option value="antibodies">Antibodies</option>
<option value="antioxidants">Antioxidants</option>
<option value="applmech">Applied Mechanics</option>
<option value="applnano">Applied Nano</option>
<option value="applsci">Applied Sciences</option>
<option value="asi">Applied System Innovation</option>
<option value="arts">Arts</option>
<option value="atmosphere">Atmosphere</option>
<option value="atoms">Atoms</option>
<option value="audiolres">Audiology Research</option>
<option value="automation">Automation</option>
 <option value="axioms">Axioms</option>
<option value="batteries">Batteries</option>
<option value="behavsci">Behavioral Sciences</option>
<option value="beverages">Beverages</option>
<option value="BDCC">Big Data and Cognitive Computing</option>
<option value="bioengineering">Bioengineering</option>
<option value="biology">Biology</option>
<option value="biomechanics">Biomechanics</option>
<option value="biomedicines">Biomedicines</option>
<option value="biomimetics">Biomimetics</option>
<option value="biomolecules">Biomolecules</option>
<option value="biosensors">Biosensors</option>
<option value="biotech">BioTech</option>
<option value="birds">Birds</option>
<option value="brainsci">Brain Sciences</option>
<option value="buildings">Buildings</option>
<option value="carbon">C</option>
<option value="cancers">Cancers</option>
<option value="cardiogenetics">Cardiogenetics</option>
<option value="catalysts">Catalysts</option>
<option value="cells">Cells</option>
<option value="ceramics">Ceramics</option>
<option value="challenges">Challenges</option>
<option value="ChemEngineering">ChemEngineering</option>
<option value="chemistry">Chemistry</option>
<option value="chemproc">Chemistry Proceedings</option>
<option value="chemosensors">Chemosensors</option>
<option value="children">Children</option>
<option value="civileng">CivilEng</option>
<option value="cleantechnol">Clean Technologies</option>
<option value="climate">Climate</option>
<option value="clinpract">Clinics and Practice</option>
<option value="clockssleep">Clocks &amp; Sleep</option>
<option value="coatings">Coatings</option>
<option value="colloids">Colloids and Interfaces</option>
<option value="compounds">Compounds</option>
<option value="computation">Computation</option>
<option value="computers">Computers</option>
<option value="condensedmatter">Condensed Matter</option>
<option value="constrmater">Construction Materials</option>
<option value="cmd">Corrosion and Materials Degradation</option>
<option value="cosmetics">Cosmetics</option>
<option value="cryptography">Cryptography</option>
<option value="crystals">Crystals</option>
<option value="curroncol">Current Oncology</option>
<option value="dairy">Dairy</option>
<option value="data">Data</option>
<option value="dentistry">Dentistry Journal</option>
<option value="dermatopathology">Dermatopathology</option>
<option value="designs">Designs</option>
<option value="diabetology">Diabetology</option>
<option value="diagnostics">Diagnostics</option>
<option value="diseases">Diseases</option>
<option value="diversity">Diversity</option>
<option value="drones">Drones</option>
<option value="earth">Earth</option>
<option value="ecologies">Ecologies</option>
<option value="econometrics">Econometrics</option>
<option value="economies">Economies</option>
 <option value="education">Education Sciences</option>
<option value="electricity">Electricity</option>
<option value="electrochem">Electrochem</option>
<option value="electronicmat">Electronic Materials</option>
<option value="electronics">Electronics</option>
<option value="encyclopedia">Encyclopedia</option>
<option value="endocrines">Endocrines</option>
<option value="energies">Energies</option>
<option value="eng">Eng</option>
<option value="engproc">Engineering Proceedings</option>
<option value="entropy">Entropy</option>
<option value="environsciproc">Environmental Sciences Proceedings</option>
<option value="environments">Environments</option>
<option value="epidemiologia">Epidemiologia</option>
<option value="epigenomes">Epigenomes</option>
<option value="ebj">European Burn Journal</option>
<option value="ejihpe">European Journal of Investigation in Health, Psychology and Education</option>
<option value="fermentation">Fermentation</option>
<option value="fibers">Fibers</option>
<option value="fire">Fire</option>
<option value="fishes">Fishes</option>
<option value="fluids">Fluids</option>
<option value="foods">Foods</option>
<option value="forecasting">Forecasting</option>
<option value="forensicsci">Forensic Sciences</option>
<option value="forests">Forests</option>
<option value="fractalfract">Fractal and Fractional</option>
<option value="fuels">Fuels</option>
<option value="futureinternet">Future Internet</option>
<option value="galaxies">Galaxies</option>
<option value="games">Games</option>
<option value="gases">Gases</option>
<option value="gastroent">Gastroenterology Insights</option>
<option value="gastrointestdisord">Gastrointestinal Disorders</option>
<option value="gels">Gels</option>
<option value="genealogy">Genealogy</option>
<option value="genes">Genes</option>
<option value="geographies">Geographies</option>
<option value="geohazards">GeoHazards</option>
<option value="geosciences">Geosciences</option>
<option value="geriatrics">Geriatrics</option>
<option value="healthcare">Healthcare</option>
<option value="hearts">Hearts</option>
<option value="hemato">Hemato</option>
<option value="heritage">Heritage</option>
<option value="histories">Histories</option>
<option value="horticulturae">Horticulturae</option>
<option value="humanities">Humanities</option>
<option value="hydrogen">Hydrogen</option>
<option value="hydrology">Hydrology</option>
<option value="idr">Infectious Disease Reports</option>
<option value="informatics">Informatics</option>
<option value="information">Information</option>
<option value="infrastructures">Infrastructures</option>
<option value="inorganics">Inorganics</option>
<option value="insects">Insects</option>
<option value="instruments">Instruments</option>
<option value="ijerph">International Journal of Environmental Research and Public Health</option>
 <option value="ijfs">International Journal of Financial Studies</option>
<option value="ijms">International Journal of Molecular Sciences</option>
<option value="IJNS">International Journal of Neonatal Screening</option>
<option value="ijtpp">International Journal of Turbomachinery, Propulsion and Power</option>
<option value="inventions">Inventions</option>
<option value="IoT">IoT</option>
<option value="ijgi">ISPRS International Journal of Geo-Information</option>
<option value="J">J</option>
<option value="jcdd">Journal of Cardiovascular Development and Disease</option>
<option value="jcm">Journal of Clinical Medicine</option>
<option value="jcs">Journal of Composites Science</option>
<option value="jcp">Journal of Cybersecurity and Privacy</option>
<option value="jdb">Journal of Developmental Biology</option>
<option value="jfb">Journal of Functional Biomaterials</option>
<option value="jfmk">Journal of Functional Morphology and Kinesiology</option>
<option value="jof">Journal of Fungi</option>
<option value="jimaging">Journal of Imaging</option>
<option value="jintelligence">Journal of Intelligence</option>
<option value="jlpea">Journal of Low Power Electronics and Applications</option>
<option value="jmmp">Journal of Manufacturing and Materials Processing</option>
<option value="jmse">Journal of Marine Science and Engineering</option>
<option value="jmp">Journal of Molecular Pathology</option>
<option value="jnt">Journal of Nanotheranostics</option>
<option value="jne">Journal of Nuclear Engineering</option>
<option value="JOItmC">Journal of Open Innovation: Technology, Market, and Complexity</option>
<option value="ohbm">Journal of Otorhinolaryngology, Hearing and Balance Medicine</option>
<option value="jpm">Journal of Personalized Medicine</option>
<option value="jor">Journal of Respiration</option>
<option value="jrfm">Journal of Risk and Financial Management</option>
<option value="jsan">Journal of Sensor and Actuator Networks</option>
<option value="jtaer">Journal of Theoretical and Applied Electronic Commerce Research</option>
<option value="jox">Journal of Xenobiotics</option>
<option value="jzbg">Journal of Zoological and Botanical Gardens</option>
<option value="journalmedia">Journalism and Media</option>
<option value="land">Land</option>
<option value="languages">Languages</option>
<option value="laws">Laws</option>
<option value="life">Life</option>
<option value="livers">Livers</option>
<option value="logistics">Logistics</option>
<option value="lubricants">Lubricants</option>
<option value="make">Machine Learning and Knowledge Extraction</option>
<option value="machines">Machines</option>
<option value="macromol">Macromol</option>
<option value="magnetochemistry">Magnetochemistry</option>
<option value="marinedrugs">Marine Drugs</option>
<option value="materials">Materials</option>
<option value="materproc">Materials Proceedings</option>
<option value="mca">Mathematical and Computational Applications</option>
<option value="mathematics">Mathematics</option>
<option value="medsci">Medical Sciences</option>
<option value="medicina">Medicina</option>
<option value="medicines">Medicines</option>
<option value="membranes">Membranes</option>
 <option value="metabolites">Metabolites</option>
<option value="metals">Metals</option>
<option value="mps">Methods and Protocols</option>
<option value="microbiolres">Microbiology Research</option>
<option value="micromachines">Micromachines</option>
<option value="microorganisms">Microorganisms</option>
<option value="minerals">Minerals</option>
<option value="mining">Mining</option>
<option value="modelling">Modelling</option>
<option value="molbank">Molbank</option>
<option value="molecules">Molecules</option>
<option value="mti">Multimodal Technologies and Interaction</option>
<option value="nanomaterials">Nanomaterials</option>
<option value="neuroglia">Neuroglia</option>
<option value="neurolint">Neurology International</option>
<option value="neurosci">NeuroSci</option>
<option value="nitrogen">Nitrogen</option>
<option value="ncrna">Non-Coding RNA</option>
<option value="nursrep">Nursing Reports</option>
<option value="nutrients">Nutrients</option>
<option value="obesities">Obesities</option>
<option value="oceans">Oceans</option>
<option value="optics">Optics</option>
<option value="oral">Oral</option>
<option value="organics">Organics</option>
<option value="osteology">Osteology</option>
<option value="particles">Particles</option>
<option value="pathogens">Pathogens</option>
<option value="pathophysiology">Pathophysiology</option>
<option value="pediatrrep">Pediatric Reports</option>
<option value="pharmaceuticals">Pharmaceuticals</option>
<option value="pharmaceutics">Pharmaceutics</option>
<option value="pharmacy">Pharmacy</option>
<option value="philosophies">Philosophies</option>
<option value="photonics">Photonics</option>
<option value="physics">Physics</option>
<option value="plants">Plants</option>
<option value="plasma">Plasma</option>
<option value="polymers">Polymers</option>
<option value="polysaccharides">Polysaccharides</option>
<option value="proceedings">Proceedings</option>
<option value="processes">Processes</option>
<option value="prosthesis">Prosthesis</option>
<option value="proteomes">Proteomes</option>
<option value="psych">Psych</option>
<option value="psychiatryint">Psychiatry International</option>
<option value="publications">Publications</option>
<option value="qubs">Quantum Beam Science</option>
<option value="quantumrep">Quantum Reports</option>
<option value="quaternary">Quaternary</option>
<option value="radiation">Radiation</option>
<option value="reactions">Reactions</option>
<option value="recycling">Recycling</option>
<option value="religions">Religions</option>
<option value="remotesensing">Remote Sensing</option>
<option value="reports">Reports</option>
<option value="reprodmed">Reproductive Medicine</option>
<option value="resources">Resources</option>
<option value="risks">Risks</option>
<option value="robotics">Robotics</option>
<option value="safety">Safety</option>
<option value="sci">Sci</option>
<option value="scipharm">Scientia Pharmaceutica</option>
<option value="sensors">Sensors</option>
<option value="separations">Separations</option>
<option value="sexes">Sexes</option>
<option value="signals">Signals</option>
<option value="sinusitis">Sinusitis</option>
<option value="smartcities">Smart Cities</option>
<option value="socsci">Social Sciences</option>
<option value="societies">Societies</option>
<option value="soilsystems">Soil Systems</option>
<option value="solids">Solids</option>
<option value="sports">Sports</option>
<option value="stats">Stats</option>
<option value="stresses">Stresses</option>
<option value="surfaces">Surfaces</option>
<option value="surgeries">Surgeries</option>
<option value="sustainability">Sustainability</option>
<option value="suschem">Sustainable Chemistry</option>
<option value="symmetry">Symmetry</option>
<option value="systems">Systems</option>
<option value="taxonomy">Taxonomy</option>
<option value="technologies">Technologies</option>
<option value="telecom">Telecom</option>
<option value="tourismhosp">Tourism and Hospitality</option>
<option value="toxics">Toxics</option>
<option value="toxins">Toxins</option>
<option value="transplantology">Transplantology</option>
<option value="tropicalmed">Tropical Medicine and Infectious Disease</option>
<option value="universe">Universe</option>
<option value="urbansci">Urban Science</option>
<option value="uro">Uro</option>
<option value="vaccines">Vaccines</option>
<option value="vehicles">Vehicles</option>
<option value="vetsci">Veterinary Sciences</option>
<option value="vibration">Vibration</option>
<option value="viruses">Viruses</option>
<option value="vision">Vision</option>
<option value="water">Water</option>
<option value="women">Women</option>
<option value="world">World</option>
<option value="wevj">World Electric Vehicle Journal</option>
</select>
<input name="email" placeholder="Enter your email address..." required="required" type="email">
<button class="genericCaptcha button button--dark UA_FooterNewsletterSubscribeButton" type="submit">Subscribe</button>
</form>
</div>
</div>
</div>
<div id="footer-copyright">
<div class="row">
<div class="columns large-6 medium-6 small-12 text-left">
 1996-2020 MDPI (Basel, Switzerland) unless otherwise stated
</div>
<div class="columns large-6 medium-6 small-12 small-text-left medium-text-right large-text-right">
<a data-dropdown="drop-view-disclaimer" aria-controls="drop-view-disclaimer" aria-expanded="false" data-options="align:top; is_hover:true; hover_timeout:2000;">
Disclaimer
</a>
<div id="drop-view-disclaimer" class="f-dropdown label__btn__dropdown label__btn__dropdown--wide text-left" data-dropdown-content="" aria-hidden="true" tabindex="-1">
The statements, opinions and data contained in the journals are solely
those of the individual authors and contributors and not of the publisher and the editor(s).
MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.
</div>
<a href="https://www.mdpi.com/about/terms-and-conditions">
Terms and Conditions
</a>
<a href="https://www.mdpi.com/about/privacy">
Privacy Policy
</a>
</div>
</div>
 </div>
</div>
<div id="cookie-notification" class="js-allow-cookies" style="display: none;">
<div class="columns large-10 medium-10 small-12">
We use cookies on our website to ensure you get the best experience.<br class="show-for-medium-up">
Read more about our cookies <a href="https://www.mdpi.com/about/privacy">here</a>.
</div>
<div class="columns large-2 medium-2 small-12 small-only-text-left text-right">
<a class="button button--default" href="https://www.mdpi.com/accept_cookies">Accept</a>
</div>
</div>
<div id="cookie-notification" class="js-survey" style="display: none;">
<div class="columns large-10 medium-10 small-12">
<strong>We have just recently launched a new version of our website.</strong><br class="show-for-meidum-up">
<span style="font-weight: 400;">
Help us to further improve by taking part in this short 5 minute survey <span style="inline-block">
<span class="show-for-medium-up js-survey-link"><a href="https://www.surveymonkey.com/r/VWJZ6VW" target="_blank" rel="noopener noreferrer">here</a>.</span>
<span class="show-for-small-only js-survey-link"><a href="https://www.surveymonkey.com/r/VWMDSNQ" target="_blank" rel="noopener noreferrer">here</a>.</span>
</span>
</span>
</div>
<div class="columns large-2 medium-2 small-12 small-only-text-left text-right">
<a class="button button--default js-disable-survey-link" href="https://www.mdpi.com/disable_survey">Never show this again</a>
</div>
</div>
</div>
<div id="main-share-modal" class="reveal-modal reveal-modal-new reveal-modal-new--small" data-reveal="" aria-labelledby="modalTitle" aria-hidden="true" role="dialog">
<div class="row">
<div class="small-12 columns">
<h2 style="margin: 0;">Share Link</h2>
</div>
<div class="small-12 columns">
<div class="social-media-links UA_ShareModalLinks" style="text-align: left;">
<a href="mailto:?&amp;subject=From%20MDPI%3A%20%22The%20Resolution%20in%20X-ray%20Crystallography%20and%20Single-Particle%20Cryogenic%20Electron%20Microscopy&quot;&amp;body=https://www.mdpi.com/761056%3A%0A%0AThe%20Resolution%20in%20X-ray%20Crystallography%20and%20Single-Particle%20Cryogenic%20Electron%20Microscopy%0A%0AAbstract%3A%20X-ray%20crystallography%20and%20single-particle%20analysis%20cryogenic%20electron%20microscopy%20are%20essential%20techniques%20for%20uncovering%20the%20three-dimensional%20structures%20of%20biological%20macromolecules.%20Both%20techniques%20rely%20on%20the%20Fourier%20transform%20to%20calculate%20experimental%20maps.%20However%2C%20one%20of%20the%20crucial%20parameters%2C%20resolution%2C%20is%20rather%20broadly%20defined.%20Here%2C%20the%20methods%20to%20determine%20the%20resolution%20in%20X-ray%20crystallography%20and%20single-particle%20analysis%20are%20summarized.%20In%20X-ray%20crystallography%2C%20it%20is%20becoming%20increasingly%20more%20common%20to%20include%20reflections%20discarded%20previously%20by%20traditionally%20used%20standards%2C%20allowing%20for%20the%20inclusion%20of%20incomplete%20and%20anisotropic%20reflections%20into%20the%20refinement%20process.%20In%20general%2C%20the%20resolution%20is%20the%20smallest%20lattice%20spacing%20given%20by%20Bragg%26rsquo%3Bs%20law%20for%20a%20particular%20set%20of%20X-ray%20diffraction%20intensities%3B%20however%2C%20typically%20the%20resolution%20is%20truncated%20by%20the%20user%20during%20the%20data%20processing%20based%20on%20certain%20parameters%20and%20later%20it%20is%20used%20during%20refinement.%20However%2C%20at%20which%20resolution%20to%20perform%20such%20a%20truncation%20is%20not%20always%20clear%20and%20this%20makes%20it%20very%20confusing%20for%20the%20novices%20entering%20the%20structural%20biology%20field.%20Furthermore%2C%20it%20is%20argued%20that%20the%20effective%20resolution%20should%20be%20also%20reported%20as%20it%20is%20a[...]" title="Email">
<i class="fa fa-envelope-square" style="font-size: 30px;"></i>
</a>
<a href="https://twitter.com/intent/tweet?text=The+Resolution+in+X-ray+Crystallography+and+Single-Particle+Cryogenic+Electron+Microscopy&amp;hashtags=&amp;url=https%3A%2F%2Fwww.mdpi.com%2F761056&amp;via=MDPIOpenAccess" onclick="if (!window.__cfRLUnblockHandlers) return false; windowOpen(this.href,600,800); return false" title="Twitter" target="_blank">
<i class="fa fa-twitter-square" style="font-size: 30px;"></i>
</a>
<a href="http://www.linkedin.com/shareArticle?mini=true&amp;url=https%3A%2F%2Fwww.mdpi.com%2F761056&amp;title=The%20Resolution%20in%20X-ray%20Crystallography%20and%20Single-Particle%20Cryogenic%20Electron%20Microscopy%26source%3Dhttps%3A%2F%2Fwww.mdpi.com%26summary%3DX-ray%20crystallography%20and%20single-particle%20analysis%20cryogenic%20electron%20microscopy%20are%20essential%20techniques%20for%20uncovering%20the%20three-dimensional%20structures%20of%20biological%20macromolecules.%20Both%20techniques%20rely%20on%20the%20Fourier%20transform%20to%20calculate%20%5B...%5D" onclick="if (!window.__cfRLUnblockHandlers) return false; windowOpen(this.href,600,800); return false" title="LinkedIn" target="_blank">
<i class="fa fa-linkedin-square" style="font-size: 30px;"></i>
</a>
<a href="https://www.facebook.com/sharer.php?u=https://www.mdpi.com/761056" title="facebook" target="_blank">
<i class="fa fa-facebook-square" style="font-size: 30px;"></i>
</a>
<a href="https://www.reddit.com/submit?url=https://www.mdpi.com/761056" title="Reddit" target="_blank">
<i class="fa fa-reddit-square" style="font-size: 30px;"></i>
</a>
<a href="https://www.mendeley.com/import/?url=https://www.mdpi.com/761056" title="Mendeley" target="_blank">
<i class="fa fa-mendeley-square" style="font-size: 30px;"></i>
</a>
<a href="http://www.citeulike.org/posturl?url=https://www.mdpi.com/761056" title="CiteULike" target="_blank">
<i class="fa fa-citeulike-square" style="font-size: 30px;"></i>
</a>
</div>
</div>
<div class="small-9 columns">
<input id="js-clipboard-text" readonly="readonly" value="https://www.mdpi.com/761056" type="text">
</div>
<div class="small-3 columns text-left">
<a class="button button--color js-clipboard-copy" data-clipboard-target="#js-clipboard-text">Copy</a>
</div>
</div>
<a class="close-reveal-modal" aria-label="Close">
<i class="material-icons">clear</i>
</a>
</div>
<a href="#" class="back-to-top"><span class="show-for-medium-up">Back to Top</span><span class="show-for-small">Top</span></a>
<script src="https://www.mdpi.com/assets/js/modernizr-2.8.3.min.js?5227e0738f7f421d" type="text/javascript"></script><!--<script src="https://www.mdpi.com/assets/js/modernizr-2.8.3.min.js?5227e0738f7f421d" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/jquery-1.12.4.min.js?4f252523d4af0b47" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/foundation-5.5.3.min.js?6b2ec41c18b29054" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/foundation-5.5.3.equalizer.min.js?0f6c549b75ec554c" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/jquery.multiselect.js?0edd3998731d1091" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/jquery.cycle2.min.js?63413052928f97ee" type="text/javascript"></script>-->
<!--<script type="text/javascript">
            $('select.foundation-select').multiselect({
                search: true,
                minHeight: 130,
                maxHeight: 130,
            });

            $(document).foundation({
                orbit: {
                    timer_speed: 4000,
                },
                reveal: {
                    animation: 'fadeAndPop',
                    animation_speed: 100,
                }
            });

            // old browser fix - this way the console log rows won't throw (silent) errors in browsers not supporting console log
            if (!window.console) window.console = {};
            if (!window.console.log) window.console.log = function () { };

            var currentJournalNameSystem = "crystals";

            $(document).ready(function()
            {
                $(".chosen-select").on('chosen:ready', function(event, data) {
                    var select = $(data.chosen.form_field);
                    if (select.attr('id') === 'journal-browser-volume') {
                        $(data.chosen.dropdown).addClass('UI_JournalBrowser_Volume_Options');
                    }
                    if (select.attr('id') === 'journal-browser-issue') {
                        $(data.chosen.dropdown).addClass('UI_JournalBrowser_Issue_Options');
                    }
                }).chosen({ disable_search_threshold: 7, width: "100%" });

                $(".toEncode").each(function(e) {
                    var oldHref = $(this).attr("href");
                    var newHref = oldHref.replace('.botdefense.please.enable.javaScript.','@');
                    $(this).attr("href", newHref);

                    if (!$(this).hasClass("emailCaptcha"))
                    {
                        $(this).html(newHref.replace('mailto:', ''));
                    }
                    
                                    });

                $(document).on('opened.fndtn.reveal', '[data-reveal]', function()
                {
                    $(document).foundation('equalizer', 'reflow');
                });

                // fix the images that have tag height / width defined
                // otherwise the default foundation styles overwrite the tag definitions
                $("img").each(function()
                {
                   if ($(this).attr('width') != undefined || $(this).attr('height') != undefined)
                   {
                       $(this).addClass("img-fixed");
                   }
                });

                $(".hide-show-desktop-option").click(function(e) 
                {
                    e.preventDefault();
                    var parentDiv = $(this).closest("div");

                    $.ajax({
                        url: $(this).attr('href'),
                        success: function(msg)
                        {
                            parentDiv.removeClass().hide();
                        }
                    });
                });

                /*
                * handle whole row as a link if the row contains only one visible link
                */
                $("table.new tr").hover(function()
                {
                   if ($(this).find("td:visible a").length == 1)
                   {
                       $(this).addClass("single-link");
                   }
                },
                function()
                {
                   $(this).removeClass("single-link");
                });

                $("table.new:not(.table-of-tables)").on("click", "tr.single-link", function(e)
                {
                    var target = $(e.target);
                    if (!e.ctrlKey && !target.is("a")) {
                        $(this).find("td:visible a")[0].click();
                    }
                });

                $(document).on("click", ".custom-accordion-for-small-screen-link", function(e) {
                    if ($(this).closest("#basic_search").length > 0) {
                        if ($(".search-container__advanced").first().is(":visible")) {
                            openAdvanced()
                        }
                    }

                    if (Foundation.utils.is_small_only()) {
                        if ($(this).hasClass("active")) {
                            $(this).removeClass("active");
                            $(this).next(".custom-accordion-for-small-screen-content").addClass("show-for-medium-up");
                        }
                        else {
                            $(this).addClass("active");
                            $(this).next(".custom-accordion-for-small-screen-content").removeClass("show-for-medium-up");
                            $(document).foundation('orbit', 'reflow');
                        }
                    }

                    if (undefined !== $(this).data("callback")) {
                        var customCallback = $(this).data("callback");
                        func = window[customCallback];
                        func();
                    }
                });

                $(document).on("click", ".js-open-small-search", function(e) {
                    e.preventDefault();  
                    $(this).toggleClass("active").closest(".tab-bar").toggleClass("active");
                    $(".search-container").toggleClass("hide-for-small-down");
                });

                $(document).on("click", ".js-survey-link", function(e) {
                    localStorage.setItem("mdpi_disable_survey", true);
                    window.location.href = "/disable_survey";
                });

                $(document).on("click", ".js-disable-survey-link", function(e) {
                    localStorage.setItem("mdpi_disable_survey", true);
                });

                $(document).on("click", ".js-open-menu", function(e) {
                    $(".search-container").addClass("hide-for-small-down");

                });

                $(window).on('resize', function() {
                    recalculate_main_browser_position();
                    recalculate_responsive_moving_containers();
                });

                checkCookiesAllowed();
                updateSearchLabelVisibilities();
                recalculate_main_browser_position();
                recalculate_responsive_moving_containers();
                
                
                if (window.document.documentMode == 11) {
                    $("<link/>", { rel: "stylesheet", type: "text/css", href: "https://fonts.googleapis.com/icon?family=Material+Icons"}).appendTo("head");
                }
            });

            function recalculate_main_browser_position() {
                if (Foundation.utils.is_small_only()) {
                    if ($("#js-main-top-container").parent("#js-large-main-top-container").length > 0) {
                        $("#js-main-top-container").appendTo($("#js-small-main-top-container"));
                    }
                }
                else {
                    if ($("#js-main-top-container").parent("#js-small-main-top-container").length > 0) {
                        $("#js-main-top-container").appendTo($("#js-large-main-top-container"));
                    }
                }
            }

            function recalculate_responsive_moving_containers() {
                $(".responsive-moving-container.large").each(function() {
                    var previousParent = $(".responsive-moving-container.active[data-id='"+$(this).data("id")+"']");
                    var movingContent  = previousParent.html();

                    if (Foundation.utils.is_small_only()) {
                        var currentParent = $(".responsive-moving-container.small[data-id='"+$(this).data("id")+"']");
                    }
                    else if (Foundation.utils.is_medium_only()) {
                        var currentParent = $(".responsive-moving-container.medium[data-id='"+$(this).data("id")+"']");
                    }
                    else {
                        var currentParent = $(".responsive-moving-container.large[data-id='"+$(this).data("id")+"']");
                    }

                    if (previousParent.attr("class") !== currentParent.attr("class")) {
                        currentParent.html(movingContent);
                        previousParent.html();
                        currentParent.addClass("active");
                        previousParent.removeClass("active");
                    }
                });
            }

            // cookies allowed is checked from a) local storage and b) from server separately so that the footer bar doesn't
            // get included in the custom page caches
            function checkCookiesAllowed() {
                var cookiesEnabled = localStorage.getItem("mdpi_cookies_enabled"); 

                if (null === cookiesEnabled) {
                    $.ajax({
                        url: "/ajax_cookie_value/mdpi_cookies_accepted",
                        success: function(data)
                        {
                            if (data.value) {
                                localStorage.setItem("mdpi_cookies_enabled", true);
                                checkDisplaySurvey();
                            }
                            else {
                                $(".js-allow-cookies").show();
                            }
                        }
                    });
                }
                else {
                    checkDisplaySurvey();
                }
            }

            // cookies allowed is checked from a) local storage and b) from server separately so that the footer bar doesn't
            // get included in the custom page caches
            //
            // NOTE! survey has been closed. Not used anymore (this code can be moved to somewhere for storage in case we need similar logic with other iterations)
            // 
            function checkDisplaySurvey() {
                
                var surveyDisabled = localStorage.getItem("mdpi_disable_survey"); 
                var viewCount      = localStorage.getItem("mdpi_new_layout_views"); 
                var viewLimit      = localStorage.getItem("mdpi_new_layout_limit");

                if (null !== surveyDisabled) {
                    return;
                }

                $('head').append('<script>(function(t,e,s,o){var n,c,l;t.SMCX=t.SMCX||[],e.getElementById(o)||(n=e.getElementsByTagName(s),c=n[n.length-1],l=e.createElement(s),l.type="text\/javascript",l.async=!0,l.id=o,l.src="https:\/\/widget.surveymonkey.com\/collect\/website\/js\/tRaiETqnLgj758hTBazgdwDJB9xmyTMSURmUFHu_2B7D1Hr8R_2B7128L_2F8Pj41SL9if.js",c.parentNode.insertBefore(l,c))})(window,document,"script","smcx-sdk");<\/script>');

                return;
                // todo: the below part can be removed later on...

                if (null === viewLimit) {
                    localStorage.setItem("mdpi_new_layout_limit", Math.floor(Math.random() * 50));
                    localStorage.setItem("mdpi_new_layout_views", 0);
                }
                else {
                    viewCount++;
                    localStorage.setItem("mdpi_new_layout_views", viewCount);

                    if (viewCount >= viewLimit) {
                        $.ajax({
                            url: "/ajax_cookie_value/mdpi_disable_survey",
                            success: function(data)
                            {
                                if (data.value) {
                                    localStorage.setItem("mdpi_disable_survey", true);
                                }
                                else {
                                    $(".js-survey").show();
                                }
                            }
                        });
                    }
                }
            }
            
        </script>-->
<!--<script src="https://www.mdpi.com/assets/js/lib.js?82799462daed82e0" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/mdpi.js?c5e55b6509d6aab3" type="text/javascript"></script>-->
<!--<script type="text/javascript">var banners_url = 'https://serve.mdpi.com';</script>-->
<!--<script type="text/javascript" src="https://www.mdpi.com/assets/js/ifvisible.min.js?c621d19ecb761212"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/xmltohtml/affix.js?ac4ea55275297c15" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/clipboard.min.js?3f3688138a1b9fc4" type="text/javascript"></script>-->
<!--<script type="text/javascript">
        $(document).ready(function()
        {
            var helpFunctions = $(".middle-column__help__fixed");
            var middleColumn  = $("#middle-column");

            helpFunctions.affix({
                offset: {
                    top: function() {
                        return middleColumn.offset().top - 8 - (Foundation.utils.is_medium_only() ? 30 : 0);
                    },
                    bottom: function() {
                        return $("#footer").innerHeight() + 56 + (Foundation.utils.is_medium_only() ? 0 : 0);
                    }
                }
            });

            new ClipboardJS('.js-clipboard-copy');
        });
    </script>-->
<!--<script src="https://www.mdpi.com/assets/js/jquery-ui-1.12.0.min.js?4c2c52cc11e5a28d" type="text/javascript"></script>-->
<!--<script type="text/javascript">
        $(document).ready(function()
        {
            
            $(".link-article-menu").click(function(e) {
                e.preventDefault();

                $(this).find('span').toggle();
                $(this).next("div").toggleClass("active");
            });

                    });
    </script>-->
<!--<script src="https://www.mdpi.com/assets/js/xmltohtml/affix.js?ac4ea55275297c15" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/xmltohtml/storage.js?e9b262d3a3476d25" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/xmltohtml/jquery-scrollspy.js?09cbaec0dbb35a67" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/xmltohtml/magnific-popup.js?4a09c18460afb26c" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/xmltohtml/underscore.js?f893e294cde60c24" type="text/javascript"></script>-->
<!--<script type="text/javascript">
    $('document').ready(function(){
    //   if(1 == 0){
    //     var doi = $('.html-art-header').find('a');
    //     var text = doi.text();
    //     doi.remove();
    //     $('.html-art-header p').append(text);
    //     $('.html-art-header').html($('.html-art-header')[0].innerHTML.replace(/doi:\s+/,'doi:&nbsp;'))
    //   }

      $("#left-column").addClass("show-for-large-up");
      $("#middle-column").removeClass("medium-9").removeClass("left-bordered").addClass("medium-12");
    });
	$(document).on('DOMNodeInserted', function(e) {

        var element = $(e.target);

        if (element.hasClass('menu') && element.hasClass('html-nav') )
        {
            element.addClass("side-menu-ul");
                                            }
    });
  </script>-->
<!--<script src="https://www.mdpi.com/assets/js/jquery-ui-1.10.4.custom.min.js?9fb4d59ff745b497" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/xmltohtml/articles.js?a171333ed3a7c6c5" type="text/javascript"></script>-->
<!--<script type="text/javascript">
        repositionOpenSideBar = function() {
            $('#left-column').addClass("show-for-large-up show-for-medium-up").show();
            $('#middle-column').removeClass('large-12').removeClass('medium-12');
            $('#middle-column').addClass('large-9');
        }

        repositionCloseSideBar = function() {
            $('#left-column').removeClass("show-for-large-up show-for-medium-up").hide();
            $('#middle-column').removeClass('large-9');
            $('#middle-column').addClass('large-12').addClass('medium-12');
        }
    </script>-->
<!--<script type="text/javascript">
        $(document).ready(function() {

            $(document).on('keyup', function (e) {
                if (e.keyCode == 27) {

                    var hElem = $(this).find(".annotator-adder");

                    if (hElem.length){
                        hElem.css({'visibility':'hidden'});
                    } else {
                        document.querySelector("hypothesis-adder").shadowRoot.querySelector(".annotator-adder").style.visibility = "hidden";
                    }

                }
            });
        });
    </script>-->
<!--<script type="text/javascript">
            window.hypothesisConfig = function () {
                return {
                    sidebarAppUrl: 'https://commenting.mdpi.com/app.html',
                    showHighlights:   'whenSidebarOpen'  ,
                    openSidebar:  false  ,
                    assetRoot: 'https://commentingres.mdpi.com/hypothesis',
                    services: [{
                        apiUrl: 'https://commenting.mdpi.com/api/',
                        authority: 'mdpi',
                        grantToken: '',
                        doi: '10.3390/cryst10070580'
                    }],
                };
            };
            MathJax . Hub . Startup . signal . Interest ( function ( message ) {
                if ( message == 'End' ){
                var hypoLink = document . getElementById ( "hypothesis_frame" )
                hypoLink . setAttribute ( "src" , "https://commenting.mdpi.com/embed.js" )
                }
});
        </script>-->
<!--<script async="" id="hypothesis_frame" type="text/javascript"></script>-->
<!--[if lt IE 9]>
                        <script src="https://www.mdpi.com/assets/js/ie8/ie8.js?6eef8fcbc831f5bd"></script>
            <script src="https://www.mdpi.com/assets/js/ie8/jquery.xdomainrequest.min.js?a945caca315782b0"></script>
        <![endif]-->

<!--<script type="text/javascript">
            !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);
            },s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
            a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
            // Insert Twitter Pixel ID and Standard Event data below
            twq('init','o2pip');
            twq('track','PageView');
        </script>-->



</body></html>