<!DOCTYPE html>
<html class=" js svg" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <title>A multi-crystal method for extracting obscured 
crystallographic states from conventionally uninterpretable electron 
density | Nature Communications</title>
    
        
    
    <script type="text/javascript" async="" src="https://www.google-analytics.com/analytics.js"></script><script src="https://hm.baidu.com/hm.js?485f8e597c8915da9aca0c37dca3f39f"></script><script type="text/javascript" async="" src="//recommended.springernature.com/latest/generated/entry-point.js"></script><script async="" src="93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea0.js"></script><script src="otSDKStub.js" data-domain-script="83f2c78a-6cbc-4d1a-9088-3f8e8c4c7460"></script><script async="" src="gtm.js"></script><script id="save-data-connection-testing">
        if ("connection" in navigator) {
            if (navigator.connection.saveData === true) {
                document.documentElement.classList.add('save-data');
            }
        }
    </script>

    
        <script>
            function createLink(src) {
                if ("connection" in navigator) {
                    if (navigator.connection.saveData != true) {
                        var preloadLink = document.createElement("link");
                        preloadLink.rel = "preload";
                        preloadLink.href = src;
                        preloadLink.as = "font";
                        preloadLink.type = "font/woff2";
                        preloadLink.crossOrigin = "";
                        document.head.insertBefore(preloadLink, document.head.firstChild);
                    }
                }
            }

            createLink("/static/fonts/HardingText-Regular-Web.b167e675dc.woff2");
        </script>
    


<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=2.5,user-scalable=yes">


<script>
    (function(e){var t=e.documentElement,n=e.implementation;t.className+=' js';if(n&&n.hasFeature('http://www.w3.org/TR/SVG11/feature#Image','1.1')){t.className+=' svg'}})(document)
</script>

<script data-test="dataLayer">
    dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"proteins;screening;x-ray-crystallography","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Communications","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article"}},"article":{"doi":"10.1038/ncomms15123"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Nicholas M. Pearce","Tobias Krojer","Anthony R. Bradley","Patrick Collins","Radosław P. Nowak","Romain Talon","Brian D. Marsden","Sebastian Kelm","Jiye Shi","Charlotte M. Deane","Frank von Delft"],"publishedAt":1492992000,"publishedAtString":"2017-04-24","title":"A multi-crystal method for extracting obscured crystallographic states from conventionally uninterpretable electron density","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"ncomms","title":"nature communications","volume":"8","issue":"1"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[]},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US"}];
</script>




    
        
    
     
        

    <style>html{font-family:sans-serif;line-height:1.15;height:100%;overflow-y:scroll;font-size:62.5%;box-sizing:border-box}article,aside,header,main,nav,section{display:block}h1{font-size:30px}a,sup{vertical-align:baseline}a{background-color:transparent;text-decoration:none;color:#069}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;max-width:100%;height:auto;vertical-align:middle}svg:not(:root){overflow:hidden}button,input{font-family:sans-serif;font-size:100%;line-height:1.15;overflow:visible}button{text-transform:none}[type=submit],button{-webkit-appearance:button}button{cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;border-top-left-radius:0;border-top-right-radius:0;border-bottom-right-radius:0;border-bottom-left-radius:0}h2{font-size:24px}h3{font-size:17px}nav ul{list-style:none none}body{min-height:100%;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;line-height:1.76;color:#222;background-color:#eee;background-position:initial initial;background-repeat:initial initial}body,button,div,form,input{margin:0;padding:0}h1,h2,h3,p{padding:0}body{font-size:1.8em}p{margin:0 0 28px}ol,ul{margin-top:0;margin-bottom:28px}h1{-webkit-font-smoothing:antialiased;line-height:1.3;font-size:3.2rem;margin:0 0 24px;font-weight:700}h1,h2,h3{font-family:Harding,Palatino,serif}h2{font-size:2.4rem}h2,h3{-webkit-font-smoothing:antialiased;line-height:1.3;margin:0 0 8px;font-weight:700}h3{font-size:2rem}.cleared:after,.cleared:before{content:" ";display:table}.cleared:after{clear:both}.hide,.js .js-hide{display:none;visibility:hidden}.text-center{text-align:center}.serif{font-family:Harding,Palatino,serif}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.text14{font-size:1.4rem}.article-page{background-color:#fff;background-position:initial initial;background-repeat:initial initial}.z-index-50{z-index:50}.background-white{background-color:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{list-style:none;font-size:1.6rem;line-height:1.3;display:-webkit-flex;-webkit-flex-wrap:wrap;color:#6f6f6f;padding:0;margin:0 0 8px}.c-article-identifiers__item{border-right:1px solid #6f6f6f;margin-right:8px;padding-right:8px;list-style:none}.c-article-identifiers__item a{color:#069;text-decoration:none}.c-article-identifiers__item:last-child{margin-right:0;padding-right:0;border-right-width:0}.c-article-identifiers__open{color:#b74616}@media only screen and (min-width:768px){.c-author-popup .c-article-identifiers{-webkit-box-pack:end;-webkit-justify-content:flex-end}}.c-article-title{font-size:2.4rem;line-height:1.25;margin-bottom:16px}@media only screen and (min-width:768px){.c-article-title{font-size:3rem;line-height:1.2}}.c-author-list{font-size:1.6rem;list-style:none;margin-bottom:0;padding:0;width:100%}.c-author-list__item{margin-left:0}.c-author-list__item,.c-author-list li{display:inline;padding-right:0}.c-author-list__item svg,.c-reading-companion__figure-full-link svg{margin-left:4px}.c-article-info-details{font-size:1.6rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:-webkit-flex;-webkit-flex-wrap:wrap;line-height:1.3;font-size:1.6rem}.c-article-metrics-bar__item{-webkit-box-align:baseline;-webkit-align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right-width:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-weight:400;font-style:normal;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-author-popup__subheading{font-weight:700;float:left;padding-right:8px;margin-bottom:8px;margin-top:4px}.c-author-popup__author-list{list-style:none;font-size:1.6rem;padding:0;margin-top:0;clear:both;margin-bottom:16px}.c-author-popup__author-list>li{margin-bottom:8px}.c-author-popup__link{font-weight:700;vertical-align:baseline;color:#069;text-decoration:none}.c-author-popup .c-article-button{color:#fff;background-color:#069;border:1px solid #069;border-top-left-radius:2px;border-top-right-radius:2px;border-bottom-right-radius:2px;border-bottom-left-radius:2px;text-decoration:none;font-size:1.6rem;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;display:block;width:100%;padding-top:8px;padding-bottom:8px;text-align:center;background-position:initial initial;background-repeat:initial initial}.c-article-section{clear:both}.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:2rem;line-height:1.3;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-section__title{font-size:2.4rem;line-height:1.24}}.c-article-section__content{margin-bottom:40px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-top:0;margin-bottom:24px}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-weight:700;margin:0;padding:0;font-size:1.7rem}.c-article-authors-search__item{font-size:1.6rem}.c-article-authors-search__text{margin:0}@media only screen and (min-width:768px){.c-author-popup .c-article-authors-search__list{display:-webkit-flex;-webkit-flex-wrap:wrap;-webkit-box-pack:justify;-webkit-justify-content:space-between;-webkit-box-align:center;-webkit-align-items:center}.c-author-popup .c-article-authors-search__list-item--left{-webkit-flex-basis:40%}}.c-author-popup .c-article-authors-search__list-item--right{margin-top:16px}@media only screen and (min-width:768px){.c-author-popup .c-article-authors-search__list-item--right{text-align:right;-webkit-box-flex:1;-webkit-flex:1 1 0px;margin-top:0}}.c-article-share-box__no-sharelink-info{font-size:1.3rem;font-weight:700;padding-top:4px;margin-bottom:24px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;display:inline-block;margin-bottom:8px;font-size:1.4rem;font-weight:700;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:1.4rem;margin-bottom:8px;margin-left:10px}.c-article-body{clear:both}.c-article-body a,.c-article-body p{word-wrap:break-word}.c-article-body a{word-break:break-word}.c-article-orcid__text{font-size:1.6rem;vertical-align:middle}.c-article-orcid:before{display:inline-block;width:24px;height:24px;content:url("data:image/svg+xml;charset=utf-8,%3Csvg height='24' viewBox='0 0 24 24' width='24' xmlns='http://www.w3.org/2000/svg'%3E%3Cg fill='none' fill-rule='evenodd'%3E%3Cpath d='m0 0h24v24h-24z' fill='%23fff' opacity='0'/%3E%3Cpath d='m12 20c-4.418278 0-8-3.581722-8-8s3.581722-8 8-8 8 3.581722 8 8-3.581722 8-8 8zm-3.01799709-13.09169471c-.16628978-.16109322-.36635416-.24163862-.60019916-.24163862-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437zm-1.30173065 2.36492841v7.1712107h1.41865256v-7.1712107zm5.97732434.02508949h-2.9924703v7.14612121h2.9106249c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.8594238 0-.6854582-.1602789-1.3027023-.4808414-1.8517509-.3205625-.5490485-.7707074-.9753221-1.3504481-1.2788334-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1432294 6.06678581h-1.4681008v-4.9874504h1.4220628c.7877654 0 1.4143875.2233674 1.8798852.6701087.4654977.4467414.698243 1.05546.698243 1.8261742 0 .7434322-.2310402 1.3444779-.6931277 1.8031551-.4620874.4586773-1.0750688.6880124-1.8389625.6880124z' fill='%23999'/%3E%3C/g%3E%3C/svg%3E");margin-left:-3px;padding-right:4px;vertical-align:middle}.c-pdf-download{display:-webkit-flex;margin-bottom:24px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}@media only screen and (min-width:1024px){.c-pdf-button__container{display:none}}.c-context-bar{position:relative;width:100%;-webkit-box-shadow:rgba(51,51,51,.2) 0 0 10px 0;box-shadow:0 0 10px 0 rgba(51,51,51,.2)}.c-context-bar__title{display:none}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-pdf-download__link{display:-webkit-flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;-webkit-box-pack:justify;-webkit-justify-content:space-between;color:#fff;background-color:#069;border:1px solid #069;border-top-left-radius:2px;border-top-right-radius:2px;border-bottom-right-radius:2px;border-bottom-left-radius:2px;text-decoration:none;font-size:1.6rem;line-height:1.3;-webkit-box-flex:1;-webkit-flex:1 1 0px;padding:13px 24px;background-position:initial initial;background-repeat:initial initial}.c-reading-companion{clear:both}.c-reading-companion__sticky{max-width:582px}.c-reading-companion__scroll-pane{overflow-x:hidden;overflow-y:auto;margin:0 0 16px}.c-reading-companion__tabs{font-size:1.6rem;list-style:none;display:-webkit-flex;-webkit-box-orient:horizontal;-webkit-box-direction:normal;-webkit-flex-flow:row nowrap;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{-webkit-box-flex:1;-webkit-flex-grow:1}.c-reading-companion__tab{color:#069;border:1px solid #d5d5d5;border-left-width:0;background-color:#eee;padding:8px 8px 8px 15px;text-align:left;font-size:1.6rem;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{color:#222;background-color:#fff;border-bottom:1px solid #fff;font-weight:700}.c-reading-companion__figures-list,.c-reading-companion__references-list,.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1.6rem;padding:0}.c-reading-companion__section-item a{display:block;padding:8px 0 8px 16px;line-height:1em;overflow:hidden;white-space:nowrap;text-overflow:ellipsis;text-decoration:none}.c-reading-companion__figure-item{padding:16px 8px 16px 0;border-top:1px solid #d5d5d5;font-size:1.6rem}.c-reading-companion__figure-item:first-child{border-top-style:none;padding-top:8px}.c-reading-companion__reference-item{padding:8px 8px 8px 0;border-top:1px solid #d5d5d5;font-size:1.6rem}.c-reading-companion__reference-item:first-child{border-top-style:none}.c-reading-companion__reference-citation{display:inline;font-family:Harding,Palatino,serif}.c-reading-companion__reference-links{list-style:none;text-align:right;margin:8px 0 0;padding:0;font-weight:700;font-size:1.3rem}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{font-weight:700;font-family:Harding,Palatino,serif;display:block;margin:0 0 8px}.c-reading-companion__figure-links{margin:8px 0 0;display:-webkit-flex;-webkit-box-pack:justify;-webkit-justify-content:space-between}.c-reading-companion__figure-links>a{display:inline-block}.c-reading-companion__panel{display:none;border-top:1px solid #d5d5d5;margin-top:-9px;padding-top:9px}.c-reading-companion__panel--active{display:block}.c-ad{display:none;padding:8px;text-align:center}@media only screen and (min-width:768px){.js .c-ad{display:block}}.c-ad--728x90{background-color:#ccc}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}.c-ad__label{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-weight:400;margin-bottom:4px;color:#333;font-size:1.4rem;line-height:1.5}.js .c-popup{position:absolute;font-family:Harding,Palatino,serif;z-index:100;padding:16px;border:1px solid rgba(151,191,216,.298039);-webkit-box-shadow:hsla(0,0%,63.5%,.0980392) 0 0 5px 0;box-shadow:0 0 5px 0 hsla(0,0%,63.5%,.0980392);width:auto;border-top-left-radius:2px;border-top-right-radius:2px;border-bottom-right-radius:2px;border-bottom-left-radius:2px;background-color:#fff}.js .c-popup__close{position:absolute;display:block;top:16px;right:16px;height:16px;background-image:url("data:image/svg+xml,%0A%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z' fill='%23666' fill-rule='evenodd'/%3E%3C/svg%3E");border:0;padding-right:16px;background-position:initial initial;background-repeat:no-repeat}.js .c-popup__close-text{border:0;clip:rect(0 0 0 0);height:1px;margin:-100%;overflow:hidden;padding:0;width:1px;position:absolute!important}.js .c-popup__arrow{content:"";position:absolute;width:20px;height:20px;background-color:#fff;border-top:1px solid rgba(151,191,216,.298039);border-left:1px solid rgba(151,191,216,.298039)}.c-button{-webkit-box-align:center;-webkit-align-items:center;cursor:pointer;display:-webkit-inline-flex;margin:0;position:relative;text-decoration:none;width:auto;font-family:sans-serif;font-size:1.6rem;line-height:1.3;-webkit-box-pack:center;-webkit-justify-content:center;padding:8px;transition:.25s ease,color .25s ease,border-color .25s ease;-webkit-transition:.25s ease,color .25s ease,border-color .25s ease;border-top-left-radius:2px;border-top-right-radius:2px;border-bottom-right-radius:2px;border-bottom-left-radius:2px;background-color:transparent;background-image:none;color:#00f;border:1px solid #00f}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1.6rem;margin-bottom:16px}.c-header>:last-child{margin-bottom:0}.c-header__row-border{border-bottom:1px solid #eee;margin-bottom:8px;position:relative}.c-header__layout,.c-header__logo{display:-webkit-flex}.c-header__layout{-webkit-box-align:center;-webkit-align-items:center;-webkit-box-pack:justify;-webkit-justify-content:space-between}.c-header__logo{margin-right:16px;max-height:32px;-webkit-transform:translateZ(0)}@media only screen and (min-width:768px){.c-header__logo{margin-right:0}}.c-header__site-navigation{display:-webkit-flex;-webkit-box-align:center;-webkit-align-items:center;-webkit-box-flex:0;-webkit-flex:0 1 auto;margin-bottom:0;padding:16px;text-align:right}.c-header__site-navigation:last-child{padding:16px 0 16px 16px}.c-header__site-navigation--show-at-md{display:none}@media only screen and (min-width:768px){.c-header__site-navigation--show-at-md{display:-webkit-flex}}.c-header__site-navigation--border{border-left:1px solid #eee}.c-header__site-navigation--border .c-header__link:not(:first-child){margin-left:16px}.c-header__container{margin:0 auto;max-width:1280px;padding:0 16px}.c-header__item,.c-header__menu{display:-webkit-flex}.c-header__menu{list-style:none;padding:0;-webkit-box-align:center;-webkit-align-items:center;-webkit-flex-wrap:wrap;-webkit-box-flex:1;-webkit-flex:1 1 100%;line-height:1.2;margin:0 0 8px -8px}.c-header__item{font-weight:700;margin-right:16px;position:relative}.c-header__item:last-child{margin-right:0}.c-header__link{color:inherit;display:-webkit-flex;-webkit-box-align:center;-webkit-align-items:center;white-space:nowrap}.c-header-expander a{color:inherit}.c-header__link>svg{margin-left:4px;transition:.2s;-webkit-transition:.2s}.c-header__show-text{display:none}@media only screen and (min-width:540px){.c-header__show-text{display:inline}}.c-header__menu .c-header__link{padding:8px}.c-header-expander{background-color:#000;color:#eee;padding:24px 0 0;font-size:1.4rem}.c-header-expander button{border:1px solid #fff;color:#fff;background-color:transparent}.c-header-expander__container{margin:0 auto;max-width:1280px;padding:0 16px}.c-header-expander__keyline{border-bottom:1px solid #555;padding-bottom:16px;margin-bottom:0}.c-header-expander__heading{display:inline-block;line-height:1.3;font-size:2rem;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-weight:400}.c-header-expander__list{padding:0;margin:0}.c-header-expander__item{display:inline-block;margin-right:16px;margin-bottom:16px}.c-header-expander__item--keyline{border-left:1px solid #fff;padding-left:16px}.c-header-expander__link{color:inherit;display:-webkit-flex;-webkit-box-align:center;-webkit-align-items:center;white-space:nowrap}.c-header-expander__link>svg{margin-left:8px}@media only screen and (min-width:768px){.c-header-expander__form{max-width:600px}}.c-header-expander.has-tethered{position:absolute;top:100%;-webkit-transform:translateY(8px);z-index:1;left:0;text-align:left;width:auto;min-width:-webkit-max-content;border-top-left-radius:0;border-top-right-radius:0;border-bottom-right-radius:2px;border-bottom-left-radius:2px}@media only screen and (min-width:540px){.c-header-expander.has-tethered{width:120%}}.c-header-expander.has-tethered .c-header-expander__item{display:block;margin-right:0}.c-header-expander.has-tethered .c-header-expander__item--keyline{border-top:1px solid #fff;border-left-style:none;padding-top:16px;padding-left:0}.c-header-expander.has-tethered .c-header-expander__keyline{margin-bottom:16px;padding-top:0}.c-header-expander.has-tethered .c-header-expander__keyline:last-child,.c-header-expander.has-tethered .c-header-expander__keyline:only-child{border:0;padding-bottom:0}.c-header-expander--tray.has-tethered{padding:32px 0 24px;-webkit-transform:none;width:100%}.c-breadcrumbs{list-style:none;margin:0;padding:0;font-size:16px;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;color:#000}.c-breadcrumbs>li,.u-list-inline>li{display:inline}.c-breadcrumbs__link{color:#666}.c-breadcrumbs__chevron{margin:0 4px}.c-form-field{margin-bottom:16px}.c-form-field__input{padding:.6em 1em;width:100%}.c-skip-link{border:0;clip:rect(0 0 0 0);height:1px;margin:-100%;overflow:hidden;padding:0;width:1px;background-color:#069;color:#fff;text-align:center;font-size:1.4rem;z-index:9999;position:absolute!important}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-visually-hidden{border:0;clip:rect(0 0 0 0);height:1px;margin:-100%;overflow:hidden;padding:0;width:1px;position:absolute!important}@media print{.u-hide-print{display:none}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-icon{fill:currentColor;-webkit-transform:translate(0);display:inline-block;vertical-align:text-top}.u-float-left{float:left}.u-list-inline,.u-list-reset{list-style:none;margin:0;padding:0}.u-h3{font-size:2rem}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-display-flex{display:-webkit-flex;width:100%}.u-flex-shrink,.u-flex-static{-webkit-box-flex:0;-webkit-flex:0 1 auto}.u-flex-static{-webkit-flex:0 0 auto}.u-ma-0{margin:0}.u-mt-32{margin-top:32px}.u-mr-24{margin-right:24px}.u-mb-16{margin-bottom:16px}.u-mb-32{margin-bottom:32px}.u-ml-8{margin-left:8px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.grade-c-hide{display:block}.grade-c-show{display:none;visibility:hidden}</style>



        
        
        
            <link id="js-mustard" rel="stylesheet" href="article-nature-branded.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)" onload="this.media='only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)';this.onload=null">
            <noscript>
                <link id="js-mustard" rel="stylesheet" type="text/css" href="/static/css/article-nature-branded.bd02d71d0b.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
            </noscript>
        
        <link rel="stylesheet" type="text/css" href="article-print.css" media="print">
    



    <style>
        .c-header--brand-border {
            border-bottom: 5px solid #e63323;
        }
    </style>


<link rel="apple-touch-icon" sizes="180x180" href="https://www.nature.com/static/images/favicons/nature/apple-touch-icon.f39cb19454.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://www.nature.com/static/images/favicons/nature/favicon-32x32.3fe59ece92.png">
<link rel="icon" type="image/png" sizes="16x16" href="https://www.nature.com/static/images/favicons/nature/favicon-16x16.951651ab72.png">
<link rel="manifest" href="https://www.nature.com/static/manifest.1a481c42b1.json">
<link rel="mask-icon" href="https://www.nature.com/static/images/favicons/nature/safari-pinned-tab.69bff48fe6.svg" color="#000000">
<link rel="shortcut icon" href="https://www.nature.com/static/images/favicons/nature/favicon.62367f778b.ico">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content="/static/browserconfig.e35b3b052c.xml">
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    window.config = window.config || {};
    window.config.mustardcut = false;

    
    if (window.matchMedia && window.matchMedia('only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)').matches) {
        window.config.mustardcut = true;
    }
</script>
 
<!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js-->
<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>

<script>
    (function() {
        function deleteCookie (name, domain) {
            document.cookie = encodeURIComponent(name) +
                    '=' +
                    ';path=/' +
                    ';domain=' + domain +
                    ';expires=Thu, 01 Jan 1970 00:00:00 GMT';
        }

        var consentCookieParts = ('; ' + document.cookie).split('; OptanonConsent=');

        if (consentCookieParts.length > 1) {
            consentCookieParts.shift(); // remove redundant first part from the split array

            // onetrust can set the same cookie multiple times with different domain specificities
            for (let i=0; i<consentCookieParts.length; i++) {
                var otCookieGroups = consentCookieParts[i].split('&groups=').pop().split('&').shift();

                if (otCookieGroups.indexOf('C0001') === -1) {
                    // to be deleted when new onetrust implemented on www.springer.com pages
                    deleteCookie('OptanonConsent', '.springer.com');

                    deleteCookie('OptanonConsent', 'nature.com');
                    deleteCookie('OptanonAlertBoxClosed', 'nature.com');
                }
            }
        }
    })();
</script>
<script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                e.src = 'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js';
                e.setAttribute('data-domain-script', '83f2c78a-6cbc-4d1a-9088-3f8e8c4c7460');
            } else {
                e.src = '/static/js/cookie-consent-bundle.9d49adbc02.js';
                e.setAttribute('data-consent', h);
            }
            s.parentNode.insertBefore(e, s);
        }

        !!w.google_tag_manager ? cc() : window.addEventListener('gtm_loaded', function() {cc()});
    })(window,document,'script');
</script>
<script>
    function OptanonWrapper() {
        window.dataLayer.push({event:'OneTrustGroupsUpdated'});
        document.activeElement.blur();
    }
</script>

<script id="js-position0">
    window.idpVerifyPrefix = 'https://verify.nature.com';
    window.ra21Host = 'https://wayf.springernature.com';
</script><script id="js-position1" data-test="polyfills-js" defer="defer" src="polyfills-bundle.js"></script><script id="js-position2" data-test="nature-es6-js" defer="defer" src="nature-es6-bundle.js"></script><script id="js-position3" data-test="shared-es6-js" defer="defer" src="shared-es6-bundle.js"></script><script id="js-position4" data-test="header-150-js" defer="defer" src="article-header-150-bundle.js"></script>


    <script>
        (function() {
            window.onerror = function(message, url, line, column, error) {
                if (window.Raven) {
                    window.Raven.captureException(error);
                } else {
                    var s = document.createElement('script');
                    s.src = 'https://cdn.ravenjs.com/3.27.0/raven.min.js';
                    s.crossorigin = 'anonymous';
                    s.defer = 'defer';
                    s.onload = function() {
                        window.Raven.config('https://bddad754e8264fc7b4d73878b697b599@sentry.i-ris.io/39', {
                            whitelistUrls: [
                                'www.nature.com/'
                            ]
                        }).install();
                        window.Raven.captureException(error);
                    };
                    document.body.appendChild(s);
                }
            };

            if (window.config.mustardcut === true) {
                var index = 0;
                var scripts = [
                    {src: '/static/js/polyfills-bundle.662073af6a.js', test: 'polyfills-js'},
                    {src: '/static/js/nature-es6-bundle.2669c02c13.js', test: 'nature-es6-js'},
                    {src: '/static/js/shared-es6-bundle.c83ed51f05.js', test: 'shared-es6-js'},
                    {src: '/static/js/article-header-150-bundle.6c38860ebb.js', test: 'header-150-js'}
                ];

                var initOnceWhenReady = function() {
                    if (window.polyFills && window.initHeader && window.initEs6 && !window.inited) {
                        setTimeout(window.polyFills, 0);
                        setTimeout(window.initHeader, 0);
                        setTimeout(window.initEs6, 0);
                        window.inited = true;
                    }
                };
                var createScript = function(src, test, beforeLoad) {
                    var s = document.createElement('script');
                    s.id = 'js-position' + (index + 1);
                    s.setAttribute('data-test', test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function() {
                            if (document.readyState === 'complete' || (document.readyState !== 'loading' && !document.documentElement.doScroll)) {
                                initOnceWhenReady();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = src;
                    return s;
                };
                var insert = function(s) {
                    document.getElementById('js-position' + index).insertAdjacentElement('afterend', s);
                    ++index;
                };

                scripts.forEach(function (script) {
                    insert(createScript(script.src, script.test, true));
                });

                document.addEventListener('DOMContentLoaded', function() {
                    initOnceWhenReady();

                    var conditionalScripts = [
                        {match: 'div[data-pan-container]', src: '/static/js/pan-zoom.ac65de5f5b.js', test: 'pan-zoom-js'},
                        {match: 'math,span.mathjax-tex', src: '/static/js/math.4e116e29fc.js', test: 'math-js'}
                    ];
                    conditionalScripts.filter(function(script) {
                        return !!document.querySelector(script.match);
                    }).forEach(function(script) {
                        insert(createScript(script.src, script.test));
                    });
                }, false);
            }
        })();
    </script>


<script>(function(w,d){if(window.config.mustardcut === true){d.documentElement.className=d.documentElement.className.replace(/\s*grade-c/, "")}})(window,document)</script>




<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
    'https://www.googletagmanager.com/gtm.js?id='+i+dl;

    
    j.addEventListener('load', function() {
    var _ge = new CustomEvent('gtm_loaded', { bubbles: true });
    d.dispatchEvent(_ge);
    });

    f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-NWDMT9Q');</script>




<meta name="robots" content="noarchive">
<meta name="access" content="Yes">
<meta name="WT.cg_s" content="Article">
<meta name="WT.z_bandiera_abtest" content="a">
<meta name="WT.page_categorisation" content="Article_HTML">

    <meta name="WT.template" content="oscar">
    <meta name="WT.z_cg_type" content="Nature Research Journals">
    <meta name="WT.cg_n" content="Nature Communications">
    <meta name="dc.rights" content="©2020 Macmillan Publishers Limited. All Rights Reserved.">
    <meta name="prism.issn" content="2041-1723">


<link rel="search" href="http://www.nature.com/search">
<link rel="search" href="http://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="http://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">


    
    

    <meta name="journal_id" content="41467">

    <meta name="dc.title" content="A multi-crystal method for extracting obscured crystallographic states from conventionally uninterpretable electron density">

    <meta name="dc.source" content="Nature Communications 2017 8:1">

    <meta name="dc.format" content="text/html">

    <meta name="dc.publisher" content="Nature Publishing Group">

    <meta name="dc.date" content="2017-04-24">

    <meta name="dc.type" content="OriginalPaper">

    <meta name="dc.language" content="En">

    <meta name="dc.copyright" content="2017 The Author(s)">

    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com">

    <meta name="dc.description" content="In macromolecular crystallography, the rigorous detection of changed states (for example, ligand binding) is difficult unless signal is strong. Ambiguous (‘weak’ or ‘noisy’) density is experimentally common, since molecular states are generally only fractionally present in the crystal. Existing methodologies focus on generating maximally accurate maps whereby minor states become discernible; in practice, such map interpretation is disappointingly subjective, time-consuming and methodologically unsound. Here we report the PanDDA method, which automatically reveals clear electron density for the changed state—even from inaccurate maps—by subtracting a proportion of the confounding ‘ground state’; changed states are objectively identified from statistical analysis of density distributions. The method is completely general, implying new best practice for all changed-state studies, including the routine collection of multiple ground-state crystals. More generally, these results demonstrate: the incompleteness of atomic models; that single data sets contain insufficient information to model them fully; and that accuracy requires further map-deconvolution approaches. Building a ligand into a weak region of an electron density map of a protein is a subjective process. Here, the authors present a new method to obtain a clear electron density for a bound ligand based on multi-crystal experiments and 3D background correction.">

    <meta name="prism.issn" content="2041-1723">

    <meta name="prism.publicationName" content="Nature Communications">

    <meta name="prism.publicationDate" content="2017-04-24">

    <meta name="prism.volume" content="8">

    <meta name="prism.number" content="1">

    <meta name="prism.section" content="OriginalPaper">

    <meta name="prism.startingPage" content="1">

    <meta name="prism.endingPage" content="8">

    <meta name="prism.copyright" content="2017 The Author(s)">

    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com">

    <meta name="prism.url" content="https://www.nature.com/articles/ncomms15123">

    <meta name="prism.doi" content="doi:10.1038/ncomms15123">

    <meta name="citation_pdf_url" content="https://www.nature.com/articles/ncomms15123.pdf">

    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/ncomms15123">

    <meta name="citation_journal_title" content="Nature Communications">

    <meta name="citation_journal_abbrev" content="Nat Commun">

    <meta name="citation_publisher" content="Nature Publishing Group">

    <meta name="citation_issn" content="2041-1723">

    <meta name="citation_title" content="A multi-crystal method for extracting obscured crystallographic states from conventionally uninterpretable electron density">

    <meta name="citation_volume" content="8">

    <meta name="citation_issue" content="1">

    <meta name="citation_online_date" content="2017/04/24">

    <meta name="citation_firstpage" content="1">

    <meta name="citation_lastpage" content="8">

    <meta name="citation_article_type" content="Article">

    <meta name="citation_fulltext_world_readable" content="">

    <meta name="citation_language" content="en">

    <meta name="dc.identifier" content="doi:10.1038/ncomms15123">

    <meta name="DOI" content="10.1038/ncomms15123">

    <meta name="citation_doi" content="10.1038/ncomms15123">

    <meta name="description" content="In macromolecular crystallography, the rigorous detection of changed states (for example, ligand binding) is difficult unless signal is strong. Ambiguous (‘weak’ or ‘noisy’) density is experimentally common, since molecular states are generally only fractionally present in the crystal. Existing methodologies focus on generating maximally accurate maps whereby minor states become discernible; in practice, such map interpretation is disappointingly subjective, time-consuming and methodologically unsound. Here we report the PanDDA method, which automatically reveals clear electron density for the changed state—even from inaccurate maps—by subtracting a proportion of the confounding ‘ground state’; changed states are objectively identified from statistical analysis of density distributions. The method is completely general, implying new best practice for all changed-state studies, including the routine collection of multiple ground-state crystals. More generally, these results demonstrate: the incompleteness of atomic models; that single data sets contain insufficient information to model them fully; and that accuracy requires further map-deconvolution approaches. Building a ligand into a weak region of an electron density map of a protein is a subjective process. Here, the authors present a new method to obtain a clear electron density for a bound ligand based on multi-crystal experiments and 3D background correction.">

    <meta name="dc.creator" content="Nicholas M. Pearce">

    <meta name="dc.creator" content="Tobias Krojer">

    <meta name="dc.creator" content="Anthony R. Bradley">

    <meta name="dc.creator" content="Patrick Collins">

    <meta name="dc.creator" content="Radosław P. Nowak">

    <meta name="dc.creator" content="Romain Talon">

    <meta name="dc.creator" content="Brian D. Marsden">

    <meta name="dc.creator" content="Sebastian Kelm">

    <meta name="dc.creator" content="Jiye Shi">

    <meta name="dc.creator" content="Charlotte M. Deane">

    <meta name="dc.creator" content="Frank von Delft">

    <meta name="dc.subject" content="Proteins">

    <meta name="dc.subject" content="Screening">

    <meta name="dc.subject" content="X-ray crystallography">

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr. Sect. A Found. Crystallogr.; citation_title=A short history of SHELX; citation_author=GM Sheldrick; citation_volume=64; citation_publication_date=2007; citation_pages=112-122; citation_doi=10.1107/S0108767307043930; citation_id=CR1">

    <meta name="citation_reference" content="citation_journal_title=FEBS J.; citation_title=The R-factor gap in macromolecular crystallography: an untapped potential for insights on accurate structures; citation_author=JM Holton, S Classen, KA Frankel, JA Tainer; citation_volume=281; citation_publication_date=2014; citation_pages=4046-4060; citation_doi=10.1111/febs.12922; citation_id=CR2">

    <meta name="citation_reference" content="citation_journal_title=Elife; citation_title=Modelling dynamics in protein crystal structures by ensemble refinement; citation_author=BT Burnley, PV Afonine, PD Adams, P Gros; citation_volume=1; citation_publication_date=2012; citation_pages=e00311; citation_doi=10.7554/eLife.00311; citation_id=CR3">

    <meta name="citation_reference" content="citation_journal_title=Structure; citation_title=Crystallographic refinement by knowledge-based exploration of complex energy landscapes; citation_author=MA DePristo, PIW De Bakker, RJK Johnson, TL Blundell; citation_volume=13; citation_publication_date=2005; citation_pages=1311-1319; citation_doi=10.1016/j.str.2005.06.008; citation_id=CR4">

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr. Sect. D Biol. Crystallogr.; citation_title=Modeling discrete heterogeneity in X-ray diffraction data by fitting multi-conformers; citation_author=H Van Den Bedem, A Dhanik, JC Latombe, AM Deacon; citation_volume=65; citation_publication_date=2009; citation_pages=1107-1117; citation_doi=10.1107/S0907444909030613; citation_id=CR5">

    <meta name="citation_reference" content="citation_journal_title=Protein Sci.; citation_title=Automated electron-density sampling reveals widespread conformational polymorphism in proteins; citation_author=PT Lang; citation_volume=19; citation_publication_date=2010; citation_pages=1420-1431; citation_doi=10.1002/pro.423; citation_id=CR6">

    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Protein structural ensembles are revealed by redefining X-ray electron density noise; citation_author=PT Lang, JM Holton, JS Fraser, T Alber; citation_volume=111; citation_publication_date=2014; citation_pages=237-242; citation_doi=10.1073/pnas.1302823110; citation_id=CR7">

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr. Sect. A Found. Crystallogr.; citation_title=Improved Fourier coefficients for maps using phases from partial structures with errors; citation_author=RJ Read; citation_volume=42; citation_publication_date=1986; citation_pages=140-149; citation_doi=10.1107/S0108767386099622; citation_id=CR8">

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr. Sect. D Biol. Crystallogr.; citation_title=Techniques, tools and best practices for ligand electron-density analysis and results from their application to deposited crystal structures; citation_author=E Pozharski, CX Weichenberger, B Rupp; citation_volume=69; citation_publication_date=2013; citation_pages=150-167; citation_doi=10.1107/S0907444912044423; citation_id=CR9">

    <meta name="citation_reference" content="citation_journal_title=J. Immunol.; citation_title=Comment on Three X-ray Crystal Structure Papers; citation_author=R Stanfield, E Pozharski, B Rupp; citation_volume=196; citation_publication_date=2016; citation_pages=521-524; citation_id=CR10">

    <meta name="citation_reference" content="citation_journal_title=Nat. Struct. Biol; citation_title=Questions about the structure of the botulinum neurotoxin B light chain in complex with a target peptide; citation_author=B Rupp, B Segelke; citation_volume=8; citation_publication_date=2001; citation_pages=663-664; citation_doi=10.1038/90361; citation_id=CR11">

    <meta name="citation_reference" content="citation_journal_title=Structure; citation_title=Where freedom is given, liberties are taken; citation_author=GJ Kleywegt, TA Jones; citation_volume=3; citation_publication_date=1995; citation_pages=535-540; citation_doi=10.1016/S0969-2126(01)00187-3; citation_id=CR12">

    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Time-resolved crystallography using the Hadamard transform; citation_author=BA Yorke, GS Beddard, RL Owen, AR Pearson; citation_volume=11; citation_publication_date=2014; citation_pages=1131-1134; citation_doi=10.1038/nmeth.3139; citation_id=CR13">

    <meta name="citation_reference" content="citation_journal_title=Methods Enzymol.; citation_title=Isomorphous Difference Methods; citation_author=MA Rould, CW Carter; citation_volume=374; citation_publication_date=2003; citation_pages=145-163; citation_doi=10.1016/S0076-6879(03)74007-5; citation_id=CR14">

    <meta name="citation_reference" content="citation_journal_title=Structure; citation_title=Crystal cryocooling distorts conformational heterogeneity in a model michaelis complex of DHFR; citation_author=DA Keedy; citation_volume=22; citation_publication_date=2014; citation_pages=899-910; citation_doi=10.1016/j.str.2014.04.016; citation_id=CR15">

    <meta name="citation_reference" content="citation_journal_title=Prog. Biophys. Mol. Biol.; citation_title=Advantages of crystallographic fragment screening: Functional and mechanistic insights from a powerful platform for efficient drug discovery; citation_author=D Patel, JD Bauman, E Arnold; citation_volume=116; citation_publication_date=2014; citation_pages=92-100; citation_doi=10.1016/j.pbiomolbio.2014.08.004; citation_id=CR16">

    <meta name="citation_reference" content="citation_journal_title=Chem. Sci.; citation_title=A poised fragment library enables rapid synthetic expansion yielding the first reported inhibitors of PHIP(2), an atypical bromodomain; citation_author=OB Cox; citation_volume=7; citation_publication_date=2016; citation_pages=2322-2330; citation_doi=10.1039/C5SC03115J; citation_id=CR17">

    <meta name="citation_reference" content="citation_journal_title=J. Comput. Aided. Mol. Des.; citation_title=The consequences of translational and rotational entropy lost by small molecules on binding to proteins; citation_author=CW Murray, ML Verdonk; citation_volume=16; citation_publication_date=2002; citation_pages=741-753; citation_doi=10.1023/A:1022446720849; citation_id=CR18">

    <meta name="citation_reference" content="citation_journal_title=Chem. Med. Chem.; citation_title=Automated protein-ligand crystallography for structure-based drug design; citation_author=WTM Mooij; citation_volume=1; citation_publication_date=2006; citation_pages=827-838; citation_doi=10.1002/cmdc.200600074; citation_id=CR19">

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr. Sect. D Biol. Crystallogr.; citation_title=Statistical quality indicators for electron-density maps; citation_author=IJ Tickle; citation_volume=68; citation_publication_date=2012; citation_pages=454-467; citation_doi=10.1107/S0907444911035918; citation_id=CR20">

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr. Sect. D Biol. Crystallogr.; citation_title=Low-resolution refinement tools in REFMAC5; citation_author=RA Nicholls, F Long, GN Murshudov; citation_volume=68; citation_publication_date=2012; citation_pages=404-417; citation_doi=10.1107/S090744491105606X; citation_id=CR21">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=The Protein Data Bank; citation_author=HM Berman; citation_volume=28; citation_publication_date=2000; citation_pages=235-242; citation_doi=10.1093/nar/28.1.235; citation_id=CR22">

    <meta name="citation_reference" content="citation_journal_title=ACS Chem. Biol.; citation_title=Six Biophysical Screening Methods Miss a Large Proportion of Crystallographically Discovered Fragment Hits: A Case Study; citation_author=J Schiebel; citation_volume=11; citation_publication_date=2016; citation_pages=1693-1701; citation_doi=10.1021/acschembio.5b01034; citation_id=CR23">

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr. Sect. D Biol. Crystallogr.; citation_title=Optimal fine phi-slicing for single-photon-counting pixel detectors; citation_author=M Mueller, M Wang, C Schulze-Briese; citation_volume=68; citation_publication_date=2012; citation_pages=42-56; citation_doi=10.1107/S0907444911049833; citation_id=CR24">

    <meta name="citation_reference" content="citation_journal_title=IUCrJ; citation_title=Synchrotron radiation macromolecular crystallography: Science and spin-offs; citation_author=JR Helliwell, EP Mitchell; citation_volume=2; citation_publication_date=2015; citation_pages=283-291; citation_doi=10.1107/S205225251402795X; citation_id=CR25">

    <meta name="citation_reference" content="citation_journal_title=Structure; citation_title=High-throughput crystallography: reliable and efficient identification of fragment hits; citation_author=J Schiebel; citation_volume=24; citation_publication_date=2016; citation_pages=1398-1409; citation_doi=10.1016/j.str.2016.06.010; citation_id=CR26">

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr. Sect. D Biol. Crystallogr.; citation_title=Clustering procedures for the optimal selection of data sets from multiple crystals in macromolecular crystallography; citation_author=J Foadi; citation_volume=69; citation_publication_date=2013; citation_pages=1617-1632; citation_doi=10.1107/S0907444913012274; citation_id=CR27">

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr. Sect. D Biol. Crystallogr.; citation_title=Overview of the CCP4 suite and current developments; citation_author=MD Winn; citation_volume=67; citation_publication_date=2011; citation_pages=235-242; citation_doi=10.1107/S0907444910045749; citation_id=CR28">

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr. Sect. D Biol. Crystallogr.; citation_title=Towards automated crystallographic structure refinement with phenix.refine; citation_author=PV Afonine; citation_volume=68; citation_publication_date=2012; citation_pages=352-367; citation_doi=10.1107/S0907444912001308; citation_id=CR29">

    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr. Sect. D Biol. Crystallogr.; citation_title=PHENIX: a comprehensive Python-based system for macromolecular structure solution; citation_author=PD Adams; citation_volume=66; citation_publication_date=2010; citation_pages=213-221; citation_doi=10.1107/S0907444909052925; citation_id=CR30">

    <meta name="citation_reference" content="citation_journal_title=J. Appl. Crystallogr.; citation_title=The Computational Crystallography Toolbox: crystallographic algorithms in a reusable software framework; citation_author=RW Grosse-Kunstleve, NK Sauter, NW Moriarty, PD Adams; citation_volume=35; citation_publication_date=2002; citation_pages=126-136; citation_doi=10.1107/S0021889801017824; citation_id=CR31">

    <meta name="citation_reference" content="Krojer, T., Pearce, N. M., Bradley, A., Marsden, B. D. &amp; von Delft, F. PanDDA analysis of BAZ2B screened against Zenobia Fragment Library (HTML Summary). Zenodo. Available at: https://zenodo.org/record/290199 (2017).">

    <meta name="citation_reference" content="Krojer, T., Pearce, N. M., Bradley, A., Marsden, B. D. &amp; von Delft, F. PanDDA analysis of JMJD2D screened against Zenobia Fragment Library (HTML Summary). Zenodo. Available at: https://zenodo.org/record/290220 (2017).">

    <meta name="citation_reference" content="Krojer, T., Pearce, N. M., Collins, P., Talon, R. &amp; von Delft, F. PanDDA analysis of BRD1 screened against 3D-Fragment-Consortium Fragment Library. (HTML Summary) Zenodo. Available at: https://zenodo.org/record/290217 (2017).">

    <meta name="citation_reference" content="Krojer, T., Pearce, N. M., Collins, P., Talon, R. &amp; von Delft, F. PanDDA analysis of SP100 screened against selection of Maybridge Fragment Library (HTML Summary). Zenodo. Available at: https://zenodo.org/record/290201 (2017).">

    <meta name="citation_reference" content="Pearce, N. M., Bradley, A. R., Marsden, B. D. &amp; von Delft, F. PanDDA analysis of BAZ2B screened against Zenobia Fragment Library [Dataset]. Zenodo. Available at: https://zenodo.org/record/48768 (2016).">

    <meta name="citation_reference" content="Pearce, N. M., Bradley, A. R., Marsden, B. D. &amp; von Delft, F. PanDDA analysis of JMJD2D screened against Zenobia Fragment Library [Dataset]. Zenodo. Available at: https://zenodo.org/record/48770 (2016).">

    <meta name="citation_reference" content="Pearce, N. M., Krojer, T., Collins, P., Talon, R. &amp; von Delft, F. PanDDA analysis of BRD1 screened against 3D-Fragment Consortium Fragment Library [Dataset]. Zenodo. Available at: https://zenodo.org/record/48769 (2016).">

    <meta name="citation_reference" content="Pearce, N. M., Krojer, T., Collins, P., Talon, R. &amp; von Delft, F. PanDDA analysis of SP100 screened against selection of Maybridge Fragment Library [Dataset]. Zenodo. Available at: https://zenodo.org/record/48771 (2016).">

    <meta name="citation_author" content="Nicholas M. Pearce">

    <meta name="citation_author_institution" content="Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Oxford, UK">

    <meta name="citation_author" content="Tobias Krojer">

    <meta name="citation_author_institution" content="Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Oxford, UK">

    <meta name="citation_author" content="Anthony R. Bradley">

    <meta name="citation_author_institution" content="Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Oxford, UK">

    <meta name="citation_author" content="Patrick Collins">

    <meta name="citation_author_institution" content="Diamond Light Source Ltd, Harwell Science and Innovation Campus, Didcot, UK">

    <meta name="citation_author" content="Radosław P. Nowak">

    <meta name="citation_author_institution" content="Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Oxford, UK">

    <meta name="citation_author" content="Romain Talon">

    <meta name="citation_author_institution" content="Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Oxford, UK">

    <meta name="citation_author" content="Brian D. Marsden">

    <meta name="citation_author_institution" content="Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Oxford, UK">

    <meta name="citation_author_institution" content="Nuffield Department of Orthopaedics, Kennedy Institute of Rheumatology, Rheumatology and Musculoskeletal Sciences, University of Oxford, Oxford, UK">

    <meta name="citation_author" content="Sebastian Kelm">

    <meta name="citation_author_institution" content="UCB Pharma, Slough, UK">

    <meta name="citation_author" content="Jiye Shi">

    <meta name="citation_author_institution" content="UCB Pharma, Slough, UK">

    <meta name="citation_author" content="Charlotte M. Deane">

    <meta name="citation_author_institution" content="Department of Statistics, University of Oxford, Oxford, UK">

    <meta name="citation_author" content="Frank von Delft">

    <meta name="citation_author_institution" content="Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Oxford, UK">

    <meta name="citation_author_institution" content="Diamond Light Source Ltd, Harwell Science and Innovation Campus, Didcot, UK">

    <meta name="citation_author_institution" content="Department of Biochemistry, University of Johannesburg, Auckland Park, South Africa">

    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access">

    <meta name="twitter:site" content="@NatureComms">

    <meta name="twitter:card" content="summary">

    <meta name="twitter:image:alt" content="Content cover image">

    <meta name="WT.z_cc_license_type" content="cc_by">

    <meta name="WT.z_primary_atype" content="Research">

    <meta name="WT.z_subject_term" content="Proteins;Screening;X-ray crystallography">

    <meta name="WT.z_subject_term_id" content="proteins;screening;x-ray-crystallography">


    
        <meta property="og:url" content="https://www.nature.com/articles/ncomms15123">
        <meta property="og:type" content="article">
        <meta property="og:site_name" content="Nature Communications">
        <meta property="og:title" content="A multi-crystal method for extracting obscured crystallographic states from conventionally uninterpretable electron density">
        <meta property="og:description" content="Building a ligand into a weak region of an electron density map of a protein is a subjective process. Here, the authors present a new method to obtain a clear electron density for a bound ligand based on multi-crystal experiments and 3D background correction.">
        <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig1_HTML.jpg">
    
    <script>
        window.eligibleForRa21 = 'false'; // required by js files for displaying the cobranding box (entitlement-box.js)
    </script>
<script async="" type="text/javascript" src="inject.js"></script><link type="text/css" rel="stylesheet" href="nature_checkout.css"><script src="otBannerSdk.js" async="" type="text/javascript"></script><style id="onetrust-style">#onetrust-banner-sdk{-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}#onetrust-banner-sdk .onetrust-vendors-list-handler{cursor:pointer;color:#1f96db;font-size:inherit;font-weight:bold;text-decoration:none;margin-left:5px}#onetrust-banner-sdk .onetrust-vendors-list-handler:hover{color:#1f96db}#onetrust-banner-sdk .ot-close-icon,#onetrust-pc-sdk .ot-close-icon{background-image:url("data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4IiB3aWR0aD0iMzQ4LjMzM3B4IiBoZWlnaHQ9IjM0OC4zMzNweCIgdmlld0JveD0iMCAwIDM0OC4zMzMgMzQ4LjMzNCIgc3R5bGU9ImVuYWJsZS1iYWNrZ3JvdW5kOm5ldyAwIDAgMzQ4LjMzMyAzNDguMzM0OyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+PGc+PHBhdGggZmlsbD0iIzU2NTY1NiIgZD0iTTMzNi41NTksNjguNjExTDIzMS4wMTYsMTc0LjE2NWwxMDUuNTQzLDEwNS41NDljMTUuNjk5LDE1LjcwNSwxNS42OTksNDEuMTQ1LDAsNTYuODVjLTcuODQ0LDcuODQ0LTE4LjEyOCwxMS43NjktMjguNDA3LDExLjc2OWMtMTAuMjk2LDAtMjAuNTgxLTMuOTE5LTI4LjQxOS0xMS43NjlMMTc0LjE2NywyMzEuMDAzTDY4LjYwOSwzMzYuNTYzYy03Ljg0Myw3Ljg0NC0xOC4xMjgsMTEuNzY5LTI4LjQxNiwxMS43NjljLTEwLjI4NSwwLTIwLjU2My0zLjkxOS0yOC40MTMtMTEuNzY5Yy0xNS42OTktMTUuNjk4LTE1LjY5OS00MS4xMzksMC01Ni44NWwxMDUuNTQtMTA1LjU0OUwxMS43NzQsNjguNjExYy0xNS42OTktMTUuNjk5LTE1LjY5OS00MS4xNDUsMC01Ni44NDRjMTUuNjk2LTE1LjY4Nyw0MS4xMjctMTUuNjg3LDU2LjgyOSwwbDEwNS41NjMsMTA1LjU1NEwyNzkuNzIxLDExLjc2N2MxNS43MDUtMTUuNjg3LDQxLjEzOS0xNS42ODcsNTYuODMyLDBDMzUyLjI1OCwyNy40NjYsMzUyLjI1OCw1Mi45MTIsMzM2LjU1OSw2OC42MTF6Ii8+PC9nPjwvc3ZnPg==");background-size:contain;background-repeat:no-repeat;background-position:center;height:12px;width:12px}#onetrust-banner-sdk .powered-by-logo,#onetrust-banner-sdk .ot-pc-footer-logo a,#onetrust-pc-sdk .powered-by-logo,#onetrust-pc-sdk .ot-pc-footer-logo a{background-size:contain;background-repeat:no-repeat;background-position:center;height:25px;width:152px;display:block}#onetrust-banner-sdk h3 *,#onetrust-banner-sdk h4 *,#onetrust-banner-sdk h6 *,#onetrust-banner-sdk button *,#onetrust-banner-sdk a[data-parent-id] *,#onetrust-pc-sdk h3 *,#onetrust-pc-sdk h4 *,#onetrust-pc-sdk h6 *,#onetrust-pc-sdk button *,#onetrust-pc-sdk a[data-parent-id] *{font-size:inherit;font-weight:inherit;color:inherit}#onetrust-banner-sdk .ot-hide,#onetrust-pc-sdk .ot-hide{display:none !important}#onetrust-pc-sdk .ot-sdk-row .ot-sdk-column{padding:0}#onetrust-pc-sdk .ot-sdk-container{padding-right:0}#onetrust-pc-sdk .ot-sdk-row{flex-direction:initial;width:100%}#onetrust-pc-sdk [type="checkbox"]:checked,#onetrust-pc-sdk [type="checkbox"]:not(:checked){pointer-events:initial}#onetrust-pc-sdk [type="checkbox"]:disabled+label::before,#onetrust-pc-sdk [type="checkbox"]:disabled+label:after,#onetrust-pc-sdk [type="checkbox"]:disabled+label{pointer-events:none;opacity:0.7}#onetrust-pc-sdk #vendor-list-content{transform:translate3d(0, 0, 0)}#onetrust-pc-sdk li input[type="checkbox"]{z-index:1}#onetrust-pc-sdk li .ot-checkbox label{z-index:2}#onetrust-pc-sdk li .ot-checkbox input[type="checkbox"]{height:auto;width:auto}#onetrust-pc-sdk li .host-title a,#onetrust-pc-sdk li .ot-host-name a,#onetrust-pc-sdk li .accordion-text,#onetrust-pc-sdk li .ot-acc-txt{z-index:2;position:relative}#onetrust-pc-sdk input{margin:3px 0.1ex}#onetrust-pc-sdk .toggle-always-active{opacity:0.6;cursor:default}#onetrust-pc-sdk .screen-reader-only,#onetrust-pc-sdk .ot-scrn-rdr{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}#onetrust-pc-sdk .pc-logo,#onetrust-pc-sdk .ot-pc-logo{height:60px;width:180px;background-position:center;background-size:contain;background-repeat:no-repeat}#onetrust-pc-sdk .ot-tooltip .ot-tooltiptext{visibility:hidden;width:120px;background-color:#555;color:#fff;text-align:center;padding:5px 0;border-radius:6px;position:absolute;z-index:1;bottom:125%;left:50%;margin-left:-60px;opacity:0;transition:opacity 0.3s}#onetrust-pc-sdk .ot-tooltip .ot-tooltiptext::after{content:"";position:absolute;top:100%;left:50%;margin-left:-5px;border-width:5px;border-style:solid;border-color:#555 transparent transparent transparent}#onetrust-pc-sdk .ot-tooltip:hover .ot-tooltiptext{visibility:visible;opacity:1}#onetrust-pc-sdk .ot-tooltip{position:relative;display:inline-block;z-index:3}#onetrust-pc-sdk .ot-tooltip svg{color:grey;height:20px;width:20px}#onetrust-pc-sdk.ot-fade-in,.onetrust-pc-dark-filter.ot-fade-in{animation-name:onetrust-fade-in;animation-duration:400ms;animation-timing-function:ease-in-out}#onetrust-pc-sdk.ot-hide{display:none !important}.onetrust-pc-dark-filter.ot-hide{display:none !important}#ot-sdk-btn.ot-sdk-show-settings,#ot-sdk-btn.optanon-show-settings{color:#68b631;border:1px solid #68b631;height:auto;white-space:normal;word-wrap:break-word;padding:0.8em 2em;font-size:0.8em;line-height:1.2;cursor:pointer;-moz-transition:0.1s ease;-o-transition:0.1s ease;-webkit-transition:1s ease;transition:0.1s ease}#ot-sdk-btn.ot-sdk-show-settings:hover,#ot-sdk-btn.optanon-show-settings:hover{color:#fff;background-color:#68b631}#ot-sdk-btn.ot-sdk-show-settings:focus,#ot-sdk-btn.optanon-show-settings:focus{outline:none}.onetrust-pc-dark-filter{background:rgba(0,0,0,0.5);z-index:2147483646;width:100%;height:100%;overflow:hidden;position:fixed;top:0;bottom:0;left:0}@keyframes onetrust-fade-in{0%{opacity:0}100%{opacity:1}}@media only screen and (min-width: 426px) and (max-width: 896px) and (orientation: landscape){#onetrust-pc-sdk p{font-size:0.75em}}
#onetrust-banner-sdk,#onetrust-pc-sdk,#ot-sdk-cookie-policy{font-size:16px}#onetrust-banner-sdk *,#onetrust-banner-sdk ::after,#onetrust-banner-sdk ::before,#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before,#ot-sdk-cookie-policy *,#ot-sdk-cookie-policy ::after,#ot-sdk-cookie-policy ::before{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}#onetrust-banner-sdk div,#onetrust-banner-sdk span,#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-banner-sdk p,#onetrust-banner-sdk img,#onetrust-banner-sdk svg,#onetrust-banner-sdk button,#onetrust-banner-sdk section,#onetrust-banner-sdk a,#onetrust-banner-sdk label,#onetrust-banner-sdk input,#onetrust-banner-sdk ul,#onetrust-banner-sdk li,#onetrust-banner-sdk nav,#onetrust-banner-sdk table,#onetrust-banner-sdk thead,#onetrust-banner-sdk tr,#onetrust-banner-sdk td,#onetrust-banner-sdk tbody,#onetrust-banner-sdk .ot-main-content,#onetrust-banner-sdk .ot-toggle,#onetrust-banner-sdk #ot-content,#onetrust-banner-sdk #ot-pc-content,#onetrust-banner-sdk .checkbox,#onetrust-pc-sdk div,#onetrust-pc-sdk span,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#onetrust-pc-sdk p,#onetrust-pc-sdk img,#onetrust-pc-sdk svg,#onetrust-pc-sdk button,#onetrust-pc-sdk section,#onetrust-pc-sdk a,#onetrust-pc-sdk label,#onetrust-pc-sdk input,#onetrust-pc-sdk ul,#onetrust-pc-sdk li,#onetrust-pc-sdk nav,#onetrust-pc-sdk table,#onetrust-pc-sdk thead,#onetrust-pc-sdk tr,#onetrust-pc-sdk td,#onetrust-pc-sdk tbody,#onetrust-pc-sdk .ot-main-content,#onetrust-pc-sdk .ot-toggle,#onetrust-pc-sdk #ot-content,#onetrust-pc-sdk #ot-pc-content,#onetrust-pc-sdk .checkbox,#ot-sdk-cookie-policy div,#ot-sdk-cookie-policy span,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy p,#ot-sdk-cookie-policy img,#ot-sdk-cookie-policy svg,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy section,#ot-sdk-cookie-policy a,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy input,#ot-sdk-cookie-policy ul,#ot-sdk-cookie-policy li,#ot-sdk-cookie-policy nav,#ot-sdk-cookie-policy table,#ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy tr,#ot-sdk-cookie-policy td,#ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy .ot-main-content,#ot-sdk-cookie-policy .ot-toggle,#ot-sdk-cookie-policy #ot-content,#ot-sdk-cookie-policy #ot-pc-content,#ot-sdk-cookie-policy .checkbox{font-family:inherit;font-weight:normal;-webkit-font-smoothing:auto;letter-spacing:normal;line-height:normal;padding:0;margin:0;height:auto;min-height:0;max-height:none;width:auto;min-width:0;max-width:none;border-radius:0;border:none;clear:none;float:none;position:static;bottom:auto;left:auto;right:auto;top:auto;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;white-space:normal;background:none;overflow:visible;vertical-align:baseline;visibility:visible;z-index:auto;box-shadow:none}#onetrust-banner-sdk label:before,#onetrust-banner-sdk label:after,#onetrust-banner-sdk .checkbox:after,#onetrust-banner-sdk .checkbox:before,#onetrust-pc-sdk label:before,#onetrust-pc-sdk label:after,#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before,#ot-sdk-cookie-policy label:before,#ot-sdk-cookie-policy label:after,#ot-sdk-cookie-policy .checkbox:after,#ot-sdk-cookie-policy .checkbox:before{content:"";content:none}
#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{position:relative;width:100%;max-width:100%;margin:0 auto;padding:0 20px;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{width:100%;float:left;box-sizing:border-box;padding:0;display:initial}@media (min-width: 400px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:90%;padding:0}}@media (min-width: 550px){#onetrust-banner-sdk .ot-sdk-container,#onetrust-pc-sdk .ot-sdk-container,#ot-sdk-cookie-policy .ot-sdk-container{width:100%}#onetrust-banner-sdk .ot-sdk-column,#onetrust-banner-sdk .ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-column,#onetrust-pc-sdk .ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-columns{margin-left:4%}#onetrust-banner-sdk .ot-sdk-column:first-child,#onetrust-banner-sdk .ot-sdk-columns:first-child,#onetrust-pc-sdk .ot-sdk-column:first-child,#onetrust-pc-sdk .ot-sdk-columns:first-child,#ot-sdk-cookie-policy .ot-sdk-column:first-child,#ot-sdk-cookie-policy .ot-sdk-columns:first-child{margin-left:0}#onetrust-banner-sdk .ot-sdk-one.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-one.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-one.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-one.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-one.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-one.ot-sdk-columns{width:4.66666666667%}#onetrust-banner-sdk .ot-sdk-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-two.ot-sdk-columns{width:13.3333333333%}#onetrust-banner-sdk .ot-sdk-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-three.ot-sdk-columns{width:22%}#onetrust-banner-sdk .ot-sdk-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-four.ot-sdk-columns{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-five.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-five.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-five.ot-sdk-columns{width:39.3333333333%}#onetrust-banner-sdk .ot-sdk-six.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-six.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-six.ot-sdk-columns{width:48%}#onetrust-banner-sdk .ot-sdk-seven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-seven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-seven.ot-sdk-columns{width:56.6666666667%}#onetrust-banner-sdk .ot-sdk-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eight.ot-sdk-columns{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-nine.ot-sdk-columns{width:74%}#onetrust-banner-sdk .ot-sdk-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-ten.ot-sdk-columns{width:82.6666666667%}#onetrust-banner-sdk .ot-sdk-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-eleven.ot-sdk-columns{width:91.3333333333%}#onetrust-banner-sdk .ot-sdk-twelve.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-twelve.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-twelve.ot-sdk-columns{width:100%;margin-left:0}#onetrust-banner-sdk .ot-sdk-one-third.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-one-third.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-one-third.ot-sdk-column{width:30.6666666667%}#onetrust-banner-sdk .ot-sdk-two-thirds.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-two-thirds.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-two-thirds.ot-sdk-column{width:65.3333333333%}#onetrust-banner-sdk .ot-sdk-one-half.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-one-half.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-one-half.ot-sdk-column{width:48%}#onetrust-banner-sdk .ot-sdk-offset-by-one.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-one.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-one.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-one.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-one.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-one.ot-sdk-columns{margin-left:8.66666666667%}#onetrust-banner-sdk .ot-sdk-offset-by-two.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-two.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-two.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-two.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-two.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-two.ot-sdk-columns{margin-left:17.3333333333%}#onetrust-banner-sdk .ot-sdk-offset-by-three.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-three.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-three.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-three.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-three.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-three.ot-sdk-columns{margin-left:26%}#onetrust-banner-sdk .ot-sdk-offset-by-four.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-four.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-four.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-four.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-four.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-four.ot-sdk-columns{margin-left:34.6666666667%}#onetrust-banner-sdk .ot-sdk-offset-by-five.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-five.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-five.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-five.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-five.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-five.ot-sdk-columns{margin-left:43.3333333333%}#onetrust-banner-sdk .ot-sdk-offset-by-six.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-six.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-six.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-six.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-six.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-six.ot-sdk-columns{margin-left:52%}#onetrust-banner-sdk .ot-sdk-offset-by-seven.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-seven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-seven.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-seven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-seven.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-seven.ot-sdk-columns{margin-left:60.6666666667%}#onetrust-banner-sdk .ot-sdk-offset-by-eight.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-eight.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-eight.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-eight.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-eight.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-eight.ot-sdk-columns{margin-left:69.3333333333%}#onetrust-banner-sdk .ot-sdk-offset-by-nine.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-nine.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-nine.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-nine.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-nine.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-nine.ot-sdk-columns{margin-left:78%}#onetrust-banner-sdk .ot-sdk-offset-by-ten.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-ten.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-ten.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-ten.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-ten.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-ten.ot-sdk-columns{margin-left:86.6666666667%}#onetrust-banner-sdk .ot-sdk-offset-by-eleven.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-eleven.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-eleven.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-eleven.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-eleven.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-eleven.ot-sdk-columns{margin-left:95.3333333333%}#onetrust-banner-sdk .ot-sdk-offset-by-one-third.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-one-third.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-one-third.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-one-third.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-one-third.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-one-third.ot-sdk-columns{margin-left:34.6666666667%}#onetrust-banner-sdk .ot-sdk-offset-by-two-thirds.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-two-thirds.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-two-thirds.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-two-thirds.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-two-thirds.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-two-thirds.ot-sdk-columns{margin-left:69.3333333333%}#onetrust-banner-sdk .ot-sdk-offset-by-one-half.ot-sdk-column,#onetrust-banner-sdk .ot-sdk-offset-by-one-half.ot-sdk-columns,#onetrust-pc-sdk .ot-sdk-offset-by-one-half.ot-sdk-column,#onetrust-pc-sdk .ot-sdk-offset-by-one-half.ot-sdk-columns,#ot-sdk-cookie-policy .ot-sdk-offset-by-one-half.ot-sdk-column,#ot-sdk-cookie-policy .ot-sdk-offset-by-one-half.ot-sdk-columns{margin-left:52%}}#onetrust-banner-sdk h1,#onetrust-banner-sdk h2,#onetrust-banner-sdk h3,#onetrust-banner-sdk h4,#onetrust-banner-sdk h5,#onetrust-banner-sdk h6,#onetrust-pc-sdk h1,#onetrust-pc-sdk h2,#onetrust-pc-sdk h3,#onetrust-pc-sdk h4,#onetrust-pc-sdk h5,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h1,#ot-sdk-cookie-policy h2,#ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy h5,#ot-sdk-cookie-policy h6{margin-top:0;font-weight:600;font-family:inherit}#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem;line-height:1.2}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem;line-height:1.25}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem;line-height:1.3}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem;line-height:1.35}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem;line-height:1.5}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem;line-height:1.6}@media (min-width: 550px){#onetrust-banner-sdk h1,#onetrust-pc-sdk h1,#ot-sdk-cookie-policy h1{font-size:1.5rem}#onetrust-banner-sdk h2,#onetrust-pc-sdk h2,#ot-sdk-cookie-policy h2{font-size:1.5rem}#onetrust-banner-sdk h3,#onetrust-pc-sdk h3,#ot-sdk-cookie-policy h3{font-size:1.5rem}#onetrust-banner-sdk h4,#onetrust-pc-sdk h4,#ot-sdk-cookie-policy h4{font-size:1.5rem}#onetrust-banner-sdk h5,#onetrust-pc-sdk h5,#ot-sdk-cookie-policy h5{font-size:1.5rem}#onetrust-banner-sdk h6,#onetrust-pc-sdk h6,#ot-sdk-cookie-policy h6{font-size:1.5rem}}#onetrust-banner-sdk p,#onetrust-pc-sdk p,#ot-sdk-cookie-policy p{margin:0 0 1em 0;font-family:inherit;line-height:normal}#onetrust-banner-sdk a,#onetrust-pc-sdk a,#ot-sdk-cookie-policy a{color:#565656;text-decoration:underline}#onetrust-banner-sdk a:hover,#onetrust-pc-sdk a:hover,#ot-sdk-cookie-policy a:hover{color:#565656;text-decoration:none}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button{margin-bottom:1rem;font-family:inherit}#onetrust-banner-sdk .ot-sdk-button,#onetrust-banner-sdk button,#onetrust-banner-sdk input[type="submit"],#onetrust-banner-sdk input[type="reset"],#onetrust-banner-sdk input[type="button"],#onetrust-pc-sdk .ot-sdk-button,#onetrust-pc-sdk button,#onetrust-pc-sdk input[type="submit"],#onetrust-pc-sdk input[type="reset"],#onetrust-pc-sdk input[type="button"],#ot-sdk-cookie-policy .ot-sdk-button,#ot-sdk-cookie-policy button,#ot-sdk-cookie-policy input[type="submit"],#ot-sdk-cookie-policy input[type="reset"],#ot-sdk-cookie-policy input[type="button"]{display:inline-block;height:38px;padding:0 30px;color:#555;text-align:center;font-size:0.9em;font-weight:400;line-height:38px;letter-spacing:0.01em;text-decoration:none;white-space:nowrap;background-color:transparent;border-radius:2px;border:1px solid #bbb;cursor:pointer;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-button:hover,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:hover,#onetrust-banner-sdk input[type="submit"]:hover,#onetrust-banner-sdk input[type="reset"]:hover,#onetrust-banner-sdk input[type="button"]:hover,#onetrust-banner-sdk .ot-sdk-button:focus,#onetrust-banner-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-banner-sdk input[type="submit"]:focus,#onetrust-banner-sdk input[type="reset"]:focus,#onetrust-banner-sdk input[type="button"]:focus,#onetrust-pc-sdk .ot-sdk-button:hover,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:hover,#onetrust-pc-sdk input[type="submit"]:hover,#onetrust-pc-sdk input[type="reset"]:hover,#onetrust-pc-sdk input[type="button"]:hover,#onetrust-pc-sdk .ot-sdk-button:focus,#onetrust-pc-sdk :not(.ot-leg-btn-container)>button:focus,#onetrust-pc-sdk input[type="submit"]:focus,#onetrust-pc-sdk input[type="reset"]:focus,#onetrust-pc-sdk input[type="button"]:focus,#ot-sdk-cookie-policy .ot-sdk-button:hover,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:hover,#ot-sdk-cookie-policy input[type="submit"]:hover,#ot-sdk-cookie-policy input[type="reset"]:hover,#ot-sdk-cookie-policy input[type="button"]:hover,#ot-sdk-cookie-policy .ot-sdk-button:focus,#ot-sdk-cookie-policy :not(.ot-leg-btn-container)>button:focus,#ot-sdk-cookie-policy input[type="submit"]:focus,#ot-sdk-cookie-policy input[type="reset"]:focus,#ot-sdk-cookie-policy input[type="button"]:focus{color:#333;border-color:#888;outline:0;opacity:0.7}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-banner-sdk button.ot-sdk-button-primary,#onetrust-banner-sdk input[type="submit"].ot-sdk-button-primary,#onetrust-banner-sdk input[type="reset"].ot-sdk-button-primary,#onetrust-banner-sdk input[type="button"].ot-sdk-button-primary,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary,#onetrust-pc-sdk button.ot-sdk-button-primary,#onetrust-pc-sdk input[type="submit"].ot-sdk-button-primary,#onetrust-pc-sdk input[type="reset"].ot-sdk-button-primary,#onetrust-pc-sdk input[type="button"].ot-sdk-button-primary,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary,#ot-sdk-cookie-policy button.ot-sdk-button-primary,#ot-sdk-cookie-policy input[type="submit"].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type="reset"].ot-sdk-button-primary,#ot-sdk-cookie-policy input[type="button"].ot-sdk-button-primary{color:#fff;background-color:#33c3f0;border-color:#33c3f0}#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-banner-sdk button.ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type="submit"].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type="reset"].ot-sdk-button-primary:hover,#onetrust-banner-sdk input[type="button"].ot-sdk-button-primary:hover,#onetrust-banner-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-banner-sdk button.ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type="submit"].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type="reset"].ot-sdk-button-primary:focus,#onetrust-banner-sdk input[type="button"].ot-sdk-button-primary:focus,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:hover,#onetrust-pc-sdk button.ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type="submit"].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type="reset"].ot-sdk-button-primary:hover,#onetrust-pc-sdk input[type="button"].ot-sdk-button-primary:hover,#onetrust-pc-sdk .ot-sdk-button.ot-sdk-button-primary:focus,#onetrust-pc-sdk button.ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type="submit"].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type="reset"].ot-sdk-button-primary:focus,#onetrust-pc-sdk input[type="button"].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy button.ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type="submit"].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type="reset"].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy input[type="button"].ot-sdk-button-primary:hover,#ot-sdk-cookie-policy .ot-sdk-button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy button.ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type="submit"].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type="reset"].ot-sdk-button-primary:focus,#ot-sdk-cookie-policy input[type="button"].ot-sdk-button-primary:focus{color:#fff;background-color:#1eaedb;border-color:#1eaedb}#onetrust-banner-sdk input[type="email"],#onetrust-banner-sdk input[type="number"],#onetrust-banner-sdk input[type="search"],#onetrust-banner-sdk input[type="text"],#onetrust-banner-sdk input[type="tel"],#onetrust-banner-sdk input[type="url"],#onetrust-banner-sdk input[type="password"],#onetrust-banner-sdk textarea,#onetrust-banner-sdk select,#onetrust-pc-sdk input[type="email"],#onetrust-pc-sdk input[type="number"],#onetrust-pc-sdk input[type="search"],#onetrust-pc-sdk input[type="text"],#onetrust-pc-sdk input[type="tel"],#onetrust-pc-sdk input[type="url"],#onetrust-pc-sdk input[type="password"],#onetrust-pc-sdk textarea,#onetrust-pc-sdk select,#ot-sdk-cookie-policy input[type="email"],#ot-sdk-cookie-policy input[type="number"],#ot-sdk-cookie-policy input[type="search"],#ot-sdk-cookie-policy input[type="text"],#ot-sdk-cookie-policy input[type="tel"],#ot-sdk-cookie-policy input[type="url"],#ot-sdk-cookie-policy input[type="password"],#ot-sdk-cookie-policy textarea,#ot-sdk-cookie-policy select{height:38px;padding:6px 10px;background-color:#fff;border:1px solid #d1d1d1;border-radius:4px;box-shadow:none;box-sizing:border-box}#onetrust-banner-sdk input[type="email"],#onetrust-banner-sdk input[type="number"],#onetrust-banner-sdk input[type="search"],#onetrust-banner-sdk input[type="text"],#onetrust-banner-sdk input[type="tel"],#onetrust-banner-sdk input[type="url"],#onetrust-banner-sdk input[type="password"],#onetrust-banner-sdk textarea,#onetrust-pc-sdk input[type="email"],#onetrust-pc-sdk input[type="number"],#onetrust-pc-sdk input[type="search"],#onetrust-pc-sdk input[type="text"],#onetrust-pc-sdk input[type="tel"],#onetrust-pc-sdk input[type="url"],#onetrust-pc-sdk input[type="password"],#onetrust-pc-sdk textarea,#ot-sdk-cookie-policy input[type="email"],#ot-sdk-cookie-policy input[type="number"],#ot-sdk-cookie-policy input[type="search"],#ot-sdk-cookie-policy input[type="text"],#ot-sdk-cookie-policy input[type="tel"],#ot-sdk-cookie-policy input[type="url"],#ot-sdk-cookie-policy input[type="password"],#ot-sdk-cookie-policy textarea{-webkit-appearance:none;-moz-appearance:none;appearance:none}#onetrust-banner-sdk textarea,#onetrust-pc-sdk textarea,#ot-sdk-cookie-policy textarea{min-height:65px;padding-top:6px;padding-bottom:6px}#onetrust-banner-sdk input[type="email"]:focus,#onetrust-banner-sdk input[type="number"]:focus,#onetrust-banner-sdk input[type="search"]:focus,#onetrust-banner-sdk input[type="text"]:focus,#onetrust-banner-sdk input[type="tel"]:focus,#onetrust-banner-sdk input[type="url"]:focus,#onetrust-banner-sdk input[type="password"]:focus,#onetrust-banner-sdk textarea:focus,#onetrust-banner-sdk select:focus,#onetrust-pc-sdk input[type="email"]:focus,#onetrust-pc-sdk input[type="number"]:focus,#onetrust-pc-sdk input[type="search"]:focus,#onetrust-pc-sdk input[type="text"]:focus,#onetrust-pc-sdk input[type="tel"]:focus,#onetrust-pc-sdk input[type="url"]:focus,#onetrust-pc-sdk input[type="password"]:focus,#onetrust-pc-sdk textarea:focus,#onetrust-pc-sdk select:focus,#ot-sdk-cookie-policy input[type="email"]:focus,#ot-sdk-cookie-policy input[type="number"]:focus,#ot-sdk-cookie-policy input[type="search"]:focus,#ot-sdk-cookie-policy input[type="text"]:focus,#ot-sdk-cookie-policy input[type="tel"]:focus,#ot-sdk-cookie-policy input[type="url"]:focus,#ot-sdk-cookie-policy input[type="password"]:focus,#ot-sdk-cookie-policy textarea:focus,#ot-sdk-cookie-policy select:focus{border:1px solid #33c3f0;outline:0}#onetrust-banner-sdk label,#onetrust-banner-sdk legend,#onetrust-pc-sdk label,#onetrust-pc-sdk legend,#ot-sdk-cookie-policy label,#ot-sdk-cookie-policy legend{display:block;margin-bottom:0.5rem;font-weight:600}#onetrust-banner-sdk fieldset,#onetrust-pc-sdk fieldset,#ot-sdk-cookie-policy fieldset{padding:0;border-width:0}#onetrust-banner-sdk input[type="checkbox"],#onetrust-banner-sdk input[type="radio"],#onetrust-pc-sdk input[type="checkbox"],#onetrust-pc-sdk input[type="radio"],#ot-sdk-cookie-policy input[type="checkbox"],#ot-sdk-cookie-policy input[type="radio"]{display:inline}#onetrust-banner-sdk label>.label-body,#onetrust-pc-sdk label>.label-body,#ot-sdk-cookie-policy label>.label-body{display:inline-block;margin-left:0.5rem;font-weight:normal}#onetrust-banner-sdk ul,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ul{list-style:circle inside}#onetrust-banner-sdk ol,#onetrust-pc-sdk ol,#ot-sdk-cookie-policy ol{list-style:decimal inside}#onetrust-banner-sdk ol,#onetrust-banner-sdk ul,#onetrust-pc-sdk ol,#onetrust-pc-sdk ul,#ot-sdk-cookie-policy ol,#ot-sdk-cookie-policy ul{padding-left:0;margin-top:0}#onetrust-banner-sdk ul ul,#onetrust-banner-sdk ul ol,#onetrust-banner-sdk ol ol,#onetrust-banner-sdk ol ul,#onetrust-pc-sdk ul ul,#onetrust-pc-sdk ul ol,#onetrust-pc-sdk ol ol,#onetrust-pc-sdk ol ul,#ot-sdk-cookie-policy ul ul,#ot-sdk-cookie-policy ul ol,#ot-sdk-cookie-policy ol ol,#ot-sdk-cookie-policy ol ul{margin:1.5rem 0 1.5rem 3rem;font-size:90%}#onetrust-banner-sdk li,#onetrust-pc-sdk li,#ot-sdk-cookie-policy li{margin-bottom:1rem}#onetrust-banner-sdk code,#onetrust-pc-sdk code,#ot-sdk-cookie-policy code{padding:0.2rem 0.5rem;margin:0 0.2rem;font-size:90%;white-space:nowrap;background:#f1f1f1;border:1px solid #e1e1e1;border-radius:4px}#onetrust-banner-sdk pre>code,#onetrust-pc-sdk pre>code,#ot-sdk-cookie-policy pre>code{display:block;padding:1rem 1.5rem;white-space:pre}#onetrust-banner-sdk th,#onetrust-banner-sdk td,#onetrust-pc-sdk th,#onetrust-pc-sdk td,#ot-sdk-cookie-policy th,#ot-sdk-cookie-policy td{padding:12px 15px;text-align:left;border-bottom:1px solid #e1e1e1}#onetrust-banner-sdk .ot-sdk-u-full-width,#onetrust-pc-sdk .ot-sdk-u-full-width,#ot-sdk-cookie-policy .ot-sdk-u-full-width{width:100%;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-u-max-full-width,#onetrust-pc-sdk .ot-sdk-u-max-full-width,#ot-sdk-cookie-policy .ot-sdk-u-max-full-width{max-width:100%;box-sizing:border-box}#onetrust-banner-sdk .ot-sdk-u-pull-right,#onetrust-pc-sdk .ot-sdk-u-pull-right,#ot-sdk-cookie-policy .ot-sdk-u-pull-right{float:right}#onetrust-banner-sdk .ot-sdk-u-pull-left,#onetrust-pc-sdk .ot-sdk-u-pull-left,#ot-sdk-cookie-policy .ot-sdk-u-pull-left{float:left}#onetrust-banner-sdk hr,#onetrust-pc-sdk hr,#ot-sdk-cookie-policy hr{margin-top:3rem;margin-bottom:3.5rem;border-width:0;border-top:1px solid #e1e1e1}#onetrust-banner-sdk .ot-sdk-container:after,#onetrust-banner-sdk .ot-sdk-row:after,#onetrust-banner-sdk .ot-sdk-u-cf,#onetrust-pc-sdk .ot-sdk-container:after,#onetrust-pc-sdk .ot-sdk-row:after,#onetrust-pc-sdk .ot-sdk-u-cf,#ot-sdk-cookie-policy .ot-sdk-container:after,#ot-sdk-cookie-policy .ot-sdk-row:after,#ot-sdk-cookie-policy .ot-sdk-u-cf{content:"";display:table;clear:both}#onetrust-banner-sdk .ot-sdk-row,#onetrust-pc-sdk .ot-sdk-row,#ot-sdk-cookie-policy .ot-sdk-row{margin:0;max-width:none;display:block}
#onetrust-banner-sdk{box-shadow:0 0 18px rgba(0,0,0,.2)}#onetrust-banner-sdk.otFlat{position:fixed;z-index:2147483645;bottom:0;right:0;left:0;background-color:#fff;max-height:90%;overflow-x:hidden;overflow-y:auto}#onetrust-banner-sdk>.ot-sdk-container{overflow:hidden}#onetrust-banner-sdk::-webkit-scrollbar{width:11px}#onetrust-banner-sdk::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-banner-sdk{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-banner-sdk #onetrust-policy{margin:1.25em 0 .625em 2em;overflow:hidden}#onetrust-banner-sdk #onetrust-policy-title{font-size:1.2em;line-height:1.3;margin-bottom:10px}#onetrust-banner-sdk #onetrust-policy-text{clear:both;text-align:left;font-size:.88em;line-height:1.4}#onetrust-banner-sdk #onetrust-policy-text *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk #onetrust-policy-text a{font-weight:bold;margin-left:5px}#onetrust-banner-sdk #onetrust-policy-title,#onetrust-banner-sdk #onetrust-policy-text{color:dimgray;float:left}#onetrust-banner-sdk #onetrust-button-group-parent{min-height:1px;text-align:center}#onetrust-banner-sdk #onetrust-button-group{display:inline-block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{background-color:#68b631;color:#fff;border-color:#68b631;margin-right:1em;min-width:125px;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:12px 10px;line-height:1.2;font-size:.813em;font-weight:600}#onetrust-banner-sdk #onetrust-pc-btn-handler.cookie-setting-link{background-color:#fff;border:none;color:#68b631;text-decoration:underline;padding-right:0}#onetrust-banner-sdk #onetrust-close-btn-container{text-align:center}#onetrust-banner-sdk .onetrust-close-btn-ui{width:.8em;height:18px;margin:50% 0 0 50%;border:none}#onetrust-banner-sdk .onetrust-close-btn-ui.onetrust-lg{top:50%;margin:auto;transform:translate(-50%, -50%);position:absolute;padding:0}#onetrust-banner-sdk .banner_logo{display:none}#onetrust-banner-sdk .ot-b-addl-desc{clear:both;float:left;display:block}#onetrust-banner-sdk #banner-options{float:left;display:table;margin-right:0;margin-left:1em;width:calc(100% - 1em)}#onetrust-banner-sdk #banner-options label{margin:0;display:inline-block}#onetrust-banner-sdk .banner-option{margin-bottom:12px;border:none;float:left;padding:0}#onetrust-banner-sdk .banner-option:not(:first-child){padding:0;border:none}#onetrust-banner-sdk .banner-option-input{position:absolute;cursor:pointer;width:auto;height:20px;opacity:0}#onetrust-banner-sdk .banner-option-header{margin-bottom:6px;cursor:pointer;display:inline-block}#onetrust-banner-sdk .banner-option-header :first-child{font-size:.82em;line-height:1.4;color:dimgray;font-weight:bold;float:left}#onetrust-banner-sdk .banner-option-header .ot-arrow-container{display:inline-block;border-top:6px solid transparent;border-bottom:6px solid transparent;border-left:6px solid dimgray;margin-left:10px;margin-top:2px}#onetrust-banner-sdk .banner-option-details{display:none;font-size:.83em;line-height:1.5;padding:10px 0px 5px 10px;margin-right:10px;height:0px}#onetrust-banner-sdk .banner-option-details *{font-size:inherit;line-height:inherit;color:dimgray}#onetrust-banner-sdk .ot-arrow-container,#onetrust-banner-sdk .banner-option-details{transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-banner-sdk .banner-option-input:checked~label .banner-option-header .ot-arrow-container{transform:rotate(90deg)}#onetrust-banner-sdk .banner-option-input:checked~.banner-option-details{height:auto;display:block}#onetrust-banner-sdk .ot-dpd-container{float:left}#onetrust-banner-sdk .ot-dpd-title{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-title,#onetrust-banner-sdk .ot-dpd-desc{font-size:.88em;line-height:1.4;color:dimgray}#onetrust-banner-sdk .ot-dpd-title *,#onetrust-banner-sdk .ot-dpd-desc *{font-size:inherit;line-height:inherit}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text *{margin-bottom:0}#onetrust-banner-sdk.ot-iab-2 .onetrust-vendors-list-handler{display:block;margin-left:0;margin-top:5px;clear:both}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk #onetrust-policy-text,#onetrust-banner-sdk .ot-dpd-desc,#onetrust-banner-sdk .ot-b-addl-desc{font-size:.813em;line-height:1.5}#onetrust-banner-sdk .ot-dpd-desc{margin-bottom:10px}#onetrust-banner-sdk .ot-dpd-desc>.ot-b-addl-desc{margin-top:10px;margin-bottom:10px;font-size:1em}@media only screen and (max-width: 425px){#onetrust-banner-sdk #onetrust-policy{margin-left:0}#onetrust-banner-sdk .ot-hide-small{display:none}#onetrust-banner-sdk #onetrust-button-group{display:block}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{width:100%}#onetrust-banner-sdk .onetrust-close-btn-ui{margin:5px 0 0 0;float:right;padding:0}#onetrust-banner-sdk #onetrust-close-btn-container-mobile,#onetrust-banner-sdk #onetrust-policy-title{display:inline;float:none}#onetrust-banner-sdk #banner-options{margin:0;padding:0;width:100%}}@media only screen and (min-width: 426px)and (max-width: 896px){#onetrust-banner-sdk #onetrust-policy{margin-left:1em;margin-right:1em}#onetrust-banner-sdk .onetrust-close-btn-ui.onetrust-lg{top:25%;right:2%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:95%}#onetrust-banner-sdk.ot-iab-2 #onetrust-group-container{width:100%}#onetrust-banner-sdk #onetrust-button-group-parent{width:100%;position:relative;margin-left:0}#onetrust-banner-sdk .ot-hide-large{display:none}#onetrust-banner-sdk #onetrust-button-group button{display:inline-block}#onetrust-banner-sdk #onetrust-button-group{margin-right:0;text-align:center}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler{float:left}#onetrust-banner-sdk .has-reject-all-button #onetrust-reject-all-handler,#onetrust-banner-sdk .has-reject-all-button #onetrust-accept-btn-handler{float:right}#onetrust-banner-sdk .has-reject-all-button #onetrust-button-group{width:calc(100% - 2em);margin-right:0}#onetrust-banner-sdk .has-reject-all-button #onetrust-pc-btn-handler.cookie-setting-link{padding-left:0px;text-align:left}#onetrust-banner-sdk.ot-buttons-fw .ot-sdk-three button{width:100%;text-align:center}#onetrust-banner-sdk.ot-buttons-fw #onetrust-button-group-parent button{float:none}#onetrust-banner-sdk.ot-buttons-fw #onetrust-pc-btn-handler.cookie-setting-link{text-align:center}}@media only screen and (min-width: 550px){#onetrust-banner-sdk .banner-option:not(:first-child){border-left:1px solid #d8d8d8;padding-left:25px}}@media only screen and (min-width: 425px)and (max-width: 550px){#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group,#onetrust-banner-sdk.ot-iab-2 #onetrust-policy,#onetrust-banner-sdk.ot-iab-2 .banner-option{width:100%}}@media only screen and (min-width: 769px){#onetrust-banner-sdk .ot-hide-large{display:none}#onetrust-banner-sdk #onetrust-button-group{margin-right:30%}#onetrust-banner-sdk #banner-options{margin-left:2em;margin-right:5em;margin-bottom:1.25em;width:calc(100% - 7em)}#onetrust-banner-sdk .banner-option{float:none;display:table-cell}}@media only screen and (min-width: 1024px){#onetrust-banner-sdk #onetrust-policy{margin-left:2em}#onetrust-banner-sdk.vertical-align-content #onetrust-button-group-parent{position:absolute;top:50%;left:60%;transform:translateY(-50%)}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-title{width:50%}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text,#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:1em;width:50%;border-right:1px solid #d8d8d8;padding-right:1rem}#onetrust-banner-sdk.ot-iab-2 #onetrust-policy-text{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 :not(.ot-dpd-desc)>.ot-b-addl-desc{margin-bottom:0;padding-bottom:1em}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-container{width:45%;padding-left:1rem;display:inline-block;float:none}#onetrust-banner-sdk.ot-iab-2 .ot-dpd-title{line-height:1.7}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group-parent{left:auto;right:4%;margin-left:0}#onetrust-banner-sdk.ot-iab-2 #onetrust-button-group button{display:block}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{margin:auto;width:36%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:60%}#onetrust-banner-sdk #onetrust-button-group{margin-right:auto}#onetrust-banner-sdk #onetrust-close-btn-container{float:right}#onetrust-banner-sdk #onetrust-accept-btn-handler,#onetrust-banner-sdk #onetrust-reject-all-handler,#onetrust-banner-sdk #onetrust-pc-btn-handler{margin-top:1em}}@media only screen and (min-width: 890px){#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group-parent{padding-left:4%;margin-left:0}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group{margin-right:0;margin-top:1.25em;width:100%}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button{width:100%;margin-bottom:5px;margin-top:5px}#onetrust-banner-sdk.ot-buttons-fw:not(.ot-iab-2) #onetrust-button-group button:last-of-type{margin-bottom:20px}}@media only screen and (min-width: 1280px){#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-group-container{width:55%}#onetrust-banner-sdk:not(.ot-iab-2) #onetrust-button-group-parent{width:44%;padding-left:2%;padding-right:2%}#onetrust-banner-sdk:not(.ot-iab-2).vertical-align-content #onetrust-button-group-parent{position:absolute;left:55%}}
        #onetrust-consent-sdk #onetrust-banner-sdk {background-color: #486A7E;}
            #onetrust-consent-sdk #onetrust-policy-title,
                    #onetrust-consent-sdk #onetrust-policy-text,
                    #onetrust-consent-sdk .ot-b-addl-desc,
                    #onetrust-consent-sdk .ot-dpd-desc,
                    #onetrust-consent-sdk .ot-dpd-title,
                    #onetrust-consent-sdk #onetrust-policy-text *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk .ot-dpd-desc *:not(.onetrust-vendors-list-handler),
                    #onetrust-consent-sdk #onetrust-banner-sdk #banner-options * {
                        color: #FFFFFF;
                    }
            #onetrust-consent-sdk #onetrust-banner-sdk .banner-option-details {
                    background-color: #E9E9E9;}
            #onetrust-consent-sdk #onetrust-accept-btn-handler,
                         #onetrust-banner-sdk #onetrust-reject-all-handler {
                            background-color: #ffffff;border-color: #ffffff;
                            color: #222;
                        }#onetrust-consent-sdk #onetrust-pc-btn-handler.cookie-setting-link {
                            border-color: #486A7E;
                            background-color: #486A7E;
                            color: #ffffff
                        }#onetrust-consent-sdk #onetrust-pc-btn-handler {
                      color: #ffffff; border-color: #ffffff;
        background-color: #486A7E;
  }/* scrap ot grid column layout */
#onetrust-banner-sdk .ot-sdk-columns {
  width: auto !important;
  margin: 0 !important;
}

#onetrust-banner-sdk .ot-sdk-container {
    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif !important;
  width: auto !important;
  max-width: 1280px !important;
  margin: auto !important;
  padding: 16px !important;
}

#onetrust-banner-sdk #onetrust-policy {
  margin: auto !important;
}

#onetrust-banner-sdk #onetrust-group-container {
  width: auto !important;
}

#onetrust-banner-sdk #onetrust-policy-title {
  display: inline-block !important;
  font-size: 24px !important;
  font-size: min(max(24px, 4vw), 36px) !important;   
  margin-bottom: 16px !important;
  margin-bottom: 0.25em !important;
}

#onetrust-banner-sdk #onetrust-policy-text {
    font-size: 18px !important;
    font-size: min(max(14px, 3vw), 18px) !important;
}

#onetrust-banner-sdk #onetrust-button-group-parent {
  text-align: left !important;
  position: static !important;
  -webkit-transform: translate(0) !important;
      -ms-transform: translate(0) !important;
          transform: translate(0) !important;
}

#onetrust-banner-sdk #onetrust-button-group {
  margin: auto !important;
}

#onetrust-banner-sdk #onetrust-button-group > button {
  font-size: 16px !important;
  font-size: min(max(14px, 2.5vw), 16px) !important;
  padding: 1em 1.2em !important;
  margin: 0 0 1em !important;
font-weight: bold;
}

#onetrust-banner-sdk #onetrust-button-group > button:focus { 
  outline: 4px solid #fc0 !important; 
  opacity: 1 !important; 
}

#onetrust-banner-sdk #onetrust-close-btn-container {
  position: absolute !important;
  top: 16px !important;
  right: 16px !important;
}

/* Media Queries */
@media only screen and (min-width: 426px) {
  #onetrust-banner-sdk #onetrust-button-group > button + button {
    margin-left: 1em !important;
  }
}

@media screen and (min-width: 550px) {
   #onetrust-banner-sdk .ot-sdk-container {
    padding: 32px 16px !important;
  }
}

@media screen and (min-width: 1024px) {
  #onetrust-banner-sdk .ot-sdk-container {
    padding: 80px 16px !important;
  }
  
  /* if flex not supported, stacking is fine */
  #onetrust-banner-sdk .ot-sdk-container .ot-sdk-row {
    display: -webkit-box !important;
    display: -ms-flexbox !important;
    display: flex !important;
    -webkit-box-pack: justify !important;
        -ms-flex-pack: justify !important;
            justify-content: space-between !important;
  }


  #onetrust-banner-sdk .ot-sdk-container .ot-sdk-row > * {
    -webkit-box-flex: 0 !important;
        -ms-flex: 0 1 auto !important;
            flex: 0 1 auto !important;
  }

  #onetrust-banner-sdk #onetrust-button-group-parent {
    -ms-flex-negative: 0 !important;
        flex-shrink: 0 !important;
    -ms-flex-item-align: end !important;
        align-self: flex-end !important;
    padding-left: 48px !important;
  }
}
#onetrust-pc-sdk.otPcTab{position:fixed;width:730px;max-width:730px;height:580px;left:0;right:0;top:0;bottom:0;margin:auto;font-size:16px;z-index:2147483647;border:1px solid #d7d7d7;border-radius:2px;background-color:#fff;box-shadow:0 2px 4px 0 rgba(0,0,0,0),0 7px 14px 0 rgba(50,50,93,.1)}#onetrust-pc-sdk *,#onetrust-pc-sdk ::after,#onetrust-pc-sdk ::before{box-sizing:content-box}#onetrust-pc-sdk #ot-content{height:calc(100% - 196px)}#onetrust-pc-sdk .ot-main-content{height:100%}#onetrust-pc-sdk .privacy-notice-link{text-decoration:underline}#onetrust-pc-sdk .privacy-notice-link:focus{outline:1px solid #63a3f8}#onetrust-pc-sdk li{list-style:none}#onetrust-pc-sdk ul,#onetrust-pc-sdk li{margin:0}#onetrust-pc-sdk a{text-decoration:none}#onetrust-pc-sdk .pc-content::-webkit-scrollbar,#onetrust-pc-sdk .groups-container::-webkit-scrollbar,#onetrust-pc-sdk #vendor-list-content::-webkit-scrollbar,#onetrust-pc-sdk .vendor-content::-webkit-scrollbar,#onetrust-pc-sdk #ot-filter-modal .ot-group-options::-webkit-scrollbar{width:11px}#onetrust-pc-sdk .pc-content::-webkit-scrollbar-thumb,#onetrust-pc-sdk .groups-container::-webkit-scrollbar-thumb,#onetrust-pc-sdk #vendor-list-content::-webkit-scrollbar-thumb,#onetrust-pc-sdk .vendor-content::-webkit-scrollbar-thumb,#onetrust-pc-sdk #ot-filter-modal .ot-group-options::-webkit-scrollbar-thumb{border-radius:10px;background:#c1c1c1}#onetrust-pc-sdk .pc-content,#onetrust-pc-sdk .groups-container,#onetrust-pc-sdk #vendor-list-content,#onetrust-pc-sdk .vendor-content,#onetrust-pc-sdk #ot-filter-modal .ot-group-options{scrollbar-arrow-color:#c1c1c1;scrollbar-darkshadow-color:#c1c1c1;scrollbar-face-color:#c1c1c1;scrollbar-shadow-color:#c1c1c1}#onetrust-pc-sdk .pc-header{height:auto;padding:20px;display:inline-block;vertical-align:middle;width:calc(100% - 40px);border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk .pc-logo-container{float:left}#onetrust-pc-sdk .pc-title-container{float:right;width:calc(100% - 190px);margin-left:10px}#onetrust-pc-sdk .ot-always-active{font-size:1em;line-height:1.3;color:#3860be}#onetrust-pc-sdk .pc-close-btn-container{float:right;position:absolute;right:15px;margin-top:18px}#onetrust-pc-sdk .pc-content{position:relative;height:100%;overflow-y:auto;overflow-x:hidden}#onetrust-pc-sdk .groups-container{height:100%;overflow-y:auto}#onetrust-pc-sdk .group{cursor:pointer;border-left:10px solid transparent;background-color:#f4f4f4;border-bottom:1px solid #d7d7d7;padding-top:16px;padding-right:5px;padding-bottom:16px;padding-left:12px}#onetrust-pc-sdk .group h3{float:left;text-align:left;margin:0;color:dimgray;line-height:1.4;font-size:1em;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .active-group{border-left:10px solid #68b631;background-color:#fff;border-bottom:none;position:relative}#onetrust-pc-sdk .active-group h3{color:#263238;font-weight:bold}#onetrust-pc-sdk .description-container{word-break:break-word;word-wrap:break-word;padding-top:20px;padding-right:16px;padding-bottom:15px;position:absolute;left:30%}#onetrust-pc-sdk .group-description{word-break:break-word;word-wrap:break-word;text-align:left;font-size:.82em;line-height:1.3;margin:0}#onetrust-pc-sdk #pc-policy-text{clear:both;float:left}#onetrust-pc-sdk #pc-policy-text a{color:#3860be;cursor:pointer;font-size:1em}#onetrust-pc-sdk #pc-policy-text a:hover{color:#1883fd}#onetrust-pc-sdk #pc-policy-text *{font-size:inherit}#onetrust-pc-sdk #pc-policy-text ul li{padding:10px 0px;border-bottom:1px solid #e2e2e2}#onetrust-pc-sdk .ot-close-icon{background-image:url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB4PSIwcHgiIHk9IjBweCIgd2lkdGg9IjQ3Ljk3MSIgaGVpZ2h0PSI0Ny45NzEiIHZpZXdCb3g9IjAgMCA0Ny45NzEgNDcuOTcxIiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IDAgMCA0Ny45NzEgNDcuOTcxOyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+PGc+PHBhdGggZD0iTTI4LjIyOCwyMy45ODZMNDcuMDkyLDUuMTIyYzEuMTcyLTEuMTcxLDEuMTcyLTMuMDcxLDAtNC4yNDJjLTEuMTcyLTEuMTcyLTMuMDctMS4xNzItNC4yNDIsMEwyMy45ODYsMTkuNzQ0TDUuMTIxLDAuODhjLTEuMTcyLTEuMTcyLTMuMDctMS4xNzItNC4yNDIsMGMtMS4xNzIsMS4xNzEtMS4xNzIsMy4wNzEsMCw0LjI0MmwxOC44NjUsMTguODY0TDAuODc5LDQyLjg1Yy0xLjE3MiwxLjE3MS0xLjE3MiwzLjA3MSwwLDQuMjQyQzEuNDY1LDQ3LjY3NywyLjIzMyw0Ny45NywzLDQ3Ljk3czEuNTM1LTAuMjkzLDIuMTIxLTAuODc5bDE4Ljg2NS0xOC44NjRMNDIuODUsNDcuMDkxYzAuNTg2LDAuNTg2LDEuMzU0LDAuODc5LDIuMTIxLDAuODc5czEuNTM1LTAuMjkzLDIuMTIxLTAuODc5YzEuMTcyLTEuMTcxLDEuMTcyLTMuMDcxLDAtNC4yNDJMMjguMjI4LDIzLjk4NnoiLz48L2c+PC9zdmc+");background-size:100%;background-repeat:no-repeat;background-position:center;height:22px;width:22px;display:inline-block}#onetrust-pc-sdk .group-toggle{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .group-toggle .checkbox{background-color:#fff}#onetrust-pc-sdk .group-toggle .ot-toggle{float:right;background-color:#fff}#onetrust-pc-sdk .group-toggle .ot-toggle-group{float:right}#onetrust-pc-sdk .group-toggle .ot-toggle-group p{margin:0}#onetrust-pc-sdk .group-toggle .ot-toggle-group:focus{outline:#3b99fc auto 5px}#onetrust-pc-sdk .group-toggle .ot-toggle-group .checkbox input[type=checkbox]:focus+label::before{outline:#3b99fc auto 5px}#onetrust-pc-sdk .description-container h3{color:#263238;float:left;margin:0;font-weight:bold;font-size:1em;line-height:1.3;max-width:85%;top:3px}#onetrust-pc-sdk #pc-privacy-header{margin-bottom:10px}#onetrust-pc-sdk .category-vendors-list-container{overflow:hidden}#onetrust-pc-sdk .category-vendors-list-handler,#onetrust-pc-sdk .category-host-list-handler,#onetrust-pc-sdk .category-vendors-list-handler+a{display:block;float:left;color:#3860be;font-size:.8em;font-weight:bold;line-height:1.1;cursor:pointer}#onetrust-pc-sdk .category-vendors-list-handler:hover,#onetrust-pc-sdk .category-host-list-handler:hover,#onetrust-pc-sdk .category-vendors-list-handler+a:hover{color:#1883fd}#onetrust-pc-sdk .category-vendors-list-handler+a::after{content:"";height:15px;width:15px;background-repeat:no-repeat;margin-left:5px;float:right;background-image:url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 511.626 511.627'%3E%3Cg fill='%231276CE'%3E%3Cpath d='M392.857 292.354h-18.274c-2.669 0-4.859.855-6.563 2.573-1.718 1.708-2.573 3.897-2.573 6.563v91.361c0 12.563-4.47 23.315-13.415 32.262-8.945 8.945-19.701 13.414-32.264 13.414H82.224c-12.562 0-23.317-4.469-32.264-13.414-8.945-8.946-13.417-19.698-13.417-32.262V155.31c0-12.562 4.471-23.313 13.417-32.259 8.947-8.947 19.702-13.418 32.264-13.418h200.994c2.669 0 4.859-.859 6.57-2.57 1.711-1.713 2.566-3.9 2.566-6.567V82.221c0-2.662-.855-4.853-2.566-6.563-1.711-1.713-3.901-2.568-6.57-2.568H82.224c-22.648 0-42.016 8.042-58.102 24.125C8.042 113.297 0 132.665 0 155.313v237.542c0 22.647 8.042 42.018 24.123 58.095 16.086 16.084 35.454 24.13 58.102 24.13h237.543c22.647 0 42.017-8.046 58.101-24.13 16.085-16.077 24.127-35.447 24.127-58.095v-91.358c0-2.669-.856-4.859-2.574-6.57-1.713-1.718-3.903-2.573-6.565-2.573z'/%3E%3Cpath d='M506.199 41.971c-3.617-3.617-7.905-5.424-12.85-5.424H347.171c-4.948 0-9.233 1.807-12.847 5.424-3.617 3.615-5.428 7.898-5.428 12.847s1.811 9.233 5.428 12.85l50.247 50.248-186.147 186.151c-1.906 1.903-2.856 4.093-2.856 6.563 0 2.479.953 4.668 2.856 6.571l32.548 32.544c1.903 1.903 4.093 2.852 6.567 2.852s4.665-.948 6.567-2.852l186.148-186.148 50.251 50.248c3.614 3.617 7.898 5.426 12.847 5.426s9.233-1.809 12.851-5.426c3.617-3.616 5.424-7.898 5.424-12.847V54.818c-.001-4.952-1.814-9.232-5.428-12.847z'/%3E%3C/g%3E%3C/svg%3E")}#onetrust-pc-sdk .category-host-list-handler,#onetrust-pc-sdk .category-vendors-list-container,#onetrust-pc-sdk #pc-policy-text+.category-vendors-list-handler{margin-top:10px}#onetrust-pc-sdk .group-toggle+.category-vendors-list-container{margin-top:0px;margin-bottom:10px}#onetrust-pc-sdk .ot-always-active-group .group-toggle h3.category-header,#onetrust-pc-sdk .cookie-subgroup.ot-always-active-group>h6{max-width:70%}#onetrust-pc-sdk .group-description ul,#onetrust-pc-sdk .cookie-subgroups-container li.cookie-subgroup p ul{margin:0px;margin-left:15px;padding-bottom:8px}#onetrust-pc-sdk .group-description ul li,#onetrust-pc-sdk .cookie-subgroups-container li.cookie-subgroup p ul li{font-size:inherit;padding-top:8px;display:list-item;list-style:disc}#onetrust-pc-sdk .ot-toggle.ot-hide-tgl{visibility:hidden}#onetrust-pc-sdk .ot-toggle.ot-hide-tgl *{visibility:hidden}#onetrust-pc-sdk .cookie-subgroups-container{margin-top:15px}#onetrust-pc-sdk .cookie-subgroups-container ul.cookie-subgroups{margin:0;font-size:inherit}#onetrust-pc-sdk .cookie-subgroups-container ul.cookie-subgroups .cookie-subgroup-toggle{float:right}#onetrust-pc-sdk .cookie-subgroups-container ul.cookie-subgroups .cookie-subgroup-toggle.ot-always-active-subgroup{width:auto}#onetrust-pc-sdk .cookie-subgroups-container ul.cookie-subgroups .ot-toggle{background-color:inherit}#onetrust-pc-sdk .cookie-subgroups-container ul.cookie-subgroups li{padding:0;border:none;position:relative}#onetrust-pc-sdk .cookie-subgroups-container ul.cookie-subgroups li h6,#onetrust-pc-sdk .cookie-subgroups-container ul.cookie-subgroups li p{font-size:.82em;line-height:1.4}#onetrust-pc-sdk .cookie-subgroups-container ul.cookie-subgroups li p{color:dimgray;clear:both;float:left;margin-top:10px;margin-bottom:0;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .cookie-subgroups-container ul.cookie-subgroups li h6{color:#263238;font-weight:bold;margin-bottom:0;float:left;position:relative;top:3px}#onetrust-pc-sdk .cookie-subgroups-container ul.cookie-subgroups li.cookie-subgroup{margin-left:30px;display:inline-block;width:calc(100% - 30px)}#onetrust-pc-sdk .cookie-subgroups-container .ot-arrow{top:-3px}#onetrust-pc-sdk .ot-button-group-parent{height:34px;padding:30px;bottom:0;border-top:1px solid #d8d8d8}#onetrust-pc-sdk .ot-button-group{float:right;text-align:center}#onetrust-pc-sdk .ot-pc-footer-logo{float:left;text-align:center;background-repeat:no-repeat;background-position:center;background-size:contain;width:155px;height:25px;position:relative;top:10px}#onetrust-pc-sdk #accept-recommended-btn-handler{float:right;margin-right:16px;margin-left:15px;text-align:center}#onetrust-pc-sdk .save-preference-btn-handler{height:38px;min-width:155px;background-color:#68b631;border-radius:2px;color:#fff;font-size:.9em;line-height:1.1;text-align:center;margin-left:15px;margin-right:15px}#onetrust-pc-sdk #pc-title{float:left;margin-left:10px;margin-top:10px;max-width:85%;overflow:hidden;position:relative;line-height:1.2em;max-height:2.4em;padding-right:1em}#onetrust-pc-sdk #pc-title:before{content:"...";right:0px;bottom:0px;position:absolute}#onetrust-pc-sdk #pc-title:after{position:absolute;content:"";width:1em;height:1em;right:0px;background:#fff}#onetrust-pc-sdk #vendors-list{height:100%;width:100%}#onetrust-pc-sdk #vendors-list .accordion-header{padding-top:17px;padding-right:15px;padding-bottom:17px;padding-left:20px;display:inline-block;width:calc(100% - 35px);vertical-align:middle}#onetrust-pc-sdk #vendors-list .accordion-text{padding-top:6px;padding-right:50px;padding-bottom:10px;padding-left:50px}#onetrust-pc-sdk .vendor-content{height:100%}#onetrust-pc-sdk #vendors-list-header{padding-top:15px;padding-right:30px;padding-bottom:15px;padding-left:20px;display:inline-block;width:calc(100% - 50px);height:20px;border-bottom:1px solid #d8d8d8}#onetrust-pc-sdk #vendors-list-header input{border:1px solid #d7d7d7;height:32px;width:300px;border-radius:50px;font-size:.8em;padding-right:35px;padding-left:15px;float:left}#onetrust-pc-sdk #vendors-list-header input::placeholder{color:#d4d4d4;font-style:italic}#onetrust-pc-sdk #vendor-list-content{height:calc(100% - 86px);width:calc(100% - 183px);padding-left:80px;padding-right:100px;padding-top:20px;margin-top:8px;margin-right:3px;margin-bottom:4px;margin-left:0;overflow-x:hidden;overflow-y:auto}#onetrust-pc-sdk #ot-back-arrow{height:17px;width:20px;vertical-align:middle;display:inline-block;color:dimgray;font-size:1em;line-height:1;text-align:center}#onetrust-pc-sdk #vendors-list-title{color:dimgray;font-size:.9em;font-weight:bold;line-height:1;display:inline-block;vertical-align:middle;margin:0;padding-left:5px}#onetrust-pc-sdk #vendors-list-title *{font-size:inherit}#onetrust-pc-sdk .navigation-container{float:left}#onetrust-pc-sdk .back-btn-handler{display:inline-block}#onetrust-pc-sdk .action-container{float:right;position:relative;bottom:6px}#onetrust-pc-sdk #search-container{float:left}#onetrust-pc-sdk #search-container svg{position:absolute;right:35px;width:30px;height:30px;font-size:1em;line-height:1;top:3px}#onetrust-pc-sdk .label-text{display:none}#onetrust-pc-sdk #filter-container{display:inline-block;top:8px;position:relative;margin-left:20px}#onetrust-pc-sdk #filter-btn-handler svg{cursor:pointer;width:15px;height:15px}#onetrust-pc-sdk #select-all-container{min-width:200px;min-height:30px;padding-left:20px}#onetrust-pc-sdk #select-all-text-container{display:inline-block;padding-left:30px}#onetrust-pc-sdk #select-all-text-container p{color:#2c3643;font-size:.8em;font-weight:bold;line-height:1.1;margin:0;display:inline-block}#onetrust-pc-sdk #select-all-text-container p *{font-size:inherit}#onetrust-pc-sdk #select-all-vendors-input-container,#onetrust-pc-sdk #select-all-hosts-input-container{float:left;height:100%}#onetrust-pc-sdk #select-all-vendors-input-container label,#onetrust-pc-sdk #select-all-hosts-input-container label{height:100%;padding-left:0}#onetrust-pc-sdk .ot-checkbox input[type=checkbox]{display:block;opacity:0}#onetrust-pc-sdk .ot-checkbox input:checked~label::before{background-color:#4285f4}#onetrust-pc-sdk .ot-checkbox label{position:relative;height:20px;padding-left:30px}#onetrust-pc-sdk .ot-checkbox label::before,#onetrust-pc-sdk .ot-checkbox label::after{position:absolute;content:"";display:inline-block;border-radius:3px}#onetrust-pc-sdk .ot-checkbox label::before{height:18px;width:18px;border:1px solid #3860be;left:0px}#onetrust-pc-sdk .ot-checkbox label::after{height:5px;width:9px;border-left:3px solid;border-bottom:3px solid;transform:rotate(-45deg);-o-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-webkit-transform:rotate(-45deg);left:4px;top:5px}#onetrust-pc-sdk .ot-checkbox input[type=checkbox]+label::after{content:none;color:#fff}#onetrust-pc-sdk .ot-checkbox input[type=checkbox]:checked+label::after{content:""}#onetrust-pc-sdk .ot-checkbox input[type=checkbox]:focus+label::before{outline-style:solid;outline-width:2px;outline-style:auto}#onetrust-pc-sdk #select-all-vendors-input-container.line-through label::after,#onetrust-pc-sdk #select-all-vendors-leg-input-container.line-through label::after,#onetrust-pc-sdk #select-all-hosts-input-container.line-through label::after{height:auto;border-left:0;left:5px;top:10.5px;transform:none;-o-transform:none;-ms-transform:none;-webkit-transform:none}#onetrust-pc-sdk .ot-checkbox{float:left}#onetrust-pc-sdk .vendor-title,#onetrust-pc-sdk .host-title{color:#2c3643;font-weight:bold;font-size:.8em;line-height:1.2;margin:0;height:auto;text-align:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .host-title{position:relative;vertical-align:middle}#onetrust-pc-sdk .host-title a{font-size:1em;font-weight:bold}#onetrust-pc-sdk .cookie-name-container a{font-size:1em}#onetrust-pc-sdk .host-description{font-size:.69em;line-height:1.4;margin-top:5px;margin-bottom:5px;float:left;color:dimgray}#onetrust-pc-sdk .host-info>a{text-decoration:underline;font-size:.82em;position:relative;z-index:2;float:left;width:100%;margin-bottom:5px}#onetrust-pc-sdk .host-info .host-title+a{margin-top:5px}#onetrust-pc-sdk .vendor-info{width:88%;float:right}#onetrust-pc-sdk .vendor-purposes{margin-top:3px}#onetrust-pc-sdk .vendor-purposes p{font-size:.8em;line-height:1.2;margin-bottom:0;font-weight:500;text-align:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk input[type=checkbox]:focus+.accordion-header{outline:auto;outline-color:#007bff}#onetrust-pc-sdk #select-all-hosts-input-container input[type=checkbox],#onetrust-pc-sdk #select-all-vendors-input-container input[type=checkbox]{position:absolute}#onetrust-pc-sdk #vendors-list-container input[type=checkbox],#onetrust-pc-sdk #hosts-list-container input[type=checkbox],#onetrust-pc-sdk .cookie-subgroups input[type=checkbox]{position:absolute;cursor:pointer;width:100%;height:100%;opacity:0;margin:0;top:0;left:0}#onetrust-pc-sdk #vendors-list-container input[type=checkbox]~.accordion-header,#onetrust-pc-sdk #hosts-list-container input[type=checkbox]~.accordion-header,#onetrust-pc-sdk .cookie-subgroups input[type=checkbox]~.accordion-header{cursor:pointer}#onetrust-pc-sdk #vendors-list-container input[type=checkbox]:not(:checked)~.accordion-text,#onetrust-pc-sdk #hosts-list-container input[type=checkbox]:not(:checked)~.accordion-text,#onetrust-pc-sdk .cookie-subgroups input[type=checkbox]:not(:checked)~.accordion-text{margin-top:0;max-height:0;opacity:0;overflow:hidden;width:100%;transition:.25s ease-out;display:none}#onetrust-pc-sdk #vendors-list-container input[type=checkbox]:checked~.accordion-text,#onetrust-pc-sdk #hosts-list-container input[type=checkbox]:checked~.accordion-text,#onetrust-pc-sdk .cookie-subgroups input[type=checkbox]:checked~.accordion-text{transition:.1s ease-in;display:block}#onetrust-pc-sdk #vendors-list-container .ot-checkbox input[type=checkbox],#onetrust-pc-sdk #hosts-list-container .ot-checkbox input[type=checkbox],#onetrust-pc-sdk .cookie-subgroups .ot-checkbox input[type=checkbox]{width:auto;height:auto}#onetrust-pc-sdk #vendors-list-container .ot-checkbox label,#onetrust-pc-sdk #hosts-list-container .ot-checkbox label,#onetrust-pc-sdk .cookie-subgroups .ot-checkbox label{z-index:2}#onetrust-pc-sdk #vendors-list-container,#onetrust-pc-sdk #hosts-list-container{width:100%}#onetrust-pc-sdk #vendors-list-container li,#onetrust-pc-sdk #hosts-list-container li{border:1px solid #d7d7d7;border-radius:2px;position:relative;margin-top:10px}#onetrust-pc-sdk #vendors-list-container .ot-toggle-group{width:65%}#onetrust-pc-sdk #hosts-list-container .ot-toggle-group{width:65%}#onetrust-pc-sdk label{margin-bottom:0}#onetrust-pc-sdk .vendor-notice,#onetrust-pc-sdk .host-notice{float:right}#onetrust-pc-sdk .vendor-privacy-notice-container,#onetrust-pc-sdk .third-party-cookies-container{float:left}#onetrust-pc-sdk .vendor-privacy-notice-container{position:relative}#onetrust-pc-sdk .vendor-privacy-notice-container a:hover{text-decoration:underline}#onetrust-pc-sdk .vendor-privacy-notice,#onetrust-pc-sdk .third-party-cookie-notice{color:dimgray;font-size:.7em;line-height:.9;display:inline-block}#onetrust-pc-sdk .vendor-privacy-notice *,#onetrust-pc-sdk .third-party-cookie-notice *{font-size:inherit}#onetrust-pc-sdk .vendor-privacy-notice{position:relative;z-index:2}#onetrust-pc-sdk .ot-arrow{margin-left:20px}#onetrust-pc-sdk .vendor-options{display:inline-block;width:100%;padding-bottom:10px;border-radius:2px;background-color:#f8f8f8}#onetrust-pc-sdk .vendor-options p{font-size:.69em;text-align:left;display:table-cell;vertical-align:middle;word-break:break-word;word-wrap:break-word;margin:0;padding-left:15px;color:#2e3644}#onetrust-pc-sdk #hosts-list-container .host-info{float:left;width:calc(100% - 50px)}#onetrust-pc-sdk #hosts-list-container .host-option-group{display:inline-block;width:100%;margin:0;font-size:inherit}#onetrust-pc-sdk #hosts-list-container .host-option-group li.vendor-host{border:none;font-size:.8em;color:dimgray;float:left;text-align:left;padding:10px;margin-bottom:10px;width:calc(100% - 10px);background-color:#f8f8f8}#onetrust-pc-sdk #hosts-list-container .host-option-group li.vendor-host a{color:dimgray}#onetrust-pc-sdk #hosts-list-container .host-option-group li>div div{font-size:.81em;padding:5px 0}#onetrust-pc-sdk #hosts-list-container .host-option-group li>div div:nth-child(1){width:30%;float:left}#onetrust-pc-sdk #hosts-list-container .host-option-group li>div div:nth-child(2){width:70%;float:left;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk .vendor-purpose-groups .vendor-option-purpose{border-top:1px solid #e9e9e9;border-bottom:1px solid #e9e9e9;margin-bottom:10px;min-height:30px;max-height:50px;width:100%;display:table}#onetrust-pc-sdk .vendor-purpose-groups .vendor-option-purpose:first-child,#onetrust-pc-sdk .vendor-purpose-groups .vendor-option-purpose:first-of-type{border-top:none}#onetrust-pc-sdk .vendor-purpose-groups .vendor-option-purpose p{font-weight:bold}#onetrust-pc-sdk .vendor-purpose-groups .vendor-consent-group{display:inline-block;width:100%;margin-bottom:10px}#onetrust-pc-sdk .vendor-purpose-groups .legitimate-interest-group .consent-category{float:left}#onetrust-pc-sdk .vendor-purpose-groups .vendor-opt-out-handler{text-decoration:none;float:right;font-size:.69em;line-height:1.4;color:#3860be;right:15px;position:relative}#onetrust-pc-sdk .vendor-purpose-groups .vendor-opt-out-handler span{font-size:.69em;line-height:1.4}#onetrust-pc-sdk .vendor-purpose-groups .vendor-opt-out-handler svg{width:15px;height:15px;vertical-align:middle}#onetrust-pc-sdk #no-results{text-align:center;margin-top:30px}#onetrust-pc-sdk #no-results p{font-size:1em;color:#2e3644;word-break:break-word;word-wrap:break-word}#onetrust-pc-sdk #no-results p span{font-weight:bold}#onetrust-pc-sdk .ot-toggle .checkbox label:after{cursor:pointer}#onetrust-pc-sdk .ot-group-option{height:25px}#onetrust-pc-sdk .checkbox{position:relative;display:inline-block}#onetrust-pc-sdk .checkbox label{background:#d6d6d6;border:none;height:10px;width:35px;position:relative;display:inline-block;border-radius:46px;transition:.4s}#onetrust-pc-sdk .checkbox label:after{content:"";position:absolute;width:20px;height:20px;border-radius:100%;left:0;top:-5px;z-index:2;background:#7b7b7b;transition:.4s}#onetrust-pc-sdk .checkbox input{position:absolute;left:0;top:0;width:100%;height:100%;z-index:5;opacity:0;cursor:pointer}#onetrust-pc-sdk .checkbox input:checked+label:after{left:40px}#onetrust-pc-sdk .checkbox input:focus+label::after{outline:#3b99fc solid 1px}#onetrust-pc-sdk .checkbox:after,#onetrust-pc-sdk .checkbox:before{-webkit-font-feature-settings:normal;font-feature-settings:normal;-webkit-font-kerning:auto;font-kerning:auto;-webkit-font-language-override:normal;font-language-override:normal;font-stretch:normal;font-style:normal;font-synthesis:weight style;font-variant:normal;font-weight:normal;text-rendering:auto}#onetrust-pc-sdk .ot-toggle{background-color:#f8f8f8}#onetrust-pc-sdk .ot-toggle .checkbox input:checked+label{background:#cddcf2}#onetrust-pc-sdk .ot-toggle .checkbox input:checked+label:after{background:#4285f4;left:16px}#onetrust-pc-sdk .ot-toggle-group,#onetrust-pc-sdk .ot-toggle{display:inline-block;vertical-align:middle}#onetrust-pc-sdk .ot-arrow-container{float:right}#onetrust-pc-sdk .ot-arrow{top:2px;position:relative;width:10px;height:10px;margin-left:5px;transition:all 300ms ease-in 0s;-webkit-transition:all 300ms ease-in 0s;-moz-transition:all 300ms ease-in 0s;-o-transition:all 300ms ease-in 0s}#onetrust-pc-sdk input:checked~.accordion-header .ot-arrow{transform:rotate(90deg);-o-transform:rotate(90deg);-ms-transform:rotate(90deg);-webkit-transform:rotate(90deg)}#onetrust-pc-sdk button.ot-pill{font-size:.75em;text-align:center;background-color:#3860be;border-color:#3860be;font-weight:600;box-shadow:0 0 10px 1px #cce1ff;width:170px;max-width:180px;color:#fff;height:auto;white-space:normal;word-break:break-word;word-wrap:break-word;padding:10px;line-height:1.2;letter-spacing:.05em}#onetrust-pc-sdk button.ot-pill:first-child{margin-top:10px}#onetrust-pc-sdk.ot-leg-opt-out .description-container{padding-top:0px;margin-top:20px;padding-right:0px;border-radius:3px;overflow:hidden;padding-bottom:10px}#onetrust-pc-sdk.ot-leg-opt-out .ot-leg-border-color{border:1px solid #e9e9e9}#onetrust-pc-sdk.ot-leg-opt-out .ot-leg-border-color .cookie-subgroups-container{border-top:1px solid #e9e9e9}#onetrust-pc-sdk.ot-leg-opt-out .ot-category-desc{padding-bottom:10px}#onetrust-pc-sdk.ot-leg-opt-out .group-toggle{padding-left:10px;width:calc(100% - 20px);padding-top:10px;margin-bottom:0px;padding-bottom:8px}#onetrust-pc-sdk.ot-leg-opt-out .cookie-subgroups-container{padding-top:10px;margin-top:0px}#onetrust-pc-sdk.ot-leg-opt-out .description-container>*:not(.group-toggle){padding-left:10px;padding-right:10px}#onetrust-pc-sdk.ot-leg-opt-out .leg-int-header{overflow:hidden;padding-top:7.5px;padding-bottom:7.5px;background-color:#f8f8f8;border:none;border-bottom:1px solid #e9e9e9}#onetrust-pc-sdk.ot-leg-opt-out .leg-int-header .leg-int-title{float:right;font-size:13px}#onetrust-pc-sdk.ot-leg-opt-out .leg-int-header span:first-child{text-align:left;max-width:80px}#onetrust-pc-sdk.ot-leg-opt-out .leg-int-header span:last-child{padding-right:10px;max-width:95px;text-align:center}#onetrust-pc-sdk.ot-leg-opt-out .description-container .ot-toggle-group:first-of-type{padding-left:36px;padding-right:7px}#onetrust-pc-sdk.ot-leg-opt-out .ot-always-active-group .group-toggle .ot-toggle-group:first-of-type{padding-left:0px}#onetrust-pc-sdk.ot-leg-opt-out .category-header,#onetrust-pc-sdk.ot-leg-opt-out .cookie-subgroup h6{max-width:calc(100% - 115px)}#onetrust-pc-sdk.ot-leg-opt-out #vendor-list-content:not(.host-list-content) #select-all-container{width:100%;display:inline-block;padding:0}#onetrust-pc-sdk.ot-leg-opt-out #vendor-list-content:not(.host-list-content) #select-all-container .ot-checkbox{float:right}#onetrust-pc-sdk.ot-leg-opt-out .leg-int-sel-all-hdr{float:right;position:relative;height:20px;right:55px}#onetrust-pc-sdk.ot-leg-opt-out .consent-hdr,#onetrust-pc-sdk.ot-leg-opt-out .leg-int-hdr{float:right;font-size:.8em;position:relative}#onetrust-pc-sdk.ot-leg-opt-out .leg-int-hdr{margin-right:30px}#onetrust-pc-sdk.ot-leg-opt-out .ot-vendor-consent-tgl{margin-left:62px;margin-right:38px}#onetrust-pc-sdk.ot-leg-opt-out .ot-vendor-consent-tgl label,#onetrust-pc-sdk.ot-leg-opt-out .ot-leg-int-tgl label{width:22px;padding:0}#onetrust-pc-sdk.ot-leg-opt-out .ot-leg-int-tgl-only{margin-right:122px}#onetrust-pc-sdk.ot-leg-opt-out #select-all-vendors-leg-input-container{display:block;width:21px;height:21px;position:relative;float:left;right:30px;clear:both}#onetrust-pc-sdk.ot-leg-opt-out #select-all-vendors-leg-input-container input{position:absolute}#onetrust-pc-sdk.ot-leg-opt-out #select-all-vendors-input-container{float:right;width:21px;height:21px;position:relative;right:70px}#onetrust-pc-sdk.ot-leg-opt-out #vendors-list-container .vendor-info{float:left;width:60%}#onetrust-pc-sdk.ot-leg-opt-out #vendors-list-container .ot-toggle-group{float:right;width:auto}#onetrust-pc-sdk.ot-leg-opt-out #vendors-list-container .ot-checkbox{position:relative}#onetrust-pc-sdk.ot-leg-btn #select-all-vendors-input-container{clear:both}#onetrust-pc-sdk #ot-triangle{border:12px solid transparent;display:none;position:absolute;z-index:2147483647;top:140px;right:40px;transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);-webkit-transform:rotate(45deg);background-color:#fff;-webkit-box-shadow:-3px -3px 5px -2px #c7c5c7;-moz-box-shadow:-3px -3px 5px -2px #c7c5c7;box-shadow:-3px -3px 5px -2px #c7c5c7}#onetrust-pc-sdk #ot-filter-modal{width:300px;position:absolute;z-index:2147483646;top:146px;height:auto;display:none;-moz-transition:.2s ease;-o-transition:.2s ease;-webkit-transition:2s ease;transition:.2s ease;overflow-x:hidden;opacity:1;right:0}#onetrust-pc-sdk #ot-filter-modal .ot-pill{width:130px;float:right}#onetrust-pc-sdk #ot-options{background-color:#fff;margin:5px;border-radius:3px;height:350px;-webkit-box-shadow:0px 0px 12px 2px #c7c5c7;-moz-box-shadow:0px 0px 12px 2px #c7c5c7;box-shadow:0px 0px 12px 2px #c7c5c7}#onetrust-pc-sdk .ot-group-options{overflow-y:auto;width:100%;max-height:65%}#onetrust-pc-sdk .ot-group-option{margin-bottom:25px;margin-left:15px;clear:both}#onetrust-pc-sdk .ot-group-option span{cursor:pointer;color:dimgray;font-size:.8em;line-height:1.1;font-weight:normal}#onetrust-pc-sdk #clear-filters-handler{float:right;max-width:200px;margin-bottom:30px;text-decoration:none}#onetrust-pc-sdk #clear-filters-handler p{float:right;font-weight:bold;color:#3860be;font-size:.9em;margin:0}#onetrust-pc-sdk #clear-filters-handler p:hover{color:#1883fd}#onetrust-pc-sdk #clear-filters-container{float:right;padding-right:15px;height:auto;margin-top:20px}#onetrust-pc-sdk #filter-apply-handler{margin-right:10px;float:right}#onetrust-pc-sdk .group-description+.ot-leg-btn-container{margin-top:0}#onetrust-pc-sdk .ot-leg-btn-container{display:inline-block;width:100%;margin-top:10px}#onetrust-pc-sdk .ot-leg-btn-container button{height:32px;padding:6.5px 8px;margin-bottom:0;line-height:18px;letter-spacing:0}#onetrust-pc-sdk .ot-leg-btn-container svg{display:none;height:14px;width:14px;padding-right:5px;vertical-align:sub}#onetrust-pc-sdk .ot-active-leg-btn{cursor:default;pointer-events:none}#onetrust-pc-sdk .ot-active-leg-btn svg{display:inline-block}#onetrust-pc-sdk .ot-remove-objection-handler{border:none;text-decoration:underline;padding:0;font-size:.82em;font-weight:600;line-height:1.4;padding-left:10px}#onetrust-pc-sdk .ot-obj-leg-btn-handler span{font-weight:bold;text-align:center;font-size:.91em;line-height:1.5}#onetrust-pc-sdk.otPcTab[dir=rtl] .ot-pc-footer-logo svg{direction:rtl}#onetrust-pc-sdk.otPcTab[dir=rtl] input~.accordion-header .ot-arrow,#onetrust-pc-sdk.otPcTab[dir=rtl] #ot-back-arrow{transform:rotate(180deg);-o-transform:rotate(180deg);-ms-transform:rotate(180deg);-webkit-transform:rotate(180deg)}#onetrust-pc-sdk.otPcTab[dir=rtl] input:checked~.accordion-header .ot-arrow{transform:rotate(270deg);-o-transform:rotate(270deg);-ms-transform:rotate(270deg);-webkit-transform:rotate(270deg)}#onetrust-pc-sdk.otPcTab[dir=rtl] #search-container svg{right:52px}#onetrust-pc-sdk.otPcTab[dir=rtl] .ot-checkbox label::after{transform:rotate(45deg);-webkit-transform:rotate(45deg);-o-transform:rotate(45deg);-ms-transform:rotate(45deg);border-left:0;border-right:3px solid}#onetrust-pc-sdk #close-pc-btn-handler.ot-close-icon{padding:0;background-color:transparent;border:none}@media(max-width: 767px){#onetrust-pc-sdk.otPcTab{width:100%;height:100%;border:none}#onetrust-pc-sdk.otPcTab .ot-sdk-container,#onetrust-pc-sdk.otPcTab .ot-button-group-parent,#onetrust-pc-sdk.otPcTab .ot-pc-footer-logo{width:100%}#onetrust-pc-sdk.otPcTab .ot-sdk-container{padding:0;margin:0}#onetrust-pc-sdk.otPcTab #pc-title{margin-left:10px;margin-top:10px;max-width:60%}#onetrust-pc-sdk.otPcTab .description-container{margin:0;padding-top:20px;padding-right:20px;padding-bottom:15px;padding-left:20px;position:relative;left:auto}#onetrust-pc-sdk.otPcTab.ot-leg-opt-out .description-container{margin-top:20px;margin-left:20px;padding:0;padding-bottom:10px}#onetrust-pc-sdk.otPcTab .groups-container{max-height:none;height:auto;overflow:hidden}#onetrust-pc-sdk.otPcTab .ot-button-group-parent{padding-left:0;padding-right:0;position:relative}#onetrust-pc-sdk.otPcTab .ot-button-group,#onetrust-pc-sdk.otPcTab #accept-recommended-btn-handler{float:none}#onetrust-pc-sdk.otPcTab .save-preference-btn-handler,#onetrust-pc-sdk.otPcTab #accept-recommended-btn-handler,#onetrust-pc-sdk.otPcTab .ot-pc-refuse-all-handler{width:calc(100% - 33px)}}@media(max-width: 640px){#onetrust-pc-sdk.otPcTab .pc-header{padding:10px;width:calc(100% - 20px)}#onetrust-pc-sdk.otPcTab #ot-content{height:calc(100% - 275px)}#onetrust-pc-sdk.otPcTab .ot-button-group-parent{padding:15px;margin:0;width:calc(100% - 30px)}#onetrust-pc-sdk.otPcTab .ot-sdk-row .ot-sdk-columns{width:100%}#onetrust-pc-sdk.otPcTab .description-container{margin:0}#onetrust-pc-sdk.otPcTab.ot-leg-opt-out .description-container{margin-left:10px;width:calc(100% - 15px);margin-top:5px;margin-bottom:5px}#onetrust-pc-sdk.otPcTab .vendor-info{max-width:80%}#onetrust-pc-sdk.otPcTab #filter-container{bottom:5px;position:relative;float:right;right:15px}#onetrust-pc-sdk.otPcTab #vendor-list-content{height:calc(100% - 305px);width:calc(100% - 18px);padding-top:13px;padding-right:5px;padding-left:10px}#onetrust-pc-sdk.otPcTab #ot-triangle{top:190px}#onetrust-pc-sdk.otPcTab #ot-filter-modal{top:196px}#onetrust-pc-sdk.otPcTab .ot-pc-footer-logo a{width:auto}}@media(max-width: 600px){#onetrust-pc-sdk.otPcTab .vendor-content{overflow-y:scroll}#onetrust-pc-sdk.otPcTab #vendor-list-content{height:auto;overflow:hidden}#onetrust-pc-sdk.otPcTab #vendors-list-header{height:70px;padding:15px 0;width:100%}#onetrust-pc-sdk.otPcTab #search-container{display:inline-block;width:85%;position:absolute}#onetrust-pc-sdk.otPcTab .navigation-container{display:block;text-align:left;float:none;border-bottom:1px solid #d7d7d7;padding-bottom:15px;padding-left:15px}#onetrust-pc-sdk.otPcTab .action-container{width:calc(100% - 15px);float:none;bottom:auto;display:inline-block;padding-top:8px;padding-left:15px}#onetrust-pc-sdk.otPcTab #vendor-search-handler{width:90%}#onetrust-pc-sdk.otPcTab .ot-button-group-parent{display:inline-block;height:auto;margin-bottom:10px}}@media(max-width: 425px){#onetrust-pc-sdk.otPcTab #vendors-list .accordion-text{padding-top:6px;padding-right:25px;padding-bottom:10px;padding-left:25px}#onetrust-pc-sdk.otPcTab #vendors-list-container .ot-toggle-group,#onetrust-pc-sdk.otPcTab #hosts-list-container .ot-toggle-group{width:100%}#onetrust-pc-sdk.otPcTab #vendors-list-container .host-notice,#onetrust-pc-sdk.otPcTab #vendors-list-container .vendor-notice,#onetrust-pc-sdk.otPcTab #hosts-list-container .host-notice,#onetrust-pc-sdk.otPcTab #hosts-list-container .vendor-notice{width:100%;float:left;margin-left:30px}#onetrust-pc-sdk.otPcTab #vendors-list-container .ot-arrow-container,#onetrust-pc-sdk.otPcTab #hosts-list-container .ot-arrow-container{float:left}#onetrust-pc-sdk.otPcTab #vendors-list-container .vendor-info{float:left;width:100%;max-width:85%}#onetrust-pc-sdk.otPcTab #pc-title{max-width:100%}#onetrust-pc-sdk.otPcTab.ot-leg-opt-out .cookie-subgroups-container li.cookie-subgroup{margin-left:10px;width:calc(100% - 10px)}#onetrust-pc-sdk.otPcTab.ot-leg-opt-out #vendors-list-container .ot-toggle-group{width:auto;float:right}#onetrust-pc-sdk.otPcTab.ot-leg-opt-out #vendors-list-container .ot-arrow-container{float:right}#onetrust-pc-sdk.otPcTab.ot-leg-opt-out #vendors-list-container .vendor-info{max-width:47%}}@media only screen and (min-width: 320px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk.otPcTab{height:100%}#onetrust-pc-sdk.otPcTab #ot-content{height:calc(100% - 162px)}#onetrust-pc-sdk.otPcTab .pc-header{padding:10px;width:calc(100% - 20px)}#onetrust-pc-sdk.otPcTab .ot-button-group-parent{padding:20px;width:calc(100% - 40px)}#onetrust-pc-sdk.otPcTab .vendor-content{overflow-y:scroll}#onetrust-pc-sdk.otPcTab #vendor-list-content{height:auto;overflow:hidden}#onetrust-pc-sdk.otPcTab .ot-button-group,#onetrust-pc-sdk.otPcTab #accept-recommended-btn-handler{float:right}#onetrust-pc-sdk.otPcTab .ot-pc-footer-logo,#onetrust-pc-sdk.otPcTab .save-preference-btn-handler,#onetrust-pc-sdk.otPcTab #accept-recommended-btn-handler{width:auto}}@media only screen and (min-width: 320px)and (max-width: 640px)and (orientation: landscape){#onetrust-pc-sdk.otPcTab .ot-sdk-row .ot-sdk-columns{width:100%}#onetrust-pc-sdk.otPcTab.ot-leg-opt-out .description-container{width:calc(100% - 15px)}}@media only screen and (min-width: 641px)and (max-width: 896px)and (orientation: landscape){#onetrust-pc-sdk.otPcTab .description-container{position:absolute;left:30%}}@media(max-width: 425px),(max-width: 896px)and (max-height: 425px)and (orientation: landscape){#onetrust-pc-sdk.otPcTab .pc-header{padding-right:20px}#onetrust-pc-sdk.otPcTab .pc-logo-container{margin-left:0px;margin-top:5px}#onetrust-pc-sdk.otPcTab .pc-logo{height:40px;width:150px}#onetrust-pc-sdk.otPcTab .pc-close-btn-container{margin-top:10px}#onetrust-pc-sdk.otPcTab .pc-close-btn-container .ot-close-icon{width:12px;height:12px}#onetrust-pc-sdk.otPcTab .group-toggle{float:right;margin-left:10px;width:auto}#onetrust-pc-sdk.otPcTab.ot-leg-opt-out .group-toggle{margin-left:0px;padding-right:10px}#onetrust-pc-sdk.otPcTab #pc-privacy-header,#onetrust-pc-sdk.otPcTab .group-toggle .category-header{display:none}}@media only screen and (min-device-width: 768px)and (max-device-width: 1024px)and (orientation: landscape)and (-webkit-min-device-pixel-ratio: 1){#onetrust-pc-sdk.otPcTab #vendors-list-header input{min-width:250px;max-width:300px}}
            #onetrust-consent-sdk #onetrust-pc-sdk,
                #onetrust-consent-sdk #search-container,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-switch.ot-toggle,
                #onetrust-consent-sdk #onetrust-pc-sdk group-toggle .checkbox,
                #onetrust-consent-sdk #onetrust-pc-sdk #pc-title:after
                 {
                    background-color: #FFFFFF;
                }
               
            #onetrust-consent-sdk #onetrust-pc-sdk h3,
                #onetrust-consent-sdk #onetrust-pc-sdk h4,
                #onetrust-consent-sdk #onetrust-pc-sdk h5,
                #onetrust-consent-sdk #onetrust-pc-sdk h6,
                #onetrust-consent-sdk #onetrust-pc-sdk p,
                #onetrust-consent-sdk #onetrust-pc-sdk #vendors-list-container .vendor-options p,
                #onetrust-consent-sdk #onetrust-pc-sdk #pc-policy-text,
                #onetrust-consent-sdk #onetrust-pc-sdk #pc-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .leg-int-title,
                #onetrust-consent-sdk #onetrust-pc-sdk .leg-int-sel-all-hdr span,
                #onetrust-consent-sdk #onetrust-pc-sdk #hosts-list-container .vendor-host,
                #onetrust-consent-sdk #onetrust-pc-sdk #ot-filter-modal #modal-header,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-checkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #vendors-list #select-all-container p,
                #onetrust-consent-sdk #onetrust-pc-sdk #vendors-list #vendors-list-title,
                #onetrust-consent-sdk #onetrust-pc-sdk #vendors-list .back-btn-handler p,
                #onetrust-consent-sdk #onetrust-pc-sdk #vendors-list .vendor-title,
                #onetrust-consent-sdk #onetrust-pc-sdk #vendors-list #vendors-list-container .consent-category,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-label-status,
                #onetrust-consent-sdk #onetrust-pc-sdk .ot-chkbox label span,
                #onetrust-consent-sdk #onetrust-pc-sdk #clear-filters-handler 
                {
                    color: #222222;
                }
             #onetrust-consent-sdk #onetrust-pc-sdk .privacy-notice-link,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler + a,
                    #onetrust-consent-sdk #onetrust-pc-sdk .category-host-list-handler,
                    #onetrust-consent-sdk #onetrust-pc-sdk .vendor-privacy-notice,
                    #onetrust-consent-sdk #onetrust-pc-sdk #hosts-list-container .host-title a,
                    #onetrust-consent-sdk #onetrust-pc-sdk #hosts-list-container .accordion-header .host-view-cookies,
                    #onetrust-consent-sdk #onetrust-pc-sdk #hosts-list-container .vendor-host a
                    {
                        color: #004b83;
                    }
             #onetrust-consent-sdk #onetrust-banner-sdk a[href]
                        {
                            color: #ffffff;
                        }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-vendors-list-handler:hover { opacity: .7;}
            
        #onetrust-consent-sdk #onetrust-pc-sdk 
            button:not(#clear-filters-handler):not(.ot-close-icon):not(#filter-btn-handler):not(.ot-remove-objection-handler):not(.ot-obj-leg-btn-handler),
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-active-leg-btn {
                background-color: #183642;border-color: #183642;
                color: #fff;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .active-group {
                border-color: #183642;
            }
            
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-remove-objection-handler{
                background-color: transparent;
                border:1px solid transparent;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .ot-leg-btn-container .ot-inactive-leg-btn {
                background-color: #FFFFFF;
                color: #78808E; border-color: #78808E;
            }
            #onetrust-consent-sdk #onetrust-pc-sdk .category-menu-switch-handler {
                    background-color: #f2f2f2
                }#onetrust-consent-sdk #onetrust-pc-sdk .active-group {
                    background-color: #ffffff
                }.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}.ot-sdk-cookie-policy h3,.ot-sdk-cookie-policy h4,.ot-sdk-cookie-policy h6,.ot-sdk-cookie-policy p,.ot-sdk-cookie-policy li,.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy th,.ot-sdk-cookie-policy #cookie-policy-description,.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}.ot-sdk-cookie-policy h4{font-size:1.2em}.ot-sdk-cookie-policy h6{font-size:1em;margin-top:2em}.ot-sdk-cookie-policy th{min-width:75px}.ot-sdk-cookie-policy a,.ot-sdk-cookie-policy a:hover{background:#fff}.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}.ot-sdk-cookie-policy .ot-mobile-border{display:none}.ot-sdk-cookie-policy section{margin-bottom:2em}.ot-sdk-cookie-policy table{border-collapse:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy{font-family:inherit;font-size:16px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h3,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h4,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title{color:dimgray}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup{margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group-desc,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-table-header,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td{font-size:.9em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td span,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td a{font-size:inherit}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group{font-size:1em;margin-bottom:.6em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-title{margin-bottom:1.2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy>section{margin-bottom:1em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th{min-width:75px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a:hover{background:#fff}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead{background-color:#f6f6f4;font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-mobile-border{display:none}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy section{margin-bottom:2em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li{list-style:disc;margin-left:1.5em}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-subgroup ul li h4{display:inline-block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{border-collapse:inherit;margin:auto;border:1px solid #d7d7d7;border-radius:5px;border-spacing:initial;width:100%;overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border-bottom:1px solid #d7d7d7;border-right:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr th:last-child,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr td:last-child{border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:25%}.ot-sdk-cookie-policy[dir=rtl]{text-align:left}#ot-sdk-cookie-policy h3{font-size:1.5em}@media only screen and (max-width: 530px){.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) table,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tbody,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) th,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td,.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{display:block}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) thead tr{position:absolute;top:-9999px;left:-9999px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr{margin:0 0 1em 0}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd),.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) tr:nth-child(odd) a{background:#f6f6f4}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td{border:none;border-bottom:1px solid #eee;position:relative;padding-left:50%}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{position:absolute;height:100%;left:6px;width:40%;padding-right:10px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) .ot-mobile-border{display:inline-block;background-color:#e4e4e4;position:absolute;height:100%;top:0;left:45%;width:2px}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) td:before{content:attr(data-label);font-weight:bold}.ot-sdk-cookie-policy:not(#ot-sdk-cookie-policy-v2) li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table{overflow:hidden}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table td{border:none;border-bottom:1px solid #d7d7d7}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tbody,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{display:block}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-host,#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table .ot-cookies-type{width:auto}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy tr{margin:0 0 1em 0}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{height:100%;width:40%;padding-right:10px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td:before{content:attr(data-label);font-weight:bold}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li{word-break:break-word;word-wrap:break-word}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy thead tr{position:absolute;top:-9999px;left:-9999px;z-index:-9999}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td{border-bottom:1px solid #d7d7d7;border-right:0px}#ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table tr:last-child td:last-child{border-bottom:0px}}
                
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h5,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy h6,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy li,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy p,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy a,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy span,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy td,
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-description {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy th {
                        color: #696969;
                    }
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy .ot-sdk-cookie-policy-group {
                        color: #696969;
                    }
                    
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy #cookie-policy-title {
                            color: #696969;
                        }
                    
            
                    #ot-sdk-cookie-policy-v2.ot-sdk-cookie-policy table th {
                            background-color: #F8F8F8;
                        }
                    
            
                      @keyframes slide-down-custom {
                          0% {
                              bottom: 0px !important;
                          }
                          100% {
                              bottom: 0px;
                          }
                      }
                      @-webkit-keyframes slide-down-custom {
                          0% {
                              bottom: 0px !important;
                          }
                          100% {
                              bottom: 0px;
                          }
                      }
                      @-moz-keyframes slide-down-custom {
                          0% {
                              bottom: 0px !important;
                          }
                          100% {
                              bottom: 0px;
                          }
                      }
                      </style></head>
<body class="article-page">

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-NWDMT9Q"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div role="banner" class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    
    <a class="c-skip-link u-hide-print" href="#content">Skip to main content</a>

    
        

        

        <aside class="c-ad c-ad--728x90">
            <div class="c-ad__inner" data-container-type="banner-advert">
                <p class="c-ad__label">Advertisement</p>
                    
            
            
                <div id="article-doubleclickad-container">
    <div id="div-gpt-ad-top-1" class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-gpt-unitpath="/285/nature_communications/article" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=ncomms15123;doi=10.1038/ncomms15123;techmeta=101,119,82,9;subjmeta=1266,45,507,535,612,631,92;kwrd=Proteins,Screening,X-ray crystallography">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature_communications/article&amp;sz=728x90&amp;c=-1308498918&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Dncomms15123%26doi%3D10.1038/ncomms15123%26techmeta%3D101,119,82,9%26subjmeta%3D1266,45,507,535,612,631,92%26kwrd%3DProteins,Screening,X-ray crystallography">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature_communications/article&amp;sz=728x90&amp;c=-1308498918&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Dncomms15123%26doi%3D10.1038/ncomms15123%26techmeta%3D101,119,82,9%26subjmeta%3D1266,45,507,535,612,631,92%26kwrd%3DProteins,Screening,X-ray crystallography"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>
</div>

            
            
        
            </div>
        </aside>
    

    

    
        <div class="hide-print text-orange content grade-c-show">
            
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>
    
        </div>
    


    
    <header class="c-header c-header--brand-border" id="header" data-header="">
        <div class="c-header__row-border">
            <div class="c-header__container">
                <div class="c-header__layout">
                    <a href="https://www.nature.com/ncomms" data-track="click" data-track-action="home" data-track-category="nature-150-split-header" data-track-label="image">
                        <picture class="c-header__logo">
                            <source srcset="//media.springernature.com/full/nature-cms/uploads/product/ncomms/header-03d2e325c0a02f6df509e5730e9be304.svg" media="(min-width: 769px)">
                            <img src="header-7001f06bc3fe2437048388e9f2f44215.svg" alt="Nature Communications">
                        </picture>
                    </a>
                    <div class="c-header__layout">
                        
                            <div class="c-header__site-navigation c-header__site-navigation--show-at-md" data-test="siteindex-link">
                                <a class="c-header__link" href="https://www.nature.com/siteindex" data-track="click" data-track-category="nature-150-split-header" data-track-action="open nature research index" data-track-label="link">
                                    <span>View all Nature Research journals</span>
                                </a>
                            </div>
                        
                        <div class="c-header__site-navigation c-header__site-navigation--border">
                            <a class="c-header__link" href="javascript:;" data-header-expander="" data-test="search-link" data-track="click" data-track-category="nature-150-split-header" data-track-action="open search tray" data-track-label="button" role="button" aria-pressed="false" aria-expanded="false">
                                <span>Search</span><svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg>
                            </a><div id="search-menu" class="c-header-expander c-header-expander--tray u-hide-print u-js-hide has-tethered" data-track-component="nature-150-split-header" aria-hidden="true">
        <div class="c-header-expander__container">
            <h2 class="u-visually-hidden">Search</h2>
            <div data-test="inline-search">
                <div class="c-header-expander__keyline u-mb-16">
                    <form action="/search" method="get" role="search" class="c-header-expander__form" autocomplete="off" data-dynamic-track-label="" data-track="submit" data-track-action="search" data-track-label="form">
                        <label class="c-header-expander__heading" for="keywords">Article Search</label>
                        <div class="c-form-field u-display-flex">
                            <input class="c-form-field__input u-flex-shrink" id="keywords" name="q" placeholder="Search by keywords or author" data-test="search-keywords" type="text">
                            <button type="submit" class="c-button c-button--contrast u-flex-static u-ml-8" data-test="search-submit">Search</button>
                        </div>
                        <p class="u-ma-0">
                            <a href="https://www.nature.com/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">
                                Advanced search
                            </a>
                        </p>
                    </form>
                </div>
                <div class="c-header-expander__keyline">
                    <h3 class="c-header-expander__heading">Quick links</h3>
                    <ul class="u-list-reset">
                        <li class="u-display-inline-block u-mr-24"><a href="https://www.nature.com/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
                        <li class="u-display-inline-block u-mr-24"><a href="https://www.nature.com/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
                        <li class="u-display-inline-block u-mr-24"><a href="https://www.nature.com/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
                        <li class="u-display-inline-block u-mr-24"><a href="https://www.nature.com/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
                    </ul>
                </div>
            </div>
        </div>
    </div>
                            <a href="https://www.nature.com/nams/svc/myaccount" id="my-account" class="c-header__link placeholder" data-test="login-link" data-track="click" data-track-action="my account" data-track-category="nature-150-split-header" data-track-label="link" style="display: none;">
    <span>My Account</span><svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M10.238 16.905a7.96 7.96 0 003.53-1.48c-.874-2.514-2.065-3.936-3.768-4.319V9.83a3.001 3.001 0 10-2 0v1.277c-1.703.383-2.894 1.805-3.767 4.319A7.96 7.96 0 009 17c.419 0 .832-.032 1.238-.095zm4.342-2.172a8 8 0 10-11.16 0c.757-2.017 1.84-3.608 3.49-4.322a4 4 0 114.182 0c1.649.714 2.731 2.305 3.488 4.322zM9 18A9 9 0 119 0a9 9 0 010 18z" fill="#333" fill-rule="evenodd"></path></svg>
</a>
<a href="https://idp.nature.com/authorize/natureuser?client_id=grover&amp;redirect_uri=https%3A%2F%2Fwww.nature.com%2Farticles%2Fncomms15123" id="login-button" style="" class="c-header__link placeholder" data-test="login-link" data-track="click" data-track-action="login" data-track-category="nature-150-split-header" data-track-label="link">
    <span>Login</span><svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M10.238 16.905a7.96 7.96 0 003.53-1.48c-.874-2.514-2.065-3.936-3.768-4.319V9.83a3.001 3.001 0 10-2 0v1.277c-1.703.383-2.894 1.805-3.767 4.319A7.96 7.96 0 009 17c.419 0 .832-.032 1.238-.095zm4.342-2.172a8 8 0 10-11.16 0c.757-2.017 1.84-3.608 3.49-4.322a4 4 0 114.182 0c1.649.714 2.731 2.305 3.488 4.322zM9 18A9 9 0 119 0a9 9 0 010 18z" fill="#333" fill-rule="evenodd"></path></svg>
</a>

                        </div>
                    </div>
                </div>
            </div>
        </div>
        
            <div class="c-header__container" data-test="c-header__container">
                <ul class="c-header__menu">
                    
                        <li class="c-header__item" data-test="explore-content-button">
                            <a href="javascript:;" class="c-header__link c-header__link--dropdown" data-header-expander="" data-test="menu-button" data-track="click" data-track-category="nature-150-split-header" data-track-action="open explore expander" data-track-label="button" role="button" aria-pressed="false" aria-expanded="false">
                                <span>Explore <span class="c-header__show-text">our content</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                            </a><nav class="u-hide-print c-header-expander u-js-hide has-tethered" aria-labelledby="Explore-our-content" data-test="Explore-our-content" id="explore" data-track-component="nature-150-split-header" aria-hidden="true">
            <div class="c-header-expander__container">
                <div class="c-header-expander__keyline">
                    <h2 id="Explore-our-content" class="c-header-expander__heading u-js-hide">Explore our content</h2>
                    <ul class="c-header-expander__list">
                        
                            
                                <li class="c-header-expander__item">
                                    <a class="c-header-expander__link" href="https://www.nature.com/ncomms/news-and-comment" data-track="click" data-track-action="news &amp; comment" data-track-label="link">
                                        News &amp; Comment
                                    </a>
                                </li>
                            
                                <li class="c-header-expander__item">
                                    <a class="c-header-expander__link" href="https://www.nature.com/ncomms/research" data-track="click" data-track-action="research" data-track-label="link">
                                        Research
                                    </a>
                                </li>
                            
                        
                        
                            
                                <li class="c-header-expander__item">
                                    <a class="c-header-expander__link" href="https://www.nature.com/ncomms/articles" data-track="click" data-track-action="browse articles" data-track-label="link">
                                        Browse Articles
                                    </a>
                                </li>
                            
                                <li class="c-header-expander__item">
                                    <a class="c-header-expander__link" href="https://www.nature.com/ncomms/collections" data-track="click" data-track-action="browse collections" data-track-label="link">
                                        Browse Collections
                                    </a>
                                </li>
                            
                                <li class="c-header-expander__item">
                                    <a class="c-header-expander__link" href="https://www.nature.com/ncomms/browse-subjects" data-track="click" data-track-action="browse subjects" data-track-label="link">
                                        Browse Subjects
                                    </a>
                                </li>
                            
                        
                        
                            <li class="c-header-expander__item c-header-expander__item--keyline">
                                <a class="c-header-expander__link" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=264" data-track="click" data-track-action="Sign up for alerts" data-track-label="link">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </nav>
                        </li>
                    
                    <li class="c-header__item">
                        <a href="javascript:;" class="c-header__link c-header__link--dropdown" data-header-expander="" data-test="menu-button" data-track="click" data-track-category="nature-150-split-header" data-track-action="open journal information expander" data-track-label="button" role="button" aria-pressed="false" aria-expanded="false">
                            <span>Journal info<span class="c-header__show-text">rmation</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                        </a><nav class="u-hide-print c-header-expander u-js-hide has-tethered" aria-labelledby="Journal-information" id="journal-info" data-track-component="nature-150-split-header" aria-hidden="true">
            <div class="c-header-expander__container">
                <div class="c-header-expander__keyline">
                    <h2 id="Journal-information" class="c-header-expander__heading u-js-hide">Journal information</h2>
                    <ul class="c-header-expander__list">
                        
                            <li class="c-header-expander__item">
                                <a class="c-header-expander__link" href="https://www.nature.com/ncomms/about" data-track="click" data-track-action="about the journal" data-track-label="link">
                                    About the journal
                                </a>
                            </li>
                        
                            <li class="c-header-expander__item">
                                <a class="c-header-expander__link" href="https://www.nature.com/ncomms/submit" data-track="click" data-track-action="submit" data-track-label="link">
                                    Submit
                                </a>
                            </li>
                        
                            <li class="c-header-expander__item">
                                <a class="c-header-expander__link" href="https://www.nature.com/ncomms/journal-policies" data-track="click" data-track-action="journal policies" data-track-label="link">
                                    Journal policies
                                </a>
                            </li>
                        
                            <li class="c-header-expander__item">
                                <a class="c-header-expander__link" href="https://www.nature.com/ncomms/editorshighlights" data-track="click" data-track-action="editors' highlights" data-track-label="link">
                                    Editors' Highlights
                                </a>
                            </li>
                        
                            <li class="c-header-expander__item">
                                <a class="c-header-expander__link" href="https://www.nature.com/ncomms/contact" data-track="click" data-track-action="contact" data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                            <li class="c-header-expander__item">
                                <a class="c-header-expander__link" href="https://www.nature.com/ncomms/videos" data-track="click" data-track-action="videos" data-track-label="link">
                                    Videos
                                </a>
                            </li>
                        
                            <li class="c-header-expander__item">
                                <a class="c-header-expander__link" href="https://www.nature.com/ncomms/top50" data-track="click" data-track-action="top articles of 2019" data-track-label="link">
                                    Top Articles of 2019
                                </a>
                            </li>
                        
                        
                            <li class="c-header-expander__item c-header-expander__item--keyline">
                                <a class="c-header-expander__link" href="http://mts-ncomms.nature.com/" data-track="click" data-track-action="Submit manuscript" data-track-label="link">Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </nav>
                    </li>
                    
                </ul>
            </div>
        
    </header>

    
        
    
    <div class="u-mb-16">
        <div class="u-container">
            <ol class="c-breadcrumbs">
            <li class="c-breadcrumbs__item" id="breadcrumb0" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb" itemref="breadcrumb1"><a class="c-breadcrumbs__link" href="https://www.nature.com/" itemprop="url" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="title">nature</span></a><svg class="u-icon c-breadcrumbs__chevron" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></svg></li><li class="c-breadcrumbs__item" id="breadcrumb1" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb" itemref="breadcrumb2"><a class="c-breadcrumbs__link" href="https://www.nature.com/ncomms" itemprop="url" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature communications"><span itemprop="title">nature communications</span></a><svg class="u-icon c-breadcrumbs__chevron" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></svg></li><li class="c-breadcrumbs__item" id="breadcrumb2" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb" itemref="breadcrumb3"><a class="c-breadcrumbs__link" href="https://www.nature.com/ncomms/articles?type=article" itemprop="url" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="title">articles</span></a><svg class="u-icon c-breadcrumbs__chevron" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></svg></li><li class="c-breadcrumbs__item" id="breadcrumb3" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb" itemref="breadcrumb4"><span itemprop="title">article</span></li>
            </ol>
        </div>
    </div>
    

    
    
    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container" data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            
                <div class="c-context-bar u-hide" data-component="context-bar" aria-hidden="true">
                    <div class="c-context-bar__container u-container">
                        <div class="c-context-bar__title">
                            A multi-crystal method for extracting 
obscured crystallographic states from conventionally uninterpretable 
electron density
                        </div>
                        
    
    <div class="c-pdf-download u-clear-both">
        <a href="https://www.nature.com/articles/ncomms15123.pdf" class="c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" download="">
            <span>Download PDF</span>
            <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"></use></svg>
        </a>
    </div>
    

                    </div>
                </div>
            
        
        <article itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" lang="en">
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
    
    
        <li class="c-article-identifiers__item">
            <span class="c-article-identifiers__open" data-test="open-access">Open Access</span>
        </li>
    
    

                        <li class="c-article-identifiers__item"><a href="#article-info" data-track="click" data-track-action="publication date" data-track-label="link">Published: <time datetime="2017-04-24" itemprop="datePublished">24 April 2017</time></a></li>
                    </ul>

                    
                    <h1 class="c-article-title" data-test="article-title" data-article-title="" itemprop="name headline">A multi-crystal method for extracting obscured crystallographic states from conventionally uninterpretable electron density</h1>
                    <ul class="c-author-list js-etal-collapsed js-no-scroll" data-etal="25" data-etal-small="3" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="javascript:;" data-author-popup="auth-Nicholas_M_-Pearce" aria-pressed="false" aria-expanded="false">Nicholas M. Pearce</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Structural Genomics Consortium, University of Oxford"><meta itemprop="address" content="grid.4991.5, 0000 0004 1936 8948, Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK"></span></sup>, </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="javascript:;" data-author-popup="auth-Tobias-Krojer" aria-pressed="false" aria-expanded="false">Tobias Krojer</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Structural Genomics Consortium, University of Oxford"><meta itemprop="address" content="grid.4991.5, 0000 0004 1936 8948, Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK"></span></sup>, </li><li class="c-author-list__item js-smaller-author-etal" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="javascript:;" data-author-popup="auth-Anthony_R_-Bradley" aria-pressed="false" aria-expanded="false">Anthony R. Bradley</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Structural Genomics Consortium, University of Oxford"><meta itemprop="address" content="grid.4991.5, 0000 0004 1936 8948, Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK"></span></sup>, </li><li class="c-author-list__item js-smaller-author-etal" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="javascript:;" data-author-popup="auth-Patrick-Collins" aria-pressed="false" aria-expanded="false">Patrick Collins</a></span><span class="u-js-hide">&nbsp;
            <a class="js-orcid" itemprop="url" href="http://orcid.org/0000-0002-6265-9922"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6265-9922</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Diamond Light Source Ltd, Harwell Science and Innovation Campus"><meta itemprop="address" content="grid.18785.33, 0000 0004 1764 0696, Diamond Light Source Ltd, Harwell Science and Innovation Campus, Didcot, OX11 0QX, UK"></span></sup>, </li><li class="c-author-list__item js-smaller-author-etal" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="javascript:;" data-author-popup="auth-Rados_aw_P_-Nowak" aria-pressed="false" aria-expanded="false">Radosław P. Nowak</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Structural Genomics Consortium, University of Oxford"><meta itemprop="address" content="grid.4991.5, 0000 0004 1936 8948, Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK"></span></sup>, </li><li class="c-author-list__item js-smaller-author-etal" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="javascript:;" data-author-popup="auth-Romain-Talon" aria-pressed="false" aria-expanded="false">Romain Talon</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Structural Genomics Consortium, University of Oxford"><meta itemprop="address" content="grid.4991.5, 0000 0004 1936 8948, Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK"></span></sup>, </li><li class="c-author-list__item js-smaller-author-etal" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="javascript:;" data-author-popup="auth-Brian_D_-Marsden" aria-pressed="false" aria-expanded="false">Brian D. Marsden</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff3" tabindex="-1">3</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Structural Genomics Consortium, University of Oxford"><meta itemprop="address" content="grid.4991.5, 0000 0004 1936 8948, Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK"></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Kennedy Institute of Rheumatology, Rheumatology and Musculoskeletal Sciences, University of Oxford"><meta itemprop="address" content="grid.4991.5, 0000 0004 1936 8948, Nuffield Department of Orthopaedics, Kennedy Institute of Rheumatology, Rheumatology and Musculoskeletal Sciences, University of Oxford, Roosevelt Drive, Oxford, OX3 7FY, UK"></span></sup>, </li><li class="c-author-list__item js-smaller-author-etal" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="javascript:;" data-author-popup="auth-Sebastian-Kelm" aria-pressed="false" aria-expanded="false">Sebastian Kelm</a></span><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="UCB Pharma"><meta itemprop="address" content="grid.418727.f, 0000 0004 5903 3819, UCB Pharma, 208 Bath Road, Slough, SL1 3WE, UK"></span></sup>, </li><li class="c-author-list__item js-smaller-author-etal" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="javascript:;" data-author-popup="auth-Jiye-Shi" aria-pressed="false" aria-expanded="false">Jiye Shi</a></span><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="UCB Pharma"><meta itemprop="address" content="grid.418727.f, 0000 0004 5903 3819, UCB Pharma, 208 Bath Road, Slough, SL1 3WE, UK"></span></sup>, </li><li class="c-author-list__show-more u-js-hide js-mq480-show-inline"><a class="js-etal" href="javascript:;" aria-label="Show all 11 authors for this article" title="Show all 11 authors for this article">[…]</a></li><li class="c-author-list__item js-smaller-author-etal" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="javascript:;" data-author-popup="auth-Charlotte_M_-Deane" aria-pressed="false" aria-expanded="false">Charlotte M. Deane</a></span><span class="u-js-hide">&nbsp;
            <a class="js-orcid" itemprop="url" href="http://orcid.org/0000-0003-1388-2252"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1388-2252</a></span><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="University of Oxford"><meta itemprop="address" content="grid.4991.5, 0000 0004 1936 8948, Department of Statistics, University of Oxford, 24-29 St Giles, Oxford, OX1 3LB, UK"></span></sup> &amp; </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="javascript:;" data-author-popup="auth-Frank-von_Delft" data-corresp-id="c1" aria-pressed="false" aria-expanded="false">Frank von Delft<svg width="16" height="16" class="u-icon"><use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-email"></use></svg></a></span><span class="u-js-hide">&nbsp;
            <a class="js-orcid" itemprop="url" href="http://orcid.org/0000-0003-0378-0017"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-0378-0017</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff6" tabindex="-1">6</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Structural Genomics Consortium, University of Oxford"><meta itemprop="address" content="grid.4991.5, 0000 0004 1936 8948, Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK"></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Diamond Light Source Ltd, Harwell Science and Innovation Campus"><meta itemprop="address" content="grid.18785.33, 0000 0004 1764 0696, Diamond Light Source Ltd, Harwell Science and Innovation Campus, Didcot, OX11 0QX, UK"></span><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="University of Johannesburg"><meta itemprop="address" content="grid.412988.e, 0000 0001 0109 131X, Department of Biochemistry, University of Johannesburg, Auckland Park, 2006, South Africa"></span></sup>&nbsp;</li><li class="c-author-list__show-less u-js-hide"><a class="js-etal" href="javascript:;" aria-label="Show fewer authors for this article" title="Show fewer authors for this article">-Show fewer authors</a></li></ul>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="https://www.nature.com/ncomms"><i data-test="journal-title">Nature Communications</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;8</b>, Article&nbsp;number:&nbsp;<span data-test="article-number">15123</span> (<span data-test="article-publication-year">2017</span>)
            <a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
    <div class="c-article-metrics-bar__wrapper u-clear-both">
        <ul class="c-article-metrics-bar u-list-reset">
            
                <li class=" c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__count">4431 <span class="c-article-metrics-bar__label">Accesses</span></p>
                </li>
            
            
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__count">62 <span class="c-article-metrics-bar__label">Citations</span></p>
                </li>
            
            
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">89 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                    </li>
                
            
            <li class="c-article-metrics-bar__item">
                <p class="c-article-metrics-bar__details"><a href="https://www.nature.com/articles/ncomms15123/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
            </li>
        </ul>
    </div>

                    
                </header>

                
    

                
    

    

            </div>

            <div class="c-article-body">
                <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>In
 macromolecular crystallography, the rigorous detection of changed 
states (for example, ligand binding) is difficult unless signal is 
strong. Ambiguous (‘weak’ or ‘noisy’) density is experimentally common, 
since molecular states are generally only fractionally present in the 
crystal. Existing methodologies focus on generating maximally accurate 
maps whereby minor states become discernible; in practice, such map 
interpretation is disappointingly subjective, time-consuming and 
methodologically unsound. Here we report the PanDDA method, which 
automatically reveals clear electron density for the changed state—even 
from inaccurate maps—by subtracting a proportion of the confounding 
‘ground state’; changed states are objectively identified from 
statistical analysis of density distributions. The method is completely 
general, implying new best practice for all changed-state studies, 
including the routine collection of multiple ground-state crystals. More
 generally, these results demonstrate: the incompleteness of atomic 
models; that single data sets contain insufficient information to model 
them fully; and that accuracy requires further map-deconvolution 
approaches.</p></div></div></section>
                <noscript>
                    
                </noscript>
                
                    <div class="c-pdf-button__container">
                        
                            
    
    <div class="c-pdf-download u-clear-both">
        <a href="https://www.nature.com/articles/ncomms15123.pdf" class="c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" download="">
            <span>Download PDF</span>
            <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"></use></svg>
        </a>
    </div>
    

                        
                    </div>
                
                
                    <section aria-labelledby="Sec1" data-title="Introduction"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Introduction</h2><div class="c-article-section__content" id="Sec1-content"><p>Besides
 its use for resolving the overall three-dimensional (3D) structure of 
bio-molecules, macromolecular X-ray crystallography (MX) is deployed 
extensively to observe small changes to known structures, especially 
compound binding in ligand-discovery and -development projects. Arriving
 at the final model once initial electron density estimates are 
available (after ‘phasing’), relies on a long-established and rarely 
questioned paradigm: cycling between building atoms into the current 
density estimate and computationally optimizing the model against the 
measured data (‘refinement’). The latter improves the calculated phases 
and yields more detailed density that should reveal additional model 
omissions and errors; the process is assumed to converge on a model that
 fully describes the crystal’s content.</p><p>In practice, convergence 
is never convincingly achieved. Much density both strong and weak 
invariably remains unexplained (‘noisy’), hence the aphorism that 
‘refinement […] is never finished, only abandoned’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Sheldrick, G. M. A short history of SHELX. Acta Crystallogr. Sect. A Found. Crystallogr. 64, 112–122 (2007)." href="https://www.nature.com/articles/ncomms15123#ref-CR1" id="ref-link-section-d25626e556">1</a></sup>, and hence too the ‘R-factor gap’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Holton, J. M., Classen, S., Frankel, K. A. &amp; Tainer, J. A. The R-factor gap in macromolecular crystallography: an untapped potential for insights on accurate structures. FEBS J. 281, 4046–4060 (2014)." href="https://www.nature.com/articles/ncomms15123#ref-CR2" id="ref-link-section-d25626e560">2</a></sup>,
 which has obdurately resisted all methodology advances. More recent 
work has shown that conventional single-conformation models are too 
simplistic to describe the crystal<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Burnley, B. T., Afonine, P. V., Adams, P. D. &amp; Gros, P. Modelling dynamics in protein crystal structures by ensemble refinement. Elife 1, e00311 (2012)." href="https://www.nature.com/articles/ncomms15123#ref-CR3" id="ref-link-section-d25626e564">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="DePristo, M. A., De Bakker, P. I. W., Johnson, R. J. K. &amp; Blundell, T. L. Crystallographic refinement by knowledge-based exploration of complex energy landscapes. Structure 13, 1311–1319 (2005)." href="https://www.nature.com/articles/ncomms15123#ref-CR4" id="ref-link-section-d25626e567">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Van Den Bedem, H., Dhanik, A., Latombe, J. C. &amp; Deacon, A. M. Modeling discrete heterogeneity in X-ray diffraction data by fitting multi-conformers. Acta Crystallogr. Sect. D Biol. Crystallogr. 65, 1107–1117 (2009)." href="https://www.nature.com/articles/ncomms15123#ref-CR5" id="ref-link-section-d25626e570">5</a></sup>;
 and that electron density features far weaker than the conventional 
cut-off reflect model deficiencies rather than measurement error<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Lang, P. T. et al. Automated electron-density sampling reveals widespread conformational polymorphism in proteins. Protein Sci. 19, 1420–1431 (2010)." href="https://www.nature.com/articles/ncomms15123#ref-CR6" id="ref-link-section-d25626e574">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Lang, P. T., Holton, J. M., Fraser, J. S. &amp; Alber, T. Protein structural ensembles are revealed by redefining X-ray electron density noise. Proc. Natl Acad. Sci. USA 111, 237–242 (2014)." href="https://www.nature.com/articles/ncomms15123#ref-CR7" id="ref-link-section-d25626e577">7</a></sup>.</p><p>Evidently then, near convergence, conventionally calculated (sigmaA-weighted<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Read, R. J. Improved Fourier coefficients for maps using phases from partial structures with errors. Acta Crystallogr. Sect. A Found. Crystallogr. 42, 140–149 (1986)." href="https://www.nature.com/articles/ncomms15123#ref-CR8" id="ref-link-section-d25626e584">8</a></sup>)
 density derived from a single data set is necessary but insufficient to
 complete the model, as it shows a superposition of states that is 
currently impossible to de-convolute algorithmically. Nearly complete 
models with discrete yet uninterpretable superpositions are common in 
systematic studies of perturbations involving few atoms, such as ligand 
binding, photochemical changes or radiation damage. Since even strong 
biophysical effects are contingent on crystal packing or integrity, only
 a subset of the crystal may transition away from the ground state, even
 after extensive optimization of the experiment. Finally, all current 
modelling approaches ultimately rely on shape matching, and density 
superpositions are susceptible to interpretation errors and bias<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Pozharski, E., Weichenberger, C. X. &amp; Rupp, B. Techniques, tools and best practices for ligand electron-density analysis and results from their application to deposited crystal structures. Acta Crystallogr. Sect. D Biol. Crystallogr. 69, 150–167 (2013)." href="https://www.nature.com/articles/ncomms15123#ref-CR9" id="ref-link-section-d25626e588">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Stanfield, R., Pozharski, E. &amp; Rupp, B. Comment on Three X-ray Crystal Structure Papers. J. Immunol. 196, 521–524 (2016)." href="https://www.nature.com/articles/ncomms15123#ref-CR10" id="ref-link-section-d25626e591">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Rupp, B. &amp; Segelke, B. Questions about the structure of the botulinum neurotoxin B light chain in complex with a target peptide. Nat. Struct. Biol 8, 663–664 (2001)." href="https://www.nature.com/articles/ncomms15123#ref-CR11" id="ref-link-section-d25626e594">11</a></sup> (such as the problem of the ‘Ligand of Desire’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Pozharski, E., Weichenberger, C. X. &amp; Rupp, B. Techniques, tools and best practices for ligand electron-density analysis and results from their application to deposited crystal structures. Acta Crystallogr. Sect. D Biol. Crystallogr. 69, 150–167 (2013)." href="https://www.nature.com/articles/ncomms15123#ref-CR9" id="ref-link-section-d25626e598">9</a></sup>).</p><p>Existing methods to auto-generate multi-conformer models<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="DePristo, M. A., De Bakker, P. I. W., Johnson, R. J. K. &amp; Blundell, T. L. Crystallographic refinement by knowledge-based exploration of complex energy landscapes. Structure 13, 1311–1319 (2005)." href="https://www.nature.com/articles/ncomms15123#ref-CR4" id="ref-link-section-d25626e605">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Van Den Bedem, H., Dhanik, A., Latombe, J. C. &amp; Deacon, A. M. Modeling discrete heterogeneity in X-ray diffraction data by fitting multi-conformers. Acta Crystallogr. Sect. D Biol. Crystallogr. 65, 1107–1117 (2009)." href="https://www.nature.com/articles/ncomms15123#ref-CR5" id="ref-link-section-d25626e608">5</a></sup>
 are not relevant when changes are chemical, and moreover have had 
little take-up, presumably because neither is explicit modelling 
involved nor have robust validation criteria emerged to allay 
long-cultivated fears of over-fitting<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Kleywegt, G. J. &amp; Jones, T. A. Where freedom is given, liberties are taken. Structure 3, 535–540 (1995)." href="https://www.nature.com/articles/ncomms15123#ref-CR12" id="ref-link-section-d25626e612">12</a></sup>. Approaches from time-resolved crystallography<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Yorke, B. A., Beddard, G. S., Owen, R. L. &amp; Pearson, A. R. Time-resolved crystallography using the Hadamard transform. Nat. Methods 11, 1131–1134 (2014)." href="https://www.nature.com/articles/ncomms15123#ref-CR13" id="ref-link-section-d25626e616">13</a></sup> apply only to specialized experiments.</p><p>In
 this work, we show that unencumbered views of the changed, non-ground 
state can be obtained by recasting the problem as a multi-data set, 3D 
background-correction problem (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/ncomms15123#Fig1">Fig. 1</a>),
 which allows the relevant signal to be extracted from conventional 
single-data set density. An accurate estimate of the background can be 
obtained by averaging near-convergence density, in real space and after 
local alignment, from dozens (&gt;30) of independently measured but 
approximately identical ground-state crystals. Subtraction of a suitable
 fraction of this background estimate from the near-convergence density 
of a data set containing a putative changed state, yields a residual 
partial-difference map that we call an event map, which is in general 
fully interpretable:</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Schematic of how outlier identification and background subtraction reveals changed-state density."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 1: Schematic of how outlier identification and background subtraction reveals changed-state density.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/ncomms15123/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig1_HTML.jpg?as=webp"><img aria-describedby="figure-1-desc" src="41467_2017_Article_BFncomms15123_Fig1_HTML.jpg" alt="figure1" loading="lazy" width="685" height="347"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>(<b>a</b>)
 Analysis of the aligned electron density distribution for the same 
voxel (yellow dots) identifies data sets which differ from the 
ensemble—outliers—such as those containing a bound ligand or other 
‘changed state’, for example the changed state in <b>c</b>. (<b>b</b>) 
Averaging over multiple ‘ground-state’ data sets further provides an 
accurate estimate of the ground-state density, leading to <b>d</b>. With pixel intensity representing electron density strength, (<b>c</b>)
 shows an identified location, at which the density is a superposition 
of changed-state (20%) and ground-state (80%) densities; the changed 
state is obscured by the superposed ground state. (<b>d</b>) shows only the ground-state density, adjusted by applying a weighting (BDC=0.8). (<b>e</b>)
 The density that remains after subtracting the background yields an 
estimate of the changed state which is in general fully interpretable.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="https://www.nature.com/articles/ncomms15123/figures/1" data-track-dest="link:Figure1 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><div id="Equ1" class="c-article-equation"><div class="c-article-equation__content"><img src="41467_2017_Article_BFncomms15123_Equ1_HTML.png" class="u-display-block" alt=""></div></div><p>Identifying
 the optimal Background Density Correction factor (BDC) is essential for
 extracting the best signal, as illustrated schematically in <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/ncomms15123#Fig2">Fig. 2</a>, which also illustrates the problems with using conventional maps for the identification of minor crystallographic states.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Minor conformations are obscured in conventional maps but are revealed by background correction."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 2: Minor conformations are obscured in conventional maps but are revealed by background correction.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/ncomms15123/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig2_HTML.jpg?as=webp"><img aria-describedby="figure-2-desc" src="41467_2017_Article_BFncomms15123_Fig2_HTML.jpg" alt="figure2" loading="lazy" width="685" height="387"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p>1D simulations are used to illustrate 3D electron density. (<b>a</b>)
 The actual crystal contains 80% major (black) and 20% minor (orange) 
states, which are largely dissimilar (correlation: 0.42). (<b>b</b>) Conventional (2mF<sub>o</sub>–DF<sub>c</sub>)
 maps (blue) show only the superposition, which resembles the major far 
more than the interesting minor state (correlations: 0.98 and 0.59; in 
practice, the scale is arbitrary). Isomorphous difference (F<sub>o</sub>–F<sub>o</sub>)
 maps (green) show the subtraction of the full-occupancy major state 
from the observed data set, and do not resemble the minor state either, 
except where the major state has low density (right side). (<b>c</b>) 
‘Event maps’ (scaled for comparison), generated as in equation (1) for 
different values of BDC, reveal the minor state optimally for only one 
value of BDC (0.8, indicated in red). BDC=0.0 corresponds to the 
observed density, and BDC=1.0 to a F<sub>o</sub>–F<sub>o</sub> map.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="https://www.nature.com/articles/ncomms15123/figures/2" data-track-dest="link:Figure2 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div></div></div></section><section aria-labelledby="Sec2" data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">The PanDDA algorithm</h3><p>Our
 new method—Pan-Dataset Density Analysis (PanDDA)—comprises: the 
characterization of a set of related crystallographic data sets of the 
same crystal form; the identification of (binding) events; and the 
subtraction of ground state density to reveal clear density for events.</p><p>The method builds on the principle of isomorphous difference (F<sub>o</sub>–F<sub>o</sub>) maps<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Rould, M. A. &amp; Carter, C. W. Isomorphous Difference Methods. Methods Enzymol. 374, 145–163 (2003)." href="https://www.nature.com/articles/ncomms15123#ref-CR14" id="ref-link-section-d25626e743">14</a></sup>,
 but analyses many maps simultaneously by first locally aligning maps in
 real space to bypass the requirement of strict isomorphism, and then 
directly comparing the best estimate of true electron density, namely 
sigmaA-weighted (2mF<sub>o</sub>–DF<sub>c</sub>) maps from late-stage refinement, which ensures that maps are correctly scaled.</p><p>Using multiple maps allows a <i>Z</i>-score
 measure to be calculated at each point in every data set, that reflects
 how significantly the data set deviates from the ensemble of data sets 
at that point in space. <i>Z</i>-scores are assembled into spatial <i>Z</i>-maps, and clusters of large <i>Z</i>-scores
 are an objective and statistically meaningful measure for potentially 
interesting crystallographic signal—events—such as a binding ligand. 
Using <i>Z</i>-maps addresses the common pitfall of over-interpreting density that is in fact ground state density, since in such cases, <i>Z</i>-scores will be small. Equally importantly, <i>Z</i>-maps
 also make it possible to identify weak changed states (for example, 
low-occupancy ligands) that do not yield strong difference (mF<sub>o</sub>–DF<sub>c</sub>) density.</p><p>Finally,
 the precise localization of each change enables reliable background 
subtraction at that site, where the optimal BDC is estimated as the 
value for which the ground state-subtracted map is locally least 
correlated to the ground-state map, relative to a normalizing global 
correlation across the unit cell (see Methods section). Using an 
averaged ground-state map for subtraction, as opposed to a single 
ground-state map, reduces experimental noise in the ground-state 
estimate and thereby also in the event map. Furthermore, averaging over 
multiple data sets minimizes the influence of stochastic variation 
between the data sets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Keedy, D. A. et al. Crystal cryocooling distorts conformational heterogeneity in a model michaelis complex of DHFR. Structure 22, 899–910 (2014)." href="https://www.nature.com/articles/ncomms15123#ref-CR15" id="ref-link-section-d25626e783">15</a></sup> (characterized and discussed in <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Methods</a>).
 Finally, the averaging generates an estimate of the ground state that 
can be used directly as density, bypassing the need for any subjective 
modelling and map interpretation. The BDC is determined algorithmically 
and objectively, and results in event map density approximating only the
 changed configuration of the site, including protein backbone and 
side-chain conformations induced by the change.</p><h3 class="c-article__sub-heading" id="Sec4">Application to crystallographic fragment screening data</h3><p>We
 demonstrate the power of the method by applying it to the most 
demanding type of changed-state study, namely crystallographic fragment 
screening<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Patel, D., Bauman, J. D. &amp; Arnold, E. Advantages of crystallographic fragment screening: Functional and mechanistic insights from a powerful platform for efficient drug discovery. Prog. Biophys. Mol. Biol. 116, 92–100 (2014)." href="https://www.nature.com/articles/ncomms15123#ref-CR16" id="ref-link-section-d25626e798">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Cox, O. B. et al. A poised fragment library enables rapid synthetic expansion yielding the first reported inhibitors of PHIP(2), an atypical bromodomain. Chem. Sci. 7, 2322–2330 (2016)." href="https://www.nature.com/articles/ncomms15123#ref-CR17" id="ref-link-section-d25626e801">17</a></sup>,
 which attempts to observe in electron density the rare and often low 
occupancy binding events that occur when a relatively large (200–1,000) 
library of weak-binding ‘fragment’ compounds (150–300 Da, 100 μM–10 mM)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Murray, C. W. &amp; Verdonk, M. L. The consequences of translational and rotational entropy lost by small molecules on binding to proteins. J. Comput. Aided. Mol. Des. 16, 741–753 (2002)." href="https://www.nature.com/articles/ncomms15123#ref-CR18" id="ref-link-section-d25626e805">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Mooij, W. T. M. et al. Automated protein-ligand crystallography for structure-based drug design. Chem. Med. Chem. 1, 827–838 (2006)." href="https://www.nature.com/articles/ncomms15123#ref-CR19" id="ref-link-section-d25626e808">19</a></sup>
 are added individually or as cocktails to a series of equivalent 
crystals. Conventionally, the analysis is challenging as it involves 
inspecting a lot of 3D space—the whole unit cell in each data set—for 
convincing evidence of bound fragments (‘hits’). In contrast, PanDDA 
directly eliminates the thousands of strong electron density blobs with 
no statistical significance, objectively identifying only regions that 
are unique to each data set; the ground-state data sets are provided by 
the many hit-free crystals.</p><p>Applied to a series of fragment screens (<a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/ncomms15123#Tab1">Table 1</a>), PanDDA yielded markedly more hits than manual inspection of density, far more quickly and all with high confidence (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/ncomms15123#Fig3">Figs 3</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/ncomms15123#Fig4">4</a>; <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Figs 1–6</a>), in both known binding sites and new allosteric sites (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/ncomms15123#Fig4">Fig. 4d</a>).
 Several fragments induced significant reordering of sections of the 
protein that could only be modelled with PanDDA event maps (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/ncomms15123#Fig4">Fig. 4a–c</a>, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Fig. 1a–c</a>), whilst also enabling the identification of mislabelled ligands and the discovery of experimental errors (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Figs 1d–f and 3d–f</a>).
 Models erroneously built into misleading conventional density could be 
discarded with statistical confidence, and the binding of chemically 
elaborated hit compounds could be analysed more reliably. Full 
experimental details and complete descriptions are provided in <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Note 1</a>.
 The method also effectively disambiguates density in conventional 
ligand-binding studies with ligands co-crystallized and a sub-optimal 
number of ground-state data sets (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Note 2</a>).</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption class="c-article-table__figcaption"><b id="Tab1" data-test="table-caption">Table 1 Hit rates from fragment screens before and after use of PanDDA.</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="https://www.nature.com/articles/ncomms15123/tables/1"><span>Full size table</span><svg width="16" height="16" class="u-icon"><use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="PanDDA maps clearly show detail obscured by conventional maps."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 3: PanDDA maps clearly show detail obscured by conventional maps.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/ncomms15123/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig3_HTML.jpg?as=webp"><img aria-describedby="figure-3-desc" src="41467_2017_Article_BFncomms15123_Fig3_HTML.jpg" alt="figure3" loading="lazy" width="685" height="251"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>JMJD2D fragment screening data set x401 at 1.48 Å. (<b>a</b>,<b>b</b>) Conventional maps (2mF<sub>o</sub>–DF<sub>c</sub>, blue, contour as indicated; mF<sub>o</sub>–DF<sub>c</sub>, green/red, ±3<i>σ</i>) are dominated by the NOG co-factor analogue bound in the majority fraction of the crystal, whereas (<b>c</b>) the event map (blue, 2<i>σ</i>, BDC=0.9) and the <i>Z</i>-map (green/red, <i>±</i>4) unambiguously reveal both ligand and associated changes in protein conformations.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="https://www.nature.com/articles/ncomms15123/figures/3" data-track-dest="link:Figure3 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="PanDDA maps reveal complex minor conformations and identify allosteric binders."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 4: PanDDA maps reveal complex minor conformations and identify allosteric binders.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/ncomms15123/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig4_HTML.jpg?as=webp"><img aria-describedby="figure-4-desc" src="41467_2017_Article_BFncomms15123_Fig4_HTML.jpg" alt="figure4" loading="lazy" width="685" height="222"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>In JMJD2D data set x402, at 1.45 Å, (<b>a</b>) conventional maps (contoured as in <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/ncomms15123#Fig3">Fig. 3a</a>)
 show a complex superposition of bound and unbound states that make it 
impossible to identify the bound state (the known unbound state is 
shown). (<b>b</b>) However, in PanDDA maps (contoured as in <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/ncomms15123#Fig3">Fig. 3c</a>, BDC=0.8) the bound conformation can be modelled easily (as shown). (<b>c</b>) Final models for the unbound (yellow) and bound (magenta) conformations highlight the large conformational change. (<b>d</b>)
 Fragments are detected to bind all over the surface of JMJD2D, 
revealing potential allosteric sites, including the peptide-binding 
groove (site A) and the large helix reordering (site B).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="https://www.nature.com/articles/ncomms15123/figures/4" data-track-dest="link:Figure4 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><p>Strikingly, detection of weak binding events is simple even when phases are far from convergence (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/ncomms15123#Fig5">Fig. 5</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Weak ligand identification remains straightforward when phases are degraded."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 5: Weak ligand identification remains straightforward when phases are degraded.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/ncomms15123/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig5_HTML.jpg?as=webp"><img aria-describedby="figure-5-desc" src="41467_2017_Article_BFncomms15123_Fig5_HTML.jpg" alt="figure5" loading="lazy" width="685" height="243"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p>BAZ2B data sets were re-analysed using a deliberately sabotaged reference model, introducing a <span class="stix">∼</span>30° phase error and increasing R<sub>work</sub> and R<sub>free</sub> by <span class="stix">∼</span>12%
 for all dimple-refined data sets. Shown here is the weak hit (refined 
occupancy: 0.64) in data set x492, contoured for different maps as 
labelled: (<b>a</b>,<b>b</b>) 1.78 Å 2mF<sub>o</sub>–DF<sub>c</sub> (blue, 1<i>σ</i>) and mF<sub>o</sub>–DF<sub>c</sub> (green/red, ±3<i>σ</i>). (<b>c</b>,<b>d</b>) 1.79 Å event (blue, 2<i>σ</i>) and <i>Z</i>-maps (green/red, ±3). R<sub>work</sub>/R<sub>free</sub>
 are 0.18/0.21 and 0.30/0.32 for best and degraded phases, respectively.
 BDCs for best and degraded phases are 0.77 and 0.73, respectively. 
Whereas with standard maps, degraded phases remove all evidence of an 
unmodelled change, in PanDDA maps, ligand identification is no more 
difficult, even if the quality of the density is predictably reduced.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="https://www.nature.com/articles/ncomms15123/figures/5" data-track-dest="link:Figure5 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec5">Model validation</h3><p>Model validation is a long-established bedrock of crystallographic analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Kleywegt, G. J. &amp; Jones, T. A. Where freedom is given, liberties are taken. Structure 3, 535–540 (1995)." href="https://www.nature.com/articles/ncomms15123#ref-CR12" id="ref-link-section-d25626e1290">12</a></sup>,
 and crucially requires a model that is numerically stable in 
refinement. As ligands—but especially fragments—invariably bind at 
sub-unitary occupancy, we generate an atomic ensemble model that 
reflects the crystal content implied by the density correction: the 
changed state modelled from event maps is combined with the ground-state
 model, with initial occupancy of the changed state set to 2*(1-BDC) 
(discussed in <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Methods</a>).
 Incorporation of the ground state into the model enforces our Bayesian 
prior knowledge of the crystal, that the ligand is most likely not bound
 to all copies of the protein in the crystal, and a superposition of the
 two states is thus the most likely situation. These ensemble models are
 indeed well-behaved in refinement, provided the ground state can be 
easily represented by an atomic model.</p><p>After refinement, some 
ligands built into strong event density would be considered invalid by 
comparison of the model and the refined density (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Fig. 7</a>), or the subjective but best-practice criterion<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Pozharski, E., Weichenberger, C. X. &amp; Rupp, B. Techniques, tools and best practices for ligand electron-density analysis and results from their application to deposited crystal structures. Acta Crystallogr. Sect. D Biol. Crystallogr. 69, 150–167 (2013)." href="https://www.nature.com/articles/ncomms15123#ref-CR9" id="ref-link-section-d25626e1303">9</a></sup>
 of visual assessment of agreement between model and conventional OMIT 
maps. As this is counterintuitive, given the clarity of the event maps, 
we instead formulated the following strong objective validation 
principles: </p><ol class="u-list-style-none">
                    <li>
                      <span class="u-custom-list-number">1</span>
                      
                        <p>The changed-state partial model must conform to calculable numerical criteria (<a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/ncomms15123#Tab2">Table 2</a>).
 We adopt established requirements: a strong correlation between the 
model and the observed density (real-space correlation coefficient, 
RSCC&gt;0.7) and that ligand B-factors must be comparable to those of 
surrounding residues. We also apply a new metric, that modelling and 
refinement should result in negligible difference density around the 
site (real-space <i>Z</i>-difference score, RSZD&lt;3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Tickle, I. J. Statistical quality indicators for electron-density maps. Acta Crystallogr. Sect. D Biol. Crystallogr. 68, 454–467 (2012)." href="https://www.nature.com/articles/ncomms15123#ref-CR20" id="ref-link-section-d25626e1323">20</a></sup>,
 and further require that the model must not move under refinement (low 
heavy-atom root-mean-squared deviation before versus after). These 
metrics are fully defined in Methods and shown for all models in <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Tables 1–4</a>.</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-2"><figure><figcaption class="c-article-table__figcaption"><b id="Tab2" data-test="table-caption">Table 2 Acceptable values of ligand validation scores.</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="https://www.nature.com/articles/ncomms15123/tables/2"><span>Full size table</span><svg width="16" height="16" class="u-icon"><use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
                      
                    </li>
                    <li>
                      <span class="u-custom-list-number">2</span>
                      
                        <p>The ground state partial model is considered 
an immutable component of the crystal, with a status similar to common 
restraints (for example, geometry or non-crystallographic symmetry), as 
in general there is not enough diffraction information to propose 
otherwise. Thus, the ground state model needs to be fully complete 
before incorporation into the ensemble, and during further cycles of 
model building, it may not be altered, as it is a strong Bayesian prior.
 To stabilize refinement, it may need to be strongly restrained to the 
original ground state model (by external restraints using, for example, 
PROSMART<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Nicholls, R. A., Long, F. &amp; Murshudov, G. N. Low-resolution refinement tools in REFMAC5. Acta Crystallogr. Sect. D Biol. Crystallogr. 68, 404–417 (2012)." href="https://www.nature.com/articles/ncomms15123#ref-CR21" id="ref-link-section-d25626e1448">21</a></sup>).</p>
                      
                    </li>
                    <li>
                      <span class="u-custom-list-number">3</span>
                      
                        <p>The primary event density must always be 
available when disseminating such models. All crystallographic data used
 for the PanDDA analysis must also be made available so that the 
analysis may be reproduced.</p>
                      
                    </li>
                  </ol><p>The group deposition feature recently added by the PDB<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Berman, H. M. et al. The Protein Data Bank. Nucleic Acids Res. 28, 235–242 (2000)." href="https://www.nature.com/articles/ncomms15123#ref-CR22" id="ref-link-section-d25626e1469">22</a></sup>
 makes it realistic to deposit all the many changed- and ground-state 
structures, as well as event maps (see Data Availability). On the other 
hand, standard mechanisms for presenting the validation evidence 
described above are yet to be finalized. Refinement programs do not yet 
support some external restraints that we predict will be important for 
numerical stability at low resolution or for very low occupancy at high 
resolution, in particular restraining relative B-factors to stabilize 
occupancy refinement; this is the subject of future work.</p><p>In 
general, only the changed state will be of primary scientific interest 
in the refined model, with the ground state essentially an experimental 
artefact. Unlike the artefacts inherent in any crystal structure, here 
they are explicitly declared and need not be inferred by further 
analysis. Structure repositories, whether public (PDB) or internal, 
would ideally support this by removing the ground state for normal use; 
this is only possible when states are logically labelled, as discussed 
in Methods.</p></div></div></section><section aria-labelledby="Sec6" data-title="Discussion"><div class="c-article-section" id="Sec6-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec6">Discussion</h2><div class="c-article-section__content" id="Sec6-content"><p>The
 PanDDA algorithm fundamentally revises current methods through a more 
correct treatment of the crystallographic data, not only yielding 
dramatically improved signal-to-noise, but also providing rigorous 
measures of confidence in identified signal. This allows far more subtle
 changes to be modelled, whose importance will be experiment- and 
context-dependent: in ligand development, evidence of weak binding is 
now known to be productive for optimizing binding potency<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Schiebel, J. et al. Six Biophysical Screening Methods Miss a Large Proportion of Crystallographically Discovered Fragment Hits: A Case Study. ACS Chem. Biol. 11, 1693–1701 (2016)." href="https://www.nature.com/articles/ncomms15123#ref-CR23" id="ref-link-section-d25626e1485">23</a></sup>.
 More generally, occupancy is subject to diffusion- and other 
solid-state effects inherent to the crystalline state, and will be an 
imperfect proxy for the scientific import of a change of interest. What 
matters most is that any changed state can be viewed as objectively and 
modelled as accurately as possible, which is what the PanDDA approach 
allows.</p><p>We thus propose a new standard practice for ligand binding
 and other changed-state studies, namely the collection of a series of 
ground state data sets before proceeding with the putative changed-state
 data sets, to provide the contrast necessary to identify the changes of
 interest.</p><p>Retrospective analysis indicates that <span class="stix">∼</span>30 data sets are required for full convergence of the statistical model (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Methods</a>), an experiment that can be completed within hours at modern synchrotron beamlines with fast pixel detectors<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Mueller, M., Wang, M. &amp; Schulze-Briese, C. Optimal fine phi-slicing for single-photon-counting pixel detectors. Acta Crystallogr. Sect. D Biol. Crystallogr. 68, 42–56 (2012)." href="https://www.nature.com/articles/ncomms15123#ref-CR24" id="ref-link-section-d25626e1498">24</a></sup> and sample automation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Helliwell, J. R. &amp; Mitchell, E. P. Synchrotron radiation macromolecular crystallography: Science and spin-offs. IUCrJ 2, 283–291 (2015)." href="https://www.nature.com/articles/ncomms15123#ref-CR25" id="ref-link-section-d25626e1502">25</a></sup>,
 and that needs to be performed only once per crystal form. To address 
such an experiment's other bottleneck, the logistics of analysing large 
numbers of data sets, the PanDDA implementation includes graphical tools
 and various command-line options.</p><p>This number of data sets is 
required for identification of subtle changes from the ground state to 
be sensitive and robust, by ensuring that the <i>Z</i>-map represents a 
true statistical measure of changed-state signal. However, the 
background correction itself still works when fewer than 30 data sets 
are available (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Note 2</a>),
 the trade-off being potentially reduced quality of the event maps. 
Future work will address whether the number of required data sets can be
 identified <i>a priori</i> for a given crystal system.</p><p>The PanDDA
 method is applicable and effective at any resolution, though at lower 
resolutions, as maps become less precise, higher occupancies of changed 
states will in general be required for them to be detected by <i>Z</i>-score.
 What matters most is the consistency of ground-state data sets so that 
they can be represented well by an average; therefore, in regions of 
crystals that vary considerably, such as crystal contacts, statistical 
confidence is reduced similarly to low resolutions.</p><p>As the 
algorithm currently uses a contrast-maximization approach to estimate 
BDC, event map density for changes appears somewhat stronger than 
density for unchanged atoms (typically, surrounding protein). In 
practice, this is not problematic, as the density for the changed states
 is generally clear, and unchanged conformations do not require 
modelling anyway. Establishing a BDC procedure that evens out this 
difference will require accounting for phase bias in the event maps, but
 falls outside the scope of this work.</p><p>In principle, the PanDDA 
approach will allow comparisons between different crystal forms of the 
same protein. However, since functionally important conformational 
changes are not only common in such cases but by their nature affect the
 functionally interesting regions, algorithmic treatment of the local 
alignment is complex and the topic of future work.</p><p>Our results 
upend a long-held tenet in macromolecular crystallographic model 
building, that to visualize subtle features requires optimal phase 
estimates and thus a model as complete and globally error-free as 
possible<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Schiebel, J. et al. High-throughput crystallography: reliable and efficient identification of fragment hits. Structure 24, 1398–1409 (2016)." href="https://www.nature.com/articles/ncomms15123#ref-CR26" id="ref-link-section-d25626e1534">26</a></sup>.
 Conscientiously observed, this places a heavy time burden on the 
analysing scientist as it demands multiple iterations of modelling for 
each data set. The PanDDA approach makes this both practically and 
theoretically unnecessary: a single local modelling step fully validates
 an interpretation, even when the model retains minor problems 
elsewhere.</p><p>More generally, we submit that a qualitative shift in 
approaches to generating crystallographic models is now due. PanDDA 
addresses one class of experiments, those involving induced local 
changes, but all problems of uninterpretable density, and indeed some of
 the R-factor gap<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Holton, J. M., Classen, S., Frankel, K. A. &amp; Tainer, J. A. The R-factor gap in macromolecular crystallography: an untapped potential for insights on accurate structures. FEBS J. 281, 4046–4060 (2014)." href="https://www.nature.com/articles/ncomms15123#ref-CR2" id="ref-link-section-d25626e1541">2</a></sup>,
 should be addressable by analogous map deconvolution methods. 
Multi-data set experiments are no longer difficult; nevertheless, 
existing tools for treating them focus on pursuing a single, 
representative data set through averaging<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Foadi, J. et al. Clustering procedures for the optimal selection of data sets from multiple crystals in macromolecular crystallography. Acta Crystallogr. Sect. D Biol. Crystallogr. 69, 1617–1632 (2013)." href="https://www.nature.com/articles/ncomms15123#ref-CR27" id="ref-link-section-d25626e1545">27</a></sup>.
 Instead, what now appears key is to establish methods for targeted 
perturbations of poorly ordered regions, along with rigorous algorithms 
for reconstructing and visualizing discrete states, and for subsequent 
model validation.</p></div></div></section><section aria-labelledby="Sec7" data-title="Methods"><div class="c-article-section" id="Sec7-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec7">Methods</h2><div class="c-article-section__content" id="Sec7-content"><p>The PanDDA algorithm is schematically outlined in <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Fig. 9</a> (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Methods</a>).</p><h3 class="c-article__sub-heading" id="Sec8">Data set preparation</h3><p>The
 input to PanDDA is a series of refined crystallographic data sets, each
 consisting of a refined structure and associated diffraction data, 
including 2mF<sub>o</sub>–DF<sub>c</sub> structure factors. These can 
come from any refinement program, as long as all data sets are refined 
using the same initial atomic model and the same protocol. All models of
 the protein must be identical, up to the numbering and labelling of 
atoms. All data sets used in this paper were prepared using the Dimple 
pipeline (part of CCP4 (ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Winn, M. D. et al. Overview of the CCP4 suite and current developments. Acta Crystallogr. Sect. D Biol. Crystallogr. 67, 235–242 (2011)." href="https://www.nature.com/articles/ncomms15123#ref-CR28" id="ref-link-section-d25626e1573">28</a>)), from reference models including solvent molecules; there is no requirement to remove solvent atoms from known binding sites.</p><h3 class="c-article__sub-heading" id="Sec9">Structure and map alignment</h3><p>To
 allow map voxels to be compared between crystals that are not exactly 
isomorphous, maps are aligned using the refined models as reference 
points.</p><p>The input protein structures are aligned using a flexible alignment algorithm (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Methods</a>).
 Sections of the protein are aligned separately, to give alignment 
matrices for that section. The alignments generated from the structures 
are stored and are used to transform and thereby align the electron 
density maps.</p><h3 class="c-article__sub-heading" id="Sec10">Handling variations of map resolutions</h3><p>To
 allow map voxels to be compared between crystals, maps have to be 
calculated at the same level of detail, even though crystals can 
diffract to a wide range of resolutions. For analysing a specific data 
set, its full resolution is used; but for contributing to the analysis 
of a different data set, higher resolution data sets are truncated to 
the resolution of the target data set, while lower resolution data sets 
are ignored. Therefore, we analyse the collection of data sets at a 
number of resolutions, and high resolution data sets are used multiple 
times for characterization at lower resolutions, but will only be 
analysed once, at their highest possible resolution. Maps are 
recalculated using truncated diffraction data at each different 
resolution limit. Thus, if processing in resolution bins of 1.0, 1.5, 2,
 and 2.5 Å, a 1.2 Å data set would be analysed at 1.5 Å, but also be 
used to build distributions at 2 Å and 2.5 Å.</p><p>Fourier terms 
omitted in a given map, as happens when reflections are unobserved and 
then effectively set to zero, lead to systematic changes in electron 
density throughout the unit cell that strongly affect the outlier 
analysis; strong low-resolution terms are particularly problematic. 
Therefore, reflections in all data sets are truncated to the set of 
miller indices common to all data sets; and for map calculation, all 
missing Fourier terms are estimated as DF<sub>c</sub>, which refinement programs perform automatically as long as the indices are correctly included in the reflection files.</p><p>Truncated 2mF<sub>o</sub>–DF<sub>c</sub>
 structure factors are Fourier-transformed to generate maps. These maps 
are aligned using the alignment transformations from the flexible 
alignment.</p><h3 class="c-article__sub-heading" id="Sec11">Statistical model</h3><p>Once
 maps for a particular resolution have been aligned, a statistical model
 is parameterized using the electron density of the ground-state data 
sets. The aligned maps are placed on an isotropic Cartesian grid, and 
the electron density is sampled at each grid point of each data set. The
 model treats the observed value of the electron density in data set <i>i</i>, at grid point <i>m</i>, as being sampled from a distribution</p><div id="Equ2" class="c-article-equation"><div class="c-article-equation__content"><img src="41467_2017_Article_BFncomms15123_Equ2_HTML.png" class="u-display-block" alt=""></div></div><p>where <img src="41467_2017_Article_BFncomms15123_IEq1_HTML.png" style="width:32px;max-width:none;" alt=""> models the natural variation in the electron density at point <i>m</i>, independent of data set, and <i>ɛ</i><sub><i>i</i></sub> represents the experimental uncertainty in the electron density in data set <i>i</i>. The variability of the <img src="41467_2017_Article_BFncomms15123_IEq2_HTML.png" style="width:32px;max-width:none;" alt="">term
 accounts for the fact that the crystals are not identical, and that 
small local fluctuations may exist between the crystals. These areas are
 most likely to be in the crystal contacts, or flexible areas of the 
protein. <img src="41467_2017_Article_BFncomms15123_IEq3_HTML.png" style="width:32px;max-width:none;" alt="">represents
 the ‘true’ (unmeasurable) electron density for this crystal form, of 
which each crystal (and associated data set) is a sample.</p><p>The 
simplest model is to assume that both the uncertainty in electron 
density values as well as variation in electron density at a point 
arising from differences between the crystals, can be modelled by a 
normal distribution. Therefore, if</p><div id="Equ3" class="c-article-equation"><div class="c-article-equation__content"><img src="41467_2017_Article_BFncomms15123_Equ3_HTML.png" class="u-display-block" alt=""></div></div><p>then</p><div id="Equ4" class="c-article-equation"><div class="c-article-equation__content"><img src="41467_2017_Article_BFncomms15123_Equ4_HTML.png" class="u-display-block" alt=""></div></div><p>where <i>μ</i><sub><i>m</i></sub> is the mean value of the electron density at point <i>m</i>, <i>s</i><sub><i>m</i></sub> is the variance of the ‘true’ electron density at point <i>m</i>, and <i>σ</i><sub><i>i</i></sub> is the uncertainty in data set <i>i</i>. Under this model, the parameters <i>μ</i><sub><i>m</i></sub> are estimated by taking the un-weighted average of all of the ground state densities.</p><p>The mean ground state map is used to estimate the data set uncertainty, <i>σ</i><sub><i>i</i></sub>,
 for all data sets as follows. Subtracting the mean map from each data 
set map we obtain a mean difference map. By assuming that the 
experimental and model uncertainty in the electron density map are the 
major contributors to deviations from the mean map, the histogram of the
 mean-difference map values is used to estimate the total uncertainty of
 the data set. Calculating the quantiles of a theoretical normal 
distribution <img src="41467_2017_Article_BFncomms15123_IEq4_HTML.png" style="width:54px;max-width:none;" alt="">
 and plotting them against the quantiles from the mean-difference map, 
yields a Q-Q plot where the slope of the central portion of the map 
(between the ±1.5 theoretical quantiles) gives an estimate of the 
uncertainty of the data set (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Fig. 11a</a>). This is equivalent to the method used in Tickle (2012) for calculating the uncertainty of an electron density map<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Tickle, I. J. Statistical quality indicators for electron-density maps. Acta Crystallogr. Sect. D Biol. Crystallogr. 68, 454–467 (2012)." href="https://www.nature.com/articles/ncomms15123#ref-CR20" id="ref-link-section-d25626e1751">20</a></sup>.</p><p>To estimate <i>s</i><sub><i>m</i></sub>, a maximum likelihood method is applied on our model in (4), using the observed values <img src="41467_2017_Article_BFncomms15123_IEq5_HTML.png" style="width:54px;max-width:none;" alt="">, as well as estimates for <i>σ</i><sub><i>i</i></sub> and <i>μ</i><sub><i>m</i></sub> for the ground-state data sets (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Methods</a>).
 An example comparison of the ‘raw’ standard deviations of the grid 
points (simple s.d. of electron density values, not accounting for 
observation error) and the ‘adjusted’ values is shown in <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Fig. 12</a>.
 This adjustment results in the majority of points having no variation 
that is not accounted for by the data set uncertainties; the remaining 
points have non-negligible variation, with non-zero <i>s</i><sub><i>m</i></sub>, and these indicate naturally variable regions.</p><h3 class="c-article__sub-heading" id="Sec12">Calculation of <i>Z</i>-maps</h3><p>The
 parameterized statistical model allows the identification of areas of 
individual data set maps that deviate significantly from the mean map: 
‘events’. Z-scores are calculated by</p><div id="Equ5" class="c-article-equation"><div class="c-article-equation__content"><img src="41467_2017_Article_BFncomms15123_Equ5_HTML.png" class="u-display-block" alt=""></div></div><p>where large <i>Z</i>-scores indicate significant deviations from the mean map. The distributions of <i>Z</i>-scores for a particular data set have improved normality compared to the simple differences from the mean (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Fig. 11b</a>), as expected.</p><p>Regions of individual data sets are identified as significant by contouring <i>Z</i>-maps at <i>Z</i>=2.5, and filtering remaining blobs by a minimum peak value of <i>Z</i>=3 and a minimum volume of 10 Å<sup>3</sup> (volume of a water molecule is <span class="stix">∼</span>30 Å<sup>3</sup>).
 Neighbouring blobs are grouped together if the minimum distance between
 them is &lt;5 Å. These parameters were identified on the BAZ2B data 
set, and found appropriate in subsequent studies and are therefore the 
current program defaults.</p><h3 class="c-article__sub-heading" id="Sec13">Calculation of event maps</h3><p>For
 identified events, the background density correction (BDC) factor is 
estimated as follows. Different fractions of the mean map are subtracted
 from the data set map, and the correlation between the resulting map 
and the mean map is calculated both globally and for the area around the
 event, defined by the blob identified in the <i>Z</i>-map expanded by 1 Å.</p><p>Globally,
 the data set map looks similar to the mean map, so plotting the global 
correlation against the subtracted fraction yields a signal-to-noise 
curve, dropping off at a speed related to the noise in the data set 
(green dashed line, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Fig. 15</a>).
 Locally to the identified site, however, the data set map is a 
superposition between something similar to the mean map and something 
that is unrelated (for example, density of bound ligand). As more of the
 mean map is subtracted, the local correlation between the mean map and 
the resulting map (black dashed line, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Fig. 15</a>)
 will decrease faster than the global correlation. Subtracting the local
 correlation curve from the global correlation curve, BDC is estimated 
where the difference between these two correlation curves is maximized 
(blue solid line, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Fig. 15</a>). The final event map is calculated as in equation (1).</p><h3 class="c-article__sub-heading" id="Sec14">Model building and refinement</h3><p>Interesting sites are identified by <i>Z</i>-maps and modelling is performed using a combination of <i>Z</i>-maps and event maps, similarly to the way that mF<sub>o</sub>–DF<sub>c</sub> maps may be used to guide the modelling of 2mF<sub>o</sub>–DF<sub>c</sub> maps. Modelling takes place in the aligned reference frame, as defined in <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Methods</a>.</p><p>After
 modelling of the changed state, the new conformations of the protein 
are merged with the ground state model. Atoms in the ground state that 
are not present or have moved in the changed state are assigned to a 
previously unused conformer (for example, C). Similarly, atoms in the 
changed state model that are not present in the ground state, or have 
moved, are assigned another unused conformer (for example, D). Atoms 
that are not changed between the two states remain unaltered. The 
resulting ensemble models are then back-transformed, using the flexible 
alignments, to the original crystallographic frame for refinement.</p><p>The models in <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/ncomms15123#Tab1">Table 1</a> have then been refined as an ensemble using phenix.refine<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Afonine, P. V. et al. Towards automated crystallographic structure refinement with phenix.refine. Acta Crystallogr. Sect. D Biol. Crystallogr. 68, 352–367 (2012)." href="https://www.nature.com/articles/ncomms15123#ref-CR29" id="ref-link-section-d25626e1902">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr. Sect. D Biol. Crystallogr. 66, 213–221 (2010)." href="https://www.nature.com/articles/ncomms15123#ref-CR30" id="ref-link-section-d25626e1905">30</a></sup>,
 under conventional resolution-dependant refinement protocols, with 
constrained occupancy groups corresponding to the bound and unbound 
structures to ensure that the occupancies of the bound and unbound 
states sum to unity.</p><p>Because of the methodical way in which the 
ensembles are generated, the changed state model can be extracted simply
 by removing the atoms corresponding to the changed ground state atoms 
(that is, conformer C in the above example).</p><h3 class="c-article__sub-heading" id="Sec15">Validation</h3><p>The atomic model of the changed state is validated by four quality metrics (<a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/ncomms15123#Tab2">Table 2</a>). Two are electron density scores, generated by <i>EDSTATS</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Tickle, I. J. Statistical quality indicators for electron-density maps. Acta Crystallogr. Sect. D Biol. Crystallogr. 68, 454–467 (2012)." href="https://www.nature.com/articles/ncomms15123#ref-CR20" id="ref-link-section-d25626e1925">20</a></sup>:
 real-space correlation coefficient (RSCC) reflects the fit of the atoms
 to the experimental density, and should typically be greater than 0.7; 
while real-space <i>Z</i>-difference score (RSZD) measures the amount of
 difference density that is found around these atoms, and should be 
below 3. The B-factor ratio measures the consistency of the model with 
surrounding protein, and is calculated from the B-factors of the changed
 atoms and all side-chain atoms within 4 Å, respectively. Large values 
(&gt;3) reflect poor evidence for the model, and intermediate values 
(1.5+) indicate errors in refinement or modelling; for weakly-binding 
ligands, systematically large ratios may be justifiable. Coordinate 
root-mean-squared deviation (RMSD) compares the positions of all atoms 
built into event density, with their positions after final refinement, 
and should be below 1 Å.</p><h3 class="c-article__sub-heading" id="Sec16">Implementation</h3><p>PanDDA is implemented in Python and relies heavily on the CCTBX<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Grosse-Kunstleve, R. W., Sauter, N. K., Moriarty, N. W. &amp; Adams, P. D. The Computational Crystallography Toolbox: crystallographic algorithms in a reusable software framework. J. Appl. Crystallogr. 35, 126–136 (2002)." href="https://www.nature.com/articles/ncomms15123#ref-CR31" id="ref-link-section-d25626e1940">31</a></sup>.
 It has been tested extensively for robustness and usability by users of
 Diamond’s XChem fragment screening facility. Source code is available 
on bitbucket (<a href="https://bitbucket.org/pandda/pandda">https://bitbucket.org/pandda/pandda</a>) or as part of CCP4 (ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Winn, M. D. et al. Overview of the CCP4 suite and current developments. Acta Crystallogr. Sect. D Biol. Crystallogr. 67, 235–242 (2011)." href="https://www.nature.com/articles/ncomms15123#ref-CR28" id="ref-link-section-d25626e1950">28</a>). A manual and tutorial are available at <a href="https://pandda.bitbucket.io/">https://pandda.bitbucket.io</a>. Processing 200–500 data sets on a 3.7 GHz Quad-Core Intel Xeon with 32 GB of RAM takes <span class="stix">∼</span>3–10+ hours depending on resolution binning and size of crystallographic unit cell.</p><h3 class="c-article__sub-heading" id="Sec17">Data availability</h3><p>Models
 were built and refined for those ligands that could be uniquely 
identified in the event maps, except for those that interact extensively
 with the crystal contacts and are therefore unlikely to be biologically
 relevant. Modelled data sets (those in <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/ncomms15123#Tab1">Table 1</a>)
 and unmodelled data sets have been deposited in the PDB using the new 
group deposition system (PDB codes for each data set are stated in <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Table 5</a>);
 structure factors for event maps are included within each mmCIF file 
downloadable from the PDB. PDB group deposition IDs for the ligand-bound
 structures are G_1002018 (BAZ2B), G_1002020 (JMJD2D), G_1002022 (BRD1),
 and G_1002024 (SP100); group IDs for the automatically refined 
structures are G_1002019 (BAZ2B), G_1002021 (JMJD2D), G_1002023 (BRD1), 
and G_1002025 (SP100). However, since navigation of this large numbers 
of structures and event maps remains an obstacle to interpretation, 
interactive summary pages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Krojer, T., Pearce, N. M., Bradley, A., Marsden, B. D. &amp; von Delft, F. PanDDA analysis of BAZ2B screened against Zenobia Fragment Library (HTML Summary). Zenodo. Available at: https://zenodo.org/record/290199 (2017)." href="https://www.nature.com/articles/ncomms15123#ref-CR32" id="ref-link-section-d25626e1976">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Krojer, T., Pearce, N. M., Bradley, A., Marsden, B. D. &amp; von Delft, F. PanDDA analysis of JMJD2D screened against Zenobia Fragment Library (HTML Summary). Zenodo. Available at: https://zenodo.org/record/290220 (2017)." href="https://www.nature.com/articles/ncomms15123#ref-CR33" id="ref-link-section-d25626e1979">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Krojer, T., Pearce, N. M., Collins, P., Talon, R. &amp; von Delft, F. PanDDA analysis of BRD1 screened against 3D-Fragment-Consortium Fragment Library. (HTML Summary) Zenodo. Available at: https://zenodo.org/record/290217 (2017)." href="https://www.nature.com/articles/ncomms15123#ref-CR34" id="ref-link-section-d25626e1982">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Krojer, T., Pearce, N. M., Collins, P., Talon, R. &amp; von Delft, F. PanDDA analysis of SP100 screened against selection of Maybridge Fragment Library (HTML Summary). Zenodo. Available at: https://zenodo.org/record/290201 (2017)." href="https://www.nature.com/articles/ncomms15123#ref-CR35" id="ref-link-section-d25626e1985">35</a></sup> for each fragment screen have been uploaded to Zenodo (<a href="https://zenodo.org/">https://zenodo.org</a>); zip files of all of the crystallographic data have also been uploaded<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Pearce, N. M., Bradley, A. R., Marsden, B. D. &amp; von Delft, F. PanDDA analysis of BAZ2B screened against Zenobia Fragment Library [Dataset]. Zenodo. Available at: https://zenodo.org/record/48768 (2016)." href="https://www.nature.com/articles/ncomms15123#ref-CR36" id="ref-link-section-d25626e1996">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Pearce, N. M., Bradley, A. R., Marsden, B. D. &amp; von Delft, F. PanDDA analysis of JMJD2D screened against Zenobia Fragment Library [Dataset]. Zenodo. Available at: https://zenodo.org/record/48770 (2016)." href="https://www.nature.com/articles/ncomms15123#ref-CR37" id="ref-link-section-d25626e1999">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Pearce, N. M., Krojer, T., Collins, P., Talon, R. &amp; von Delft, F. PanDDA analysis of BRD1 screened against 3D-Fragment Consortium Fragment Library [Dataset]. Zenodo. Available at: https://zenodo.org/record/48769 (2016)." href="https://www.nature.com/articles/ncomms15123#ref-CR38" id="ref-link-section-d25626e2002">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Pearce, N. M., Krojer, T., Collins, P., Talon, R. &amp; von Delft, F. PanDDA analysis of SP100 screened against selection of Maybridge Fragment Library [Dataset]. Zenodo. Available at: https://zenodo.org/record/48771 (2016)." href="https://www.nature.com/articles/ncomms15123#ref-CR39" id="ref-link-section-d25626e2005">39</a></sup> (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/ncomms15123#MOESM3535">Supplementary Table 5</a>). All other data are available from the corresponding author upon reasonable request.</p></div></div></section><section aria-labelledby="Sec18" data-title="Additional information"><div class="c-article-section" id="Sec18-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec18">Additional information</h2><div class="c-article-section__content" id="Sec18-content"><p><b>How to cite this article:</b> Pearce, N. M. <i>et al</i>. A multi-crystal method for extracting obscured crystallographic states from conventionally uninterpretable electron density. <i>Nat. Commun.</i> <b>8,</b> 15123 doi: 10.1038/ncomms15123 (2017).</p><p><b>Publisher’s note</b>: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section>
                

                <section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references"><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="GM. Sheldrick, "><meta itemprop="datePublished" content="2007"><meta itemprop="headline" content="Sheldrick, G. M. A short history of SHELX. Acta Crystallogr. Sect. A Found. Crystallogr. 64, 112–122 (2007)."><span class="c-article-references__counter">1</span><p class="c-article-references__text" id="ref-CR1">Sheldrick, G. M. A short history of SHELX. <i>Acta Crystallogr. Sect. A Found. Crystallogr.</i> <b>64</b>, 112–122 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2008AcCrA..64..112S" aria-label="View reference 1 on ADS">ADS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0108767307043930" aria-label="View reference 1">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 1 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20short%20history%20of%20SHELX&amp;journal=Acta%20Crystallogr.%20Sect.%20A%20Found.%20Crystallogr.&amp;volume=64&amp;pages=112-122&amp;publication_year=2007&amp;author=Sheldrick%2CGM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="JM. Holton, S. Classen, KA. Frankel, JA. Tainer, "><meta itemprop="datePublished" content="2014"><meta itemprop="headline" content="Holton, J. M., Classen, S., Frankel, K. A. &amp; Tainer, J. A. The R-factor gap in macromolecular crystallography:"><span class="c-article-references__counter">2</span><p class="c-article-references__text" id="ref-CR2">Holton,
 J. M., Classen, S., Frankel, K. A. &amp; Tainer, J. A. The R-factor gap
 in macromolecular crystallography: an untapped potential for insights 
on accurate structures. <i>FEBS J.</i> <b>281</b>, 4046–4060 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhsFKnsb3M" aria-label="View reference 2 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Ffebs.12922" aria-label="View reference 2">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 2 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20R-factor%20gap%20in%20macromolecular%20crystallography%3A%20an%20untapped%20potential%20for%20insights%20on%20accurate%20structures&amp;journal=FEBS%20J.&amp;volume=281&amp;pages=4046-4060&amp;publication_year=2014&amp;author=Holton%2CJM&amp;author=Classen%2CS&amp;author=Frankel%2CKA&amp;author=Tainer%2CJA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="BT. Burnley, PV. Afonine, PD. Adams, P. Gros, "><meta itemprop="datePublished" content="2012"><meta itemprop="headline" content="Burnley, B. T., Afonine, P. V., Adams, P. D. &amp; Gros, P. Modelling dynamics in protein crystal structures by en"><span class="c-article-references__counter">3</span><p class="c-article-references__text" id="ref-CR3">Burnley,
 B. T., Afonine, P. V., Adams, P. D. &amp; Gros, P. Modelling dynamics 
in protein crystal structures by ensemble refinement. <i>Elife</i> <b>1</b>, e00311 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.7554%2FeLife.00311" aria-label="View reference 3">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 3 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Modelling%20dynamics%20in%20protein%20crystal%20structures%20by%20ensemble%20refinement&amp;journal=Elife&amp;volume=1&amp;publication_year=2012&amp;author=Burnley%2CBT&amp;author=Afonine%2CPV&amp;author=Adams%2CPD&amp;author=Gros%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="MA. DePristo, PIW. De Bakker, RJK. Johnson, TL. Blundell, "><meta itemprop="datePublished" content="2005"><meta itemprop="headline" content="DePristo, M. A., De Bakker, P. I. W., Johnson, R. J. K. &amp; Blundell, T. L. Crystallographic refinement by knowl"><span class="c-article-references__counter">4</span><p class="c-article-references__text" id="ref-CR4">DePristo,
 M. A., De Bakker, P. I. W., Johnson, R. J. K. &amp; Blundell, T. L. 
Crystallographic refinement by knowledge-based exploration of complex 
energy landscapes. <i>Structure</i> <b>13</b>, 1311–1319 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2MXpvFentrs%253D" aria-label="View reference 4 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.str.2005.06.008" aria-label="View reference 4">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 4 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Crystallographic%20refinement%20by%20knowledge-based%20exploration%20of%20complex%20energy%20landscapes&amp;journal=Structure&amp;volume=13&amp;pages=1311-1319&amp;publication_year=2005&amp;author=DePristo%2CMA&amp;author=De%20Bakker%2CPIW&amp;author=Johnson%2CRJK&amp;author=Blundell%2CTL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="H. Van Den Bedem, A. Dhanik, JC. Latombe, AM. Deacon, "><meta itemprop="datePublished" content="2009"><meta itemprop="headline" content="Van Den Bedem, H., Dhanik, A., Latombe, J. C. &amp; Deacon, A. M. Modeling discrete heterogeneity in X-ray diffrac"><span class="c-article-references__counter">5</span><p class="c-article-references__text" id="ref-CR5">Van
 Den Bedem, H., Dhanik, A., Latombe, J. C. &amp; Deacon, A. M. Modeling 
discrete heterogeneity in X-ray diffraction data by fitting 
multi-conformers. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>65</b>, 1107–1117 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnJ" aria-label="View reference 5 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0907444909030613" aria-label="View reference 5">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 5 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Modeling%20discrete%20heterogeneity%20in%20X-ray%20diffraction%20data%20by%20fitting%20multi-conformers&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=65&amp;pages=1107-1117&amp;publication_year=2009&amp;author=Van%20Den%20Bedem%2CH&amp;author=Dhanik%2CA&amp;author=Latombe%2CJC&amp;author=Deacon%2CAM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="PT. Lang, "><meta itemprop="datePublished" content="2010"><meta itemprop="headline" content="Lang, P. T. et al. Automated electron-density sampling reveals widespread conformational polymorphism in prote"><span class="c-article-references__counter">6</span><p class="c-article-references__text" id="ref-CR6">Lang, P. T. et al. Automated electron-density sampling reveals widespread conformational polymorphism in proteins. <i>Protein Sci.</i> <b>19</b>, 1420–1431 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXotVWhs7s%253D" aria-label="View reference 6 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1002%2Fpro.423" aria-label="View reference 6">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 6 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20electron-density%20sampling%20reveals%20widespread%20conformational%20polymorphism%20in%20proteins&amp;journal=Protein%20Sci.&amp;volume=19&amp;pages=1420-1431&amp;publication_year=2010&amp;author=Lang%2CPT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="PT. Lang, JM. Holton, JS. Fraser, T. Alber, "><meta itemprop="datePublished" content="2014"><meta itemprop="headline" content="Lang, P. T., Holton, J. M., Fraser, J. S. &amp; Alber, T. Protein structural ensembles are revealed by redefining "><span class="c-article-references__counter">7</span><p class="c-article-references__text" id="ref-CR7">Lang,
 P. T., Holton, J. M., Fraser, J. S. &amp; Alber, T. Protein structural 
ensembles are revealed by redefining X-ray electron density noise. <i>Proc. Natl Acad. Sci. USA</i> <b>111</b>, 237–242 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXps1Gisg%253D%253D" aria-label="View reference 7 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2014PNAS..111..237L" aria-label="View reference 7 on ADS">ADS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1073%2Fpnas.1302823110" aria-label="View reference 7">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 7 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20structural%20ensembles%20are%20revealed%20by%20redefining%20X-ray%20electron%20density%20noise&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=237-242&amp;publication_year=2014&amp;author=Lang%2CPT&amp;author=Holton%2CJM&amp;author=Fraser%2CJS&amp;author=Alber%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="RJ. Read, "><meta itemprop="datePublished" content="1986"><meta itemprop="headline" content="Read, R. J. Improved Fourier coefficients for maps using phases from partial structures with errors. Acta Crys"><span class="c-article-references__counter">8</span><p class="c-article-references__text" id="ref-CR8">Read, R. J. Improved Fourier coefficients for maps using phases from partial structures with errors. <i>Acta Crystallogr. Sect. A Found. Crystallogr.</i> <b>42</b>, 140–149 (1986).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0108767386099622" aria-label="View reference 8">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 8 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20Fourier%20coefficients%20for%20maps%20using%20phases%20from%20partial%20structures%20with%20errors&amp;journal=Acta%20Crystallogr.%20Sect.%20A%20Found.%20Crystallogr.&amp;volume=42&amp;pages=140-149&amp;publication_year=1986&amp;author=Read%2CRJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="E. Pozharski, CX. Weichenberger, B. Rupp, "><meta itemprop="datePublished" content="2013"><meta itemprop="headline" content="Pozharski, E., Weichenberger, C. X. &amp; Rupp, B. Techniques, tools and best practices for ligand electron-densit"><span class="c-article-references__counter">9</span><p class="c-article-references__text" id="ref-CR9">Pozharski,
 E., Weichenberger, C. X. &amp; Rupp, B. Techniques, tools and best 
practices for ligand electron-density analysis and results from their 
application to deposited crystal structures. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>69</b>, 150–167 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXitFaiu74%253D" aria-label="View reference 9 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0907444912044423" aria-label="View reference 9">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 9 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Techniques%2C%20tools%20and%20best%20practices%20for%20ligand%20electron-density%20analysis%20and%20results%20from%20their%20application%20to%20deposited%20crystal%20structures&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=69&amp;pages=150-167&amp;publication_year=2013&amp;author=Pozharski%2CE&amp;author=Weichenberger%2CCX&amp;author=Rupp%2CB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="R. Stanfield, E. Pozharski, B. Rupp, "><meta itemprop="datePublished" content="2016"><meta itemprop="headline" content="Stanfield, R., Pozharski, E. &amp; Rupp, B. Comment on Three X-ray Crystal Structure Papers. J. Immunol. 196, 521–"><span class="c-article-references__counter">10</span><p class="c-article-references__text" id="ref-CR10">Stanfield, R., Pozharski, E. &amp; Rupp, B. Comment on Three X-ray Crystal Structure Papers. <i>J. Immunol.</i> <b>196</b>, 521–524 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XltFertA%253D%253D" aria-label="View reference 10 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26747564" aria-label="View reference 10 on PubMed" rel="nofollow">PubMed</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 10 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Comment%20on%20Three%20X-ray%20Crystal%20Structure%20Papers&amp;journal=J.%20Immunol.&amp;volume=196&amp;pages=521-524&amp;publication_year=2016&amp;author=Stanfield%2CR&amp;author=Pozharski%2CE&amp;author=Rupp%2CB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="B. Rupp, B. Segelke, "><meta itemprop="datePublished" content="2001"><meta itemprop="headline" content="Rupp, B. &amp; Segelke, B. Questions about the structure of the botulinum neurotoxin B light chain in complex with"><span class="c-article-references__counter">11</span><p class="c-article-references__text" id="ref-CR11">Rupp,
 B. &amp; Segelke, B. Questions about the structure of the botulinum 
neurotoxin B light chain in complex with a target peptide. <i>Nat. Struct. Biol</i> <b>8</b>, 663–664 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3MXls1Ontbs%253D" aria-label="View reference 11 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2F90361" aria-label="View reference 11">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 11 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Questions%20about%20the%20structure%20of%20the%20botulinum%20neurotoxin%20B%20light%20chain%20in%20complex%20with%20a%20target%20peptide&amp;journal=Nat.%20Struct.%20Biol&amp;volume=8&amp;pages=663-664&amp;publication_year=2001&amp;author=Rupp%2CB&amp;author=Segelke%2CB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="GJ. Kleywegt, TA. Jones, "><meta itemprop="datePublished" content="1995"><meta itemprop="headline" content="Kleywegt, G. J. &amp; Jones, T. A. Where freedom is given, liberties are taken. Structure 3, 535–540 (1995)."><span class="c-article-references__counter">12</span><p class="c-article-references__text" id="ref-CR12">Kleywegt, G. J. &amp; Jones, T. A. Where freedom is given, liberties are taken. <i>Structure</i> <b>3</b>, 535–540 (1995).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2MXmvVyrsrs%253D" aria-label="View reference 12 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS0969-2126%2801%2900187-3" aria-label="View reference 12">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 12 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Where%20freedom%20is%20given%2C%20liberties%20are%20taken&amp;journal=Structure&amp;volume=3&amp;pages=535-540&amp;publication_year=1995&amp;author=Kleywegt%2CGJ&amp;author=Jones%2CTA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="BA. Yorke, GS. Beddard, RL. Owen, AR. Pearson, "><meta itemprop="datePublished" content="2014"><meta itemprop="headline" content="Yorke, B. A., Beddard, G. S., Owen, R. L. &amp; Pearson, A. R. Time-resolved crystallography using the Hadamard tr"><span class="c-article-references__counter">13</span><p class="c-article-references__text" id="ref-CR13">Yorke, B. A., Beddard, G. S., Owen, R. L. &amp; Pearson, A. R. Time-resolved crystallography using the Hadamard transform. <i>Nat. Methods</i> <b>11</b>, 1131–1134 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhs1yhtrbO" aria-label="View reference 13 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnmeth.3139" aria-label="View reference 13">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 13 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Time-resolved%20crystallography%20using%20the%20Hadamard%20transform&amp;journal=Nat.%20Methods&amp;volume=11&amp;pages=1131-1134&amp;publication_year=2014&amp;author=Yorke%2CBA&amp;author=Beddard%2CGS&amp;author=Owen%2CRL&amp;author=Pearson%2CAR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="MA. Rould, CW. Carter, "><meta itemprop="datePublished" content="2003"><meta itemprop="headline" content="Rould, M. A. &amp; Carter, C. W. Isomorphous Difference Methods. Methods Enzymol. 374, 145–163 (2003)."><span class="c-article-references__counter">14</span><p class="c-article-references__text" id="ref-CR14">Rould, M. A. &amp; Carter, C. W. Isomorphous Difference Methods. <i>Methods Enzymol.</i> <b>374</b>, 145–163 (2003).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXotlygsQ%253D%253D" aria-label="View reference 14 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2FS0076-6879%2803%2974007-5" aria-label="View reference 14">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 14 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Isomorphous%20Difference%20Methods&amp;journal=Methods%20Enzymol.&amp;volume=374&amp;pages=145-163&amp;publication_year=2003&amp;author=Rould%2CMA&amp;author=Carter%2CCW">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="DA. Keedy, "><meta itemprop="datePublished" content="2014"><meta itemprop="headline" content="Keedy, D. A. et al. Crystal cryocooling distorts conformational heterogeneity in a model michaelis complex of "><span class="c-article-references__counter">15</span><p class="c-article-references__text" id="ref-CR15">Keedy, D. A. et al. Crystal cryocooling distorts conformational heterogeneity in a model michaelis complex of DHFR. <i>Structure</i> <b>22</b>, 899–910 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXpslCgsb4%253D" aria-label="View reference 15 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.str.2014.04.016" aria-label="View reference 15">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 15 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Crystal%20cryocooling%20distorts%20conformational%20heterogeneity%20in%20a%20model%20michaelis%20complex%20of%20DHFR&amp;journal=Structure&amp;volume=22&amp;pages=899-910&amp;publication_year=2014&amp;author=Keedy%2CDA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="D. Patel, JD. Bauman, E. Arnold, "><meta itemprop="datePublished" content="2014"><meta itemprop="headline" content="Patel, D., Bauman, J. D. &amp; Arnold, E. Advantages of crystallographic fragment screening: Functional and mechan"><span class="c-article-references__counter">16</span><p class="c-article-references__text" id="ref-CR16">Patel,
 D., Bauman, J. D. &amp; Arnold, E. Advantages of crystallographic 
fragment screening: Functional and mechanistic insights from a powerful 
platform for efficient drug discovery. <i>Prog. Biophys. Mol. Biol.</i> <b>116</b>, 92–100 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhtlGntbjL" aria-label="View reference 16 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.pbiomolbio.2014.08.004" aria-label="View reference 16">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 16 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Advantages%20of%20crystallographic%20fragment%20screening%3A%20Functional%20and%20mechanistic%20insights%20from%20a%20powerful%20platform%20for%20efficient%20drug%20discovery&amp;journal=Prog.%20Biophys.%20Mol.%20Biol.&amp;volume=116&amp;pages=92-100&amp;publication_year=2014&amp;author=Patel%2CD&amp;author=Bauman%2CJD&amp;author=Arnold%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="OB. Cox, "><meta itemprop="datePublished" content="2016"><meta itemprop="headline" content="Cox, O. B. et al. A poised fragment library enables rapid synthetic expansion yielding the first reported inhi"><span class="c-article-references__counter">17</span><p class="c-article-references__text" id="ref-CR17">Cox,
 O. B. et al. A poised fragment library enables rapid synthetic 
expansion yielding the first reported inhibitors of PHIP(2), an atypical
 bromodomain. <i>Chem. Sci.</i> <b>7</b>, 2322–2330 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXitVGrtbnF" aria-label="View reference 17 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1039%2FC5SC03115J" aria-label="View reference 17">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 17 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20poised%20fragment%20library%20enables%20rapid%20synthetic%20expansion%20yielding%20the%20first%20reported%20inhibitors%20of%20PHIP%282%29%2C%20an%20atypical%20bromodomain&amp;journal=Chem.%20Sci.&amp;volume=7&amp;pages=2322-2330&amp;publication_year=2016&amp;author=Cox%2COB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="CW. Murray, ML. Verdonk, "><meta itemprop="datePublished" content="2002"><meta itemprop="headline" content="Murray, C. W. &amp; Verdonk, M. L. The consequences of translational and rotational entropy lost by small molecule"><span class="c-article-references__counter">18</span><p class="c-article-references__text" id="ref-CR18">Murray,
 C. W. &amp; Verdonk, M. L. The consequences of translational and 
rotational entropy lost by small molecules on binding to proteins. <i>J. Comput. Aided. Mol. Des.</i> <b>16</b>, 741–753 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXht1Wktrw%253D" aria-label="View reference 18 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2002JCAMD..16..741M" aria-label="View reference 18 on ADS">ADS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1023%2FA%3A1022446720849" aria-label="View reference 18">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 18 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20consequences%20of%20translational%20and%20rotational%20entropy%20lost%20by%20small%20molecules%20on%20binding%20to%20proteins&amp;journal=J.%20Comput.%20Aided.%20Mol.%20Des.&amp;volume=16&amp;pages=741-753&amp;publication_year=2002&amp;author=Murray%2CCW&amp;author=Verdonk%2CML">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="WTM. Mooij, "><meta itemprop="datePublished" content="2006"><meta itemprop="headline" content="Mooij, W. T. M. et al. Automated protein-ligand crystallography for structure-based drug design. Chem. Med. Ch"><span class="c-article-references__counter">19</span><p class="c-article-references__text" id="ref-CR19">Mooij, W. T. M. et al. Automated protein-ligand crystallography for structure-based drug design. <i>Chem. Med. Chem.</i> <b>1</b>, 827–838 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXjsVCmtbc%253D" aria-label="View reference 19 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1002%2Fcmdc.200600074" aria-label="View reference 19">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 19 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20protein-ligand%20crystallography%20for%20structure-based%20drug%20design&amp;journal=Chem.%20Med.%20Chem.&amp;volume=1&amp;pages=827-838&amp;publication_year=2006&amp;author=Mooij%2CWTM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="IJ. Tickle, "><meta itemprop="datePublished" content="2012"><meta itemprop="headline" content="Tickle, I. J. Statistical quality indicators for electron-density maps. Acta Crystallogr. Sect. D Biol. Crysta"><span class="c-article-references__counter">20</span><p class="c-article-references__text" id="ref-CR20">Tickle, I. J. Statistical quality indicators for electron-density maps. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>68</b>, 454–467 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38Xlt1Gns70%253D" aria-label="View reference 20 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0907444911035918" aria-label="View reference 20">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 20 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Statistical%20quality%20indicators%20for%20electron-density%20maps&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=68&amp;pages=454-467&amp;publication_year=2012&amp;author=Tickle%2CIJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="RA. Nicholls, F. Long, GN. Murshudov, "><meta itemprop="datePublished" content="2012"><meta itemprop="headline" content="Nicholls, R. A., Long, F. &amp; Murshudov, G. N. Low-resolution refinement tools in REFMAC5. Acta Crystallogr. Sec"><span class="c-article-references__counter">21</span><p class="c-article-references__text" id="ref-CR21">Nicholls, R. A., Long, F. &amp; Murshudov, G. N. Low-resolution refinement tools in REFMAC5. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>68</b>, 404–417 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38Xlt1Gnsr8%253D" aria-label="View reference 21 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS090744491105606X" aria-label="View reference 21">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 21 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Low-resolution%20refinement%20tools%20in%20REFMAC5&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=68&amp;pages=404-417&amp;publication_year=2012&amp;author=Nicholls%2CRA&amp;author=Long%2CF&amp;author=Murshudov%2CGN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="HM. Berman, "><meta itemprop="datePublished" content="2000"><meta itemprop="headline" content="Berman, H. M. et al. The Protein Data Bank. Nucleic Acids Res. 28, 235–242 (2000)."><span class="c-article-references__counter">22</span><p class="c-article-references__text" id="ref-CR22">Berman, H. M. et al. The Protein Data Bank. <i>Nucleic Acids Res.</i> <b>28</b>, 235–242 (2000).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" aria-label="View reference 22 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2000spam.book.....B" aria-label="View reference 22 on ADS">ADS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1093%2Fnar%2F28.1.235" aria-label="View reference 22">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 22 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Protein%20Data%20Bank&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=235-242&amp;publication_year=2000&amp;author=Berman%2CHM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Schiebel, "><meta itemprop="datePublished" content="2016"><meta itemprop="headline" content="Schiebel, J. et al. Six Biophysical Screening Methods Miss a Large Proportion of Crystallographically Discover"><span class="c-article-references__counter">23</span><p class="c-article-references__text" id="ref-CR23">Schiebel,
 J. et al. Six Biophysical Screening Methods Miss a Large Proportion of 
Crystallographically Discovered Fragment Hits: A Case Study. <i>ACS Chem. Biol.</i> <b>11</b>, 1693–1701 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XltVagt7c%253D" aria-label="View reference 23 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1021%2Facschembio.5b01034" aria-label="View reference 23">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 23 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Six%20Biophysical%20Screening%20Methods%20Miss%20a%20Large%20Proportion%20of%20Crystallographically%20Discovered%20Fragment%20Hits%3A%20A%20Case%20Study&amp;journal=ACS%20Chem.%20Biol.&amp;volume=11&amp;pages=1693-1701&amp;publication_year=2016&amp;author=Schiebel%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="M. Mueller, M. Wang, C. Schulze-Briese, "><meta itemprop="datePublished" content="2012"><meta itemprop="headline" content="Mueller, M., Wang, M. &amp; Schulze-Briese, C. Optimal fine phi-slicing for single-photon-counting pixel detectors"><span class="c-article-references__counter">24</span><p class="c-article-references__text" id="ref-CR24">Mueller, M., Wang, M. &amp; Schulze-Briese, C. Optimal fine phi-slicing for single-photon-counting pixel detectors. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>68</b>, 42–56 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXhs1Krtr7E" aria-label="View reference 24 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0907444911049833" aria-label="View reference 24">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 24 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Optimal%20fine%20phi-slicing%20for%20single-photon-counting%20pixel%20detectors&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=68&amp;pages=42-56&amp;publication_year=2012&amp;author=Mueller%2CM&amp;author=Wang%2CM&amp;author=Schulze-Briese%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="JR. Helliwell, EP. Mitchell, "><meta itemprop="datePublished" content="2015"><meta itemprop="headline" content="Helliwell, J. R. &amp; Mitchell, E. P. Synchrotron radiation macromolecular crystallography: Science and spin-offs"><span class="c-article-references__counter">25</span><p class="c-article-references__text" id="ref-CR25">Helliwell, J. R. &amp; Mitchell, E. P. Synchrotron radiation macromolecular crystallography: Science and spin-offs. <i>IUCrJ</i> <b>2</b>, 283–291 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXjs1Gmsr4%253D" aria-label="View reference 25 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS205225251402795X" aria-label="View reference 25">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 25 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Synchrotron%20radiation%20macromolecular%20crystallography%3A%20Science%20and%20spin-offs&amp;journal=IUCrJ&amp;volume=2&amp;pages=283-291&amp;publication_year=2015&amp;author=Helliwell%2CJR&amp;author=Mitchell%2CEP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Schiebel, "><meta itemprop="datePublished" content="2016"><meta itemprop="headline" content="Schiebel, J. et al. High-throughput crystallography: reliable and efficient identification of fragment hits. S"><span class="c-article-references__counter">26</span><p class="c-article-references__text" id="ref-CR26">Schiebel, J. et al. High-throughput crystallography: reliable and efficient identification of fragment hits. <i>Structure</i> <b>24</b>, 1398–1409 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28Xht1eju7fN" aria-label="View reference 26 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.str.2016.06.010" aria-label="View reference 26">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 26 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=High-throughput%20crystallography%3A%20reliable%20and%20efficient%20identification%20of%20fragment%20hits&amp;journal=Structure&amp;volume=24&amp;pages=1398-1409&amp;publication_year=2016&amp;author=Schiebel%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Foadi, "><meta itemprop="datePublished" content="2013"><meta itemprop="headline" content="Foadi, J. et al. Clustering procedures for the optimal selection of data sets from multiple crystals in macrom"><span class="c-article-references__counter">27</span><p class="c-article-references__text" id="ref-CR27">Foadi,
 J. et al. Clustering procedures for the optimal selection of data sets 
from multiple crystals in macromolecular crystallography. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>69</b>, 1617–1632 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhtFygsrfK" aria-label="View reference 27 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0907444913012274" aria-label="View reference 27">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 27 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Clustering%20procedures%20for%20the%20optimal%20selection%20of%20data%20sets%20from%20multiple%20crystals%20in%20macromolecular%20crystallography&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=69&amp;pages=1617-1632&amp;publication_year=2013&amp;author=Foadi%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="MD. Winn, "><meta itemprop="datePublished" content="2011"><meta itemprop="headline" content="Winn, M. D. et al. Overview of the CCP4 suite and current developments. Acta Crystallogr. Sect. D Biol. Crysta"><span class="c-article-references__counter">28</span><p class="c-article-references__text" id="ref-CR28">Winn, M. D. et al. Overview of the CCP4 suite and current developments. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>67</b>, 235–242 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" aria-label="View reference 28 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2011AIPC.1374..235W" aria-label="View reference 28 on ADS">ADS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0907444910045749" aria-label="View reference 28">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 28 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Overview%20of%20the%20CCP4%20suite%20and%20current%20developments&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=67&amp;pages=235-242&amp;publication_year=2011&amp;author=Winn%2CMD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="PV. Afonine, "><meta itemprop="datePublished" content="2012"><meta itemprop="headline" content="Afonine, P. V. et al. Towards automated crystallographic structure refinement with phenix.refine. Acta Crystal"><span class="c-article-references__counter">29</span><p class="c-article-references__text" id="ref-CR29">Afonine, P. V. et al. Towards automated crystallographic structure refinement with phenix.refine. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>68</b>, 352–367 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" aria-label="View reference 29 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0907444912001308" aria-label="View reference 29">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 29 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20automated%20crystallographic%20structure%20refinement%20with%20phenix.refine&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=68&amp;pages=352-367&amp;publication_year=2012&amp;author=Afonine%2CPV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="PD. Adams, "><meta itemprop="datePublished" content="2010"><meta itemprop="headline" content="Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Cr"><span class="c-article-references__counter">30</span><p class="c-article-references__text" id="ref-CR30">Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>66</b>, 213–221 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" aria-label="View reference 30 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0907444909052925" aria-label="View reference 30">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 30 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=PHENIX%3A%20a%20comprehensive%20Python-based%20system%20for%20macromolecular%20structure%20solution&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=66&amp;pages=213-221&amp;publication_year=2010&amp;author=Adams%2CPD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="RW. Grosse-Kunstleve, NK. Sauter, NW. Moriarty, PD. Adams, "><meta itemprop="datePublished" content="2002"><meta itemprop="headline" content="Grosse-Kunstleve, R. W., Sauter, N. K., Moriarty, N. W. &amp; Adams, P. D. The Computational Crystallography Toolb"><span class="c-article-references__counter">31</span><p class="c-article-references__text" id="ref-CR31">Grosse-Kunstleve,
 R. W., Sauter, N. K., Moriarty, N. W. &amp; Adams, P. D. The 
Computational Crystallography Toolbox: crystallographic algorithms in a 
reusable software framework. <i>J. Appl. Crystallogr.</i> <b>35</b>, 126–136 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD38XltVajug%253D%253D" aria-label="View reference 31 on CAS">CAS</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1107%2FS0021889801017824" aria-label="View reference 31">Article</a>&nbsp;
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 31 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Computational%20Crystallography%20Toolbox%3A%20crystallographic%20algorithms%20in%20a%20reusable%20software%20framework&amp;journal=J.%20Appl.%20Crystallogr.&amp;volume=35&amp;pages=126-136&amp;publication_year=2002&amp;author=Grosse-Kunstleve%2CRW&amp;author=Sauter%2CNK&amp;author=Moriarty%2CNW&amp;author=Adams%2CPD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Krojer, T., Pearce, N. M., Bradley, A., Marsden, B. D. &amp; von Delft, F. PanDDA analysis of BAZ2B screened again"><span class="c-article-references__counter">32</span><p class="c-article-references__text" id="ref-CR32">Krojer, T., Pearce, N. M., Bradley, A., Marsden, B. D. &amp; von Delft, F. <i>PanDDA analysis of BAZ2B screened against Zenobia Fragment Library</i> (HTML Summary). Zenodo. Available at: https://zenodo.org/record/290199 (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Krojer, T., Pearce, N. M., Bradley, A., Marsden, B. D. &amp; von Delft, F. PanDDA analysis of JMJD2D screened agai"><span class="c-article-references__counter">33</span><p class="c-article-references__text" id="ref-CR33">Krojer, T., Pearce, N. M., Bradley, A., Marsden, B. D. &amp; von Delft, F. <i>PanDDA analysis of JMJD2D screened against Zenobia Fragment Library</i> (HTML Summary). Zenodo. Available at: https://zenodo.org/record/290220 (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Krojer, T., Pearce, N. M., Collins, P., Talon, R. &amp; von Delft, F. PanDDA analysis of BRD1 screened against 3D-"><span class="c-article-references__counter">34</span><p class="c-article-references__text" id="ref-CR34">Krojer, T., Pearce, N. M., Collins, P., Talon, R. &amp; von Delft, F. <i>PanDDA analysis of BRD1 screened against 3D-Fragment-Consortium Fragment Library</i>. (HTML Summary) Zenodo. Available at: https://zenodo.org/record/290217 (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Krojer, T., Pearce, N. M., Collins, P., Talon, R. &amp; von Delft, F. PanDDA analysis of SP100 screened against se"><span class="c-article-references__counter">35</span><p class="c-article-references__text" id="ref-CR35">Krojer, T., Pearce, N. M., Collins, P., Talon, R. &amp; von Delft, F. <i>PanDDA analysis of SP100 screened against selection of Maybridge Fragment Library</i> (HTML Summary). Zenodo. Available at: https://zenodo.org/record/290201 (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Pearce, N. M., Bradley, A. R., Marsden, B. D. &amp; von Delft, F. PanDDA analysis of BAZ2B screened against Zenobi"><span class="c-article-references__counter">36</span><p class="c-article-references__text" id="ref-CR36">Pearce,
 N. M., Bradley, A. R., Marsden, B. D. &amp; von Delft, F. PanDDA 
analysis of BAZ2B screened against Zenobia Fragment Library [Dataset]. 
Zenodo. Available at: https://zenodo.org/record/48768 (2016).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Pearce, N. M., Bradley, A. R., Marsden, B. D. &amp; von Delft, F. PanDDA analysis of JMJD2D screened against Zenob"><span class="c-article-references__counter">37</span><p class="c-article-references__text" id="ref-CR37">Pearce,
 N. M., Bradley, A. R., Marsden, B. D. &amp; von Delft, F. PanDDA 
analysis of JMJD2D screened against Zenobia Fragment Library [Dataset]. 
Zenodo. Available at: https://zenodo.org/record/48770 (2016).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Pearce, N. M., Krojer, T., Collins, P., Talon, R. &amp; von Delft, F. PanDDA analysis of BRD1 screened against 3D-"><span class="c-article-references__counter">38</span><p class="c-article-references__text" id="ref-CR38">Pearce,
 N. M., Krojer, T., Collins, P., Talon, R. &amp; von Delft, F. PanDDA 
analysis of BRD1 screened against 3D-Fragment Consortium Fragment 
Library [Dataset]. Zenodo. Available at: https://zenodo.org/record/48769
 (2016).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Pearce, N. M., Krojer, T., Collins, P., Talon, R. &amp; von Delft, F. PanDDA analysis of SP100 screened against se"><span class="c-article-references__counter">39</span><p class="c-article-references__text" id="ref-CR39">Pearce,
 N. M., Krojer, T., Collins, P., Talon, R. &amp; von Delft, F. PanDDA 
analysis of SP100 screened against selection of Maybridge Fragment 
Library [Dataset]. Zenodo. Available at: https://zenodo.org/record/48771
 (2016).</p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" href="https://www.nature.com/articles/ncomms15123-references.ris">Download references<svg width="16" height="16" class="u-icon"><use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p></div></div></div></section><section aria-labelledby="Ack1" data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We
 thank Randy Read and Garib Murshudov for many productive conversations,
 and Luis Ospina for discussions regarding the statistical model. All 
datasets were collected at Diamond Light Source beamline I03, and at 
Diamond Light Source beamline I04-1 as part of the SGC-Diamond I04-1 
XChem partnership. NMP and CMD recognize funding from EPSRC grant 
EP/G037280/1, UCB Pharma and Diamond Light Source. The SGC is a 
registered charity (No. 1097737) that receives funds from AbbVie, Bayer,
 Boehringer Ingelheim, the Canada Foundation for Innovation, the 
Canadian Institutes for Health Research, Genome Canada, GlaxoSmithKline,
 Janssen, Lilly Canada, the Novartis Research Foundation, the Ontario 
Ministry of Economic Development and Innovation, Pfizer, Takeda and the 
Wellcome Trust (092809/Z/10/Z).</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK</p><p class="c-article-author-affiliation__authors-list">Nicholas
 M. Pearce,&nbsp;Tobias Krojer,&nbsp;Anthony R. Bradley,&nbsp;Radosław 
P. Nowak,&nbsp;Romain Talon,&nbsp;Brian D. Marsden&nbsp;&amp;&nbsp;Frank
 von Delft</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Diamond Light Source Ltd, Harwell Science and Innovation Campus, Didcot, OX11 0QX, UK</p><p class="c-article-author-affiliation__authors-list">Patrick Collins&nbsp;&amp;&nbsp;Frank von Delft</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Nuffield
 Department of Orthopaedics, Kennedy Institute of Rheumatology, 
Rheumatology and Musculoskeletal Sciences, University of Oxford, 
Roosevelt Drive, Oxford, OX3 7FY, UK</p><p class="c-article-author-affiliation__authors-list">Brian D. Marsden</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">UCB Pharma, 208 Bath Road, Slough, SL1 3WE, UK</p><p class="c-article-author-affiliation__authors-list">Sebastian Kelm&nbsp;&amp;&nbsp;Jiye Shi</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Department of Statistics, University of Oxford, 24-29 St Giles, Oxford, OX1 3LB, UK</p><p class="c-article-author-affiliation__authors-list">Charlotte M. Deane</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Department of Biochemistry, University of Johannesburg, Auckland Park, 2006, South Africa</p><p class="c-article-author-affiliation__authors-list">Frank von Delft</p></li></ol><div class="js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Nicholas_M_-Pearce"><span class="c-article-authors-search__title u-h3 js-search-name">Nicholas M. Pearce</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Nicholas%20M.+Pearce%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nicholas%20M.+Pearce" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nicholas%20M.+Pearce%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Tobias-Krojer"><span class="c-article-authors-search__title u-h3 js-search-name">Tobias Krojer</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Tobias+Krojer%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tobias+Krojer" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tobias+Krojer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Anthony_R_-Bradley"><span class="c-article-authors-search__title u-h3 js-search-name">Anthony R. Bradley</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Anthony%20R.+Bradley%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Anthony%20R.+Bradley" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Anthony%20R.+Bradley%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Patrick-Collins"><span class="c-article-authors-search__title u-h3 js-search-name">Patrick Collins</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Patrick+Collins%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Patrick+Collins" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Patrick+Collins%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Rados_aw_P_-Nowak"><span class="c-article-authors-search__title u-h3 js-search-name">Radosław P. Nowak</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Rados%C5%82aw%20P.+Nowak%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Rados%C5%82aw%20P.+Nowak" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Rados%C5%82aw%20P.+Nowak%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Romain-Talon"><span class="c-article-authors-search__title u-h3 js-search-name">Romain Talon</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Romain+Talon%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Romain+Talon" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Romain+Talon%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Brian_D_-Marsden"><span class="c-article-authors-search__title u-h3 js-search-name">Brian D. Marsden</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Brian%20D.+Marsden%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Brian%20D.+Marsden" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Brian%20D.+Marsden%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Sebastian-Kelm"><span class="c-article-authors-search__title u-h3 js-search-name">Sebastian Kelm</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Sebastian+Kelm%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sebastian+Kelm" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sebastian+Kelm%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Jiye-Shi"><span class="c-article-authors-search__title u-h3 js-search-name">Jiye Shi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Jiye+Shi%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jiye+Shi" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jiye+Shi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Charlotte_M_-Deane"><span class="c-article-authors-search__title u-h3 js-search-name">Charlotte M. Deane</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Charlotte%20M.+Deane%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Charlotte%20M.+Deane" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Charlotte%20M.+Deane%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-Frank-von_Delft"><span class="c-article-authors-search__title u-h3 js-search-name">Frank von Delft</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Frank+von%20Delft%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Frank+von%20Delft" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Frank+von%20Delft%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>N.M.P.,
 C.M.D. and F. v.D. designed and conducted the research. N.M.P. 
developed and implemented the algorithm. A.R.B., P.C., T.K., R.P.N. and 
R.T. performed the experiments that yielded the crystallographic data. 
N.M.P. analysed the processed data. N.M.P., T.K. and R.T. established 
model validation criteria. T.K. enabled the deposition of the structures
 in the PDB and generated the HTML summary pages. B.D.M., S.K. and J.S. 
were involved in discussions of the project and revising the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" rel="nofollow" href="https://www.nature.com/articles/ncomms15123/email/correspondent/c1/new">Frank von Delft</a>.</p></div></div></section><section aria-labelledby="ethics" data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading">Competing interests</h3>
                <p>The authors declare no competing financial interests.</p>
              
            </div></div></section><section aria-labelledby="Sec19" data-title="Supplementary information"><div class="c-article-section" id="Sec19-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec19">Supplementary information</h2><div class="c-article-section__content" id="Sec19-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div>
                  
                <div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3535"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_BFncomms15123_MOESM3535_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary figures, supplementary tables, supplementary notes and supplementary methods. (PDF 9813 kb)</p></div></div>
                  
                <div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3536"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_BFncomms15123_MOESM3536_ESM.pdf" data-supp-info-image="">Peer Review File (PDF 82 kb)</a></h3></div></div></div></div></section><section aria-labelledby="rightslink" data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p>This work is licensed under a Creative Commons 
Attribution 4.0 International License. The images or other third party 
material in this article are included in the article’s Creative Commons 
license, unless indicated otherwise in the credit line; if the material 
is not included under the Creative Commons license, users will need to 
obtain permission from the license holder to reproduce the material. To 
view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license" itemprop="license">http://creativecommons.org/licenses/by/4.0/</a></p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=A%20multi-crystal%20method%20for%20extracting%20obscured%20crystallographic%20states%20from%20conventionally%20uninterpretable%20electron%20density&amp;author=Nicholas%20M.%20Pearce%20et%20al&amp;contentID=10.1038%2Fncomms15123&amp;copyright=The%20Author%28s%29&amp;publication=2041-1723&amp;publicationDate=2017-04-24&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/ncomms15123" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/ncomms15123" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img alt="Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>" width="57" height="81"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Pearce, N., Krojer, T., Bradley, A. <i>et al.</i> A multi-crystal method for extracting obscured crystallographic states from conventionally uninterpretable electron density.
                    <i>Nat Commun</i> <b>8, </b>15123 (2017). https://doi.org/10.1038/ncomms15123</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" href="https://www.nature.com/articles/ncomms15123.ris">Download citation<svg width="16" height="16" class="u-icon"><use xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2016-11-01">01 November 2016</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2017-03-03">03 March 2017</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2017-04-24">24 April 2017</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--doi"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value"><a href="https://doi.org/10.1038/ncomms15123" data-track="click" data-track-action="view doi" data-track-label="link" itemprop="sameAs">https://doi.org/10.1038/ncomms15123</a></span></p></li></ul><div data-component="share-box"><div class="c-article-share-box"><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" id="get-share-url" data-track="click" data-track-label="button" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" aria-hidden="true"><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" aria-hidden="true"><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"> 		</p><button class="js-copy-share-url c-article-share-box__button--link-like" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">Provided by the Springer Nature SharedIt content-sharing initiative</p></div></div><div data-component="article-info-list">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="https://www.nature.com/subjects/proteins" data-track="click" data-track-action="view subject" data-track-label="link" itemprop="about">Proteins</a></li><li class="c-article-subject-list__subject"><a href="https://www.nature.com/subjects/screening" data-track="click" data-track-action="view subject" data-track-label="link" itemprop="about">Screening</a></li><li class="c-article-subject-list__subject"><a href="https://www.nature.com/subjects/x-ray-crystallography" data-track="click" data-track-action="view subject" data-track-label="link" itemprop="about">X-ray crystallography</a></li>
        </ul>
    </div></div></div></div></div></section>

                
    <section aria-labelledby="further-reading">
      <div class="c-article-section js-article-section" id="further-reading-section">
        <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">Further reading</h2>
        <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
        <ul class="c-article-further-reading__list" id="further-reading-list">
          
            <li class="c-article-further-reading__item js-ref-item" id="further-reading-item-1">
              <h3 class="c-article-further-reading__title">
                <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Discovery of allosteric binding sites by crystallographic fragment screening" href="https://doi.org/10.1016/j.sbi.2020.08.004" id="title-link-1">
                  Discovery of allosteric binding sites by crystallographic fragment screening
                </a>
              </h3>
              
                <ul data-test="author-list" class="c-article-further-reading__authors-list js-list-authors-3 js-etal-collapsed js-no-scroll" data-component-authors-activator="authors-list" id="authors-1">
                  <li><span class="js-separator"></span><span>Tobias Krojer</span></li><li><span class="js-separator">, </span><span>James S Fraser</span></li><li><span class="js-separator">&nbsp;&amp;&nbsp;</span><span>Frank von Delft</span></li></ul>
              
              <p class="c-article-further-reading__journal-title" id="journal-name-1">
                <i>Current Opinion in Structural Biology</i>
                (2020)
              </p>
            </li>
          
            <li class="c-article-further-reading__item js-ref-item" id="further-reading-item-2">
              <h3 class="c-article-further-reading__title">
                <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:FragMAX: the fragment-screening platform at the MAX IV Laboratory" href="https://doi.org/10.1107/S205979832000889X" id="title-link-2">
                  FragMAX: the fragment-screening platform at the MAX IV Laboratory
                </a>
              </h3>
              
                <ul data-test="author-list" class="c-article-further-reading__authors-list js-list-authors-3 js-etal-collapsed js-no-scroll" data-component-authors-activator="authors-list" id="authors-2">
                  <li><span class="js-separator"></span><span>Gustavo M. A. Lima</span></li><li><span class="js-separator">, </span><span>Vladimir O. Talibov</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Elmir Jagudin</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Céleste Sele</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Maria Nyblom</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Wolfgang Knecht</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Derek T. Logan</span></li><li class="c-author-list__show-more "><a class="js-etal" href="javascript:;" aria-label="Show all 9 authors for this article" title="Show all 9 authors for this article">[…]</a></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Tove Sjögren</span></li><li><span class="js-separator">&nbsp;&amp;&nbsp;</span><span>Uwe Mueller</span></li><li class="c-author-list__show-less u-js-hide"><a class="js-etal" href="javascript:;" aria-label="Show fewer authors for this article" title="Show fewer authors for this article">-Show fewer authors</a></li></ul>
              
              <p class="c-article-further-reading__journal-title" id="journal-name-2">
                <i>Acta Crystallographica Section D Structural Biology</i>
                (2020)
              </p>
            </li>
          
            <li class="c-article-further-reading__item js-ref-item" id="further-reading-item-3">
              <h3 class="c-article-further-reading__title">
                <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Mining the PDB for Tractable Cases Where X-ray Crystallography Combined with Fragment Screens Can Be Used to Systematically Design Protein–Protein Inhibitors: Two Test Cases Illustrated by IL1β-IL1R and p38α–TAB1 Complexes" href="https://doi.org/10.1021/acs.jmedchem.0c00403" id="title-link-3">
                  Mining the PDB for Tractable Cases Where X-ray 
Crystallography Combined with Fragment Screens Can Be Used to 
Systematically Design Protein–Protein Inhibitors: Two Test Cases 
Illustrated by IL1β-IL1R and p38α–TAB1 Complexes
                </a>
              </h3>
              
                <ul data-test="author-list" class="c-article-further-reading__authors-list js-list-authors-3 js-etal-collapsed js-no-scroll" data-component-authors-activator="authors-list" id="authors-3">
                  <li><span class="js-separator"></span><span>Charlie Nichols</span></li><li><span class="js-separator">, </span><span>Joseph Ng</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Annika Keshu</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Geoff Kelly</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Maria R. Conte</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Michael S. Marber</span></li><li class="c-author-list__show-more "><a class="js-etal" href="javascript:;" aria-label="Show all 8 authors for this article" title="Show all 8 authors for this article">[…]</a></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Franca Fraternali</span></li><li><span class="js-separator">&nbsp;&amp;&nbsp;</span><span>Gian F. De Nicola</span></li><li class="c-author-list__show-less u-js-hide"><a class="js-etal" href="javascript:;" aria-label="Show fewer authors for this article" title="Show fewer authors for this article">-Show fewer authors</a></li></ul>
              
              <p class="c-article-further-reading__journal-title" id="journal-name-3">
                <i>Journal of Medicinal Chemistry</i>
                (2020)
              </p>
            </li>
          
            <li class="c-article-further-reading__item js-ref-item" id="further-reading-item-4">
              <h3 class="c-article-further-reading__title">
                <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:F2X-Universal and F2X-Entry: Structurally Diverse Compound Libraries for Crystallographic Fragment Screening" href="https://doi.org/10.1016/j.str.2020.04.019" id="title-link-4">
                  F2X-Universal and F2X-Entry: Structurally Diverse Compound Libraries for Crystallographic Fragment Screening
                </a>
              </h3>
              
                <ul data-test="author-list" class="c-article-further-reading__authors-list js-list-authors-3 js-etal-collapsed js-no-scroll" data-component-authors-activator="authors-list" id="authors-4">
                  <li><span class="js-separator"></span><span>Jan Wollenhaupt</span></li><li><span class="js-separator">, </span><span>Alexander Metz</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Tatjana Barthel</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Gustavo M.A. Lima</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Andreas Heine</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Uwe Mueller</span></li><li class="c-author-list__show-more "><a class="js-etal" href="javascript:;" aria-label="Show all 8 authors for this article" title="Show all 8 authors for this article">[…]</a></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Gerhard Klebe</span></li><li><span class="js-separator">&nbsp;&amp;&nbsp;</span><span>Manfred S. Weiss</span></li><li class="c-author-list__show-less u-js-hide"><a class="js-etal" href="javascript:;" aria-label="Show fewer authors for this article" title="Show fewer authors for this article">-Show fewer authors</a></li></ul>
              
              <p class="c-article-further-reading__journal-title" id="journal-name-4">
                <i>Structure</i>
                (2020)
              </p>
            </li>
          
            <li class="c-article-further-reading__item js-ref-item" id="further-reading-item-5">
              <h3 class="c-article-further-reading__title">
                <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Structural characterization of melatonin as an inhibitor of the Wnt deacylase Notum" href="https://doi.org/10.1111/jpi.12630" id="title-link-5">
                  Structural characterization of melatonin as an inhibitor of the Wnt deacylase Notum
                </a>
              </h3>
              
                <ul data-test="author-list" class="c-article-further-reading__authors-list js-list-authors-3 js-etal-collapsed js-no-scroll" data-component-authors-activator="authors-list" id="authors-5">
                  <li><span class="js-separator"></span><span>Yuguang Zhao</span></li><li><span class="js-separator">, </span><span>Jingshan Ren</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>James Hillier</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Margaret Jones</span></li><li class="c-author-list__show-more "><a class="js-etal" href="javascript:;" aria-label="Show all 6 authors for this article" title="Show all 6 authors for this article">[…]</a></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Weixian Lu</span></li><li><span class="js-separator">&nbsp;&amp;&nbsp;</span><span>Edith Yvonne Jones</span></li><li class="c-author-list__show-less u-js-hide"><a class="js-etal" href="javascript:;" aria-label="Show fewer authors for this article" title="Show fewer authors for this article">-Show fewer authors</a></li></ul>
              
              <p class="c-article-further-reading__journal-title" id="journal-name-5">
                <i>Journal of Pineal Research</i>
                (2020)
              </p>
            </li>
          
        </ul>
      </div>
    </div>
    </section>
  

                
                    <section aria-labelledby="article-comments" data-title="Comments"><div class="c-article-section" id="article-comments-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-comments">Comments</h2><div class="c-article-section__content" id="article-comments-content"><p>By submitting a comment you agree to abide by our <a href="https://www.nature.com/info/tandc.html">Terms</a> and <a href="https://www.nature.com/info/community-guidelines.html">Community Guidelines</a>. If you find something abusive or that does not comply with our terms or guidelines please flag it as inappropriate.</p>
    <div id="disqus_thread" class="pb20" data-object="disqus" data-disqus-s3="e30= 3cfd1ca824192ac22199364b34e86d0e881da36a 1606163364" data-disqus-key="ZZDdHrSs8UQRH0LAFbG4KmcMfQdvUwYDEyYczGZhznVSuj9QDfXFgWxFA1tz53jo" data-disqus-sso-name="Nature" data-disqus-sso-button="//www.nature.com/static/images/buttons/disqus-sso-login-button.e9d0ed0791.png" data-disqus-sso-icon="//www.nature.com/static/images/favicons/nature/favicon.62367f778b.ico" data-disqus-sso-url="https://idp.nature.com/authorize/natureuser?client_id=grover&amp;redirect_uri=http%3A%2F%2Fwww.nature.com%2Farticles%2Fncomms15123%2Fdisqus" data-disqus-sso-logout="https://www.nature.com/foxtrot/svc/logout?target=https%3A%2F%2Fwww.nature.com%2Farticles%2Fncomms15123" data-disqus-shortname="ncomms" data-disqus-identifier="10.1038/ncomms15123" data-disqus-url="https://www.nature.com/articles/ncomms15123">
    </div>
</div></div></section>
                    
                

                <iframe style="width: 100%; height: 0px; bottom: 0px; right: 0px; z-index: 100000; padding: 0px; margin-top: 28px;" scrolling="no" src="https://recommended.springernature.com/v2397/generated/job-client.html?source=https://www.nature.com" name="job-client" id="job-client" title="Personalised recommendations" allowtransparency="true" frameborder="0"></iframe>
            </div>
        </article>
    </main>

    <div class="c-article-extras u-hide-print" role="complementary" data-container-type="reading-companion" data-track-component="reading companion">
        
            
                
    
    <div class="c-pdf-download u-clear-both">
        <a href="https://www.nature.com/articles/ncomms15123.pdf" class="c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" download="">
            <span>Download PDF</span>
            <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"></use></svg>
        </a>
    </div>
    

            
        

        
            
        

        

        <div class="c-reading-companion">
            <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
                <ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-track="click" data-track-action="sections tab" data-track-label="tab">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="figures tab" data-track-label="tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="references tab" data-track-label="tab">References</button></li></ul><div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" tabindex="0"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Introduction">Introduction</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Results">Results</a></li><li id="rc-sec-Sec6" class="c-reading-companion__section-item"><a href="#Sec6" data-track="click" data-track-action="section anchor" data-track-label="link:Discussion">Discussion</a></li><li id="rc-sec-Sec7" class="c-reading-companion__section-item"><a href="#Sec7" data-track="click" data-track-action="section anchor" data-track-label="link:Methods">Methods</a></li><li id="rc-sec-Sec18" class="c-reading-companion__section-item"><a href="#Sec18" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-ethics" class="c-reading-companion__section-item"><a href="#ethics" data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations">Ethics declarations</a></li><li id="rc-sec-Sec19" class="c-reading-companion__section-item"><a href="#Sec19" data-track="click" data-track-action="section anchor" data-track-label="link:Supplementary information">Supplementary information</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li><li id="rc-sec-further-reading" class="c-reading-companion__section-item"><a href="#further-reading" data-track="click" data-track-action="section anchor" data-track-label="link:Further reading">Further reading</a></li><li id="rc-sec-article-comments" class="c-reading-companion__section-item"><a href="#article-comments" data-track="click" data-track-action="section anchor" data-track-label="link:Comments">Comments</a></li></ul></div>
                    
    <div id="div-gpt-ad-right-2" class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-gpt-unitpath="/285/nature_communications/article" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=ncomms15123;doi=10.1038/ncomms15123;techmeta=101,119,82,9;subjmeta=1266,45,507,535,612,631,92;kwrd=Proteins,Screening,X-ray crystallography">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature_communications/article&amp;sz=300x250&amp;c=-482930428&amp;t=pos%3Dright%26type%3Darticle%26artid%3Dncomms15123%26doi%3D10.1038/ncomms15123%26techmeta%3D101,119,82,9%26subjmeta%3D1266,45,507,535,612,631,92%26kwrd%3DProteins,Screening,X-ray crystallography">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature_communications/article&amp;sz=300x250&amp;c=-482930428&amp;t=pos%3Dright%26type%3Darticle%26artid%3Dncomms15123%26doi%3D10.1038/ncomms15123%26techmeta%3D101,119,82,9%26subjmeta%3D1266,45,507,535,612,631,92%26kwrd%3DProteins,Screening,X-ray crystallography"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                </div>
                <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig1">Figure 1: Schematic of how outlier identification and background subtraction reveals changed-state density.</b></figcaption><picture><source type="image/webp" data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig1_HTML.jpg?as=webp"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig1_HTML.jpg" alt="figure1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/ncomms15123/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image<svg width="16" height="16" class="u-icon"><use href="#global-icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig2">Figure 2: Minor conformations are obscured in conventional maps but are revealed by background correction.</b></figcaption><picture><source type="image/webp" data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig2_HTML.jpg?as=webp"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig2_HTML.jpg" alt="figure2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/ncomms15123/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image<svg width="16" height="16" class="u-icon"><use href="#global-icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig3">Figure 3: PanDDA maps clearly show detail obscured by conventional maps.</b></figcaption><picture><source type="image/webp" data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig3_HTML.jpg?as=webp"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig3_HTML.jpg" alt="figure3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/ncomms15123/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image<svg width="16" height="16" class="u-icon"><use href="#global-icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig4">Figure 4: PanDDA maps reveal complex minor conformations and identify allosteric binders.</b></figcaption><picture><source type="image/webp" data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig4_HTML.jpg?as=webp"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig4_HTML.jpg" alt="figure4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/ncomms15123/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image<svg width="16" height="16" class="u-icon"><use href="#global-icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig5">Figure 5: Weak ligand identification remains straightforward when phases are degraded.</b></figcaption><picture><source type="image/webp" data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig5_HTML.jpg?as=webp"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fncomms15123/MediaObjects/41467_2017_Article_BFncomms15123_Fig5_HTML.jpg" alt="figure5"></picture><p class="c-reading-companion__figure-links"><a href="#Fig5" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/ncomms15123/figures/5" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image<svg width="16" height="16" class="u-icon"><use href="#global-icon-chevron-right"></use></svg></a></p></figure></li></ul></div></div>
                <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR1">Sheldrick, G. M. A short history of SHELX. <i>Acta Crystallogr. Sect. A Found. Crystallogr.</i> <b>64</b>, 112–122 (2007).</p><p class="c-reading-companion__reference-links"><a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2008AcCrA..64..112S" data-track="click" data-track-action="outbound reference" data-track-label="link">ADS</a>&nbsp;<a href="https://doi.org/10.1107%2FS0108767307043930" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20short%20history%20of%20SHELX&amp;journal=Acta%20Crystallogr.%20Sect.%20A%20Found.%20Crystallogr.&amp;volume=64&amp;pages=112-122&amp;publication_year=2007&amp;author=Sheldrick%2CGM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR2">Holton,
 J. M., Classen, S., Frankel, K. A. &amp; Tainer, J. A. The R-factor gap
 in macromolecular crystallography: an untapped potential for insights 
on accurate structures. <i>FEBS J.</i> <b>281</b>, 4046–4060 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhsFKnsb3M" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1111%2Ffebs.12922" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20R-factor%20gap%20in%20macromolecular%20crystallography%3A%20an%20untapped%20potential%20for%20insights%20on%20accurate%20structures&amp;journal=FEBS%20J.&amp;volume=281&amp;pages=4046-4060&amp;publication_year=2014&amp;author=Holton%2CJM&amp;author=Classen%2CS&amp;author=Frankel%2CKA&amp;author=Tainer%2CJA" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR3">Burnley,
 B. T., Afonine, P. V., Adams, P. D. &amp; Gros, P. Modelling dynamics 
in protein crystal structures by ensemble refinement. <i>Elife</i> <b>1</b>, e00311 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.7554%2FeLife.00311" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Modelling%20dynamics%20in%20protein%20crystal%20structures%20by%20ensemble%20refinement&amp;journal=Elife&amp;volume=1&amp;publication_year=2012&amp;author=Burnley%2CBT&amp;author=Afonine%2CPV&amp;author=Adams%2CPD&amp;author=Gros%2CP" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR4">DePristo,
 M. A., De Bakker, P. I. W., Johnson, R. J. K. &amp; Blundell, T. L. 
Crystallographic refinement by knowledge-based exploration of complex 
energy landscapes. <i>Structure</i> <b>13</b>, 1311–1319 (2005).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2MXpvFentrs%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1016%2Fj.str.2005.06.008" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Crystallographic%20refinement%20by%20knowledge-based%20exploration%20of%20complex%20energy%20landscapes&amp;journal=Structure&amp;volume=13&amp;pages=1311-1319&amp;publication_year=2005&amp;author=DePristo%2CMA&amp;author=De%20Bakker%2CPIW&amp;author=Johnson%2CRJK&amp;author=Blundell%2CTL" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR5">Van
 Den Bedem, H., Dhanik, A., Latombe, J. C. &amp; Deacon, A. M. Modeling 
discrete heterogeneity in X-ray diffraction data by fitting 
multi-conformers. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>65</b>, 1107–1117 (2009).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnJ" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1107%2FS0907444909030613" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Modeling%20discrete%20heterogeneity%20in%20X-ray%20diffraction%20data%20by%20fitting%20multi-conformers&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=65&amp;pages=1107-1117&amp;publication_year=2009&amp;author=Van%20Den%20Bedem%2CH&amp;author=Dhanik%2CA&amp;author=Latombe%2CJC&amp;author=Deacon%2CAM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR6">Lang, P. T. et al. Automated electron-density sampling reveals widespread conformational polymorphism in proteins. <i>Protein Sci.</i> <b>19</b>, 1420–1431 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXotVWhs7s%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1002%2Fpro.423" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20electron-density%20sampling%20reveals%20widespread%20conformational%20polymorphism%20in%20proteins&amp;journal=Protein%20Sci.&amp;volume=19&amp;pages=1420-1431&amp;publication_year=2010&amp;author=Lang%2CPT" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR7">Lang,
 P. T., Holton, J. M., Fraser, J. S. &amp; Alber, T. Protein structural 
ensembles are revealed by redefining X-ray electron density noise. <i>Proc. Natl Acad. Sci. USA</i> <b>111</b>, 237–242 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXps1Gisg%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2014PNAS..111..237L" data-track="click" data-track-action="outbound reference" data-track-label="link">ADS</a>&nbsp;<a href="https://doi.org/10.1073%2Fpnas.1302823110" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20structural%20ensembles%20are%20revealed%20by%20redefining%20X-ray%20electron%20density%20noise&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=237-242&amp;publication_year=2014&amp;author=Lang%2CPT&amp;author=Holton%2CJM&amp;author=Fraser%2CJS&amp;author=Alber%2CT" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR8">Read, R. J. Improved Fourier coefficients for maps using phases from partial structures with errors. <i>Acta Crystallogr. Sect. A Found. Crystallogr.</i> <b>42</b>, 140–149 (1986).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1107%2FS0108767386099622" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20Fourier%20coefficients%20for%20maps%20using%20phases%20from%20partial%20structures%20with%20errors&amp;journal=Acta%20Crystallogr.%20Sect.%20A%20Found.%20Crystallogr.&amp;volume=42&amp;pages=140-149&amp;publication_year=1986&amp;author=Read%2CRJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR9">Pozharski,
 E., Weichenberger, C. X. &amp; Rupp, B. Techniques, tools and best 
practices for ligand electron-density analysis and results from their 
application to deposited crystal structures. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>69</b>, 150–167 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXitFaiu74%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1107%2FS0907444912044423" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Techniques%2C%20tools%20and%20best%20practices%20for%20ligand%20electron-density%20analysis%20and%20results%20from%20their%20application%20to%20deposited%20crystal%20structures&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=69&amp;pages=150-167&amp;publication_year=2013&amp;author=Pozharski%2CE&amp;author=Weichenberger%2CCX&amp;author=Rupp%2CB" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR10">Stanfield, R., Pozharski, E. &amp; Rupp, B. Comment on Three X-ray Crystal Structure Papers. <i>J. Immunol.</i> <b>196</b>, 521–524 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XltFertA%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26747564" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Comment%20on%20Three%20X-ray%20Crystal%20Structure%20Papers&amp;journal=J.%20Immunol.&amp;volume=196&amp;pages=521-524&amp;publication_year=2016&amp;author=Stanfield%2CR&amp;author=Pozharski%2CE&amp;author=Rupp%2CB" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR11">Rupp,
 B. &amp; Segelke, B. Questions about the structure of the botulinum 
neurotoxin B light chain in complex with a target peptide. <i>Nat. Struct. Biol</i> <b>8</b>, 663–664 (2001).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3MXls1Ontbs%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1038%2F90361" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Questions%20about%20the%20structure%20of%20the%20botulinum%20neurotoxin%20B%20light%20chain%20in%20complex%20with%20a%20target%20peptide&amp;journal=Nat.%20Struct.%20Biol&amp;volume=8&amp;pages=663-664&amp;publication_year=2001&amp;author=Rupp%2CB&amp;author=Segelke%2CB" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR12">Kleywegt, G. J. &amp; Jones, T. A. Where freedom is given, liberties are taken. <i>Structure</i> <b>3</b>, 535–540 (1995).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2MXmvVyrsrs%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1016%2FS0969-2126%2801%2900187-3" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Where%20freedom%20is%20given%2C%20liberties%20are%20taken&amp;journal=Structure&amp;volume=3&amp;pages=535-540&amp;publication_year=1995&amp;author=Kleywegt%2CGJ&amp;author=Jones%2CTA" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR13">Yorke, B. A., Beddard, G. S., Owen, R. L. &amp; Pearson, A. R. Time-resolved crystallography using the Hadamard transform. <i>Nat. Methods</i> <b>11</b>, 1131–1134 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhs1yhtrbO" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1038%2Fnmeth.3139" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Time-resolved%20crystallography%20using%20the%20Hadamard%20transform&amp;journal=Nat.%20Methods&amp;volume=11&amp;pages=1131-1134&amp;publication_year=2014&amp;author=Yorke%2CBA&amp;author=Beddard%2CGS&amp;author=Owen%2CRL&amp;author=Pearson%2CAR" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR14">Rould, M. A. &amp; Carter, C. W. Isomorphous Difference Methods. <i>Methods Enzymol.</i> <b>374</b>, 145–163 (2003).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXotlygsQ%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1016%2FS0076-6879%2803%2974007-5" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Isomorphous%20Difference%20Methods&amp;journal=Methods%20Enzymol.&amp;volume=374&amp;pages=145-163&amp;publication_year=2003&amp;author=Rould%2CMA&amp;author=Carter%2CCW" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR15">Keedy, D. A. et al. Crystal cryocooling distorts conformational heterogeneity in a model michaelis complex of DHFR. <i>Structure</i> <b>22</b>, 899–910 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXpslCgsb4%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1016%2Fj.str.2014.04.016" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Crystal%20cryocooling%20distorts%20conformational%20heterogeneity%20in%20a%20model%20michaelis%20complex%20of%20DHFR&amp;journal=Structure&amp;volume=22&amp;pages=899-910&amp;publication_year=2014&amp;author=Keedy%2CDA" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR16">Patel,
 D., Bauman, J. D. &amp; Arnold, E. Advantages of crystallographic 
fragment screening: Functional and mechanistic insights from a powerful 
platform for efficient drug discovery. <i>Prog. Biophys. Mol. Biol.</i> <b>116</b>, 92–100 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhtlGntbjL" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1016%2Fj.pbiomolbio.2014.08.004" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Advantages%20of%20crystallographic%20fragment%20screening%3A%20Functional%20and%20mechanistic%20insights%20from%20a%20powerful%20platform%20for%20efficient%20drug%20discovery&amp;journal=Prog.%20Biophys.%20Mol.%20Biol.&amp;volume=116&amp;pages=92-100&amp;publication_year=2014&amp;author=Patel%2CD&amp;author=Bauman%2CJD&amp;author=Arnold%2CE" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR17">Cox,
 O. B. et al. A poised fragment library enables rapid synthetic 
expansion yielding the first reported inhibitors of PHIP(2), an atypical
 bromodomain. <i>Chem. Sci.</i> <b>7</b>, 2322–2330 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXitVGrtbnF" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1039%2FC5SC03115J" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20poised%20fragment%20library%20enables%20rapid%20synthetic%20expansion%20yielding%20the%20first%20reported%20inhibitors%20of%20PHIP%282%29%2C%20an%20atypical%20bromodomain&amp;journal=Chem.%20Sci.&amp;volume=7&amp;pages=2322-2330&amp;publication_year=2016&amp;author=Cox%2COB" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR18">Murray,
 C. W. &amp; Verdonk, M. L. The consequences of translational and 
rotational entropy lost by small molecules on binding to proteins. <i>J. Comput. Aided. Mol. Des.</i> <b>16</b>, 741–753 (2002).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXht1Wktrw%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2002JCAMD..16..741M" data-track="click" data-track-action="outbound reference" data-track-label="link">ADS</a>&nbsp;<a href="https://doi.org/10.1023%2FA%3A1022446720849" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20consequences%20of%20translational%20and%20rotational%20entropy%20lost%20by%20small%20molecules%20on%20binding%20to%20proteins&amp;journal=J.%20Comput.%20Aided.%20Mol.%20Des.&amp;volume=16&amp;pages=741-753&amp;publication_year=2002&amp;author=Murray%2CCW&amp;author=Verdonk%2CML" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR19">Mooij, W. T. M. et al. Automated protein-ligand crystallography for structure-based drug design. <i>Chem. Med. Chem.</i> <b>1</b>, 827–838 (2006).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXjsVCmtbc%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1002%2Fcmdc.200600074" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20protein-ligand%20crystallography%20for%20structure-based%20drug%20design&amp;journal=Chem.%20Med.%20Chem.&amp;volume=1&amp;pages=827-838&amp;publication_year=2006&amp;author=Mooij%2CWTM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR20">Tickle, I. J. Statistical quality indicators for electron-density maps. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>68</b>, 454–467 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38Xlt1Gns70%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1107%2FS0907444911035918" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Statistical%20quality%20indicators%20for%20electron-density%20maps&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=68&amp;pages=454-467&amp;publication_year=2012&amp;author=Tickle%2CIJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR21">Nicholls, R. A., Long, F. &amp; Murshudov, G. N. Low-resolution refinement tools in REFMAC5. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>68</b>, 404–417 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38Xlt1Gnsr8%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1107%2FS090744491105606X" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Low-resolution%20refinement%20tools%20in%20REFMAC5&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=68&amp;pages=404-417&amp;publication_year=2012&amp;author=Nicholls%2CRA&amp;author=Long%2CF&amp;author=Murshudov%2CGN" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR22">Berman, H. M. et al. The Protein Data Bank. <i>Nucleic Acids Res.</i> <b>28</b>, 235–242 (2000).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2000spam.book.....B" data-track="click" data-track-action="outbound reference" data-track-label="link">ADS</a>&nbsp;<a href="https://doi.org/10.1093%2Fnar%2F28.1.235" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Protein%20Data%20Bank&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=235-242&amp;publication_year=2000&amp;author=Berman%2CHM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR23">Schiebel,
 J. et al. Six Biophysical Screening Methods Miss a Large Proportion of 
Crystallographically Discovered Fragment Hits: A Case Study. <i>ACS Chem. Biol.</i> <b>11</b>, 1693–1701 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XltVagt7c%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1021%2Facschembio.5b01034" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Six%20Biophysical%20Screening%20Methods%20Miss%20a%20Large%20Proportion%20of%20Crystallographically%20Discovered%20Fragment%20Hits%3A%20A%20Case%20Study&amp;journal=ACS%20Chem.%20Biol.&amp;volume=11&amp;pages=1693-1701&amp;publication_year=2016&amp;author=Schiebel%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR24">Mueller, M., Wang, M. &amp; Schulze-Briese, C. Optimal fine phi-slicing for single-photon-counting pixel detectors. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>68</b>, 42–56 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXhs1Krtr7E" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1107%2FS0907444911049833" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Optimal%20fine%20phi-slicing%20for%20single-photon-counting%20pixel%20detectors&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=68&amp;pages=42-56&amp;publication_year=2012&amp;author=Mueller%2CM&amp;author=Wang%2CM&amp;author=Schulze-Briese%2CC" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR25">Helliwell, J. R. &amp; Mitchell, E. P. Synchrotron radiation macromolecular crystallography: Science and spin-offs. <i>IUCrJ</i> <b>2</b>, 283–291 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXjs1Gmsr4%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1107%2FS205225251402795X" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Synchrotron%20radiation%20macromolecular%20crystallography%3A%20Science%20and%20spin-offs&amp;journal=IUCrJ&amp;volume=2&amp;pages=283-291&amp;publication_year=2015&amp;author=Helliwell%2CJR&amp;author=Mitchell%2CEP" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR26">Schiebel, J. et al. High-throughput crystallography: reliable and efficient identification of fragment hits. <i>Structure</i> <b>24</b>, 1398–1409 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28Xht1eju7fN" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1016%2Fj.str.2016.06.010" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=High-throughput%20crystallography%3A%20reliable%20and%20efficient%20identification%20of%20fragment%20hits&amp;journal=Structure&amp;volume=24&amp;pages=1398-1409&amp;publication_year=2016&amp;author=Schiebel%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR27">Foadi,
 J. et al. Clustering procedures for the optimal selection of data sets 
from multiple crystals in macromolecular crystallography. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>69</b>, 1617–1632 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhtFygsrfK" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1107%2FS0907444913012274" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clustering%20procedures%20for%20the%20optimal%20selection%20of%20data%20sets%20from%20multiple%20crystals%20in%20macromolecular%20crystallography&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=69&amp;pages=1617-1632&amp;publication_year=2013&amp;author=Foadi%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR28">Winn, M. D. et al. Overview of the CCP4 suite and current developments. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>67</b>, 235–242 (2011).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2011AIPC.1374..235W" data-track="click" data-track-action="outbound reference" data-track-label="link">ADS</a>&nbsp;<a href="https://doi.org/10.1107%2FS0907444910045749" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Overview%20of%20the%20CCP4%20suite%20and%20current%20developments&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=67&amp;pages=235-242&amp;publication_year=2011&amp;author=Winn%2CMD" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR29">Afonine, P. V. et al. Towards automated crystallographic structure refinement with phenix.refine. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>68</b>, 352–367 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1107%2FS0907444912001308" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20automated%20crystallographic%20structure%20refinement%20with%20phenix.refine&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=68&amp;pages=352-367&amp;publication_year=2012&amp;author=Afonine%2CPV" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR30">Adams, P. D. et al. PHENIX: a comprehensive Python-based system for macromolecular structure solution. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <b>66</b>, 213–221 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1107%2FS0907444909052925" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=PHENIX%3A%20a%20comprehensive%20Python-based%20system%20for%20macromolecular%20structure%20solution&amp;journal=Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.&amp;volume=66&amp;pages=213-221&amp;publication_year=2010&amp;author=Adams%2CPD" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR31">Grosse-Kunstleve,
 R. W., Sauter, N. K., Moriarty, N. W. &amp; Adams, P. D. The 
Computational Crystallography Toolbox: crystallographic algorithms in a 
reusable software framework. <i>J. Appl. Crystallogr.</i> <b>35</b>, 126–136 (2002).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD38XltVajug%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a>&nbsp;<a href="https://doi.org/10.1107%2FS0021889801017824" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Computational%20Crystallography%20Toolbox%3A%20crystallographic%20algorithms%20in%20a%20reusable%20software%20framework&amp;journal=J.%20Appl.%20Crystallogr.&amp;volume=35&amp;pages=126-136&amp;publication_year=2002&amp;author=Grosse-Kunstleve%2CRW&amp;author=Sauter%2CNK&amp;author=Moriarty%2CNW&amp;author=Adams%2CPD" data-track="click" data-track-action="outbound reference" data-track-label="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR32">Krojer, T., Pearce, N. M., Bradley, A., Marsden, B. D. &amp; von Delft, F. <i>PanDDA analysis of BAZ2B screened against Zenobia Fragment Library</i> (HTML Summary). Zenodo. Available at: https://zenodo.org/record/290199 (2017).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR33">Krojer, T., Pearce, N. M., Bradley, A., Marsden, B. D. &amp; von Delft, F. <i>PanDDA analysis of JMJD2D screened against Zenobia Fragment Library</i> (HTML Summary). Zenodo. Available at: https://zenodo.org/record/290220 (2017).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR34">Krojer, T., Pearce, N. M., Collins, P., Talon, R. &amp; von Delft, F. <i>PanDDA analysis of BRD1 screened against 3D-Fragment-Consortium Fragment Library</i>. (HTML Summary) Zenodo. Available at: https://zenodo.org/record/290217 (2017).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR35">Krojer, T., Pearce, N. M., Collins, P., Talon, R. &amp; von Delft, F. <i>PanDDA analysis of SP100 screened against selection of Maybridge Fragment Library</i> (HTML Summary). Zenodo. Available at: https://zenodo.org/record/290201 (2017).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR36">Pearce,
 N. M., Bradley, A. R., Marsden, B. D. &amp; von Delft, F. PanDDA 
analysis of BAZ2B screened against Zenobia Fragment Library [Dataset]. 
Zenodo. Available at: https://zenodo.org/record/48768 (2016).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR37">Pearce,
 N. M., Bradley, A. R., Marsden, B. D. &amp; von Delft, F. PanDDA 
analysis of JMJD2D screened against Zenobia Fragment Library [Dataset]. 
Zenodo. Available at: https://zenodo.org/record/48770 (2016).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR38">Pearce,
 N. M., Krojer, T., Collins, P., Talon, R. &amp; von Delft, F. PanDDA 
analysis of BRD1 screened against 3D-Fragment Consortium Fragment 
Library [Dataset]. Zenodo. Available at: https://zenodo.org/record/48769
 (2016).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR39">Pearce,
 N. M., Krojer, T., Collins, P., Talon, R. &amp; von Delft, F. PanDDA 
analysis of SP100 screened against selection of Maybridge Fragment 
Library [Dataset]. Zenodo. Available at: https://zenodo.org/record/48771
 (2016).</p></li></ol></div></div>
            </div>
        </div>
    </div>
</div>


    
        
    

    
        
    




    




<footer role="contentinfo" class="composite-layer">
    
    <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <p class="c-meta u-ma-0">
                <span aria-level="2" class="c-meta__item" itemprop="name">
                    Nature Communications
                </span>
                <span class="c-meta__item">
                    <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">2041-1723</span> (online)
                </span>
            </p>
        </div>
    </div>
</div>


    <div itemscope="" itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        <div class="c-footer">
            <div class="u-container">
                <div class="u-hide-print" data-track-component="footer">
    <h2 aria-level="2" class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__header">
        <div class="c-footer__logo">
            <img alt="Nature Research" src="nature%2520research-white-150.svg" loading="lazy" width="200" height="26">
        </div>
        <ul class="c-menu c-menu--inherit u-mr-32">
            <li class="c-menu__item"><a class="c-menu__link" href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" data-track-label="link">About us</a></li>
            <li class="c-menu__item"><a class="c-menu__link" href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link">Press releases</a></li>
            <li class="c-menu__item"><a class="c-menu__link" href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link">Press office</a></li>
            <li class="c-menu__item"><a class="c-menu__link" href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" data-track-label="link">Contact us</a></li>
        </ul>
        <ul class="c-menu c-menu--inherit">
            <li class="c-menu__item">
                <a class="c-menu__link" href="https://www.facebook.com/nature/" aria-label="Nature on Facebook" data-track="click" data-track-action="facebook" data-track-label="link">
                    <svg class="u-icon u-mt-2 u-mb-2" role="img" aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 20 20"><path d="M2.5 20C1.1 20 0 18.9 0 17.5v-15C0 1.1 1.1 0 2.5 0h15C18.9 0 20 1.1 20 2.5v15c0 1.4-1.1 2.5-2.5 2.5h-3.7v-7.7h2.6l.4-3h-3v-2c0-.9.2-1.5 1.5-1.5h1.6V3.1c-.3 0-1.2-.1-2.3-.1-2.3 0-3.9 1.4-3.9 4v2.2H8.1v3h2.6V20H2.5z"></path></svg>
                </a>
            </li>
            <li class="c-menu__item">
                <a class="c-menu__link" href="https://twitter.com/nresearchnews?lang=en" aria-label="Nature on Twitter" data-track="click" data-track-action="twitter" data-track-label="link">
                    <svg class="u-icon" role="img" aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20"><path d="M17.6 4.1c.8-.5 1.5-1.4 1.8-2.4-.8.5-1.7.9-2.6 1-.7-.8-1.8-1.4-3-1.4-2.3 0-4.1 1.9-4.1 4.3 0 .3 0 .7.1 1-3.4 0-6.4-1.8-8.4-4.4C1 2.9.8 3.6.8 4.4c0 1.5.7 2.8 1.8 3.6C2 8 1.4 7.8.8 7.5v.1c0 2.1 1.4 3.8 3.3 4.2-.3.1-.7.2-1.1.2-.3 0-.5 0-.8-.1.5 1.7 2 3 3.8 3-1.3 1.1-3.1 1.8-5 1.8-.3 0-.7 0-1-.1 1.8 1.2 4 1.9 6.3 1.9C13.8 18.6 18 12 18 6.3v-.6c.8-.6 1.5-1.4 2-2.2-.7.3-1.5.5-2.4.6z"></path></svg>
                </a>
            </li>
            <li class="c-menu__item">
                <a class="c-menu__link" href="https://www.youtube.com/channel/UCvCLdSgYdSTpWcOgEJgi-ng" aria-label="Nature on YouTube" data-track="click" data-track-action="youtube" data-track-label="link">
                    <svg class="u-icon" role="img" aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20"><path d="M7.9 12.6V6.9l5.4 2.8c0 .1-5.4 2.9-5.4 2.9zM19.8 6s-.2-1.4-.8-2c-.8-.8-1.6-.8-2-.9-2.8-.2-7-.2-7-.2s-4.2 0-7 .2c-.4 0-1.2 0-2 .9-.6.6-.8 2-.8 2S0 7.6 0 9.2v1.5c0 1.7.2 3.3.2 3.3s.2 1.4.8 2c.8.8 1.8.8 2.2.9 1.6.1 6.8.2 6.8.2s4.2 0 7-.2c.4 0 1.2-.1 2-.9.6-.6.8-2 .8-2s.2-1.6.2-3.3V9.2c0-1.6-.2-3.2-.2-3.2z"></path></svg>
                </a>
            </li>
        </ul>
    </div>

    <div class="c-footer__grid">
        <div class="c-footer__group">
            <h3 class="c-footer__heading">Discover content</h3>
            <ul class="c-footer__list">
                <li class="c-footer__item"><a href="https://www.nature.com/siteindex" data-track="click" data-track-action="journals a-z" data-track-label="link">Journals A-Z</a></li>
                <li class="c-footer__item"><a href="https://www.nature.com/subjects/" data-track="click" data-track-action="article by subject" data-track-label="link">Articles by subject</a></li>
                <li class="c-footer__item"><a href="https://nano.nature.com/" data-track="click" data-track-action="nano" data-track-label="link">Nano</a></li>
                <li class="c-footer__item"><a href="https://www.nature.com/protocolexchange/" data-track="click" data-track-action="protocol exchange" data-track-label="link">Protocol Exchange</a></li>
                <li class="c-footer__item"><a href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link">Nature Index</a></li>
            </ul>
        </div>

        <div class="c-footer__group">
            <h3 class="c-footer__heading">Publish with us</h3>
            <ul class="c-footer__list">
                <li class="c-footer__item"><a href="https://www.nature.com/authors/author_resources/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to Authors</a></li>
                <li class="c-footer__item"><a href="https://www.nature.com/authors/peer_review/" data-track="click" data-track-action="guide to referees" data-track-label="link">Guide to Referees</a></li>
                <li class="c-footer__item"><a href="https://www.nature.com/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
                <li class="c-footer__item"><a href="http://www.nature.com/openresearch/publishing-with-npg/" data-track="click" data-track-action="open access" data-track-label="link">Open access</a></li>
                <li><a href="https://www.nature.com/reprints/" data-track="click" data-track-action="reprints and permissions" data-track-label="link">Reprints &amp; permissions</a></li>
            </ul>
        </div>

        <div class="c-footer__group">
            <h3 class="c-footer__heading">Researcher services</h3>
            <ul class="c-footer__list">
                <li class="c-footer__item"><a href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link">Research data</a></li>
                <li class="c-footer__item"><a href="https://authorservices.springernature.com/go/nr" data-track="click" data-track-action="language editing" data-track-label="link">Language editing</a></li>
                <li class="c-footer__item"><a href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link">Scientific editing</a></li>
                <li class="c-footer__item"><a href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link">Nature Masterclasses</a></li>
                <li class="c-footer__item"><a href="https://partnerships.nature.com/product/researcher-training/" data-track="click" data-track-action="nature research academies" data-track-label="link">Nature Research Academies</a></li>
            </ul>
        </div>

        <div class="c-footer__group">
            <h3 class="c-footer__heading">Libraries &amp; institutions</h3>
            <ul class="c-footer__list">
                <li class="c-footer__item"><a href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link">Librarian service &amp; tools</a></li>
                <li class="c-footer__item"><a href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link">Librarian portal</a></li>
                <li class="c-footer__item"><a href="http://www.nature.com/openresearch/about-open-access/information-for-institutions/" data-track="click" data-track-action="open research" data-track-label="link">Open research</a></li>
            </ul>
        </div>

        <div class="c-footer__group">
            <h3 class="c-footer__heading">Advertising &amp; partnerships</h3>
            <ul class="c-footer__list">
                <li class="c-footer__item"><a href="https://partnerships.nature.com/product/digital-advertising/" data-track="click" data-track-action="advertising" data-track-label="link">Advertising</a></li>
                <li class="c-footer__item"><a href="https://partnerships.nature.com/" data-track="click" data-track-action="partnerships and services" data-track-label="link">Partnerships &amp; Services</a></li>
                <li class="c-footer__item"><a href="https://partnerships.nature.com/media-kits/" data-track="click" data-track-action="media kits" data-track-label="link">Media kits</a></li>
                <li class="c-footer__item"><a href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link">Branded content</a></li>
            </ul>
        </div>

        <div class="c-footer__group">
            <h3 class="c-footer__heading">Career development</h3>
            <ul class="c-footer__list">
                <li class="c-footer__item"><a href="https://www.nature.com/naturecareers" data-track="click" data-track-action="nature careers" data-track-label="link">Nature Careers</a></li>
                <li class="c-footer__item"><a href="https://www.nature.com/natureconferences/" data-track="click" data-track-action="nature conferences" data-track-label="link">Nature<span class="visually-hidden"> </span> Conferences</a></li>
                <li class="c-footer__item"><a href="https://www.nature.com/natureevents/" data-track="click" data-track-action="nature events" data-track-label="link">Nature<span class="visually-hidden"> </span> events</a></li>
            </ul>
        </div>

        <div class="c-footer__group">
            <h3 class="c-footer__heading">Regional websites</h3>
            <ul class="c-footer__list">
                <li class="c-footer__item"><a href="http://www.naturechina.com/" data-track="click" data-track-action="nature china" data-track-label="link">Nature China</a></li>
                <li class="c-footer__item"><a href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link">Nature India</a></li>
                <li class="c-footer__item"><a href="https://www.natureasia.com/ja-jp/" data-track="click" data-track-action="nature japan" data-track-label="link">Nature Japan</a></li>
                <li class="c-footer__item"><a href="https://www.natureasia.com/ko-kr/" data-track="click" data-track-action="nature korea" data-track-label="link">Nature Korea</a></li>
                <li class="c-footer__item"><a href="https://www.nature.com/nmiddleeast/" data-track="click" data-track-action="nature middle east" data-track-label="link">Nature Middle East</a></li>
            </ul>
        </div>

        <div class="c-footer__group">
            <h3 class="c-footer__heading">Legal &amp; Privacy</h3>
            <ul class="c-footer__list">
                <li class="c-footer__item"><a href="https://www.nature.com/info/privacy.html" data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy Policy</a></li>
                <li class="c-footer__item"><a href="https://www.nature.com/info/cookies.html" data-track="click" data-track-action="use of cookies" data-track-label="link">Use of cookies</a></li>
                <li class="c-footer__item"><a class="optanon-toggle-display" href="javascript:;" data-track="click" data-track-action="manage cookies" data-track-label="link">Manage cookies/Do not sell my data</a></li>
                <li class="c-footer__item"><a href="https://www.nature.com/info/legal_notice.html" data-track="click" data-track-action="legal notice" data-track-label="link">Legal notice</a></li>
                <li class="c-footer__item"><a href="https://www.nature.com/info/accessibility_statement.html" data-track="click" data-track-action="accessibility statement" data-track-label="link">Accessibility statement</a></li>
                <li class="c-footer__item"><a href="https://www.nature.com/info/tandc.html" data-track="click" data-track-action="terms and conditions" data-track-label="link">Terms &amp; Conditions</a></li>
                <li class="c-footer__item"><a href="https://www.springernature.com/ccpa" data-track="click" data-track-action="california privacy statement" data-track-label="link">California Privacy Statement</a></li>
            </ul>
        </div>
    </div>
</div>


            </div>
        </div>
    </div>

    <div class="c-corporate-footer">
    <div class="u-container">
        <img src="sn-logo-white.svg" alt="Springer Nature" loading="lazy" width="140" height="14">
        <p class="c-corporate-footer__legal" data-test="copyright">© 2020 Springer Nature Limited</p>
    </div>
</div>

    
    <svg class="u-hide hide">
        <symbol id="global-icon-chevron-right" viewBox="0 0 16 16">
            <path d="M7.782 7L5.3 4.518c-.393-.392-.4-1.022-.02-1.403a1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L7.782 9l1.013-.998z" fill-rule="evenodd"></path>
        </symbol>
        <symbol id="global-icon-download" viewBox="0 0 16 16">
            <path d="M2 14c0-.556.449-1 1.002-1h9.996a.999.999 0 110 2H3.002A1.006 1.006 0 012 14zM9 2v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 010 1.417l-4.177 4.177a1.001 1.001 0 01-1.416 0L3.115 7.715a.991.991 0 01-.016-1.4 1 1 0 011.42.003L7 8.8V2c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z" fill-rule="evenodd"></path>
        </symbol>
        <symbol id="global-icon-email" viewBox="0 0 18 18">
            <path d="M1.995 2h14.01A2 2 0 0118 4.006v9.988A2 2 0 0116.005 16H1.995A2 2 0 010 13.994V4.006A2 2 0 011.995 2zM1 13.994A1 1 0 001.995 15h14.01A1 1 0 0017 13.994V4.006A1 1 0 0016.005 3H1.995A1 1 0 001 4.006zM9 11L2 7V5.557l7 4 7-4V7z" fill-rule="evenodd"></path>
        </symbol>
        <symbol id="global-icon-institution" viewBox="0 0 18 18">
            <path d="M14 8a1 1 0 011 1v6h1.5a.5.5 0 01.5.5v.5h.5a.5.5 0 01.5.5V18H0v-1.5a.5.5 0 01.5-.5H1v-.5a.5.5 0 01.5-.5H3V9a1 1 0 112 0v6h8V9a1 1 0 011-1zM6 8l2 1v4l-2 1zm6 0v6l-2-1V9zM9.573.401l7.036 4.925A.92.92 0 0116.081 7H1.92a.92.92 0 01-.528-1.674L8.427.401a1 1 0 011.146 0zM9 2.441L5.345 5h7.31z" fill-rule="evenodd"></path>
        </symbol>
        <symbol id="global-icon-search" viewBox="0 0 22 22">
            <path fill-rule="evenodd" d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z"></path>
        </symbol>
        <symbol id="icon-info" viewBox="0 0 18 18">
            <path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path>
        </symbol>
    </svg>

</footer>




    
        

<div class="c-site-messages message u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif" data-component-id="nature-briefing-banner" data-component-expirydays="30" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="redesign banner visible">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="redesign banner dismiss">
        <!--?xml version="1.0" encoding="UTF-8" standalone="no"?-->
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="nature-briefing-logo-n150-white.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="/briefing/signup/formfeedback" method="post" data-location="banner" data-track="submit" data-track-action="transmit-form">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" name="track_originReferralPoint" value="DirectEmailBannerRedesign2020" type="hidden">
                        <input id="briefing-banner-signup-form-input-track-formType" name="track_formType" value="DirectEmailBanner" type="hidden">
                        <label class="nature-briefing-banner__email-label" for="banner-EmailAddressInput">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" id="banner-EmailAddressInput" name="email" placeholder="e.g. jo.smith@university.ac.uk" required="true" aria-required="true" data-test-element="briefing-emailbanner-email-input" type="email">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" name="gdpr" value="1" data-test-element="briefing-emailbanner-gdpr-checkbox" required="" type="checkbox">
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="redesign banner dismiss">
        <!--?xml version="1.0" encoding="UTF-8" standalone="no"?-->
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif" data-track="click" data-track-category="nature briefing" data-track-label="redesign banner CTA to site" data-test-element="briefing-banner-link" target="_blank" rel="noreferrer noopener" href="https://www.nature.com/briefing/signup/?origin=Nature&amp;originReferralPoint=EmailBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>
    




<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="ping" async=""></script>
<img src="ncomms15123" alt="" class="visually-hidden" width="1" height="1">



<script src="nature.js"></script><div role="region" id="popup-auth-Nicholas_M_-Pearce" class="c-popup c-author-popup u-font-family-serif u-js-hide" aria-labelledby="auth-Nicholas_M_-Pearce" aria-hidden="true"><section><h3 id="author-auth-Nicholas_M_-Pearce" class="c-author-popup__subheading" tabindex="0">Nicholas M. Pearce</h3><ul class="c-author-popup__author-list"><li>Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK</li></ul><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Nicholas%20M.+Pearce%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nicholas%20M.+Pearce" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nicholas%20M.+Pearce%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></section><button class="c-popup__close"><span class="c-popup__close-text">Close</span></button><div class="c-popup__arrow"></div></div><div role="region" id="popup-auth-Tobias-Krojer" class="c-popup c-author-popup u-font-family-serif u-js-hide" aria-labelledby="auth-Tobias-Krojer" aria-hidden="true"><section><h3 id="author-auth-Tobias-Krojer" class="c-author-popup__subheading" tabindex="0">Tobias Krojer</h3><ul class="c-author-popup__author-list"><li>Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK</li></ul><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Tobias+Krojer%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tobias+Krojer" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tobias+Krojer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></section><button class="c-popup__close"><span class="c-popup__close-text">Close</span></button><div class="c-popup__arrow"></div></div><div role="region" id="popup-auth-Anthony_R_-Bradley" class="c-popup c-author-popup u-font-family-serif u-js-hide" aria-labelledby="auth-Anthony_R_-Bradley" aria-hidden="true"><section><h3 id="author-auth-Anthony_R_-Bradley" class="c-author-popup__subheading" tabindex="0">Anthony R. Bradley</h3><ul class="c-author-popup__author-list"><li>Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK</li></ul><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Anthony%20R.+Bradley%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Anthony%20R.+Bradley" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Anthony%20R.+Bradley%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></section><button class="c-popup__close"><span class="c-popup__close-text">Close</span></button><div class="c-popup__arrow"></div></div><div role="region" id="popup-auth-Patrick-Collins" class="c-popup c-author-popup u-font-family-serif u-js-hide" aria-labelledby="auth-Patrick-Collins" aria-hidden="true"><section><h3 id="author-auth-Patrick-Collins" class="c-author-popup__subheading" tabindex="0">Patrick Collins</h3><a class="c-article-orcid" href="http://orcid.org/0000-0002-6265-9922" target="_blank" rel="noopener"><span class="c-article-orcid__text">View ORCID ID profile</span></a><ul class="c-author-popup__author-list"><li>Diamond Light Source Ltd, Harwell Science and Innovation Campus, Didcot, OX11 0QX, UK</li></ul><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Patrick+Collins%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Patrick+Collins" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Patrick+Collins%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></section><button class="c-popup__close"><span class="c-popup__close-text">Close</span></button><div class="c-popup__arrow"></div></div><div role="region" id="popup-auth-Rados_aw_P_-Nowak" class="c-popup c-author-popup u-font-family-serif u-js-hide" aria-labelledby="auth-Rados_aw_P_-Nowak" aria-hidden="true"><section><h3 id="author-auth-Rados_aw_P_-Nowak" class="c-author-popup__subheading" tabindex="0">Radosław P. Nowak</h3><ul class="c-author-popup__author-list"><li>Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK</li></ul><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Rados%C5%82aw%20P.+Nowak%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Rados%C5%82aw%20P.+Nowak" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Rados%C5%82aw%20P.+Nowak%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></section><button class="c-popup__close"><span class="c-popup__close-text">Close</span></button><div class="c-popup__arrow"></div></div><div role="region" id="popup-auth-Romain-Talon" class="c-popup c-author-popup u-font-family-serif u-js-hide" aria-labelledby="auth-Romain-Talon" aria-hidden="true"><section><h3 id="author-auth-Romain-Talon" class="c-author-popup__subheading" tabindex="0">Romain Talon</h3><ul class="c-author-popup__author-list"><li>Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK</li></ul><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Romain+Talon%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Romain+Talon" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Romain+Talon%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></section><button class="c-popup__close"><span class="c-popup__close-text">Close</span></button><div class="c-popup__arrow"></div></div><div role="region" id="popup-auth-Brian_D_-Marsden" class="c-popup c-author-popup u-font-family-serif u-js-hide" aria-labelledby="auth-Brian_D_-Marsden" aria-hidden="true"><section><h3 id="author-auth-Brian_D_-Marsden" class="c-author-popup__subheading" tabindex="0">Brian D. Marsden</h3><ul class="c-author-popup__author-list"><li>Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK</li><li>Nuffield
 Department of Orthopaedics, Kennedy Institute of Rheumatology, 
Rheumatology and Musculoskeletal Sciences, University of Oxford, 
Roosevelt Drive, Oxford, OX3 7FY, UK</li></ul><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Brian%20D.+Marsden%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Brian%20D.+Marsden" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Brian%20D.+Marsden%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></section><button class="c-popup__close"><span class="c-popup__close-text">Close</span></button><div class="c-popup__arrow"></div></div><div role="region" id="popup-auth-Sebastian-Kelm" class="c-popup c-author-popup u-font-family-serif u-js-hide" aria-labelledby="auth-Sebastian-Kelm" aria-hidden="true"><section><h3 id="author-auth-Sebastian-Kelm" class="c-author-popup__subheading" tabindex="0">Sebastian Kelm</h3><ul class="c-author-popup__author-list"><li>UCB Pharma, 208 Bath Road, Slough, SL1 3WE, UK</li></ul><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Sebastian+Kelm%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sebastian+Kelm" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sebastian+Kelm%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></section><button class="c-popup__close"><span class="c-popup__close-text">Close</span></button><div class="c-popup__arrow"></div></div><div role="region" id="popup-auth-Jiye-Shi" class="c-popup c-author-popup u-font-family-serif u-js-hide" aria-labelledby="auth-Jiye-Shi" aria-hidden="true"><section><h3 id="author-auth-Jiye-Shi" class="c-author-popup__subheading" tabindex="0">Jiye Shi</h3><ul class="c-author-popup__author-list"><li>UCB Pharma, 208 Bath Road, Slough, SL1 3WE, UK</li></ul><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Jiye+Shi%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jiye+Shi" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jiye+Shi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></section><button class="c-popup__close"><span class="c-popup__close-text">Close</span></button><div class="c-popup__arrow"></div></div><div role="region" id="popup-auth-Charlotte_M_-Deane" class="c-popup c-author-popup u-font-family-serif u-js-hide" aria-labelledby="auth-Charlotte_M_-Deane" aria-hidden="true"><section><h3 id="author-auth-Charlotte_M_-Deane" class="c-author-popup__subheading" tabindex="0">Charlotte M. Deane</h3><a class="c-article-orcid" href="http://orcid.org/0000-0003-1388-2252" target="_blank" rel="noopener"><span class="c-article-orcid__text">View ORCID ID profile</span></a><ul class="c-author-popup__author-list"><li>Department of Statistics, University of Oxford, 24-29 St Giles, Oxford, OX1 3LB, UK</li></ul><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Charlotte%20M.+Deane%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Charlotte%20M.+Deane" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Charlotte%20M.+Deane%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></section><button class="c-popup__close"><span class="c-popup__close-text">Close</span></button><div class="c-popup__arrow"></div></div><div role="region" id="popup-auth-Frank-von_Delft" class="c-popup c-author-popup u-font-family-serif u-js-hide" aria-labelledby="auth-Frank-von_Delft" aria-hidden="true"><section><h3 id="author-auth-Frank-von_Delft" class="c-author-popup__subheading" tabindex="0">Frank von Delft</h3><a class="c-article-orcid" href="http://orcid.org/0000-0003-0378-0017" target="_blank" rel="noopener"><span class="c-article-orcid__text">View ORCID ID profile</span></a><ul class="c-author-popup__author-list"><li>Nuffield Department of Medicine, Structural Genomics Consortium, University of Oxford, Roosevelt Drive, Oxford, OX3 7DQ, UK</li><li>Diamond Light Source Ltd, Harwell Science and Innovation Campus, Didcot, OX11 0QX, UK</li><li>Department of Biochemistry, University of Johannesburg, Auckland Park, 2006, South Africa</li><li><a href="https://www.nature.com/articles/ncomms15123/email/correspondent/c1/new" class="c-author-popup__link" rel="nofollow">Contact Frank von Delft</a></li></ul><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.nature.com/search?author=%22Frank+von%20Delft%22" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Frank+von%20Delft" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Frank+von%20Delft%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></section><button class="c-popup__close"><span class="c-popup__close-text">Close</span></button><div class="c-popup__arrow"></div></div><script type="text/javascript" id="">Bootstrapper=window.Bootstrapper||{};Bootstrapper.npg=Bootstrapper.npg||{};Bootstrapper.npg.utils=Bootstrapper.npg.utils||{};
Bootstrapper.npg.utils.Consent=function(){var c=function(){for(var b=document.cookie.split(/;\s*/),a=0;b[a];++a)if(-1!==b[a].indexOf("OptanonConsent\x3d"))return b[a].split("\x3d").slice(1).join("\x3d");return""};c=unescape(c());var d=(/groups=([^&]+)/.exec(c)||["",""])[1].split(",");return{STRICTLY_NECESSARY:1,PERFORMANCE:2,FUNCTIONAL:3,TARGETING:4,allow:function(b){if(b===this.STRICTLY_NECESSARY)return!0;for(var a=0;d[a];++a)if(d[a]===b+":0")return!1;return!0}}}();</script><script type="text/javascript" id="">var filterMeta=function(a){for(var b=[],e=0;a[e];++e)a[e].name&&b.push(a[e]);return b},translateMeta=function(a){var b={"WT.cg_n":"product_name","WT.site_id":"site_id","WT.site_id_name":"site_id_name","WT.registrant_id":"user_id"};return b[a]||a},getMeta=function(a){var b=filterMeta(document.getElementsByTagName("meta")||[]),e=document.getElementsByTagName("body")||[],g=function(){for(var f=b.length,c={},d;f--;)if(d=translateMeta(b[f].name))c[d]?("string"===typeof c[d]&&(c[d]=[c[d]]),c[d].push(b[f].content)):
c[d]=b[f].content||"";c.keywords&&(c.keywords=c.keywords.replace(/,/g,";"));a(c)};e.length?g():b.length&&g()};getMeta(function(a){window._pageMetaData=a});</script><script type="text/javascript" id="">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="//scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script type="text/javascript" id="">window.dataLayer.push({event:"ga-client-id-pushed-to-datalayer",gaClientId:null});</script><script type="text/javascript" id="">window.permutive&&window.permutive.consent&&permutive.consent({opt_in:!1});</script><script type="text/javascript" id="">window.permutive&&window.permutive.consent&&permutive.consent({opt_in:!1});</script><script type="text/javascript" id="">window.permutive&&window.permutive.consent&&permutive.consent({opt_in:!1});</script><script type="text/javascript" id="">window.permutive&&window.permutive.consent&&permutive.consent({opt_in:!1});</script><script type="text/javascript" id="">var event=document.createEvent("Event");event.initEvent("OneTrustGroupsUpdated",!0,!0);document.dispatchEvent(event);</script><script type="text/javascript" id="">window.permutive&&window.permutive.consent&&permutive.consent({opt_in:!0,token:"CONSENT_CAPTURED"});</script><script type="text/javascript" id="">(function(){var a=document,b=a.getElementsByTagName("script")[0];a=a.createElement("script");a.type="text/javascript";a.async=!0;a.src=google_tag_manager["GTM-NWDMT9Q"].macro(44);b.parentNode.insertBefore(a,b)})();</script><script type="text/javascript" id="">var _hmt=_hmt||[];(function(){var a=document.createElement("script");a.src="https://hm.baidu.com/hm.js?485f8e597c8915da9aca0c37dca3f39f";var b=document.getElementsByTagName("script")[0];b.parentNode.insertBefore(a,b)})();</script><script type="text/javascript" id="">function splitKeys(b){var k=[],d="",g=[],l=b.split(";");for(b=0;b<l.length;++b){void 0!==n&&(d=n);var h=l[b].split("\x3d");var n=h[0];n!==d&&(0<d.length&&0<k.length&&g.push([d,k]),k=[]);if(2===h.length&&""!==h[0]&&""!==h[1]){var q=h[1].split(",");for(h=0;h<q.length;++h)k.push(q[h])}}0<n.length&&0<k.length&&g.push([n,k]);return g}
function getScript(b,k){var d=document.createElement("script"),g=document.getElementsByTagName("script")[0];d.async=1;d.onload=d.onreadystatechange=function(l,h){if(h||!d.readyState||/loaded|complete/.test(d.readyState))d.onload=d.onreadystatechange=null,d=void 0,h||k&&k()};d.src=b;g.parentNode.insertBefore(d,g)}
function splitSizes(b){var k=[];if(null!==b){var d=-1!==b.indexOf("|")?b.split("|"):b.split(",");for(b=0;b<d.length;++b){var g=d[b].split("x");var l=parseInt(g[0],10);var h=parseInt(g[1],10);2===g.length&&!isNaN(l)&&!isNaN(h)&&0<=l&&0<=h&&k.push([l,h])}}return k}function hasClass(b,k){var d,g=b.className.split(/\s+/);b=0;for(d=g.length;b<d;++b)if(g[b]===k)return!0;return!1}
function getAdContainers(){if("function"===typeof document.getElementsByClassName)return document.getElementsByClassName("div-gpt-ad");var b,k=[],d=document.getElementsByTagName("div");for(b=0;d[b];++b)hasClass(d[b],"div-gpt-ad")&&k.push(d[b]);return k}function debounce(b,k){var d=null,g=null;return function(){var l=this,h=+new Date,n=arguments;d&&h<d+k?(clearTimeout(g),g=setTimeout(function(){d=h;b.apply(l,n)},k)):(d=h,b.apply(l,n))}}
function addResizeEvent(b){window.addEventListener?window.addEventListener("resize",b,!1):window.attachEvent("resize",b)}function addScrollEvent(b){window.addEventListener?window.addEventListener("scroll",b,!1):window.attachEvent("onscroll",b)}function removeScrollEvent(b){window.removeEventListener?window.removeEventListener("scroll",b,!1):window.detachEvent("scroll",b)}function serveAdsFor(b){return-1!==b.indexOf("/naturejobs")?!1:!0}
function initAds(b,k){var d=[],g,l=null;if(-1===b.location.search.indexOf("hide_ads\x3dtrue")&&serveAdsFor(b.location.pathname)){var h=function(a){var e=Math.max(k.clientHeight,b.innerHeight||0);return n(a,function(c){if(c.isOutOfPage||c.forceLoadOnInit)return!0;var f=document.getElementById(c.divId),m=f.getBoundingClientRect();m=m.top-300;m=e>m;var p=null!==f.offsetParent;c=c.sizeArray&&c.sizeArray.length&&c.sizeArray[0].length&&2===c.sizeArray[0][0];f=f.getAttribute("data-ad-type")?-1===f.getAttribute("data-ad-type").indexOf("top"):
!1;var t=770>b.innerWidth;return m&&p&&!1===t||c||m&&p&&t&&f})},n=function(a,e){for(var c=a.length,f=[];c--;)e(a[c],c)&&(f.push(a[c].slot),a.splice(c,1));f.length&&googletag.pubads().refresh(f);return a},q={"career feature":!0,"career news":!0,"career q\x26a":!0,"career brief":!0,"career column":!0,spotlight:!0,"career guide":!0,"technology feature":!0,"nature careers podcast":!0},y=function(){var a=b.dataLayer?google_tag_manager["GTM-NWDMT9Q"].macro(45):null;if(a)return a;if(-1!==b.location.hostname.indexOf("guide.labanimal"))return a=
b.location.pathname,a=-1!==a.indexOf("categ")?"products":-1!==a.indexOf("supplier")?"suppliers":"homepage","laban/guide.labanimal/"+a;a=b.location.pathname.replace(/^\/+/,"").split("/");var e=b.location.hostname.split(".").slice(1).join(".");return a.length&&""!==a[0]?a[0]:e},z=function(a){var e="/270604982";0!==a.indexOf("/")&&(a="/"+a);0===a.indexOf("/285/")&&(a=a.replace(/^\/285\/[^\/]+/,e+"/nature/"+y()));a=a.replace("/nature/laban","/laban");a=a.replace("/nature/nature.com/index","/nature/nature/homepage");
a=a.replace("/collections/collections","/collections");a=a.replace("/search/search_results","/nature/search");a=a.replace(/\/article$/,"/articles");a=a.replace(/\/nature\/authors\/.*/,"/nature/nature/authors");-1!==b.location.hostname.indexOf("blogs")&&(a=a.replace(/\/nature\/.*/,"/nature/nature/blogs"));-1!==b.location.hostname.indexOf("natureindex")&&(a=a.replace(/\/nature\/.*/,"/nature/nature_index"),"/"===b.location.pathname&&(a+="/homepage"));window.dataLayer&&q[google_tag_manager["GTM-NWDMT9Q"].macro(46)]&&(a=a.replace(/\/articles$/,
"/naturecareers"));a:{if(/^\/nature\/articles\/?$/.test(window.location.pathname)){var c=(/^.*?(?:\?|&)type=([^&]+)/.exec(b.location.search)||["",""])[1];if(q[c.replace(/-/g," ").replace(/ and /,"\x26")]){c=!0;break a}}c=!1}c&&(a=a.replace(/\/article-list$/,"/naturecareers"));(c=document.querySelector('meta[name\x3d"brandedcontent"]'))&&"true"===c.getAttribute("content")&&(a=e+"/nature/brandedcontent");return a},A=function(a){for(var e={},c=0;a[c];++c){var f=a[c].size;var m="2x2"===a[c].size?window.dataLayer&&
"core media"===google_tag_manager["GTM-NWDMT9Q"].macro(47)&&-1!==b.location.pathname.indexOf("/articles/")?3:0:0;e[f]={count:m,name:a[c].name}}return e}([{size:"728x90",name:"LB"},{size:"300x250",name:"MPU"},{size:"160x600",name:"SKY"},{size:"970x250",name:"BB"},{size:"2x2",name:"NATIVE"},{size:"300x100",name:"REC"},{size:"180x150",name:"EVENTS"},{size:"160x60",name:"TILE"}]),r=function(a,e,c,f){for(var m=!1,p=0;a[p];++p)a[p][0]===e&&null!==c&&(a[p][1]=f?a[p][1].concat(c):[c],m=!0);m||null===c||a.push([e,[c]]);return a},
u=function(a,e,c){for(var f=0;a[f];++f)a[f][0]===e&&(a[f][0]=c);return a},B=function(a,e){var c=a;var f=-1!==b.location.search.indexOf("test\x3dads")?";adtype\x3dtest":"";c=c.getAttribute("data-gpt-targeting");f&&-1===c.indexOf(f)&&(c+=f);(f=google_tag_manager["GTM-NWDMT9Q"].macro(48))&&(c+=";bpid\x3d"+f.replace(/;/g,","));0===b.location.pathname.indexOf("/collections/")&&(f=document.querySelector("span.hero-title-inner"))&&(c+=";sponsored\x3d"+f.innerText.replace(/^\s+/,"").replace(/\s+$/,"").replace(/\s+/g,"_").replace(/\W+/g,
""));c=u(splitKeys(c),"artid","articleid");c=u(c,"kw","search");a=a.getAttribute("data-gpt-sizes");a=(a=A[a])?a.name+ ++a.count:null;a=r(c,"pos",a);e=r(a,"tile",e+1);a=b.location.pathname.split("/");a=3===a.length&&"subjects"===a[1]?a[2]:null;e=r(e,"subject",a);e=r(e,"article",window.dataLayer&&q[google_tag_manager["GTM-NWDMT9Q"].macro(49)]?"naturecareers":null);a=window.dataLayer?google_tag_manager["GTM-NWDMT9Q"].macro(50):null;e=r(e,"collectionID",a);return e=r(e,"type",window.dataLayer&&"core media"===google_tag_manager["GTM-NWDMT9Q"].macro(51)?"fronthalf":null,
!0)},v=function(a){for(var e=[],c=0;a[c];++c){var f=a[c];e.push({divId:f.getAttribute("id"),adUnitPath:z(f.getAttribute("data-gpt-unitpath")),sizeArray:splitSizes(f.getAttribute("data-gpt-sizes")),targeting:B(f,c),isOutOfPage:hasClass(f,"out-of-page"),forceLoadOnInit:!1,refreshed:!1})}return e},w=function(){googletag.cmd.push(function(){googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["GTM-NWDMT9Q"].macro(55)?0:1);googletag.pubads().disableInitialLoad();googletag.enableServices();var a={};googletag.pubads().addEventListener("slotRenderEnded",
function(e){var c=e.slot&&e.slot.getSlotElementId?e.slot.getSlotElementId():null,f;a[c]=!e.isEmpty;(a["div-gpt-ad-native-2"]||a["div-gpt-ad-native-1"])&&(f=document.querySelector(".c-paid-content"))&&f.classList.remove("hide");"div-gpt-ad-billboard-2"===c&&!1===e.isEmpty&&(f=document.getElementById(c),f.parentNode.parentNode.classList.add("pb40"),f.parentNode.parentNode.classList.remove("pb20"),f.parentNode.parentNode.classList.remove("hide"));c&&e.isEmpty&&(f=document.getElementById(c),f.parentNode.parentNode.classList.remove("pb20"),
f.parentNode.classList.remove("ad-with-label"))})});googletag.cmd.push(function(){for(var a=0;d[a];++a)try{d[a].slot=d[a].isOutOfPage?googletag.defineOutOfPageSlot(d[a].adUnitPath,d[a].divId).addService(googletag.pubads()):googletag.defineSlot(d[a].adUnitPath,d[a].sizeArray,d[a].divId).addService(googletag.pubads());for(var e=0,c=d[a].targeting.length;e<c;++e)if(2===d[a].targeting[e].length&&""!==d[a].targeting[e][0]&&""!==d[a].targeting[e][1]){if("pos"===d[a].targeting[e][0]&&0===d[a].targeting[e][1][0].indexOf("BB")){g=
d[a].slot;var f=googletag.sizeMapping().addSize([970,250],[3,3]).addSize([770,100],[4,4]).addSize([0,0],[5,5]).build();g.defineSizeMapping(f);g.setCollapseEmptyDiv(!0,!1)}d[a].slot.setTargeting(d[a].targeting[e][0],d[a].targeting[e][1])}}catch(m){console.log("failed to create slot for",d[a])}});googletag.cmd.push(function(){for(var a=0;d[a];++a)googletag.display(d[a].divId)})},x=function(){l&&removeScrollEvent(l);googletag.cmd.push(function(){d=h(d)});l=debounce(function(){googletag.cmd.push(function(){d=
h(d);d.length||(removeScrollEvent(l),l=null)})},250);addScrollEvent(l)};getScript("//www.googletagservices.com/tag/js/gpt.js",function(){b.googletag=b.googletag||{};b.googletag.cmd=b.googletag.cmd||[];d=v(getAdContainers());w();x();var a=969<b.innerWidth,e=769<b.innerWidth&&!a,c=770>b.innerWidth,f=debounce(function(){var m=b.innerWidth;970<=m&&!a?(a=!0,c=e=!1,googletag.cmd.push(function(){googletag.pubads().refresh([g])})):770>m&&!c?(e=a=!1,c=!0,googletag.cmd.push(function(){googletag.pubads().refresh([g])})):
970>m&&769<m&&!e&&(a=!1,e=!0,c=!1,googletag.cmd.push(function(){googletag.pubads().refresh([g])}))},250);addResizeEvent(f);document.addEventListener("refreshads",function(){googletag.destroySlots();d=v(getAdContainers());w();x()},!1)})}}-1===window.location.hostname.indexOf("nature.com")&&("complete"===document.readyState||"loaded"===document.readyState||"interactive"===document.readyState?initAds(window,document.documentElement):document.addEventListener("load",function(){initAds(window,document.documentElement)}));
document.addEventListener("permutiveready",function(){initAds(window,document.documentElement)},!1);</script>
<script type="text/javascript" id="">document.addEventListener("accessdetailsloaded",function(a){a=a.detail;var b={event:"update-access-details"};if(a){var c=a.institutional_business_partner_ids&&a.institutional_business_partner_ids.join?a.institutional_business_partner_ids.join(";"):"",d=a.resolved_by&&a.resolved_by.join?a.resolved_by.join(";"):"";b.user={};b.user.profile={};b.user.profile.profileInfo={resolvedBy:d||null,bpid:c||null};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy=
{}}window.dataLayer.push(b)},!1);</script><script type="text/javascript" id="">(function(a){if("function"===typeof window.CustomEvent)return!1;var c=function(d,b){b=b||{};var e=document.createEvent("CustomEvent");e.initCustomEvent(d,b.bubbles||!1,b.cancelable||!1,b.detail||a);return e};c.prototype=window.Event.prototype;window.CustomEvent=c})();var parse=function(a,c){try{return 200===a?JSON.parse(c):null}catch(d){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)};
if(-1!==window.location.hostname.indexOf(".nature.com")){var transport=new XMLHttpRequest;transport.open("GET","https://idp.nature.com/exposed-details",!0);transport.withCredentials=!0;transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><div id="onetrust-consent-sdk"><div class="onetrust-pc-dark-filter ot-hide ot-fade-in"></div><div id="onetrust-pc-sdk" class="ot-sdk-container otPcTab ot-hide ot-fade-in ot-sdk-not-webkit" role="dialog" aria-modal="true" aria-labelledby="pc-title" lang="en"><!-- pc header --><div class="pc-header"><!-- Header logo --><div class="pc-logo-container"><div class="pc-logo" role="img" aria-label="Company Logo" style="background-image: url(&quot;https://cdn.cookielaw.org/logos/38bdeb28-e6fb-46a2-bd31-34e5d82df3c7/83f2c78a-6cbc-4d1a-9088-3f8e8c4c7460/3dd0df1b-38ae-4795-935b-710ed4296526/logo-natureresearch-stacked.png&quot;)"></div></div><div class="pc-title-container"><h2 id="pc-title">Privacy Preference Center</h2><div class="pc-close-btn-container"><button id="close-pc-btn-handler" class="pc-close-button ot-close-icon" aria-label="Close"></button></div></div></div><!-- content --><div id="ot-content"><!-- Groups / Sub groups with cookies --><div class="ot-main-content pc-content"><div class="ot-sdk-container groups-container"><div class="ot-sdk-row" role="tablist" aria-label="Cookie Categories"><!-- About Privacy container --><div class="ot-sdk-column"><div class="ot-sdk-four ot-sdk-columns group active-group category-menu-switch-handler" role="tab" tabindex="0" aria-selected="true" aria-controls="pc-tab-description"><h3 id="privacy-text">Your Privacy</h3></div></div><div class="ot-sdk-eight ot-sdk-columns description-container" id="pc-tab-description" tabindex="0" role="tabpanel" aria-labelledby="pc-privacy-header"><h3 id="pc-privacy-header">Your Privacy</h3><p id="pc-policy-text" class="group-description">When
 you visit any website, it may store or retrieve information on your 
browser, mostly in the form of cookies. This information might be about 
you, your preferences or your device and is mostly used to make the site
 work as you expect it to. The information does not usually directly 
identify you, but it can give you a more personalized web experience. 
Because we respect your right to privacy, you can choose not to allow 
some types of cookies. Click on the different category headings to find 
out more and change our default settings. However, blocking some types 
of cookies may impact your experience of the site and the services we 
are able to offer.</p></div><ul class="category-group"><li class="category-item ot-always-active-group" data-optanongroupid="C0001"><div class="ot-sdk-column"><div class="ot-sdk-four ot-sdk-columns group category-menu-switch-handler" role="tab" tabindex="-1" aria-selected="false" aria-controls="ot-desc-id-C0001"><h3 id="ot-header-id-C0001">Strictly Necessary Cookies</h3></div></div><div class="ot-sdk-eight ot-sdk-columns description-container ot-hide ot-always-active-group" role="tabpanel" tabindex="0" id="ot-desc-id-C0001"><div class="group-toggle"><h3 class="category-header">Strictly Necessary Cookies</h3><div class="ot-toggle-group"><div class="ot-always-active">Always Active</div><div class="ot-toggle ot-hide-tgl"><div class="checkbox"><!-- DYNAMICALLY GENERATE Input ID  --> <input id="ot-group-id-C0001" class="category-switch-handler" role="switch" aria-checked="true" aria-controls="ot-desc-id-C0001" aria-labelledby="ot-header-id-C0001" aria-hidden="true" name="ot-group-id-C0001" data-optanongroupid="C0001" checked="checked" type="checkbox"> <label for="ot-group-id-C0001"><span class="label-text">Strictly Necessary Cookies</span></label><!-- DYNAMICALLY GENERATE Input ID  --></div></div></div></div><p class="group-description ot-category-desc">These
 cookies are necessary for the website to function and cannot be 
switched off in our systems. They are usually only set in response to 
actions made by you which amount to a request for services, such as 
setting your privacy preferences, logging in or filling in forms. You 
can set your browser to block or alert you about these cookies, but some
 parts of the site will not then work. These cookies do not store any 
personally identifiable information.</p><!-- sub groups --><div class="category-host-list-container"><a class="category-host-list-btn category-host-list-handler" role="button" href="javascript:void(0)" data-parent-id="C0001">Cookies Details‎</a></div></div></li><li class="category-item" data-optanongroupid="C0002"><div class="ot-sdk-column"><div class="ot-sdk-four ot-sdk-columns group category-menu-switch-handler" role="tab" tabindex="-1" aria-selected="false" aria-controls="ot-desc-id-C0002"><h3 id="ot-header-id-C0002">Performance Cookies</h3></div></div><div class="ot-sdk-eight ot-sdk-columns description-container ot-hide" role="tabpanel" tabindex="0" id="ot-desc-id-C0002"><div class="group-toggle"><h3 class="category-header">Performance Cookies</h3><div class="ot-toggle-group"><div class="ot-toggle"><div class="checkbox"><!-- DYNAMICALLY GENERATE Input ID  --> <input id="ot-group-id-C0002" class="category-switch-handler" role="switch" aria-checked="true" aria-controls="ot-desc-id-C0002" aria-labelledby="ot-header-id-C0002" name="ot-group-id-C0002" data-optanongroupid="C0002" checked="checked" type="checkbox"> <label for="ot-group-id-C0002"><span class="label-text">Performance Cookies</span></label><!-- DYNAMICALLY GENERATE Input ID  --></div></div></div></div><p class="group-description ot-category-desc">These
 cookies allow us to count visits and traffic sources so we can measure 
and improve the performance of our site. They help us to know which 
pages are the most and least popular and see how visitors move around 
the site. All information these cookies collect is aggregated and 
therefore anonymous. If you do not allow these cookies we will not know 
when you have visited our site, and will not be able to monitor its 
performance.</p><!-- sub groups --><div class="category-host-list-container"><a class="category-host-list-btn category-host-list-handler" role="button" href="javascript:void(0)" data-parent-id="C0002">Cookies Details‎</a></div></div></li><li class="category-item" data-optanongroupid="C0003"><div class="ot-sdk-column"><div class="ot-sdk-four ot-sdk-columns group category-menu-switch-handler" role="tab" tabindex="-1" aria-selected="false" aria-controls="ot-desc-id-C0003"><h3 id="ot-header-id-C0003">Functional Cookies</h3></div></div><div class="ot-sdk-eight ot-sdk-columns description-container ot-hide" role="tabpanel" tabindex="0" id="ot-desc-id-C0003"><div class="group-toggle"><h3 class="category-header">Functional Cookies</h3><div class="ot-toggle-group"><div class="ot-toggle"><div class="checkbox"><!-- DYNAMICALLY GENERATE Input ID  --> <input id="ot-group-id-C0003" class="category-switch-handler" role="switch" aria-checked="true" aria-controls="ot-desc-id-C0003" aria-labelledby="ot-header-id-C0003" name="ot-group-id-C0003" data-optanongroupid="C0003" checked="checked" type="checkbox"> <label for="ot-group-id-C0003"><span class="label-text">Functional Cookies</span></label><!-- DYNAMICALLY GENERATE Input ID  --></div></div></div></div><p class="group-description ot-category-desc">These
 cookies enable the website to provide enhanced functionality and 
personalisation. They may be set by us or by third party providers whose
 services we have added to our pages. If you do not allow these cookies 
then some or all of these services may not function properly.</p><!-- sub groups --><div class="category-host-list-container"><a class="category-host-list-btn category-host-list-handler" role="button" href="javascript:void(0)" data-parent-id="C0003">Cookies Details‎</a></div></div></li><li class="category-item" data-optanongroupid="C0004"><div class="ot-sdk-column"><div class="ot-sdk-four ot-sdk-columns group category-menu-switch-handler" role="tab" tabindex="-1" aria-selected="false" aria-controls="ot-desc-id-C0004"><h3 id="ot-header-id-C0004">Targeting Cookies</h3></div></div><div class="ot-sdk-eight ot-sdk-columns description-container ot-hide" role="tabpanel" tabindex="0" id="ot-desc-id-C0004"><div class="group-toggle"><h3 class="category-header">Targeting Cookies</h3><div class="ot-toggle-group"><div class="ot-toggle"><div class="checkbox"><!-- DYNAMICALLY GENERATE Input ID  --> <input id="ot-group-id-C0004" class="category-switch-handler" role="switch" aria-checked="true" aria-controls="ot-desc-id-C0004" aria-labelledby="ot-header-id-C0004" name="ot-group-id-C0004" data-optanongroupid="C0004" checked="checked" type="checkbox"> <label for="ot-group-id-C0004"><span class="label-text">Targeting Cookies</span></label><!-- DYNAMICALLY GENERATE Input ID  --></div></div></div></div><p class="group-description ot-category-desc">These
 cookies may be set through our site by our advertising partners. They 
may be used by those companies to build a profile of your interests and 
show you relevant adverts on other sites. They do not store directly 
personal information, but are based on uniquely identifying your browser
 and internet device. If you do not allow these cookies, you will 
experience less targeted advertising.</p><!-- sub groups --><div class="category-host-list-container"><a class="category-host-list-btn category-host-list-handler" role="button" href="javascript:void(0)" data-parent-id="C0004">Cookies Details‎</a></div></div></li><li class="category-item" data-optanongroupid="C0008"><div class="ot-sdk-column"><div class="ot-sdk-four ot-sdk-columns group category-menu-switch-handler" role="tab" tabindex="-1" aria-selected="false" aria-controls="ot-desc-id-C0008"><h3 id="ot-header-id-C0008">Targeting (1st Party)</h3></div></div><div class="ot-sdk-eight ot-sdk-columns description-container ot-hide" role="tabpanel" tabindex="0" id="ot-desc-id-C0008"><div class="group-toggle"><h3 class="category-header">Targeting (1st Party)</h3><div class="ot-toggle-group"><div class="ot-toggle"><div class="checkbox"><!-- DYNAMICALLY GENERATE Input ID  --> <input id="ot-group-id-C0008" class="category-switch-handler" role="switch" aria-checked="true" aria-controls="ot-desc-id-C0008" aria-labelledby="ot-header-id-C0008" name="ot-group-id-C0008" data-optanongroupid="C0008" checked="checked" type="checkbox"> <label for="ot-group-id-C0008"><span class="label-text">Targeting (1st Party)</span></label><!-- DYNAMICALLY GENERATE Input ID  --></div></div></div></div><p class="group-description ot-category-desc">These
 cookies may be set through our site by ourselves. They may be used by 
ourselves to build a profile of your interests and show you relevant 
content or adverts on our sites. They do not store directly personal 
information, but are based on uniquely identifying your browser and 
internet device. If you do not allow these cookies, you may experience 
less personalised content and/or advertising.</p><!-- sub groups --><div class="category-host-list-container"><a class="category-host-list-btn category-host-list-handler" role="button" href="javascript:void(0)" data-parent-id="C0008">Cookies Details‎</a></div></div></li><li class="category-item" data-optanongroupid="C0009"><div class="ot-sdk-column"><div class="ot-sdk-four ot-sdk-columns group category-menu-switch-handler" role="tab" tabindex="-1" aria-selected="false" aria-controls="ot-desc-id-C0009"><h3 id="ot-header-id-C0009">Targeting (3rd Party)</h3></div></div><div class="ot-sdk-eight ot-sdk-columns description-container ot-hide" role="tabpanel" tabindex="0" id="ot-desc-id-C0009"><div class="group-toggle"><h3 class="category-header">Targeting (3rd Party)</h3><div class="ot-toggle-group"><div class="ot-toggle"><div class="checkbox"><!-- DYNAMICALLY GENERATE Input ID  --> <input id="ot-group-id-C0009" class="category-switch-handler" role="switch" aria-checked="false" aria-controls="ot-desc-id-C0009" aria-labelledby="ot-header-id-C0009" name="ot-group-id-C0009" data-optanongroupid="C0009" type="checkbox"> <label for="ot-group-id-C0009"><span class="label-text">Targeting (3rd Party)</span></label><!-- DYNAMICALLY GENERATE Input ID  --></div></div></div></div><p class="group-description ot-category-desc">These
 cookies may be set through our site by our advertising partners. They 
may be used by those companies to build a profile of your interests and 
show you relevant adverts on other sites. They do not store directly 
personal information, but are based on uniquely identifying your browser
 and internet device. If you do not allow these cookies, you will 
experience less targeted advertising.</p><!-- sub groups --><div class="category-host-list-container"><a class="category-host-list-btn category-host-list-handler" role="button" href="javascript:void(0)" data-parent-id="C0009">Cookies Details‎</a></div></div></li></ul></div></div></div><!-- Vendors / Hosts --><section id="vendors-list" class="ot-hide"><div class="vendor-content"><nav id="vendors-list-header"><div class="navigation-container"><a class="back-btn-handler" role="button" href="javascript:void(0)"><div id="ot-back-arrow"><svg xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="15px" height="15px" viewBox="0 0 444.531 444.531" xml:space="preserve"><title>Back Button</title><g><path fill="#2c3643" d="M213.13,222.409L351.88,83.653c7.05-7.043,10.567-15.657,10.567-25.841c0-10.183-3.518-18.793-10.567-25.835
                    l-21.409-21.416C323.432,3.521,314.817,0,304.637,0s-18.791,3.521-25.841,10.561L92.649,196.425
                    c-7.044,7.043-10.566,15.656-10.566,25.841s3.521,18.791,10.566,25.837l186.146,185.864c7.05,7.043,15.66,10.564,25.841,10.564
                    s18.795-3.521,25.834-10.564l21.409-21.412c7.05-7.039,10.567-15.604,10.567-25.697c0-10.085-3.518-18.746-10.567-25.978
                    L213.13,222.409z"></path></g></svg></div><h3 class="pc-back-button-text" id="vendors-list-title">Advertising Cookies</h3></a></div><div class="action-container"><div id="search-container"><input id="vendor-search-handler" aria-label="Vendor Search" placeholder="Search..." name="vendor-search-handler" type="text"> <svg width="30" height="30" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 -30 110 110"><path fill="#2e3644" d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23
              s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92
              c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17
              s-17-7.626-17-17S14.61,6,23.984,6z"></path></svg></div><div id="filter-container"><a id="filter-btn-handler" role="button" aria-label="Filter" href="javascript:void(0)" aria-haspopup="true"><svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="15" height="15" viewBox="0 0 402.577 402.577" style="enable-background:new 0 0 402.577 402.577;" xml:space="preserve"><title>Filter Button</title><g><path fill="#2c3643" d="M400.858,11.427c-3.241-7.421-8.85-11.132-16.854-11.136H18.564c-7.993,0-13.61,3.715-16.846,11.136
                            c-3.234,7.801-1.903,14.467,3.999,19.985l140.757,140.753v138.755c0,4.955,1.809,9.232,5.424,12.854l73.085,73.083
                            c3.429,3.614,7.71,5.428,12.851,5.428c2.282,0,4.66-0.479,7.135-1.43c7.426-3.238,11.14-8.851,11.14-16.845V172.166L396.861,31.413
                            C402.765,25.895,404.093,19.231,400.858,11.427z"></path></g></svg></a></div></div></nav><section id="vendor-list-content"><div class="ot-sdk-row"><div class="ot-sdk-column"><div id="select-all-container"><div class="ot-checkbox"><div class="leg-int-sel-all-hdr"><span class="consent-hdr">Consent</span> <span class="leg-int-hdr">Leg.Interest</span></div><!-- select all vendor leg.int toggle container --><div id="select-all-vendors-leg-input-container"><input id="select-all-vendor-leg-handler" class="ot-group-option-box" type="checkbox"> <label for="select-all-vendor-leg-handler"><span class="label-text">Select All Vendors</span></label></div><!-- select all vendor consent toggle container --><div id="select-all-vendors-input-container"><input id="select-all-vendor-groups-handler" class="ot-group-option-box" type="checkbox"> <label for="select-all-vendor-groups-handler"><span class="label-text">Select All Vendors</span></label></div><!-- Hosts select all input container --><div id="select-all-hosts-input-container"><input id="select-all-hosts-groups-handler" class="ot-group-option-box" type="checkbox"> <label for="select-all-hosts-groups-handler"><span class="label-text">Select All Hosts</span></label></div><div id="select-all-text-container"><p>Select All</p></div></div></div><ul id="hosts-list-container"><li class="host-item"><input class="host-box" aria-expanded="false" role="button" ot-accordion="true" aria-label="33Across" type="checkbox"><section class="accordion-header"><div class="ot-toggle-group"><!-- Checkbox --><div class="ot-checkbox ot-host-tgl"><input id="REPLACE-WITH-DYANMIC-HOST-ID" class="host-checkbox-handler ot-group-option-box" type="checkbox"> <label for="REPLACE-WITH-DYANMIC-HOST-ID"><span class="label-text">REPLACE-WITH-DYANMIC-HOST-ID</span></label></div><!-- Checkbox END --><div class="host-info"><h3 class="host-title">33Across</h3><h4 class="host-description">33Across</h4></div></div><div class="host-notice"><div class="third-party-cookies-container"><a class="third-party-cookie-notice host-view-cookies" href="javascript:void(0)" role="presentation" aria-hidden="true" tabindex="-1">View Third Party Cookies</a></div><div class="ot-arrow-container"><svg class="ot-arrow" x="0px" y="0px" width="10px" height="10px" viewBox="0 0 451.846 451.847" style="enable-background:new 0 0 451.846 451.847;" xml:space="preserve"><g><path fill="#7b7b7b" d="M345.441,248.292L151.154,442.573c-12.359,12.365-32.397,12.365-44.75,0c-12.354-12.354-12.354-32.391,0-44.744
                        L278.318,225.92L106.409,54.017c-12.354-12.359-12.354-32.394,0-44.748c12.354-12.359,32.391-12.359,44.75,0l194.287,194.284
                        c6.177,6.18,9.262,14.271,9.262,22.366C354.708,234.018,351.617,242.115,345.441,248.292z"></path></g></svg></div></div></section><div class="accordion-text"><div class="host-options"><!-- HOST LIST VIEW UPDATE *** --><ul class="host-option-group"><li class="vendor-host"><div class="cookie-name-container"><div>Name</div><div>cookie name</div></div></li></ul><!-- HOST LIST VIEW UPDATE END *** --></div></div></li></ul></div></div></section></div><div id="ot-triangle"></div><section id="ot-filter-modal"><div id="ot-options"><div id="clear-filters-container"><a href="javascript:void(0)" id="clear-filters-handler" role="button"><p>Clear Filters</p></a></div><div class="ot-group-options"><div class="ot-group-option"><div class="ot-checkbox"><input id="storage-access-group" class="ot-group-option-box category-filter-handler" type="checkbox"> <label for="storage-access-group"><span>Information storage and access</span></label></div></div></div><button id="filter-apply-handler" class="ot-pill">Apply</button></div></section></section></div><!-- footer --><div class="ot-button-group-parent"><div class="ot-button-group"><button class="save-preference-btn-handler onetrust-close-btn-handler">Save Settings</button>  <button id="accept-recommended-btn-handler">Allow All Cookies</button></div><div class="ot-pc-footer-logo"><a class="powered-by-logo" href="https://onetrust.com/poweredbyonetrust" target="_blank" rel="noopener noreferrer" aria-label="Powered by One Trust" style="background-image: url(&quot;https://cdn.cookielaw.org/logos/static/poweredBy_ot_logo.svg&quot;)"></a></div></div><iframe class="ot-text-resize" title="onetrust-text-resize" style="position:absolute;top:-50000px;width:100em;" aria-hidden="true"></iframe></div><div id="onetrust-banner-sdk" class="otFlat bottom vertical-align-content" role="dialog" aria-labelledby="onetrust-policy-title" aria-describedby="onetrust-policy-text" style="bottom: -99px; animation-name: slide-down-custom; animation-duration: 1000ms; animation-fill-mode: forwards; animation-timing-function: ease-out;"><div class="ot-sdk-container"><div class="ot-sdk-row"><div id="onetrust-group-container" class="ot-sdk-eight ot-sdk-columns"><div class="banner_logo"></div><div id="onetrust-policy"><h3 id="onetrust-policy-title">Your privacy</h3><!-- Mobile Close Button --><div id="onetrust-close-btn-container-mobile" class="ot-hide-large"></div><!-- Mobile Close Button END--><p id="onetrust-policy-text">We
 use cookies to ensure the functionality of our website, to personalize 
content and advertising, to provide social media features, and to 
analyze our traffic. If you allow us to do so, we also inform our social
 media, advertising and analysis partners about your use of our website.
 You can decide for yourself which categories you want to deny or allow.
 Please note that based on your settings not all functionalities of the 
site are available.&nbsp;<a href="https://www.nature.com/info/privacy.html" tabindex="0">View our privacy policy</a></p></div></div><div id="onetrust-button-group-parent" class="ot-sdk-three ot-sdk-columns"><div id="onetrust-button-group"><button id="onetrust-pc-btn-handler" tabindex="0">Manage Settings</button>  <button id="onetrust-accept-btn-handler" tabindex="0">Accept All Cookies</button></div></div><!-- Close Button --><div id="onetrust-close-btn-container" class="ot-sdk-one ot-sdk-column ot-hide-small"></div><!-- Close Button END--></div></div></div></div><script type="text/javascript" id="">var event=document.createEvent("Event");event.initEvent("OneTrustGroupsUpdated",!0,!0);document.dispatchEvent(event);</script><script type="text/javascript" id="">window.permutive&&window.permutive.consent&&permutive.consent({opt_in:!0,token:"CONSENT_CAPTURED"});</script><iframe style="width: 0px; height: 0px; bottom: 0px; right: 0px; z-index: 100000; position: fixed;" scrolling="no" src="https://recommended.springernature.com/v2397/generated/client.html?source=https://www.nature.com" name="uptodate-client" id="uptodate-client" title="Personalised recommendations" allowtransparency="true" frameborder="0"></iframe></body></html>